# accSDA

Version: 1.0.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘rARPACK’ ‘sparseLDA’
      All declared Imports should be used.
    ```

# adapr

Version: 2.0.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘plotly’ ‘shinydashboard’
      All declared Imports should be used.
    ```

# adegenet

Version: 2.1.1

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.7Mb
      sub-directories of 1Mb or more:
        R       2.0Mb
        data    1.3Mb
        files   1.7Mb
    ```

# adklakedata

Version: 0.6.1

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘adklakedata-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: adk_lake_shapes
    > ### Title: Return path to Lake Polygons Shapefile
    > ### Aliases: adk_lake_shapes
    > 
    > ### ** Examples
    > 
    > library(sf)
    Error in library(sf) : there is no package called ‘sf’
    Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘sf’
    ```

# afex

Version: 0.22-1

## In both

*   checking Rd cross-references ... NOTE
    ```
    Packages unavailable to check Rd xrefs: ‘ez’, ‘ascii’
    ```

# affycoretools

Version: 1.52.2

## In both

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘oligo’
    ```

*   checking for unstated dependencies in vignettes ... NOTE
    ```
    '::' or ':::' import not declared from: ‘XML’
    ```

# afmToolkit

Version: 0.0.1

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘DBI’ ‘assertthat’ ‘tibble’
      All declared Imports should be used.
    ```

# ahpsurvey

Version: 0.2.2

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘knitr’ ‘tidyr’
      All declared Imports should be used.
    ```

# aire.zmvm

Version: 0.6.1

## In both

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 52 marked UTF-8 strings
    ```

# ALA4R

Version: 1.6.0

## In both

*   R CMD check timed out
    

# alphahull

Version: 2.1

## In both

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'alphahull.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.55 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# ameco

Version: 0.2.9

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 16.2Mb
      sub-directories of 1Mb or more:
        data  16.1Mb
    ```

# ammistability

Version: 0.1.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    0 words of pdfTeX memory
    0 indirect objects
    No pages of output.
    Transcript written on etex.log.
    fmtutil [INFO]: /Users/max/Library/TinyTeX/texmf-var/web2c/pdftex/etex.fmt installed.
    fmtutil [INFO]: Disabled formats: 1
    fmtutil [INFO]: Successfully rebuilt formats: 14
    fmtutil [INFO]: Total formats: 15
    fmtutil [INFO]: exiting with status 0
    tlmgr install wrapfig
    Unknown directive ...l
    tlmgr path add
    Unknown directive ...l
    ! LaTeX Error: File `wrapfig.sty' not found.
    
    ! Emergency stop.
    <read *> 
    
    Error: processing vignette 'Introduction.Rmd' failed with diagnostics:
    Failed to compile Introduction.tex. See Introduction.log for more info.
    Execution halted
    ```

# aMNLFA

Version: 0.1

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘devtools’ ‘gridExtra’
      All declared Imports should be used.
    ```

# amplican

Version: 1.2.1

## In both

*   checking whether package ‘amplican’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘IRanges’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/amplican/new/amplican.Rcheck/00install.out’ for details.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 13.3Mb
      sub-directories of 1Mb or more:
        doc  12.5Mb
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘CrispRVariants’
    ```

# ampliQueso

Version: 1.18.0

## In both

*   checking whether package ‘ampliQueso’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘Rsamtools’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘DelayedArray’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/ampliQueso/new/ampliQueso.Rcheck/00install.out’ for details.
    ```

*   checking examples ... WARNING
    ```
    Found the following significant warnings:
      Warning: working directory was changed to ‘/Users/max/github/ggplot2/revdep/checks.noindex/ampliQueso/new/ampliQueso.Rcheck/ampliQueso’, resetting
    ```

*   checking package dependencies ... NOTE
    ```
    Depends: includes the non-default packages:
      ‘rnaSeqMap’ ‘knitr’ ‘rgl’ ‘ggplot2’ ‘gplots’ ‘parallel’ ‘doParallel’
      ‘foreach’ ‘VariantAnnotation’ ‘genefilter’ ‘statmod’ ‘xtable’
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Description field: should contain one or more complete sentences.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘samr’
      All declared Imports should be used.
    Packages in Depends field not imported from:
      ‘VariantAnnotation’ ‘doParallel’ ‘foreach’ ‘genefilter’ ‘ggplot2’
      ‘gplots’ ‘knitr’ ‘parallel’ ‘rgl’ ‘statmod’ ‘xtable’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ':::' call which should be '::': ‘rnaSeqMap:::newSeqReads’
      See the note in ?`:::` about the use of this operator.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    compareCoverages: no visible global function definition for
      ‘read.table’
    getCountTable: no visible global function definition for ‘read.table’
    runAQReport: no visible global function definition for ‘read.table’
    runAQReport: no visible global function definition for ‘rowttests’
    runAQReport: no visible global function definition for ‘p.adjust’
    runAQReport: no visible global function definition for ‘%do%’
    runAQReport: no visible global function definition for ‘foreach’
    runAQReport: no visible global function definition for ‘aggregate’
    runAQReport: no visible global function definition for ‘knit’
    Undefined global functions or variables:
      %do% %dopar% aggregate combn detectCores end foreach hasArg knit
      makeCluster p.adjust permp qual ranges read.table readVcf ref
      registerDoParallel rowttests seqnames start stopCluster str strand
      width
    Consider adding
      importFrom("methods", "hasArg")
      importFrom("stats", "aggregate", "end", "p.adjust", "start")
      importFrom("utils", "combn", "read.table", "str")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    
        rowSds, rowVars
    
    Loading required package: statmod
    Loading required package: xtable
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'ampliQueso.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.30 \ifthenelse
                    {\boolean{Sweave@inconsolata}}{%^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# AMR

Version: 0.4.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘hms’
      All declared Imports should be used.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 67 marked UTF-8 strings
    ```

# Anaquin

Version: 2.4.0

## In both

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Package listed in more than one of Depends, Imports, Suggests, Enhances:
      ‘ggplot2’
    A package should be listed in only one of these fields.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘DESeq2’
      All declared Imports should be used.
    ```

*   checking Rd files ... NOTE
    ```
    prepare_Rd: plotConjoint.Rd:26-27: Dropping empty section \examples
    ```

# AneuFinder

Version: 1.8.0

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    > ### ** Examples
    > 
    > ## Get a BED file, bin it and run GC correction
    > bedfile <- system.file("extdata", "KK150311_VI_07.bam.bed.gz", package="AneuFinderData")
    > binned <- binReads(bedfile, assembly='mm10', binsize=1e6,
    +                   chromosomes=c(1:19,'X','Y'))
    Reading file KK150311_VI_07.bam.bed.gz ... 0.7s
    Fetching chromosome lengths from UCSC ... 1.54s
    Subsetting chromosomes ... 0.01s
    Filtering reads ... 0.07s
    Subsetting specified chromosomes ... 0.02s
    Calculating complexity ... 0.19s
    Removing duplicate reads ... 0.12s
    Calculating coverage ... 0.92s
    Making fixed-width bins for bin size 1e+06 ... 0.02s
    Splitting into strands ... 0.01s
    Counting overlaps for binsize_1e+06 ... 0.13s
    > plot(binned[[1]], type=1)
    Error in levels(dfplot.seg$state) : object 'dfplot.seg' not found
    Calls: plot ... manual_scale -> <Anonymous> -> stateColors -> intersect -> levels
    Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
         ## End(Not run)
         
    
    Reading file hg19_diploid.bam.bed.gz ... 2.84s
    Fetching chromosome lengths from UCSC ...Warning in FUN(genome = names(SUPPORTED_UCSC_GENOMES)[idx], circ_seqs = supported_genome$circ_seqs,  :
      NCBI seqlevel was set to NA for hg19 UCSC seqlevel(s) not in the NCBI assembly:
      chrM
     1.87s
    Subsetting chromosomes ... 0.06s
    Filtering reads ... 0.18s
    Subsetting specified chromosomes ... 0.02s
    Calculating complexity ... 1.14s
    Removing duplicate reads ... 0.87s
    Calculating coverage ... 0.52s
    Making fixed-width bins for bin size 1e+05 ... 0.07s
    Splitting into strands ... 0.02s
    Counting overlaps for binsize_1e+05 ... 0.24s
    Quitting from lines 110-132 (AneuFinder.Rnw) 
    Error: processing vignette 'AneuFinder.Rnw' failed with diagnostics:
    object 'dfplot.seg' not found
    Execution halted
    ```

## Newly fixed

*   R CMD check timed out
    

## In both

*   checking whether package ‘AneuFinder’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/AneuFinder/new/AneuFinder.Rcheck/00install.out’ for details.
    ```

*   checking R code for possible problems ... NOTE
    ```
    Aneufinder: multiple local function definitions for ‘parallel.helper’
      with different formal arguments
    ```

# annotatr

Version: 1.6.0

## In both

*   checking R code for possible problems ... NOTE
    ```
    plot_coannotations: no visible binding for global variable ‘.’
    plot_numerical_coannotations: no visible binding for global variable
      ‘.’
    Undefined global functions or variables:
      .
    ```

# ANOM

Version: 0.5

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    
    The following object is masked from 'package:MASS':
    
        geyser
    
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'ANOM.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `doi.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.56 \usepackage
                    {parskip}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# anomalize

Version: 0.1.1

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.3Mb
      sub-directories of 1Mb or more:
        help   4.7Mb
    ```

# anomalyDetection

Version: 0.2.5

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘caret’
      All declared Imports should be used.
    ```

# antitrust

Version: 0.99.10

## In both

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'manual.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `scrreprt.cls' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: cls)
    
    ! Emergency stop.
    <read *> 
             
    l.4 \usepackage
                   [letterpaper]{geometry}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# aoristic

Version: 0.6

## In both

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Description field: should contain one or more complete sentences.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      ‘slot’
    aoristic.grid: no visible global function definition for ‘browseURL’
    aoristic.shp: no visible global function definition for ‘aggregate’
    aoristic.shp: no visible global function definition for ‘slot’
    aoristic.shp : <anonymous>: no visible global function definition for
      ‘as’
    aoristic.shp : <anonymous>: no visible global function definition for
      ‘slot’
    aoristic.shp: no visible global function definition for ‘browseURL’
    Undefined global functions or variables:
      aggregate as browseURL colorRampPalette contourLines dev.off image
      par png quantile slot
    Consider adding
      importFrom("grDevices", "colorRampPalette", "contourLines", "dev.off",
                 "png")
      importFrom("graphics", "image", "par")
      importFrom("methods", "as", "slot")
      importFrom("stats", "aggregate", "quantile")
      importFrom("utils", "browseURL")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

# apsimr

Version: 1.2

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'apsimr-basics.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `microtype.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.43 ^^M
            
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking Rd cross-references ... NOTE
    ```
    Packages unavailable to check Rd xrefs: ‘sensitivity’, ‘APSIMBatch’
    ```

# ArchaeoPhases

Version: 1.3

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘DT’
      All declared Imports should be used.
    ```

# archetypes

Version: 2.2-0

## In both

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' calls in package code:
      ‘MASS’ ‘TSP’
      Please use :: or requireNamespace() instead.
      See section 'Suggested packages' in the 'Writing R Extensions' manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    xyplot.stepArchetypes: no visible global function definition for ‘gray’
    xyplot.stepArchetypes: no visible global function definition for ‘plot’
    xyplot.stepArchetypes: no visible global function definition for
      ‘points’
    xyplot.stepArchetypes: no visible global function definition for
      ‘lines’
    xyplot.weightedArchetypes: no visible binding for global variable
      ‘gray’
    Undefined global functions or variables:
      TSP abline arrows axis box chull cmdscale combn dist ecdf ginv gray
      head history layout lines matlines matplot median mtext optim par
      plot points polygon rgb rnorm sd solve_TSP symbols tail text
    Consider adding
      importFrom("grDevices", "chull", "gray", "rgb")
      importFrom("graphics", "abline", "arrows", "axis", "box", "layout",
                 "lines", "matlines", "matplot", "mtext", "par", "plot",
                 "points", "polygon", "symbols", "text")
      importFrom("stats", "cmdscale", "dist", "ecdf", "median", "optim",
                 "rnorm", "sd")
      importFrom("utils", "combn", "head", "history", "tail")
    to your NAMESPACE file.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    Warning in method(..., k = k[i]) : k=14: zs > maxKappa
    Warning in method(..., k = k[i]) : k=14: zs > maxKappa
    Warning in method(..., k = k[i]) : k=15: zs > maxKappa
    Warning in axis(1, at = a, ...) : "axes" is not a graphical parameter
    Loading required package: grid
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'archetypes.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.55 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# archivist

Version: 2.3.1

## In both

*   checking package dependencies ... NOTE
    ```
    Package which this enhances but not available for checking: ‘archivist.github’
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘archivist.github’
    ```

# arena2r

Version: 1.0.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘purrr’ ‘shinyBS’ ‘shinydashboard’ ‘shinyjs’
      All declared Imports should be used.
    ```

# ARPobservation

Version: 1.1

## In both

*   checking R code for possible problems ... NOTE
    ```
    ...
    reported_observations: no visible global function definition for
      ‘aggregate’
    reported_observations: no visible binding for global variable ‘var’
    rgamma_eq : <anonymous>: no visible global function definition for
      ‘uniroot’
    rgamma_eq: no visible global function definition for ‘runif’
    rgamma_mix_eq : <anonymous>: no visible global function definition for
      ‘uniroot’
    rgamma_mix_eq: no visible global function definition for ‘runif’
    rweibull_eq : <anonymous>: no visible global function definition for
      ‘uniroot’
    rweibull_eq: no visible global function definition for ‘runif’
    smooth_cov: no visible global function definition for ‘dist’
    Undefined global functions or variables:
      aggregate dist integrate nobs pgamma qnorm quantile rbinom rexp
      rgamma rnorm runif rweibull uniroot var
    Consider adding
      importFrom("stats", "aggregate", "dist", "integrate", "nobs", "pgamma",
                 "qnorm", "quantile", "rbinom", "rexp", "rgamma", "rnorm",
                 "runif", "rweibull", "uniroot", "var")
    to your NAMESPACE file.
    ```

# ashr

Version: 2.2-7

## In both

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking: ‘REBayes’ ‘Rmosek’
    ```

# asremlPlus

Version: 2.0-12

## In both

*   checking package dependencies ... NOTE
    ```
    Package which this enhances but not available for checking: ‘asreml’
    ```

# ASSIGN

Version: 1.16.0

## In both

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

# asVPC

Version: 1.0.2

## In both

*   checking R code for possible problems ... NOTE
    ```
    ...
    VPC.graph : findQuantile: no visible global function definition for
      ‘quantile’
    VPC.graph : findQuantileCI : <anonymous>: no visible global function
      definition for ‘quantile’
    asVPC.binW: no visible global function definition for ‘median’
    asVPC.binW : <anonymous>: no visible global function definition for
      ‘quantile’
    asVPC.binW: no visible global function definition for ‘quantile’
    asVPC.distanceW: no visible global function definition for ‘median’
    asVPC.distanceW : <anonymous>: no visible global function definition
      for ‘quantile’
    asVPC.distanceW: no visible global function definition for ‘quantile’
    makeCOVbin: no visible global function definition for ‘quantile’
    makeCOVbin: no visible global function definition for ‘median’
    read_Simdata: no visible global function definition for ‘read.table’
    Undefined global functions or variables:
      median quantile read.table
    Consider adding
      importFrom("stats", "median", "quantile")
      importFrom("utils", "read.table")
    to your NAMESPACE file.
    ```

# autocogs

Version: 0.1.1

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘MASS’ ‘broom’ ‘diptest’ ‘ggplot2’ ‘hexbin’ ‘moments’
      All declared Imports should be used.
    ```

# autoimage

Version: 2.0

## In both

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘akima’
    ```

# automap

Version: 1.0-14

## In both

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' calls in package code:
      ‘ggplot2’ ‘gpclib’ ‘maptools’
      Please use :: or requireNamespace() instead.
      See section 'Suggested packages' in the 'Writing R Extensions' manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      ‘geom_point’
    posPredictionInterval: no visible global function definition for
      ‘median’
    posPredictionInterval: no visible global function definition for
      ‘qnorm’
    summary.autoKrige.cv: no visible global function definition for ‘cor’
    summary.autoKrige.cv: no visible global function definition for ‘sd’
    summary.autoKrige.cv: no visible global function definition for ‘IQR’
    Undefined global functions or variables:
      IQR aes_string as.formula chull coord_equal cor facet_wrap fortify
      geom_path geom_point ggplot grey is median modifyList qnorm quantile
      scale_color_gradient2 scale_size_continuous scale_x_continuous
      scale_y_continuous sd
    Consider adding
      importFrom("grDevices", "chull", "grey")
      importFrom("methods", "is")
      importFrom("stats", "IQR", "as.formula", "cor", "median", "qnorm",
                 "quantile", "sd")
      importFrom("utils", "modifyList")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

# av

Version: 0.2

## In both

*   checking whether package ‘av’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/av/new/av.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘av’ ...
** package ‘av’ successfully unpacked and MD5 sums checked
Using PKG_CFLAGS=-I/usr/local/opt/ffmpeg/include
Using PKG_LIBS=-L/usr/local/opt/ffmpeg/lib -lavfilter
------------------------- ANTICONF ERROR ---------------------------
Configuration failed because libavfilter was not found. Try installing:
 * deb: libavfilter-dev (Debian, Ubuntu 18.04 and up)
 * rpm: ffmpeg-devel (https://rpmfusion.org) (Fedora, CentOS, RHEL)
 * csw: ffmpeg_dev (Solaris)
For Ubuntu versions Trusty (14.04) and Xenial (16.04) use this PPA:
   sudo add-apt-repository -y ppa:jonathonf/ffmpeg-3
   sudo apt-get update
   sudo apt-get install -y libavfilter-dev
If libavfilter is already installed, check that 'pkg-config' is in your
PATH and PKG_CONFIG_PATH contains a libavfilter.pc file. If pkg-config
is unavailable you can set INCLUDE_DIR and LIB_DIR manually via:
R CMD INSTALL --configure-vars='INCLUDE_DIR=... LIB_DIR=...'
--------------------------------------------------------------------
ERROR: configuration failed for package ‘av’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/av/new/av.Rcheck/av’

```
### CRAN

```
* installing *source* package ‘av’ ...
** package ‘av’ successfully unpacked and MD5 sums checked
Using PKG_CFLAGS=-I/usr/local/opt/ffmpeg/include
Using PKG_LIBS=-L/usr/local/opt/ffmpeg/lib -lavfilter
------------------------- ANTICONF ERROR ---------------------------
Configuration failed because libavfilter was not found. Try installing:
 * deb: libavfilter-dev (Debian, Ubuntu 18.04 and up)
 * rpm: ffmpeg-devel (https://rpmfusion.org) (Fedora, CentOS, RHEL)
 * csw: ffmpeg_dev (Solaris)
For Ubuntu versions Trusty (14.04) and Xenial (16.04) use this PPA:
   sudo add-apt-repository -y ppa:jonathonf/ffmpeg-3
   sudo apt-get update
   sudo apt-get install -y libavfilter-dev
If libavfilter is already installed, check that 'pkg-config' is in your
PATH and PKG_CONFIG_PATH contains a libavfilter.pc file. If pkg-config
is unavailable you can set INCLUDE_DIR and LIB_DIR manually via:
R CMD INSTALL --configure-vars='INCLUDE_DIR=... LIB_DIR=...'
--------------------------------------------------------------------
ERROR: configuration failed for package ‘av’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/av/old/av.Rcheck/av’

```
# BaalChIP

Version: 1.6.0

## In both

*   R CMD check timed out
    

*   checking whether package ‘BaalChIP’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘Rsamtools’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/BaalChIP/new/BaalChIP.Rcheck/00install.out’ for details.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 200.0Mb
      sub-directories of 1Mb or more:
        data   96.1Mb
        doc     1.6Mb
        test  101.9Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    applyBayes: no visible binding for global variable ‘SNP_id’
    plot.filt.barplot: no visible binding for global variable ‘cellname’
    plot.filt.barplot: no visible binding for global variable ‘value’
    plot.filt.barplot: no visible binding for global variable ‘variable’
    plot.filt.boxplot: no visible binding for global variable ‘variable’
    plot.filt.boxplot: no visible binding for global variable ‘value’
    plot.filt.boxplot: no visible binding for global variable ‘coltype’
    plot.filt.pie: no visible binding for global variable ‘variable’
    plot.filt.pie: no visible binding for global variable ‘value.mean’
    plot.simul: no visible binding for global variable ‘readslen’
    plot.simul: no visible binding for global variable ‘perc_right’
    plotadjustment: no visible binding for global variable ‘value’
    plotadjustment: no visible binding for global variable ‘variable’
    Undefined global functions or variables:
      SNP_id cellname coltype perc_right readslen value value.mean variable
    ```

# BacArena

Version: 1.7

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  9.0Mb
      sub-directories of 1Mb or more:
        R      1.0Mb
        data   7.0Mb
    ```

# BACCT

Version: 1.0

## In both

*   checking whether package ‘BACCT’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/BACCT/new/BACCT.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘BACCT’ ...
** package ‘BACCT’ successfully unpacked and MD5 sums checked
** R
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rjags', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/BACCT/rjags/libs/rjags.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/BACCT/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/BACCT/rjags/libs/rjags.so
  Reason: image not found
ERROR: lazy loading failed for package ‘BACCT’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/BACCT/new/BACCT.Rcheck/BACCT’

```
### CRAN

```
* installing *source* package ‘BACCT’ ...
** package ‘BACCT’ successfully unpacked and MD5 sums checked
** R
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rjags', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/BACCT/rjags/libs/rjags.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/BACCT/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/BACCT/rjags/libs/rjags.so
  Reason: image not found
ERROR: lazy loading failed for package ‘BACCT’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/BACCT/old/BACCT.Rcheck/BACCT’

```
# backShift

Version: 0.1.4.2

## In both

*   checking Rd cross-references ... NOTE
    ```
    Packages unavailable to check Rd xrefs: ‘InvariantCausalPrediction’, ‘CompareCausalNetworks’
    ```

# bacon

Version: 1.8.0

## In both

*   checking R code for possible problems ... NOTE
    ```
    .hist: no visible binding for global variable ‘..density..’
    .qq: no visible binding for global variable ‘column’
    Undefined global functions or variables:
      ..density.. column
    ```

# ballr

Version: 0.2.2

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘ballr-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: NBAPlayerPerGameStats
    > ### Title: NBA Player Career Statistics
    > ### Aliases: NBAPlayerPerGameStats
    > 
    > ### ** Examples
    > 
    > NBAPlayerPerGameStats("/players/d/davisan02.html") # Anthony Davis
    Error in rvest::html_table(pg, fill = T)[[2]] : subscript out of bounds
    Calls: NBAPlayerPerGameStats
    Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 83-86 (use-ballr.Rmd) 
    Error: processing vignette 'use-ballr.Rmd' failed with diagnostics:
    subscript out of bounds
    Execution halted
    ```

# bamdit

Version: 3.2.1

## In both

*   checking whether package ‘bamdit’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/bamdit/new/bamdit.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘bamdit’ ...
** package ‘bamdit’ successfully unpacked and MD5 sums checked
** R
** data
** inst
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rjags', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/bamdit/rjags/libs/rjags.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/bamdit/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/bamdit/rjags/libs/rjags.so
  Reason: image not found
ERROR: lazy loading failed for package ‘bamdit’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/bamdit/new/bamdit.Rcheck/bamdit’

```
### CRAN

```
* installing *source* package ‘bamdit’ ...
** package ‘bamdit’ successfully unpacked and MD5 sums checked
** R
** data
** inst
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rjags', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/bamdit/rjags/libs/rjags.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/bamdit/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/bamdit/rjags/libs/rjags.so
  Reason: image not found
ERROR: lazy loading failed for package ‘bamdit’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/bamdit/old/bamdit.Rcheck/bamdit’

```
# banter

Version: 0.9.3

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘ranger’
      All declared Imports should be used.
    ```

# BASiCS

Version: 1.2.1

## In both

*   checking whether package ‘BASiCS’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘DelayedArray’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/BASiCS/new/BASiCS.Rcheck/00install.out’ for details.
    ```

# bayesdfa

Version: 0.1.0

## In both

*   checking examples ... ERROR
    ```
    ...
    Chain 1: Iteration: 400 / 500 [ 80%]  (Sampling)
    Chain 1: Iteration: 450 / 500 [ 90%]  (Sampling)
    Chain 1: Iteration: 500 / 500 [100%]  (Sampling)
    Chain 1:  Elapsed Time: 1.45066 seconds (Warm-up)
    Chain 1:                1.42116 seconds (Sampling)
    Chain 1:                2.87183 seconds (Total)
    
    SAMPLING FOR MODEL 'dfa' NOW (CHAIN 2).
    Chain 2: Gradient evaluation took 4.1e-05 seconds
    Chain 2: 1000 transitions using 10 leapfrog steps per transition would take 0.41 seconds.
    Chain 2: Iteration:   1 / 500 [  0%]  (Warmup)
    [1] "Error in sampler$call_sampler(args_list[[i]]) : "
    [2] "  c++ exception (unknown reason)"                
    error occurred during calling the sampler; sampling not done
    Stan model 'dfa' does not contain samples.
    Stan model 'dfa' does not contain samples.
    Stan model 'dfa' does not contain samples.
    Error in names(object) <- nm : 
      'names' attribute [1] must be the same length as the vector [0]
    Calls: fit_dfa ... find_inverted_chains -> <Anonymous> -> melt.default -> setNames
    Execution halted
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      [1] "Error in sampler$call_sampler(args_list[[i]]) : "
      [2] "  c++ exception (unknown reason)"                
      Stan model 'dfa' does not contain samples.
      Stan model 'dfa' does not contain samples.
      ── 1. Error: (unknown) (@test-fit.R#8)  ────────────────────────────────────────
      attempt to set an attribute on NULL
      1: fit_dfa(y = y, num_trends = 1, iter = 800, chains = 1) at testthat/test-fit.R:8
      2: rstan::monitor(e)
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 0 SKIPPED: 0 FAILED: 1
      1. Error: (unknown) (@test-fit.R#8) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.2Mb
      sub-directories of 1Mb or more:
        libs   4.4Mb
    ```

# bayesDP

Version: 1.3.2

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘Rcpp’
      All declared Imports should be used.
    ```

# bayesplot

Version: 1.6.0

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  6.4Mb
      sub-directories of 1Mb or more:
        R     1.8Mb
        doc   4.0Mb
    ```

# BayesRS

Version: 0.1.3

## In both

*   checking whether package ‘BayesRS’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/BayesRS/new/BayesRS.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘BayesRS’ ...
** package ‘BayesRS’ successfully unpacked and MD5 sums checked
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rjags', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/BayesRS/rjags/libs/rjags.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/BayesRS/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/BayesRS/rjags/libs/rjags.so
  Reason: image not found
ERROR: lazy loading failed for package ‘BayesRS’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/BayesRS/new/BayesRS.Rcheck/BayesRS’

```
### CRAN

```
* installing *source* package ‘BayesRS’ ...
** package ‘BayesRS’ successfully unpacked and MD5 sums checked
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rjags', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/BayesRS/rjags/libs/rjags.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/BayesRS/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/BayesRS/rjags/libs/rjags.so
  Reason: image not found
ERROR: lazy loading failed for package ‘BayesRS’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/BayesRS/old/BayesRS.Rcheck/BayesRS’

```
# baystability

Version: 0.1.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘ggfortify’ ‘ggplot2’ ‘matrixStats’ ‘reshape2’ ‘scales’
      All declared Imports should be used.
    ```

# bbmle

Version: 1.0.20

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
      NaNs produced
    Warning in dbinom(x = c(0, 2, 1, 3, 4, 5, 0, 0, 0, 0, 1, 0, 0, 0, 0), prob = c(0.253207428692937,  :
      NaNs produced
    Warning in dbinom(x = c(0, 2, 1, 3, 4, 5, 0, 0, 0, 0, 1, 0, 0, 0, 0), prob = c(2.24718525432755,  :
      NaNs produced
    Warning in dbinom(x = c(0, 2, 1, 3, 4, 5, 0, 0, 0, 0, 1, 0, 0, 0, 0), prob = c(0.219940849413471,  :
      NaNs produced
    Warning in dbinom(x = c(0, 2, 1, 3, 4, 5, 0, 0, 0, 0, 1, 0, 0, 0, 0), prob = c(2.76406402172104,  :
      NaNs produced
    Warning in dbinom(x = c(0, 2, 1, 3, 4, 5, 0, 0, 0, 0, 1, 0, 0, 0, 0), prob = c(0.191138285903703,  :
      NaNs produced
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'mle2.tex' failed.
    LaTeX errors:
    ! Package babel Error: Unknown option `english'. Either you misspelled it
    (babel)                or the language definition file english.ldf was not foun
    d.
    
    See the babel package documentation for explanation.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# BCEA

Version: 2.2-6

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘INLA’
    ```

# Bclim

Version: 3.1.2

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘ggplot2’
      All declared Imports should be used.
    ```

# bcmaps

Version: 0.17.1

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘sf’
    
    Package suggested but not available for checking: ‘bcmaps.rdata’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# bcrm

Version: 0.4.7

## In both

*   checking examples ... ERROR
    ```
    ...
    > ##D plot(Power.LN.bcrm.compare.sim,threep3=TRUE)
    > ## End(Not run)
    > 
    > ## A 2-parameter model, using priors as specified in Neuenschwander et al 2008.
    > ## Posterior mean used to choose the next dose
    > ## Standardised doses using reference dose, 250mg
    > sdose<-log(dose/250)
    > ## Bivariate lognormal prior for two parameters
    > mu<-c(2.15,0.52)
    > Sigma<-rbind(c(0.84^2,0.134),c(0.134,0.80^2))
    > ## Using rjags (requires JAGS to be installed)
    > TwoPLogistic.mean.bcrm<-bcrm(stop=list(nmax=18),data=data,sdose=sdose
    +   ,dose=dose,ff="logit2",prior.alpha=list(4,mu,Sigma),target.tox=target.tox
    +   ,constrain=FALSE,pointest="mean",method="rjags")
    Error: .onLoad failed in loadNamespace() for 'rjags', details:
      call: dyn.load(file, DLLpath = DLLpath, ...)
      error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/bcrm/rjags/libs/rjags.so':
      dlopen(/Users/max/github/ggplot2/revdep/library.noindex/bcrm/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
      Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/bcrm/rjags/libs/rjags.so
      Reason: image not found
    Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘BRugs’
    ```

# bde

Version: 1.0.1

## In both

*   checking R code for possible problems ... NOTE
    ```
    ...
    rsample,BoundedDensity: no visible global function definition for
      ‘runif’
    setNormalizationConst,MacroBetaChen99Kernel: no visible global function
      definition for ‘integrate’
    setNormalizationConst,MacroBetaHirukawaTSKernel: no visible global
      function definition for ‘integrate’
    setNormalizationConstant,MacroBetaHirukawaJLNKernel: no visible global
      function definition for ‘integrate’
    setNormalizationConstants,MicroBetaChen99Kernel :
      density.kernelFunction: no visible global function definition for
      ‘dbeta’
    setNormalizationConstants,MicroBetaChen99Kernel : <anonymous>: no
      visible global function definition for ‘integrate’
    show,BoundedDensity: no visible global function definition for ‘str’
    show,KernelDensity: no visible global function definition for ‘str’
    Undefined global functions or variables:
      dbeta integrate runif str
    Consider adding
      importFrom("stats", "dbeta", "integrate", "runif")
      importFrom("utils", "str")
    to your NAMESPACE file.
    ```

# bea.R

Version: 1.0.6

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘Rcpp’ ‘chron’ ‘colorspace’ ‘gtable’ ‘htmltools’ ‘htmlwidgets’
      ‘httpuv’ ‘magrittr’ ‘munsell’ ‘plyr’ ‘scales’ ‘stringi’ ‘xtable’
      ‘yaml’
      All declared Imports should be used.
    ```

# beadarray

Version: 2.30.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
        'citation("Biobase")', and for packages
        'citation("pkgname")'.
    
    Loading required package: ggplot2
    Welcome to beadarray version 2.30.0
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'ImageProcessing.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `beramono.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.87 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘beadarrayExampleData’
    ```

*   checking top-level files ... NOTE
    ```
    File
      LICENSE
    is not mentioned in the DESCRIPTION file.
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' calls in package code:
      ‘Nozzle.R1’ ‘affy’ ‘ggbio’ ‘hwriter’ ‘lumi’ ‘vsn’
      Please use :: or requireNamespace() instead.
      See section 'Suggested packages' in the 'Writing R Extensions' manual.
    Packages in Depends field not imported from:
      ‘ggplot2’ ‘methods’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    Unexported objects imported by ':::' calls:
      ‘BeadDataPackR:::combineFiles’ ‘BeadDataPackR:::readHeader’
      ‘Biobase:::assayDataStorageMode’
      See the note in ?`:::` about the use of this operator.
    There are ::: calls to the package's namespace in its code. A package
      almost never needs to use ::: for its own objects:
      ‘illuminaForeground_6x6’ ‘locsIndicesToGrid’ ‘obtainLocs’
      ‘simpleXMLparse’
    ```

*   checking foreign function calls ... NOTE
    ```
    Foreign function call to a different package:
      .Call("roundLocsFileValues", ..., PACKAGE = "BeadDataPackR")
    See chapter ‘System and foreign language interfaces’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      geom_hline geom_point geom_tile geom_vline ggplot ggsave ggtitle
      hwrite hwriteImage is jpeg loess lowess lumiHumanIDMapping_dbconn
      lumiMouseIDMapping_dbconn lumiRatIDMapping_dbconn lumiT menu
      metaTemplate model.matrix new newCustomReport newFigure newParagraph
      newSection newTable normalize.invariantset normalize.qspline openPage
      opts p.adjust packageDescription pdf platformSigs plotIdeogram png
      predict qplot rainbow read.csv read.table rgb rsn runif
      scale_fill_discrete scale_fill_gradient setTxtProgressBar stat_binhex
      theme theme_blank theme_bw tracks txtProgressBar value value.1 vsn2
      write.csv write.table writeReport xlab ylab
    Consider adding
      importFrom("grDevices", "col2rgb", "dev.off", "jpeg", "pdf", "png",
                 "rainbow", "rgb")
      importFrom("methods", "as", "callNextMethod", "is", "new")
      importFrom("stats", "aggregate", "approx", "density", "loess",
                 "lowess", "model.matrix", "p.adjust", "predict", "runif")
      importFrom("utils", "data", "menu", "packageDescription", "read.csv",
                 "read.table", "setTxtProgressBar", "txtProgressBar",
                 "write.csv", "write.table")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

# BEARscc

Version: 1.0.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Warning in has_utility("pdfcrop") :
      pdfcrop not installed or not in PATH
    tlmgr search --file --global '/beramono.sty'
    Unknown directive ...l
    Warning in system2("tlmgr", args, ...) :
      running command ''tlmgr' search --file --global '/beramono.sty'' had status 25
    ! LaTeX Error: File `beramono.sty' not found.
    
    ! Emergency stop.
    <read *> 
    
    Error: processing vignette 'BEARscc.Rmd' failed with diagnostics:
    Failed to compile BEARscc.tex. See BEARscc.log for more info.
    Execution halted
    ```

# behaviorchange

Version: 0.0.2

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘magrittr’
      All declared Imports should be used.
    ```

# benchmark

Version: 0.3-6

## In both

*   checking whether package ‘benchmark’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘ggplot2::sym’ by ‘relations::sym’ when loading ‘benchmark’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/benchmark/new/benchmark.Rcheck/00install.out’ for details.
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' call to ‘multcomp’ in package code.
      Please use :: or requireNamespace() instead.
      See section 'Suggested packages' in the 'Writing R Extensions' manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      ‘algorithms’
    stripchart.AlgorithmPerformance: no visible binding for global variable
      ‘value’
    stripchart.AlgorithmPerformance: no visible binding for global variable
      ‘samples’
    Undefined global functions or variables:
      abline addEdge agopen algorithms as.formula axis barplot box cancor
      characteristics col2rgb getDefaultAttrs gray is layout legend lines
      lm median mi.plugin mtext na.omit new par points predict rect
      relation_is_strict_weak_order reshape rgb samples segments terms
      theme_text value
    Consider adding
      importFrom("grDevices", "col2rgb", "gray", "rgb")
      importFrom("graphics", "abline", "axis", "barplot", "box", "layout",
                 "legend", "lines", "mtext", "par", "points", "rect",
                 "segments")
      importFrom("methods", "is", "new")
      importFrom("stats", "as.formula", "cancor", "lm", "median", "na.omit",
                 "predict", "reshape", "terms")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

# binom

Version: 1.1-1

## In both

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Description field: should contain one or more complete sentences.
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' calls in package code:
      ‘ggplot2’ ‘lattice’ ‘polynom’ ‘tcltk’
      Please use :: or requireNamespace() instead.
      See section 'Suggested packages' in the 'Writing R Extensions' manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      definition for ‘tkdestroy’
    var.probit: no visible global function definition for ‘qnorm’
    var.probit: no visible global function definition for ‘dnorm’
    Undefined global functions or variables:
      abline aes_string aggregate approx axis binomial box col.whitebg
      current.panel.limits dbeta dbinom dnorm facet_wrap find geom_polygon
      ggplot heat.colors integral legend lines llines lpoints lpolygon
      ltext optim panel.abline panel.levelplot panel.xyplot pbeta plot
      pnorm polynomial predict qbeta qnorm rbinom reorder spline tclObj
      tclVar theme_bw tkbutton tkcheckbutton tkdestroy tkframe tklabel
      tkpack tkradiobutton tkscale tktoplevel tkwm.title trellis.par.get
      trellis.par.set uniroot xlab xlim xyplot ylab ylim
    Consider adding
      importFrom("grDevices", "heat.colors")
      importFrom("graphics", "abline", "axis", "box", "legend", "lines",
                 "plot")
      importFrom("stats", "aggregate", "approx", "binomial", "dbeta",
                 "dbinom", "dnorm", "optim", "pbeta", "pnorm", "predict",
                 "qbeta", "qnorm", "rbinom", "reorder", "spline", "uniroot")
      importFrom("utils", "find")
    to your NAMESPACE file.
    ```

# biobroom

Version: 1.12.1

## In both

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' call to ‘DESeq2’ in package code.
      Please use :: or requireNamespace() instead.
      See section 'Suggested packages' in the 'Writing R Extensions' manual.
    Missing or unexported object: ‘dplyr::tbl_dt’
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      for ‘colData’
    tidy.deSet: no visible global function definition for ‘exprs<-’
    tidy.deSet: no visible binding for global variable ‘value’
    tidy.deSet: no visible binding for global variable ‘gene’
    tidy.deSet: no visible global function definition for ‘pData’
    tidy.qvalue: no visible binding for global variable ‘smoothed’
    tidy.qvalue: no visible binding for global variable ‘pi0’
    tidy.qvalue: no visible binding for global variable ‘lambda’
    tidy_matrix: no visible binding for global variable ‘value’
    tidy_matrix: no visible binding for global variable ‘gene’
    Undefined global functions or variables:
      . DGEList calcNormFactors colData counts design end estimate
      estimateSizeFactors exprs<- fData<- gene gr is lambda model.matrix
      p.adjust pData pData<- pi0 protein rowRanges sample.id seqnames
      setNames smoothed start tbl_dt term value voom voomWithQualityWeights
    Consider adding
      importFrom("methods", "is")
      importFrom("stats", "end", "model.matrix", "p.adjust", "setNames",
                 "start")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

# BiocOncoTK

Version: 1.0.3

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.9Mb
      sub-directories of 1Mb or more:
        data   4.0Mb
        doc    1.7Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      ‘seqlengths’
    checkCache_patel: no visible global function definition for
      ‘BiocFileCache’
    chrbounds_basic: no visible global function definition for ‘seqlengths’
    rainfall: no visible global function definition for ‘genome’
    rainfall: no visible global function definition for ‘ggplot’
    rainfall: no visible global function definition for ‘aes’
    rainfall: no visible global function definition for ‘geom_point’
    rainfall: no visible global function definition for ‘theme’
    rainfall: no visible global function definition for ‘element_blank’
    rainfall: no visible global function definition for ‘ylab’
    rainfall: no visible global function definition for ‘xlab’
    rainfall: no visible global function definition for ‘geom_vline’
    rainfall: no visible global function definition for
      ‘scale_x_continuous’
    rainfall: no visible global function definition for ‘ggtitle’
    rainfall: no visible global function definition for ‘geom_text’
    Undefined global functions or variables:
      BiocFileCache aes element_blank genome geom_point geom_text
      geom_vline ggplot ggtitle scale_x_continuous seqlengths theme xlab
      ylab
    ```

# biomod2

Version: 3.3-7

## In both

*   checking DESCRIPTION meta-information ... NOTE
    ```
    BugReports field is not a suitable URL but contains an email address
      which will be used as from R 3.4.0
    ```

*   checking dependencies in R code ... NOTE
    ```
    Missing or unexported object: ‘gam::step.gam’
    ```

# BioNetStat

Version: 1.0.4

## In both

*   checking whether package ‘BioNetStat’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘IRanges’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/BioNetStat/new/BioNetStat.Rcheck/00install.out’ for details.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.1Mb
      sub-directories of 1Mb or more:
        extdata   1.9Mb
        shiny     1.6Mb
    ```

# bioplots

Version: 0.0.1

## In both

*   checking files in ‘vignettes’ ... NOTE
    ```
    Package has no Sweave vignette sources and no VignetteBuilder field.
    ```

# BioStatR

Version: 2.0.0

## In both

*   checking R code for possible problems ... NOTE
    ```
    ...
    cvar: no visible global function definition for ‘sd’
    eta2: no visible global function definition for ‘lm’
    eta2: no visible global function definition for ‘as.formula’
    gg_qqplot: no visible binding for global variable ‘qnorm’
    gg_qqplot: no visible global function definition for ‘quantile’
    panel.hist: no visible global function definition for ‘par’
    panel.hist: no visible global function definition for ‘hist’
    panel.hist: no visible global function definition for ‘rect’
    plotcdf2: no visible global function definition for ‘colorRampPalette’
    plotcdf2 : jet.colors: no visible global function definition for ‘gray’
    plotcdf2: no visible global function definition for ‘persp’
    poi.ci: no visible global function definition for ‘qchisq’
    Undefined global functions or variables:
      as.formula colorRampPalette gray hist lm par persp qchisq qf qnorm
      quantile rect sd
    Consider adding
      importFrom("grDevices", "colorRampPalette", "gray")
      importFrom("graphics", "hist", "par", "persp", "rect")
      importFrom("stats", "as.formula", "lm", "qchisq", "qf", "qnorm",
                 "quantile", "sd")
    to your NAMESPACE file.
    ```

# biotmle

Version: 1.4.0

## In both

*   checking R code for possible problems ... NOTE
    ```
    .biotmle: no visible global function definition for ‘new’
    Undefined global functions or variables:
      new
    Consider adding
      importFrom("methods", "new")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

# blandr

Version: 0.5.1

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘R6’ ‘knitr’
      All declared Imports should be used.
    ```

# blkbox

Version: 1.0

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘bigrf’
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘glmnet’ ‘gtools’ ‘knitr’ ‘nnet’ ‘parallel’ ‘rJava’ ‘reshape’
      ‘rmarkdown’ ‘shinyjs’
      All declared Imports should be used.
    Missing or unexported object: ‘xgboost::predict’
    ```

# blockseg

Version: 0.5.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘shiny’
      All declared Imports should be used.
    ```

# BloodCancerMultiOmics2017

Version: 1.0.2

## In both

*   R CMD check timed out
    

*   checking installed package size ... NOTE
    ```
      installed size is 115.4Mb
      sub-directories of 1Mb or more:
        data     80.0Mb
        doc      26.5Mb
        extdata   8.5Mb
    ```

# blorr

Version: 0.1.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘descriptr’
      All declared Imports should be used.
    ```

# blscrapeR

Version: 3.1.3

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      > library(testthat)
      > library(blscrapeR)
      > 
      > test_check("blscrapeR")
      ── 1. Error: (unknown) (@test_qcew_api.R#8)  ───────────────────────────────────
      cannot open URL 'https://data.bls.gov/cew/data/api/2013/1/industry/5112.csv'
      1: download.file(url, temp, quiet = TRUE) at testthat/test_qcew_api.R:8
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 2 SKIPPED: 0 FAILED: 1
      1. Error: (unknown) (@test_qcew_api.R#8) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

# bmlm

Version: 1.3.8

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘methods’
      All declared Imports should be used.
    ```

# bmmix

Version: 0.1-2

## In both

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Description field: should contain one or more complete sentences.
    ```

*   checking R code for possible problems ... NOTE
    ```
    bmmix : LL.y: no visible global function definition for ‘dmultinom’
    bmmix : LL.x : <anonymous>: no visible global function definition for
      ‘dmultinom’
    bmmix : LPrior.alpha: no visible global function definition for ‘dexp’
    bmmix : alpha.move: no visible global function definition for ‘rnorm’
    bmmix : alpha.move: no visible global function definition for ‘runif’
    bmmix : phi.move: no visible global function definition for ‘rnorm’
    bmmix : phi.move: no visible global function definition for ‘runif’
    bmmix: no visible global function definition for ‘read.table’
    Undefined global functions or variables:
      dexp dmultinom read.table rnorm runif
    Consider adding
      importFrom("stats", "dexp", "dmultinom", "rnorm", "runif")
      importFrom("utils", "read.table")
    to your NAMESPACE file.
    ```

# BMSC

Version: 0.1.1

## In both

*   checking whether package ‘BMSC’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/BMSC/new/BMSC.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘BMSC’ ...
** package ‘BMSC’ successfully unpacked and MD5 sums checked
** libs
"/Library/Frameworks/R.framework/Resources/bin/Rscript" -e "source(file.path('..', 'tools', 'make_cc.R')); make_cc(commandArgs(TRUE))" stan_files/linReg.stan
"/Library/Frameworks/R.framework/Resources/bin/Rscript" -e "source(file.path('..', 'tools', 'make_cc.R')); make_cc(commandArgs(TRUE))" stan_files/linRegHorseHoe.stan
"/Library/Frameworks/R.framework/Resources/bin/Rscript" -e "source(file.path('..', 'tools', 'make_cc.R')); make_cc(commandArgs(TRUE))" stan_files/linRegHorseHoeUnConstr.stan
"/Library/Frameworks/R.framework/Resources/bin/Rscript" -e "source(file.path('..', 'tools', 'make_cc.R')); make_cc(commandArgs(TRUE))" stan_files/linRegUnConstr.stan
Wrote C++ file "stan_files/linRegUnConstr.cc"
Wrote C++ file "stan_files/linReg.cc"
Wrote C++ file "stan_files/linRegHorseHoe.cc"
Error in readRDS("/var/folders/lb/xhxqmcrd7gv302_b1pdfykh80000gn/T//Rtmp2BA6ai/file6fc2694beaf9") : 
  error reading from connection
Calls: .Last -> readRDS
Execution halted
Wrote C++ file "stan_files/linRegHorseHoeUnConstr.cc"
make: *** [stan_files/linReg.cc] Error 1
make: *** Waiting for unfinished jobs....
rm stan_files/linRegUnConstr.cc stan_files/linRegHorseHoe.cc stan_files/linRegHorseHoeUnConstr.cc stan_files/linReg.cc
ERROR: compilation failed for package ‘BMSC’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/BMSC/new/BMSC.Rcheck/BMSC’

```
### CRAN

```
* installing *source* package ‘BMSC’ ...
** package ‘BMSC’ successfully unpacked and MD5 sums checked
** libs
"/Library/Frameworks/R.framework/Resources/bin/Rscript" -e "source(file.path('..', 'tools', 'make_cc.R')); make_cc(commandArgs(TRUE))" stan_files/linReg.stan
"/Library/Frameworks/R.framework/Resources/bin/Rscript" -e "source(file.path('..', 'tools', 'make_cc.R')); make_cc(commandArgs(TRUE))" stan_files/linRegHorseHoe.stan
"/Library/Frameworks/R.framework/Resources/bin/Rscript" -e "source(file.path('..', 'tools', 'make_cc.R')); make_cc(commandArgs(TRUE))" stan_files/linRegHorseHoeUnConstr.stan
"/Library/Frameworks/R.framework/Resources/bin/Rscript" -e "source(file.path('..', 'tools', 'make_cc.R')); make_cc(commandArgs(TRUE))" stan_files/linRegUnConstr.stan
Wrote C++ file "stan_files/linRegUnConstr.cc"
Wrote C++ file "stan_files/linRegHorseHoeUnConstr.cc"
Wrote C++ file "stan_files/linRegHorseHoe.cc"
Wrote C++ file "stan_files/linReg.cc"
Error in readRDS("/var/folders/lb/xhxqmcrd7gv302_b1pdfykh80000gn/T//Rtmp2BA6ai/file6fc24add57a9") : 
  error reading from connection
Calls: .Last -> readRDS
Execution halted
make: *** [stan_files/linRegHorseHoeUnConstr.cc] Error 1
make: *** Waiting for unfinished jobs....
rm stan_files/linRegUnConstr.cc stan_files/linRegHorseHoe.cc stan_files/linRegHorseHoeUnConstr.cc stan_files/linReg.cc
ERROR: compilation failed for package ‘BMSC’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/BMSC/old/BMSC.Rcheck/BMSC’

```
# BNSP

Version: 2.0.8

## In both

*   checking whether package ‘BNSP’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/BNSP/new/BNSP.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘BNSP’ ...
** package ‘BNSP’ successfully unpacked and MD5 sums checked
** libs
/usr/local/clang4/bin/clang -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/BNSP/cubature/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c BNSP_init.c -o BNSP_init.o
/usr/local/clang4/bin/clang -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/BNSP/cubature/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c OneResLtnt.c -o OneResLtnt.o
/usr/local/clang4/bin/clang -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/BNSP/cubature/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c mvrmGAM.c -o mvrmGAM.o
OneResLtnt.c:29:10: fatal error: 'gsl/gsl_matrix.h' file not found
#include <gsl/gsl_matrix.h>
         ^~~~~~~~~~~~~~~~~~
mvrmGAM.c:26:10: fatal error: 'gsl/gsl_matrix.h' file not found
#include <gsl/gsl_matrix.h>
         ^~~~~~~~~~~~~~~~~~
1 error generated.
make: *** [mvrmGAM.o] Error 1
make: *** Waiting for unfinished jobs....
1 error generated.
make: *** [OneResLtnt.o] Error 1
ERROR: compilation failed for package ‘BNSP’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/BNSP/new/BNSP.Rcheck/BNSP’

```
### CRAN

```
* installing *source* package ‘BNSP’ ...
** package ‘BNSP’ successfully unpacked and MD5 sums checked
** libs
/usr/local/clang4/bin/clang -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/BNSP/cubature/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c BNSP_init.c -o BNSP_init.o
/usr/local/clang4/bin/clang -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/BNSP/cubature/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c OneResLtnt.c -o OneResLtnt.o
/usr/local/clang4/bin/clang -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/BNSP/cubature/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c mvrmGAM.c -o mvrmGAM.o
mvrmGAM.c:26:10: fatal error: 'gsl/gsl_matrix.h' file not found
OneResLtnt.c:#include <gsl/gsl_matrix.h>
         ^~~~~~~~~~~~~~~~~~
29:10: fatal error: 'gsl/gsl_matrix.h' file not found
#include <gsl/gsl_matrix.h>
         ^~~~~~~~~~~~~~~~~~
1 error generated.
make: *** [mvrmGAM.o] Error 1
make: *** Waiting for unfinished jobs....
1 error generated.
make: *** [OneResLtnt.o] Error 1
ERROR: compilation failed for package ‘BNSP’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/BNSP/old/BNSP.Rcheck/BNSP’

```
# bodenmiller

Version: 0.1

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  8.9Mb
      sub-directories of 1Mb or more:
        data   8.7Mb
    ```

# bootnet

Version: 1.1.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘psych’
      All declared Imports should be used.
    ```

# bossMaps

Version: 0.1.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘rgdal’ ‘tidyr’
      All declared Imports should be used.
    ```

# BPRMeth

Version: 1.6.0

## In both

*   checking whether package ‘BPRMeth’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/BPRMeth/new/BPRMeth.Rcheck/00install.out’ for details.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.9Mb
      sub-directories of 1Mb or more:
        doc       2.5Mb
        extdata   1.5Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘BiocStyle’
      All declared Imports should be used.
    ```

# BrailleR

Version: 0.29.1

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘installr’
    ```

# brainGraph

Version: 2.2.0

## In both

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking: ‘RGtk2’ ‘cairoDevice’
    ```

# branchpointer

Version: 1.6.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    Loading required package: lattice
    Loading required package: ggplot2
    Warning in has_utility("pdfcrop") :
      pdfcrop not installed or not in PATH
    filtering for SNPs in branchpoint windows
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'branchpointer.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `beramono.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.87 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.6Mb
      sub-directories of 1Mb or more:
        R         4.2Mb
        extdata   1.1Mb
    ```

# brazilmaps

Version: 0.1.0

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘sf’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# breathtestcore

Version: 0.4.5

## In both

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘breathteststan’
    ```

# BreedingSchemeLanguage

Version: 0.9.6

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘lme4’
      All declared Imports should be used.
    ```

# brms

Version: 2.6.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error: processing vignette 'brms_multilevel.ltx' failed with diagnostics:
    Running 'texi2dvi' on 'brms_multilevel.ltx' failed.
    LaTeX errors:
    ! LaTeX Error: File `microtype.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.43 ^^M
            
    !  ==> Fatal error occurred, no output PDF file produced!
    Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  7.4Mb
      sub-directories of 1Mb or more:
        R     4.1Mb
        doc   2.4Mb
    ```

# broom.mixed

Version: 0.2.3

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘glmmADMB’
    ```

# bsam

Version: 1.1.2

## In both

*   checking whether package ‘bsam’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/bsam/new/bsam.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘bsam’ ...
** package ‘bsam’ successfully unpacked and MD5 sums checked
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
Error: package or namespace load failed for ‘rjags’:
 .onLoad failed in loadNamespace() for 'rjags', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/bsam/rjags/libs/rjags.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/bsam/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/bsam/rjags/libs/rjags.so
  Reason: image not found
Error : package ‘rjags’ could not be loaded
ERROR: lazy loading failed for package ‘bsam’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/bsam/new/bsam.Rcheck/bsam’

```
### CRAN

```
* installing *source* package ‘bsam’ ...
** package ‘bsam’ successfully unpacked and MD5 sums checked
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
Error: package or namespace load failed for ‘rjags’:
 .onLoad failed in loadNamespace() for 'rjags', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/bsam/rjags/libs/rjags.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/bsam/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/bsam/rjags/libs/rjags.so
  Reason: image not found
Error : package ‘rjags’ could not be loaded
ERROR: lazy loading failed for package ‘bsam’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/bsam/old/bsam.Rcheck/bsam’

```
# BTSPAS

Version: 2014.0901

## In both

*   checking whether package ‘BTSPAS’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/BTSPAS/new/BTSPAS.Rcheck/00install.out’ for details.
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘BRugs’
    ```

## Installation

### Devel

```
* installing *source* package ‘BTSPAS’ ...
** package ‘BTSPAS’ successfully unpacked and MD5 sums checked
** R
** demo
** inst
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rjags', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/BTSPAS/rjags/libs/rjags.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/BTSPAS/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/BTSPAS/rjags/libs/rjags.so
  Reason: image not found
ERROR: lazy loading failed for package ‘BTSPAS’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/BTSPAS/new/BTSPAS.Rcheck/BTSPAS’

```
### CRAN

```
* installing *source* package ‘BTSPAS’ ...
** package ‘BTSPAS’ successfully unpacked and MD5 sums checked
** R
** demo
** inst
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rjags', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/BTSPAS/rjags/libs/rjags.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/BTSPAS/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/BTSPAS/rjags/libs/rjags.so
  Reason: image not found
ERROR: lazy loading failed for package ‘BTSPAS’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/BTSPAS/old/BTSPAS.Rcheck/BTSPAS’

```
# BubbleTree

Version: 2.10.0

## In both

*   checking whether package ‘BubbleTree’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/BubbleTree/new/BubbleTree.Rcheck/00install.out’ for details.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 29.2Mb
      sub-directories of 1Mb or more:
        data  23.4Mb
        doc    5.3Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    annoByOverlap,Annotate: no visible binding for global variable
      'queryHits'
    Undefined global functions or variables:
      queryHits
    ```

# CAFE

Version: 1.16.0

## In both

*   checking whether package ‘CAFE’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘biovizBase’ was built under R version 3.5.1
      Warning: package 'GenomicRanges' was built under R version 3.5.1
      Warning: package 'IRanges' was built under R version 3.5.1
      Warning: package 'ggbio' was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/CAFE/new/CAFE.Rcheck/00install.out’ for details.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Description field: should contain one or more complete sentences.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    facetPlot: no visible global function definition for 'dev.off'
    facetPlot: no visible global function definition for 'na.omit'
    fisher.method: no visible global function definition for 'pchisq'
    fisherExact: no visible global function definition for 'fisher.test'
    makelevels: no visible global function definition for 'na.omit'
    rawPlot: no visible global function definition for 'png'
    rawPlot: no visible global function definition for 'dev.off'
    rawPlot: no visible global function definition for 'data'
    slidPlot: no visible global function definition for 'png'
    slidPlot: no visible global function definition for 'dev.off'
    slidPlot: no visible global function definition for 'data'
    Undefined global functions or variables:
      chisq.test data dev.off download.file fisher.test na.omit p.adjust
      pchisq png read.table setTxtProgressBar txtProgressBar
    Consider adding
      importFrom("grDevices", "dev.off", "png")
      importFrom("stats", "chisq.test", "fisher.test", "na.omit", "p.adjust",
                 "pchisq")
      importFrom("utils", "data", "download.file", "read.table",
                 "setTxtProgressBar", "txtProgressBar")
    to your NAMESPACE file.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    
    
    Writing plot to FacetSlidMul
    
    
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'CAFE-manual.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `beramono.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.87 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# caffsim

Version: 0.2.2

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘markdown’
      All declared Imports should be used.
    ```

# CAGEr

Version: 1.22.3

## Newly fixed

*   R CMD check timed out
    

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  6.9Mb
      sub-directories of 1Mb or more:
        R     2.0Mb
        doc   3.1Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    clusterCTSS,CAGEset : getCTSSdataSE: no visible binding for '<<-'
      assignment to ‘idx’
    clusterCTSS,CAGEset : getCTSSdataSE: no visible binding for global
      variable ‘idx’
    clusterCTSS,CAGEset : getCTSSdatadf: no visible binding for '<<-'
      assignment to ‘idx’
    clusterCTSS,CAGEset : getCTSSdatadf: no visible binding for global
      variable ‘idx’
    clusterCTSS,CAGEset: no visible binding for global variable ‘idx’
    Undefined global functions or variables:
      idx
    ```

# cancensus

Version: 0.1.7

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘sf’
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘sf’
    ```

# CAnD

Version: 1.12.0

## In both

*   checking R code for possible problems ... NOTE
    ```
    CAnD : pairedTtest: no visible global function definition for ‘t.test’
    barPlotAncest: no visible binding for global variable ‘id’
    barPlotAncest: no visible binding for global variable ‘value’
    barPlotAncest: no visible binding for global variable ‘variable’
    calc_combP: no visible binding for global variable ‘var’
    calc_combP: no visible global function definition for ‘pchisq’
    plotPvals: no visible binding for global variable ‘numPs’
    plotPvals: no visible binding for global variable ‘pval’
    Undefined global functions or variables:
      id numPs pchisq pval t.test value var variable
    Consider adding
      importFrom("stats", "pchisq", "t.test", "var")
    to your NAMESPACE file.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'CAnD.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `beramono.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.87 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# capm

Version: 0.13.5

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘sf’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# caret

Version: 6.0-80

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  7.4Mb
      sub-directories of 1Mb or more:
        R        2.0Mb
        data     1.5Mb
        models   2.4Mb
    ```

# cartools

Version: 0.1.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘animation’ ‘devtools’ ‘gapminder’ ‘knitr’ ‘rlist’ ‘rmarkdown’
      ‘roxygen2’ ‘sde’ ‘shiny’ ‘tidyverse’ ‘usethis’ ‘utils’
      All declared Imports should be used.
    ```

# CAST

Version: 0.3.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
         validation strategies for spatial-temporal models. To decrease
         overfitting and improve model performances, the package implements
         a forward feature selection that selects suitable predictor
         variables in view to their contribution to the target-oriented
         performance.
    
    _D_e_t_a_i_l_s:
    
         'caret' Applications for Spatio-Temporal models
    
    _A_u_t_h_o_r(_s):
    
         Hanna Meyer, Christoph Reudenbach, Marvin Ludwig, Thomas Nauss
         _Maintainer:_ Hanna Meyer <email: hanna.meyer@geo.uni-marburg.de>
    
    
    Quitting from lines 65-73 (CAST-intro.Rmd) 
    Error: processing vignette 'CAST-intro.Rmd' failed with diagnostics:
    package or namespace load failed for 'mapview' in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]):
     there is no package called 'sf'
    Execution halted
    ```

# CATALYST

Version: 1.4.2

## In both

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  9.1Mb
      sub-directories of 1Mb or more:
        data   3.1Mb
        doc    5.1Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    plotDiffHeatmap,matrix-SummarizedExperiment: no visible binding for
      global variable ‘cluster_id’
    plotDiffHeatmap,matrix-SummarizedExperiment: no visible binding for
      global variable ‘sample_id’
    Undefined global functions or variables:
      cluster_id sample_id
    ```

# cate

Version: 1.0.4

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'cate-vignette.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `mathpazo.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.56 \usepackage
                    [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# catenary

Version: 1.1.2

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘tidyverse’
      All declared Imports should be used.
    ```

# CausalImpact

Version: 1.2.3

## In both

*   checking R code for possible problems ... NOTE
    ```
    ConstructModel: warning in AddDynamicRegression(ss, formula, data =
      data, sigma.mean.prior = sigma.mean.prior): partial argument match of
      'sigma.mean.prior' to 'sigma.mean.prior.DEPRECATED'
    ```

# Causata

Version: 4.2-0

## In both

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Title field: should not end in a period.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    PredictivePowerCv: no visible global function definition for ‘sd’
    ReadCausataCsv: no visible global function definition for ‘read.csv’
    ToPmml.GlmnetModelDefinition: no visible global function definition for
      ‘coef’
    ToPmml.GlmnetModelDefinition: no visible global function definition for
      ‘terms.formula’
    ValidateModel: no visible global function definition for ‘predict’
    predict.GlmnetModelDefinition: no visible global function definition
      for ‘model.matrix’
    predict.GlmnetModelDefinition: no visible binding for global variable
      ‘contrasts’
    predict.GlmnetModelDefinition: no visible global function definition
      for ‘predict’
    Undefined global functions or variables:
      coef contrasts dbGetQuery median model.matrix na.omit predict
      quantile read.csv sd terms.formula
    Consider adding
      importFrom("stats", "coef", "contrasts", "median", "model.matrix",
                 "na.omit", "predict", "quantile", "sd", "terms.formula")
      importFrom("utils", "read.csv")
    to your NAMESPACE file.
    ```

# ccfa

Version: 1.1.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘doParallel’ ‘foreach’
      All declared Imports should be used.
    ```

# CDECRetrieve

Version: 0.1.2

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘lazyeval’ ‘purrr’ ‘roxygen2’
      All declared Imports should be used.
    ```

# cellHTS2

Version: 2.44.0

## In both

*   checking Rd cross-references ... WARNING
    ```
    Unknown package ‘cellHTS’ in Rd xrefs
    ```

*   checking running R code from vignettes ...
    ```
       ‘cellhts2.Rnw’ ... OK
       ‘cellhts2Complete.Rnw’ ... failed
       ‘twoChannels.Rnw’ ... OK
       ‘twoWay.Rnw’ ... OK
     WARNING
    Errors in running code in vignettes:
    when running code in ‘cellhts2Complete.Rnw’
      ...
    +     nrow(res)), la .... [TRUNCATED] 
    
    > library(KEGG.db)
    
      When sourcing ‘cellhts2Complete.R’:
    Error: there is no package called ‘KEGG.db’
    recover called non-interactively; frames dumped, use debugger() to view
    > proc.time()
       user  system elapsed 
     33.340   2.077  36.989 
    ```

*   checking package dependencies ... NOTE
    ```
    Depends: includes the non-default packages:
      ‘RColorBrewer’ ‘Biobase’ ‘genefilter’ ‘splots’ ‘vsn’ ‘hwriter’
      ‘locfit’ ‘grid’
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  7.3Mb
      sub-directories of 1Mb or more:
        KcViab   2.0Mb
        R        2.1Mb
        doc      1.6Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Package in Depends field not imported from: ‘genefilter’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    Unexported object imported by a ':::' call: ‘Biobase:::.showAnnotatedDataFrame’
      See the note in ?`:::` about the use of this operator.
    ```

*   checking for unstated dependencies in vignettes ... NOTE
    ```
    'library' or 'require' calls not declared from:
      ‘GO.db’ ‘KEGG.db’
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    `stat_bin()` using `bins = 30`. Pick better value with `binwidth`.
    Warning: Removed 24 rows containing non-finite values (stat_bin).
    Warning: Removed 502 rows containing non-finite values (stat_binhex).
    Warning: Removed 57 rows containing missing values (geom_hex).
    Warning: Removed 24 rows containing non-finite values (stat_qq).
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'cellhts2.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.30 \ifthenelse
                    {\boolean{Sweave@inconsolata}}{%^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# CellNOptR

Version: 1.26.0

## In both

*   checking whether package ‘CellNOptR’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘graph’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/CellNOptR/new/CellNOptR.Rcheck/00install.out’ for details.
    ```

*   checking examples ... WARNING
    ```
    Found the following significant warnings:
      Warning: working directory was changed to ‘/private/var/folders/lb/xhxqmcrd7gv302_b1pdfykh80000gn/T/RtmpQuW9GL’, resetting
      Warning: working directory was changed to ‘/private/var/folders/lb/xhxqmcrd7gv302_b1pdfykh80000gn/T/RtmpQuW9GL’, resetting
      Warning: working directory was changed to ‘/private/var/folders/lb/xhxqmcrd7gv302_b1pdfykh80000gn/T/RtmpQuW9GL’, resetting
      Warning: working directory was changed to ‘/private/var/folders/lb/xhxqmcrd7gv302_b1pdfykh80000gn/T/RtmpQuW9GL’, resetting
      Warning: working directory was changed to ‘/private/var/folders/lb/xhxqmcrd7gv302_b1pdfykh80000gn/T/RtmpQuW9GL’, resetting
      Warning: working directory was changed to ‘/private/var/folders/lb/xhxqmcrd7gv302_b1pdfykh80000gn/T/RtmpQuW9GL’, resetting
      Warning: working directory was changed to ‘/private/var/folders/lb/xhxqmcrd7gv302_b1pdfykh80000gn/T/RtmpQuW9GL’, resetting
      Warning: working directory was changed to ‘/private/var/folders/lb/xhxqmcrd7gv302_b1pdfykh80000gn/T/RtmpQuW9GL’, resetting
      Warning: working directory was changed to ‘/private/var/folders/lb/xhxqmcrd7gv302_b1pdfykh80000gn/T/RtmpQuW9GL’, resetting
      Warning: working directory was changed to ‘/private/var/folders/lb/xhxqmcrd7gv302_b1pdfykh80000gn/T/RtmpQuW9GL’, resetting
      Warning: working directory was changed to ‘/private/var/folders/lb/xhxqmcrd7gv302_b1pdfykh80000gn/T/RtmpQuW9GL’, resetting
      Warning: working directory was changed to ‘/private/var/folders/lb/xhxqmcrd7gv302_b1pdfykh80000gn/T/RtmpQuW9GL’, resetting
      Warning: working directory was changed to ‘/private/var/folders/lb/xhxqmcrd7gv302_b1pdfykh80000gn/T/RtmpQuW9GL’, resetting
      Warning: working directory was changed to ‘/private/var/folders/lb/xhxqmcrd7gv302_b1pdfykh80000gn/T/RtmpQuW9GL’, resetting
    ```

*   checking package dependencies ... NOTE
    ```
    Depends: includes the non-default packages:
      ‘RBGL’ ‘graph’ ‘hash’ ‘RCurl’ ‘Rgraphviz’ ‘XML’ ‘ggplot2’
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking installed files from ‘inst/doc’ ... NOTE
    ```
    The following files should probably not be installed:
      ‘Fig1.png’, ‘Fig5.png’
    
    Consider the use of a .Rinstignore file: see ‘Writing R Extensions’,
    or move the vignette sources from ‘inst/doc’ to ‘vignettes’.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    The following object is masked from ‘package:tools’:
    
        toHTML
    
    Loading required package: ggplot2
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'CellNOptR-vignette.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.30 \ifthenelse
                    {\boolean{Sweave@inconsolata}}{%^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# CEMiTool

Version: 1.4.2

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  8.9Mb
      sub-directories of 1Mb or more:
        data      3.1Mb
        doc       2.3Mb
        extdata   2.8Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    plot_ora,CEMiTool : <anonymous>: no visible global function definition
      for ‘head’
    plot_qq,CEMiTool: no visible binding for global variable ‘data’
    plot_sample_tree,CEMiTool: no visible global function definition for
      ‘hclust’
    plot_sample_tree,CEMiTool: no visible global function definition for
      ‘dist’
    plot_sample_tree,CEMiTool: no visible global function definition for
      ‘dev.off’
    save_plots,CEMiTool : <anonymous>: no visible global function
      definition for ‘dev.off’
    save_plots,CEMiTool: no visible global function definition for
      ‘dev.off’
    Undefined global functions or variables:
      ..eq.label.. ..rr.label.. Mean Variance as.dist cem data dev.off dist
      hclust head tail var
    Consider adding
      importFrom("grDevices", "dev.off")
      importFrom("stats", "as.dist", "dist", "hclust", "var")
      importFrom("utils", "data", "head", "tail")
    to your NAMESPACE file.
    ```

# census

Version: 0.2.0

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘census-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: getCensus
    > ### Title: Scrape US Census Data
    > ### Aliases: getCensus
    > 
    > ### ** Examples
    > 
    > radius <- 5
    > cities <- c("Boston", "Los Angeles")
    > df <- getCensus(cities, radius)
    Information from URL : http://maps.googleapis.com/maps/api/geocode/json?address=Boston&sensor=false
    Warning: geocode failed with status OVER_QUERY_LIMIT, location = "Boston"
    Information from URL : http://maps.googleapis.com/maps/api/geocode/json?address=Los%20Angeles&sensor=false
    Warning: geocode failed with status OVER_QUERY_LIMIT, location = "Los Angeles"
    Error in `[.data.frame`(geo.reply, , c("locality", "lon", "lat")) : 
      undefined columns selected
    Calls: getCensus -> [ -> [.data.frame
    Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Loading required package: ggmap
    Loading required package: ggplot2
    Loading required package: RCurl
    Loading required package: bitops
    Information from URL : http://maps.googleapis.com/maps/api/geocode/json?address=Chicago,%20IL&sensor=false
    Warning: geocode failed with status OVER_QUERY_LIMIT, location = "Chicago, IL"
    Information from URL : http://maps.googleapis.com/maps/api/geocode/json?address=New%20York&sensor=false
    Warning: geocode failed with status OVER_QUERY_LIMIT, location = "New York"
    Quitting from lines 16-23 (getting-started.Rmd) 
    Error: processing vignette 'getting-started.Rmd' failed with diagnostics:
    undefined columns selected
    Execution halted
    ```

# Census2016

Version: 0.2.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘data.table’
      All declared Imports should be used.
    ```

# ceterisParibus

Version: 0.3.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘knitr’
      All declared Imports should be used.
    ```

# CGPfunctions

Version: 0.4

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘devtools’
      All declared Imports should be used.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Packages unavailable to check Rd xrefs: ‘BSDA’, ‘janitor’
    ```

# ChainLadder

Version: 0.2.7

## In both

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    Please cite the 'systemfit' package as:
    Arne Henningsen and Jeff D. Hamann (2007). systemfit: A Package for Estimating Systems of Simultaneous Equations in R. Journal of Statistical Software 23(4), 1-40. http://www.jstatsoft.org/v23/i04/.
    
    If you have questions, suggestions, or comments regarding the 'systemfit' package, please use a forum or 'tracker' at systemfit's R-Forge site:
    https://r-forge.r-project.org/projects/systemfit/
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'ChainLadder.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.30 \ifthenelse
                    {\boolean{Sweave@inconsolata}}{%^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# ChannelAttributionApp

Version: 1.1

## In both

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Checking should be performed on sources prepared by ‘R CMD build’.
    ```

# ChemmineR

Version: 3.32.1

## In both

*   checking examples ... ERROR
    ```
    ...
    > ### Keywords: datatable,sdfset,compounds,display
    > 
    > ### ** Examples
    > 
    > 	library(ChemmineR)
    > 	library(DT)
    > 	data(sdfsample)
    > 
    > 	# this will open a browser to display the result
    > 	x=SDFDataTable(sdfsample[1:3]) 
    Error: package or namespace load failed for ‘ChemmineOB’:
     .onLoad failed in loadNamespace() for 'ChemmineOB', details:
      call: dyn.load(file, DLLpath = DLLpath, ...)
      error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/ChemmineR/ChemmineOB/libs/ChemmineOB.so':
      dlopen(/Users/max/github/ggplot2/revdep/library.noindex/ChemmineR/ChemmineOB/libs/ChemmineOB.so, 10): Library not loaded: /usr/local/opt/open-babel/lib/libopenbabel.5.dylib
      Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/ChemmineR/ChemmineOB/libs/ChemmineOB.so
      Reason: image not found
    Error in .ensureOB() : 
      ChemmineOB is required to make use of this function. This package can be installed from BioConductor with the  command 'biocLite("ChemmineOB").  It is not currently available for windows however. See http://bioconductor.org/packages/devel/bioc/html/ChemmineOB.html for more information
    Calls: SDFDataTable -> cbind -> lapply -> FUN -> sdf2image -> .ensureOB
    Execution halted
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/runTests.R’ failed.
    Last 13 lines of output:
           test.3dCoords 
           test.canonicalize 
           test.exactMassOB 
           test.genAPDescriptors 
           test.parseV3000 
           test.propOB 
           test.smartsSearchOB 
      
         test_ploting.R 
           test.openBabelPlot 
      
      
      Error in BiocGenerics:::testPackage("ChemmineR") : 
        unit tests failed for package ChemmineR
      Execution halted
    ```

*   checking whether package ‘ChemmineR’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      r_wrap.cc:1689:14: warning: explicitly assigning value of variable of type 'void *' to itself [-Wself-assign]
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/ChemmineR/new/ChemmineR.Rcheck/00install.out’ for details.
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 44-50 (ChemmineR.Rmd) 
    Error: processing vignette 'ChemmineR.Rmd' failed with diagnostics:
    package or namespace load failed for 'ChemmineOB':
     .onLoad failed in loadNamespace() for 'ChemmineOB', details:
      call: dyn.load(file, DLLpath = DLLpath, ...)
      error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/ChemmineR/ChemmineOB/libs/ChemmineOB.so':
      dlopen(/Users/max/github/ggplot2/revdep/library.noindex/ChemmineR/ChemmineOB/libs/ChemmineOB.so, 10): Library not loaded: /usr/local/opt/open-babel/lib/libopenbabel.5.dylib
      Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/ChemmineR/ChemmineOB/libs/ChemmineOB.so
      Reason: image not found
    Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘ChemmineDrugs’
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Packages listed in more than one of Depends, Imports, Suggests, Enhances:
      ‘png’ ‘ChemmineOB’
    A package should be listed in only one of these fields.
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' calls in package code:
      ‘ChemmineDrugs’ ‘ChemmineOB’ ‘RPostgreSQL’ ‘RSQLite’ ‘fmcsR’ ‘png’
      ‘snow’
      Please use :: or requireNamespace() instead.
      See section 'Suggested packages' in the 'Writing R Extensions' manual.
    Namespace in Imports field not imported from: ‘BiocGenerics’
      All declared Imports should be used.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    smartsSearchOB: no visible global function definition for
      ‘smartsSearch_OB’
    smile2sdfFile: no visible global function definition for
      ‘convertFormatFile’
    smiles2sdf: no visible global function definition for ‘convertFormat’
    smiles2sdfOB: no visible global function definition for ‘convertFormat’
    write.SMI: no visible global function definition for ‘write.table’
    Undefined global functions or variables:
      AW C1 C1.1 C2 C2.1 browseURL canonicalNumbering_OB clusterApplyLB
      clusterExport combn convertFormat convertFormatFile convertToImage
      data dev.off error exactMass_OB fingerprint_OB fmcs forEachMol pdf
      postgresqlCopyInDataframe postgresqlQuoteId postgresqlTableRef
      postgresqlgetResult postgresqlpqExec postscript prop_OB rainbow
      read.delim read.table rgb smartsSearch_OB smiles str string
      write.table
    Consider adding
      importFrom("grDevices", "dev.off", "pdf", "postscript", "rainbow",
                 "rgb")
      importFrom("utils", "browseURL", "combn", "data", "read.delim",
                 "read.table", "str", "write.table")
    to your NAMESPACE file.
    ```

*   checking compiled code ... NOTE
    ```
    File ‘ChemmineR/libs/ChemmineR.so’:
      Found ‘__ZNSt3__14cerrE’, possibly from ‘std::cerr’ (C++)
        Objects: ‘desc.o’, ‘formats.o’, ‘script.o’
      Found ‘__ZNSt3__14coutE’, possibly from ‘std::cout’ (C++)
        Object: ‘cluster.o’
    
    Compiled code should not call entry points which might terminate R nor
    write to stdout/stderr instead of to the console, nor use Fortran I/O
    nor system RNGs.
    
    See ‘Writing portable packages’ in the ‘Writing R Extensions’ manual.
    ```

# childsds

Version: 0.6.7

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘gamlss.dist’
      All declared Imports should be used.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 20 marked UTF-8 strings
    ```

# ChIPexoQual

Version: 1.4.0

## In both

*   checking whether package ‘ChIPexoQual’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘DelayedArray’ was built under R version 3.5.1
      Warning: package ‘Rsamtools’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/ChIPexoQual/new/ChIPexoQual.Rcheck/00install.out’ for details.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Failed with error:  'package 'DelayedArray' could not be loaded'
      Error in .requirePackage(package) : 
        unable to find required package 'ChIPexoQual'
      Calls: <Anonymous> ... getClass -> getClassDef -> .classEnv -> .requirePackage
      Execution halted
    ```

# ChIPQC

Version: 1.16.1

## In both

*   checking whether package ‘ChIPQC’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘DelayedArray’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/ChIPQC/new/ChIPQC.Rcheck/00install.out’ for details.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Unexported object imported by a ':::' call: ‘S4Vectors:::tabulate2’
      See the note in ?`:::` about the use of this operator.
    ```

*   checking foreign function calls ... NOTE
    ```
    Foreign function call to a different package:
      .Call("rle_sum_any", ..., PACKAGE = "chipseq")
    See chapter ‘System and foreign language interfaces’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    makeRegiPlot: no visible binding for global variable ‘Sample’
    makeRegiPlot: no visible binding for global variable ‘GenomicIntervals’
    makeRegiPlot: no visible binding for global variable ‘log2_Enrichment’
    makeSSDPlot: no visible binding for global variable ‘Sample’
    makeSSDPlot: no visible binding for global variable ‘SSD’
    makeSSDPlot: no visible global function definition for ‘geom_point’
    sampleQC: no visible global function definition for ‘seqlevels<-’
    plotCC,ChIPQCexperiment: no visible binding for global variable
      ‘Sample’
    plotCC,list: no visible binding for global variable ‘Sample’
    plotPeakProfile,ChIPQCexperiment: no visible binding for global
      variable ‘Sample’
    plotPeakProfile,list: no visible binding for global variable ‘Sample’
    Undefined global functions or variables:
      CC_Score CountsInPeaks Depth Distance FRIBL FRIP GenomicIntervals
      Reads SSD Sample Shift_Size Signal TxDb.Hsapiens.UCSC.hg38.knownGene
      geom_point log10_bp log2_Enrichment seqlengths seqlengths<-
      seqlevels<-
    Consider adding
      importFrom("stats", "SSD")
    to your NAMESPACE file.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Error in .requirePackage(package) : 
        unable to find required package 'ChIPQC'
      Calls: <Anonymous> ... getClass -> getClassDef -> .classEnv -> .requirePackage
      Execution halted
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    Fragment Length Cross-Coverage: 0.024498021220288
    Relative Cross-Coverage: 2.73653622626889
    Percentage Of Reads in GenomicFeature: 
    Percentage Of Reads in Peaks: 42.35
    Number of Peaks: 1118
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'ChIPQC.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `beramono.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.87 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# ChIPseeker

Version: 1.16.1

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.1Mb
      sub-directories of 1Mb or more:
        data   1.5Mb
        doc    2.9Mb
    ```

# ChIPSeqSpike

Version: 1.0.0

## In both

*   checking examples ... ERROR
    ```
    ...
    Processing H3K79me2_0
    	 Reading experiment bigWig file.
    	 Apply scaling factor
    	 Output bigWig file
    Processing H3K79me2_50
    	 Reading experiment bigWig file.
    	 Apply scaling factor
    	 Output bigWig file
    Processing H3K79me2_100
    	 Reading experiment bigWig file.
    	 Apply scaling factor
    	 Output bigWig file
    
    
    		 ### Step 6. Extract values ###
    Retrieving values for profiles and heatmaps.
    Extraction at pf
    Error in getPlotSetArray(tracks = files, features = gff_vec, refgenome = genome_version,  : 
      No genomes installed!
    Calls: spikePipe ... .retrieveAverageValuesList -> lapply -> lapply -> FUN -> getPlotSetArray
    Execution halted
    ```

*   checking whether package ‘ChIPSeqSpike’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘rtracklayer’ was built under R version 3.5.1
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/ChIPSeqSpike/new/ChIPSeqSpike.Rcheck/00install.out’ for details.
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
        dirname, do.call, duplicated, eval, evalq, get, grep, grepl,
        intersect, is.unsorted, lapply, lengths, mapply, match, mget,
        order, paste, pmax, pmax.int, pmin, pmin.int, rank, rbind,
        rowMeans, rowSums, rownames, sapply, setdiff, sort, table,
        tapply, union, unique, unsplit, which, which.max, which.min
    
    Loading required package: S4Vectors
    
    Attaching package: 'S4Vectors'
    
    The following object is masked from 'package:base':
    
        expand.grid
    
    Loading required package: IRanges
    Warning: package 'IRanges' was built under R version 3.5.1
    Loading required package: GenomeInfoDb
    Quitting from lines 72-98 (ChIPSeqSpike.Rmd) 
    Error: processing vignette 'ChIPSeqSpike.Rmd' failed with diagnostics:
    No genomes installed!
    Execution halted
    ```

# ChocoLattes

Version: 0.1.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘knitr’ ‘tools’
      All declared Imports should be used.
    ```

# choroplethr

Version: 3.6.3

## In both

*   checking whether package ‘choroplethr’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/choroplethr/new/choroplethr.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘choroplethr’ ...
** package ‘choroplethr’ successfully unpacked and MD5 sums checked
** R
** data
** byte-compile and prepare package for lazy loading
Error in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]) : 
  there is no package called ‘sf’
ERROR: lazy loading failed for package ‘choroplethr’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/choroplethr/new/choroplethr.Rcheck/choroplethr’

```
### CRAN

```
* installing *source* package ‘choroplethr’ ...
** package ‘choroplethr’ successfully unpacked and MD5 sums checked
** R
** data
** byte-compile and prepare package for lazy loading
Error in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]) : 
  there is no package called ‘sf’
ERROR: lazy loading failed for package ‘choroplethr’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/choroplethr/old/choroplethr.Rcheck/choroplethr’

```
# choroplethrAdmin1

Version: 1.1.1

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 18.0Mb
      sub-directories of 1Mb or more:
        data  17.9Mb
    ```

# chorrrds

Version: 0.1.7

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.4Mb
      sub-directories of 1Mb or more:
        data   5.1Mb
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 8008 marked UTF-8 strings
    ```

# chromstaR

Version: 1.6.2

## In both

*   R CMD check timed out
    

*   checking whether package ‘chromstaR’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/chromstaR/new/chromstaR.Rcheck/00install.out’ for details.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  7.2Mb
      sub-directories of 1Mb or more:
        data   4.5Mb
        doc    1.9Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    Chromstar: multiple local function definitions for ‘parallel.helper’
      with different formal arguments
    ```

# CINNA

Version: 1.1.50

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘circlize’
      All declared Imports should be used.
    ```

# cjoint

Version: 2.1.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘shinyBS’
      All declared Imports should be used.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 29 marked UTF-8 strings
    ```

# classifierplots

Version: 1.3.3

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘Rcpp’
      All declared Imports should be used.
    ```

# classify

Version: 1.3

## In both

*   checking whether package ‘classify’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/classify/new/classify.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘classify’ ...
** package ‘classify’ successfully unpacked and MD5 sums checked
** libs
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/classify/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c exp.cpp -o exp.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/classify/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c gpcm.cpp -o gpcm.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/classify/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c rcpp_w_lord.cpp -o rcpp_w_lord.o
/usr/local/clang4/bin/clang++ -dynamiclib -Wl,-headerpad_max_install_names -undefined dynamic_lookup -single_module -multiply_defined suppress -L/Library/Frameworks/R.framework/Resources/lib -L/usr/local/clang4/lib -o classify.so exp.o gpcm.o rcpp_w_lord.o -F/Library/Frameworks/R.framework/.. -framework R -Wl,-framework -Wl,CoreFoundation
ld: warning: text-based stub file /System/Library/Frameworks//CoreFoundation.framework/CoreFoundation.tbd and library file /System/Library/Frameworks//CoreFoundation.framework/CoreFoundation are out of sync. Falling back to library file for linking.
installing to /Users/max/github/ggplot2/revdep/checks.noindex/classify/new/classify.Rcheck/classify/libs
** R
** data
** inst
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rjags', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/classify/rjags/libs/rjags.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/classify/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/classify/rjags/libs/rjags.so
  Reason: image not found
ERROR: lazy loading failed for package ‘classify’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/classify/new/classify.Rcheck/classify’

```
### CRAN

```
* installing *source* package ‘classify’ ...
** package ‘classify’ successfully unpacked and MD5 sums checked
** libs
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/classify/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c exp.cpp -o exp.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/classify/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c gpcm.cpp -o gpcm.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/classify/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c rcpp_w_lord.cpp -o rcpp_w_lord.o
/usr/local/clang4/bin/clang++ -dynamiclib -Wl,-headerpad_max_install_names -undefined dynamic_lookup -single_module -multiply_defined suppress -L/Library/Frameworks/R.framework/Resources/lib -L/usr/local/clang4/lib -o classify.so exp.o gpcm.o rcpp_w_lord.o -F/Library/Frameworks/R.framework/.. -framework R -Wl,-framework -Wl,CoreFoundation
ld: warning: text-based stub file /System/Library/Frameworks//CoreFoundation.framework/CoreFoundation.tbd and library file /System/Library/Frameworks//CoreFoundation.framework/CoreFoundation are out of sync. Falling back to library file for linking.
installing to /Users/max/github/ggplot2/revdep/checks.noindex/classify/old/classify.Rcheck/classify/libs
** R
** data
** inst
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rjags', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/classify/rjags/libs/rjags.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/classify/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/classify/rjags/libs/rjags.so
  Reason: image not found
ERROR: lazy loading failed for package ‘classify’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/classify/old/classify.Rcheck/classify’

```
# ClassifyR

Version: 2.0.10

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    
    The following objects are masked from 'package:base':
    
        Filter, Find, Map, Position, Reduce, anyDuplicated, append, as.data.frame, basename, cbind, colMeans, colSums,
        colnames, dirname, do.call, duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted, lapply, lengths,
        mapply, match, mget, order, paste, pmax, pmax.int, pmin, pmin.int, rank, rbind, rowMeans, rowSums, rownames,
        sapply, setdiff, sort, table, tapply, union, unique, unsplit, which, which.max, which.min
    
    
    Attaching package: 'S4Vectors'
    
    The following object is masked from 'package:base':
    
        expand.grid
    
    Loading required package: MultiAssayExperiment
    Loading required package: BiocParallel
    Quitting from lines 103-107 (ClassifyR.Rmd) 
    Error: processing vignette 'ClassifyR.Rmd' failed with diagnostics:
    cannot open the connection
    Execution halted
    ```

*   checking R code for possible problems ... NOTE
    ```
    .posterior_probs : <anonymous>: no visible global function definition
      for ‘dmvnorm’
    plotFeatureClasses,DataFrame : <anonymous>: no visible binding for
      global variable ‘measurement’
    plotFeatureClasses,DataFrame : <anonymous>: no visible binding for
      global variable ‘..density..’
    plotFeatureClasses,DataFrame : <anonymous>: no visible binding for
      global variable ‘legends grouping’
    samplesMetricMap,list: no visible binding for global variable ‘Class’
    samplesMetricMap,list: no visible binding for global variable ‘name’
    samplesMetricMap,list: no visible binding for global variable ‘type’
    samplesMetricMap,list: no visible binding for global variable ‘Metric’
    selectionPlot,list: no visible binding for global variable ‘Freq’
    Undefined global functions or variables:
      ..density.. Class Freq Metric dmvnorm legends grouping measurement
      name type
    Consider adding
      importFrom("base", "grouping")
    to your NAMESPACE file.
    ```

# classyfire

Version: 0.1-2

## In both

*   checking R code for possible problems ... NOTE
    ```
    .boxRadial: no visible global function definition for ‘predict’
    .radialSVM: no visible global function definition for ‘predict’
    cfPredict: no visible global function definition for ‘predict’
    getPerm5Num: no visible global function definition for ‘median’
    getPerm5Num: no visible global function definition for ‘quantile’
    ggClassPred: no visible global function definition for ‘ftable’
    ggEnsHist: no visible global function definition for ‘sd’
    ggPermHist: no visible global function definition for ‘sd’
    Undefined global functions or variables:
      ftable median predict quantile sd
    Consider adding
      importFrom("stats", "ftable", "median", "predict", "quantile", "sd")
    to your NAMESPACE file.
    ```

# clhs

Version: 0.7-0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘scales’ ‘sp’
      All declared Imports should be used.
    ```

# clusterfly

Version: 0.4

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: ‘rggobi’ ‘RGtk2’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# clusterProfiler

Version: 3.8.1

## In both

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘topGO’
    ```

# CNEr

Version: 1.16.1

## In both

*   R CMD check timed out
    

*   checking compiled code ... WARNING
    ```
    ...
        Objects: ‘ucsc/axt.o’, ‘ucsc/errabort.o’, ‘ucsc/obscure.o’,
          ‘ucsc/verbose.o’, ‘ucsc/os.o’
      Found ‘___stdoutp’, possibly from ‘stdout’ (C)
        Objects: ‘ucsc/common.o’, ‘ucsc/errabort.o’, ‘ucsc/verbose.o’,
          ‘ucsc/os.o’
      Found ‘_abort’, possibly from ‘abort’ (C)
        Object: ‘ucsc/errabort.o’
      Found ‘_exit’, possibly from ‘exit’ (C)
        Objects: ‘ucsc/errabort.o’, ‘ucsc/pipeline.o’
      Found ‘_printf’, possibly from ‘printf’ (C)
        Objects: ‘ceScan.o’, ‘ucsc/pipeline.o’
      Found ‘_puts’, possibly from ‘printf’ (C), ‘puts’ (C)
        Object: ‘ucsc/pipeline.o’
      Found ‘_rand’, possibly from ‘rand’ (C)
        Object: ‘ucsc/obscure.o’
    
    Compiled code should not call entry points which might terminate R nor
    write to stdout/stderr instead of to the console, nor use Fortran I/O
    nor system RNGs.
    
    See ‘Writing portable packages’ in the ‘Writing R Extensions’ manual.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 19.4Mb
      sub-directories of 1Mb or more:
        doc       1.9Mb
        extdata  15.9Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Unexported objects imported by ':::' calls:
      ‘BiocGenerics:::replaceSlots’ ‘S4Vectors:::make_zero_col_DataFrame’
      See the note in ?`:::` about the use of this operator.
    ```

# CNPBayes

Version: 1.10.0

## In both

*   checking whether package ‘CNPBayes’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/CNPBayes/new/CNPBayes.Rcheck/00install.out’ for details.
    ```

*   checking Rd \usage sections ... WARNING
    ```
    Undocumented arguments in documentation object 'marginal_lik'
      ‘value’
    
    Functions with \usage entries need to have the appropriate \alias
    entries, and all their arguments documented.
    The \usage entries must correspond to syntactically valid R code.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Warning in has_utility("pdfcrop") :
      pdfcrop not installed or not in PATH
    tlmgr search --file --global '/beramono.sty'
    Unknown directive ...l
    Warning in system2("tlmgr", args, ...) :
      running command ''tlmgr' search --file --global '/beramono.sty'' had status 25
    ! LaTeX Error: File `beramono.sty' not found.
    
    ! Emergency stop.
    <read *> 
    
    Error: processing vignette 'Convergence.Rmd' failed with diagnostics:
    Failed to compile Convergence.tex. See Convergence.log for more info.
    Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  7.8Mb
      sub-directories of 1Mb or more:
        R      2.1Mb
        doc    3.4Mb
        libs   1.4Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    copyNumber,SingleBatchCopyNumber: no visible binding for global
      variable ‘theta.star’
    Undefined global functions or variables:
      theta.star
    ```

# CNVPanelizer

Version: 1.12.0

## In both

*   checking whether package ‘CNVPanelizer’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/CNVPanelizer/new/CNVPanelizer.Rcheck/00install.out’ for details.
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    Loading required package: GenomeInfoDb
    Calculating Background for Sample_1
    Calculating Background for Sample_2
    Calculating Background for Sample_3
    Calculating Background for Sample_4
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'CNVPanelizer.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `biblatex.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.66 \usepackage
                    [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking dependencies in R code ... NOTE
    ```
    Unexported object imported by a ':::' call: ‘utils:::format.object_size’
      See the note in ?`:::` about the use of this operator.
    ```

# CNVrd2

Version: 1.18.0

## In both

*   checking whether package ‘CNVrd2’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/CNVrd2/new/CNVrd2.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘CNVrd2’ ...
** R
** data
** inst
** byte-compile and prepare package for lazy loading
Warning: package ‘IRanges’ was built under R version 3.5.1
Warning: package ‘GenomicRanges’ was built under R version 3.5.1
Warning: package ‘DelayedArray’ was built under R version 3.5.1
Warning: package ‘Rsamtools’ was built under R version 3.5.1
Error: package or namespace load failed for ‘rjags’:
 .onLoad failed in loadNamespace() for 'rjags', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/CNVrd2/rjags/libs/rjags.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/CNVrd2/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/CNVrd2/rjags/libs/rjags.so
  Reason: image not found
Error : package ‘rjags’ could not be loaded
ERROR: lazy loading failed for package ‘CNVrd2’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/CNVrd2/new/CNVrd2.Rcheck/CNVrd2’

```
### CRAN

```
* installing *source* package ‘CNVrd2’ ...
** R
** data
** inst
** byte-compile and prepare package for lazy loading
Warning: package ‘IRanges’ was built under R version 3.5.1
Warning: package ‘GenomicRanges’ was built under R version 3.5.1
Warning: package ‘DelayedArray’ was built under R version 3.5.1
Warning: package ‘Rsamtools’ was built under R version 3.5.1
Error: package or namespace load failed for ‘rjags’:
 .onLoad failed in loadNamespace() for 'rjags', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/CNVrd2/rjags/libs/rjags.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/CNVrd2/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/CNVrd2/rjags/libs/rjags.so
  Reason: image not found
Error : package ‘rjags’ could not be loaded
ERROR: lazy loading failed for package ‘CNVrd2’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/CNVrd2/old/CNVrd2.Rcheck/CNVrd2’

```
# CNVScope

Version: 1.9.7

## In both

*   checking examples ... ERROR
    ```
    ...
    
    > ### Name: getAsymmetricBlockIndices
    > ### Title: Get Block Indices from an asymmetric (or symmetric) matrix.
    > ### Aliases: getAsymmetricBlockIndices
    > ### Keywords: CNV Hi-C HiCseg MI blockseg breakpoints imputation jointseg
    > ###   multiple
    > 
    > ### ** Examples
    > 
    > 
    > load(system.file("extdata","nbl_result_matrix_sign_small.rda",package = "CNVScope")) 
    > submatrix_tiny<-nbl_result_matrix_sign_small
    > tiny_test<-getAsymmetricBlockIndices(submatrix_tiny)
    
     *** caught segfault ***
    address 0x7f8ee94af980, cause 'memory not mapped'
    
    Traceback:
     1: HiCseg::HiCseg_linkC_R(size_mat = dim(genomicmatrix)[1], nb_change_max = nb_change_max,     distrib = distrib, mat_data = as.matrix(genomicmatrix), model = model)
     2: getAsymmetricBlockIndices(submatrix_tiny)
    An irrecoverable exception occurred. R is aborting now ...
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      'ComplexHeatmap' 'DT' 'blockseg' 'readr'
      All declared Imports should be used.
    ```

# cocktailApp

Version: 0.2.0

## In both

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 14661 marked UTF-8 strings
    ```

# codebook

Version: 0.6.3

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘graphics’ ‘pander’
      All declared Imports should be used.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 65 marked UTF-8 strings
    ```

# codingMatrices

Version: 0.3.2

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'codingMatrices.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `mathtools.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.12 ^^M
            
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# cofeatureR

Version: 1.1.1

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘tibble’
      All declared Imports should be used.
    ```

# CoGAPS

Version: 3.0.2

## In both

*   checking R code for possible problems ... NOTE
    ```
    ...
      ‘sampleS’
    cellMatchR: no visible binding for global variable ‘cluster.method’
    patternMatcher : <anonymous>: no visible binding for global variable
      ‘Samples’
    patternMatcher : <anonymous>: no visible binding for global variable
      ‘value’
    patternMatcher : <anonymous>: no visible binding for global variable
      ‘BySet’
    runFinalPhase: no visible binding for global variable ‘i’
    runFinalPhase: no visible binding for global variable ‘sampleS’
    runInitialPhase: no visible binding for global variable ‘i’
    runInitialPhase: no visible binding for global variable ‘sampleS’
    scCoGapsFromCheckpoint: no visible binding for global variable ‘i’
    scCoGapsFromCheckpoint: no visible binding for global variable
      ‘sampleS’
    sc_runFinalPhase: no visible binding for global variable ‘i’
    sc_runFinalPhase: no visible binding for global variable ‘sampleS’
    sc_runInitialPhase: no visible binding for global variable ‘i’
    sc_runInitialPhase: no visible binding for global variable ‘sampleS’
    Undefined global functions or variables:
      BySet Samples cluster.method i sampleS value
    ```

*   checking compiled code ... NOTE
    ```
    File ‘CoGAPS/libs/CoGAPS.so’:
      Found ‘_rand’, possibly from ‘rand’ (C)
        Object: ‘test-runner.o’
      Found ‘_srand’, possibly from ‘srand’ (C)
        Object: ‘test-runner.o’
    
    Compiled code should not call entry points which might terminate R nor
    write to stdout/stderr instead of to the console, nor use Fortran I/O
    nor system RNGs.
    
    See ‘Writing portable packages’ in the ‘Writing R Extensions’ manual.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
      ...
    Warning in gapsRun(SimpSim.D, SimpSim.S, nFactor = 3, nEquil = nIter, nSample = nIter) :
      gapsRun is deprecated with v3.0, use CoGAPS
    Warning in v2CoGAPS(result, ...) :
      GStoGenes is deprecated with v3.0, see CoGAPS documentation
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'CoGAPSUsersManual.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `fullpage.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.6 ^^M
           
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# cogena

Version: 1.14.0

## In both

*   checking whether package ‘cogena’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘class::somgrid’ by ‘kohonen::somgrid’ when loading ‘cogena’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/cogena/new/cogena.Rcheck/00install.out’ for details.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  6.1Mb
      sub-directories of 1Mb or more:
        doc       1.9Mb
        extdata   3.1Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    corInCluster,cogena: no visible global function definition for ‘cor’
    heatmapCluster,cogena: no visible global function definition for
      ‘topo.colors’
    heatmapCluster,cogena: no visible global function definition for
      ‘rainbow’
    heatmapCluster,cogena: no visible global function definition for ‘par’
    heatmapCluster,cogena: no visible global function definition for
      ‘legend’
    Undefined global functions or variables:
      abline as.dist axis cor data density dist hist image layout legend
      lines median mtext order.dendrogram p.adjust par phyper plot.new
      rainbow rect reorder sd text title topo.colors
    Consider adding
      importFrom("grDevices", "rainbow", "topo.colors")
      importFrom("graphics", "abline", "axis", "hist", "image", "layout",
                 "legend", "lines", "mtext", "par", "plot.new", "rect",
                 "text", "title")
      importFrom("stats", "as.dist", "cor", "density", "dist", "median",
                 "order.dendrogram", "p.adjust", "phyper", "reorder", "sd")
      importFrom("utils", "data")
    to your NAMESPACE file.
    ```

# coloc

Version: 3.1

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘flashClust’ ‘speedglm’
      All declared Imports should be used.
    ```

# colorpatch

Version: 0.1.2

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘grid’ ‘gridExtra’
      All declared Imports should be used.
    ```

# colorplaner

Version: 0.1.3

## In both

*   checking examples ... ERROR
    ```
    ...
    Running examples in ‘colorplaner-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: color_projections
    > ### Title: Color Space Projections
    > ### Aliases: YUV_projection color_projections interpolate_projection
    > ###   red_blue_projection
    > 
    > ### ** Examples
    > 
    > library(ggplot2)
    > ggplot(mtcars, aes(x = wt, y = mpg, color = disp, colour2 = hp)) +
    +   geom_point() +
    +   scale_color_colorplane(color_projection = interpolate_projection,
    +                          zero_color = "darkorange2",
    +                          horizontal_color = "mediumspringgreen",
    +                          vertical_color = "magenta")
    Error in guide_train.colorplane(guide, scale, output) : 
      unused argument (output)
    Calls: <Anonymous> ... ggplot_gtable.ggplot_built -> build_guides -> guides_train -> guide_train
    Execution halted
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 3 SKIPPED: 0 FAILED: 20
      1. Error: Basic plotting occurs without error (@test_basics.R#16) 
      2. Error: Transformed colorplane scale plots without error (@test_basics.R#37) 
      3. Error: OOB and NA data handled without error (@test_basics.R#50) 
      4. Error: Arguments can be passed to projection functions (@test_basics.R#71) 
      5. Error: Faceted plots (@test_complex_plots.R#18) 
      6. Error: Grouped plots (@test_complex_plots.R#29) 
      7. Error: Multiple data layer plots (@test_complex_plots.R#39) 
      8. Error: Colorplane only on some layers without warning (@test_complex_plots.R#52) 
      9. Error: Title options (@test_guide_options.R#20) 
      1. ...
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 35-54 (colorplaner.Rmd) 
    Error: processing vignette 'colorplaner.Rmd' failed with diagnostics:
    unused argument (output)
    Execution halted
    ```

# CommT

Version: 0.1.1

## In both

*   checking R code for possible problems ... NOTE
    ```
    CommT.anova: no visible global function definition for ‘aov’
    CommT.legendpos: no visible global function definition for ‘quantile’
    CommT.plotcolors: no visible global function definition for ‘hcl’
    Undefined global functions or variables:
      aov hcl quantile
    Consider adding
      importFrom("grDevices", "hcl")
      importFrom("stats", "aov", "quantile")
    to your NAMESPACE file.
    ```

# compcodeR

Version: 1.16.1

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    Package 'sm', version 2.2-5.6: type help(sm) for summary information
    
    This is package 'modeest' written by P. PONCET.
    For a complete list of functions, use 'library(help = "modeest")' or 'help.start()'.
    
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'compcodeR.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `subfigure.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.35 \usepackage
                    {caption}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Packages which this enhances but not available for checking: ‘rpanel’ ‘DSS’
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' call to ‘rpanel’ in package code.
      Please use :: or requireNamespace() instead.
      See section 'Suggested packages' in the 'Writing R Extensions' manual.
    Package in Depends field not imported from: ‘sm’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      ‘packageVersion’
    voom.limma.createRmd: no visible global function definition for
      ‘packageVersion’
    voom.ttest.createRmd: no visible global function definition for
      ‘packageVersion’
    vst.limma.createRmd: no visible global function definition for
      ‘packageVersion’
    vst.ttest.createRmd: no visible global function definition for
      ‘packageVersion’
    show,compData: no visible global function definition for ‘head’
    Undefined global functions or variables:
      as.dist axis cor hclust head heat.colors legend lines loess median
      na.omit packageVersion par predict rexp rnbinom rpois runif sd title
    Consider adding
      importFrom("grDevices", "heat.colors")
      importFrom("graphics", "axis", "legend", "lines", "par", "title")
      importFrom("stats", "as.dist", "cor", "hclust", "loess", "median",
                 "na.omit", "predict", "rexp", "rnbinom", "rpois", "runif",
                 "sd")
      importFrom("utils", "head", "packageVersion")
    to your NAMESPACE file.
    ```

# CompGO

Version: 1.16.0

## In both

*   checking whether package ‘CompGO’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘graph’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/CompGO/new/CompGO.Rcheck/00install.out’ for details.
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'CompGO-Intro.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `authblk.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.7 \usepackage
                   {verbatim}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      ‘nodeRenderInfo<-’
    plotZScores: no visible global function definition for ‘complete.cases’
    plotZScores: no visible global function definition for ‘setNames’
    plotZScores: no visible global function definition for ‘cor’
    slidingJaccard: no visible binding for global variable ‘useRawPvals’
    zTransformDirectory: no visible global function definition for
      ‘read.table’
    zTransformDirectory: no visible global function definition for
      ‘complete.cases’
    zTransformDirectory: no visible binding for global variable ‘var’
    Undefined global functions or variables:
      Term Var1 Var2 Z complete.cases cor dev.off dist goDag hclust
      nodeRenderInfo<- nodes p.adjust par png pnorm read.table setNames
      text useRawPvals value var
    Consider adding
      importFrom("grDevices", "dev.off", "png")
      importFrom("graphics", "par", "text")
      importFrom("stats", "complete.cases", "cor", "dist", "hclust",
                 "p.adjust", "pnorm", "setNames", "var")
      importFrom("utils", "read.table")
    to your NAMESPACE file.
    ```

# CONFESS

Version: 1.8.0

## In both

*   checking whether package ‘CONFESS’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘stats::BIC’ by ‘flowMerge::BIC’ when loading ‘CONFESS’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/CONFESS/new/CONFESS.Rcheck/00install.out’ for details.
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    Setting B = 1 performs run-based cross-validation only (keeps run(s) 1,2,3,4)
    
    -----------------------------------------------------------
    | Running the CV analysis for reference run 5 and CV 1... |
    -----------------------------------------------------------
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'vignette_tex.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `beramono.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.87 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .BBSoptions
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

# confidence

Version: 1.1-2

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    
    For more information on the usage of the Confidence tool, type:
    vignette("confidence")
    Examples of input files can be found in:
    '/Users/max/github/ggplot2/revdep/checks.noindex/confidence/new/confidence.Rcheck/confidence/extdata'
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'confidence.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `colortbl.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.268 \long
               \def\@secondoffive#1#2#3#4#5{#2}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# congressbr

Version: 0.1.3

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘congressbr-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: sen_bills
    > ### Title: Downloads and tidies information on the legislation in the
    > ###   Federal Senate
    > ### Aliases: sen_bills
    > 
    > ### ** Examples
    > 
    > pls_5_2010 <- sen_bills(type = "PLS", number = 5, year = 2010)
    Error: Column `bill_indexing` must be a 1d atomic vector or a list
    Execution halted
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 187 marked UTF-8 strings
    ```

# consensusSeekeR

Version: 1.8.0

## In both

*   checking whether package ‘consensusSeekeR’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/consensusSeekeR/new/consensusSeekeR.Rcheck/00install.out’ for details.
    ```

# contiBAIT

Version: 1.8.0

## In both

*   checking whether package ‘contiBAIT’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘Rsamtools’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/contiBAIT/new/contiBAIT.Rcheck/00install.out’ for details.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ideogramPlot.func: no visible binding for global variable ‘w’
    ideogramPlot.func: no visible binding for global variable ‘w2’
    makeChrTable: no visible global function definition for ‘queryHits’
    makeChrTable: no visible global function definition for ‘subjectHits’
    orderContigsGreedy: no visible global function definition for
      ‘makeCluster’
    orderContigsGreedy: no visible global function definition for
      ‘clusterCall’
    orderContigsGreedy: no visible global function definition for
      ‘stopCluster’
    strandSeqFreqTable: no visible global function definition for
      ‘queryHits’
    strandSeqFreqTable: no visible global function definition for
      ‘subjectHits’
    ideogramPlot,StrandReadMatrix-StrandReadMatrix-ChrTable: no visible
      binding for global variable ‘w’
    ideogramPlot,StrandReadMatrix-StrandReadMatrix-ChrTable: no visible
      binding for global variable ‘w2’
    Undefined global functions or variables:
      clusterCall makeCluster queryHits stopCluster subjectHits w w2
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    -> Generating plotting data [32/35]
    -> Generating plotting data [33/35]
    -> Generating plotting data [34/35]
    -> Generating plotting data [35/35]
    Loading required package: shape
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'contiBAIT.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `cite.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.3 ^^M
           
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# CoordinateCleaner

Version: 2.0-2

## In both

*   checking whether package ‘CoordinateCleaner’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/CoordinateCleaner/new/CoordinateCleaner.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘CoordinateCleaner’ ...
** package ‘CoordinateCleaner’ successfully unpacked and MD5 sums checked
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
Error in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]) : 
  there is no package called ‘sf’
ERROR: lazy loading failed for package ‘CoordinateCleaner’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/CoordinateCleaner/new/CoordinateCleaner.Rcheck/CoordinateCleaner’

```
### CRAN

```
* installing *source* package ‘CoordinateCleaner’ ...
** package ‘CoordinateCleaner’ successfully unpacked and MD5 sums checked
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
Error in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]) : 
  there is no package called ‘sf’
ERROR: lazy loading failed for package ‘CoordinateCleaner’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/CoordinateCleaner/old/CoordinateCleaner.Rcheck/CoordinateCleaner’

```
# coseq

Version: 1.4.0

## In both

*   checking whether package ‘coseq’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘DelayedArray’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/coseq/new/coseq.Rcheck/00install.out’ for details.
    ```

# cosinor

Version: 1.1

## In both

*   checking R code for possible problems ... NOTE
    ```
    ...
    cosinor_analyzer : <anonymous>: no visible global function definition
      for ‘as.formula’
    ggplot.cosinor.lm: no visible global function definition for ‘predict’
    predict.cosinor.lm: no visible global function definition for ‘fitted’
    predict.cosinor.lm: no visible global function definition for ‘predict’
    simulate_cosinor: no visible global function definition for ‘runif’
    simulate_cosinor: no visible global function definition for ‘rbinom’
    simulate_cosinor: no visible global function definition for ‘rnorm’
    summary.cosinor.lm: no visible global function definition for ‘vcov’
    summary.cosinor.lm: no visible global function definition for ‘qnorm’
    summary.cosinor.lm: no visible global function definition for ‘pnorm’
    test_cosinor: no visible global function definition for ‘pchisq’
    test_cosinor: no visible global function definition for ‘pnorm’
    Undefined global functions or variables:
      as.formula fitted lm na.omit pchisq pnorm predict qnorm rbinom rnorm
      runif terms vcov vitamind
    Consider adding
      importFrom("stats", "as.formula", "fitted", "lm", "na.omit", "pchisq",
                 "pnorm", "predict", "qnorm", "rbinom", "rnorm", "runif",
                 "terms", "vcov")
    to your NAMESPACE file.
    ```

# cosinor2

Version: 0.2.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    LaTeX errors found:
    ! LaTeX Error: Command \dj unavailable in encoding OT1.
    
    See the LaTeX manual or LaTeX Companion for explanation.
    Type  H <return>  for immediate help.
     ...                                              
    ! LaTeX Error: Command \dj unavailable in encoding OT1.
    
    See the LaTeX manual or LaTeX Companion for explanation.
    Type  H <return>  for immediate help.
     ...                                              
    ! LaTeX Error: Command \dj unavailable in encoding OT1.
    
    See the LaTeX manual or LaTeX Companion for explanation.
    Type  H <return>  for immediate help.
     ...                                              
    ```

# CosmoPhotoz

Version: 0.1

## In both

*   checking R code for possible problems ... NOTE
    ```
    CosmoPhotoZestimator: no visible global function definition for
      ‘predict’
    computeCombPCA: no visible global function definition for ‘prcomp’
    computeDiagPhotoZ: no visible global function definition for ‘sd’
    computeDiagPhotoZ: no visible global function definition for ‘median’
    computeDiagPhotoZ: no visible global function definition for ‘mad’
    glmPredictPhotoZ: no visible global function definition for ‘predict’
    glmTrainPhotoZ: no visible global function definition for ‘glm’
    glmTrainPhotoZ: no visible global function definition for ‘Gamma’
    glmTrainPhotoZ: no visible global function definition for
      ‘inverse.gaussian’
    plotDiagPhotoZ: no visible global function definition for ‘median’
    plotDiagPhotoZ: no visible global function definition for ‘mad’
    Undefined global functions or variables:
      Gamma glm inverse.gaussian mad median prcomp predict sd
    Consider adding
      importFrom("stats", "Gamma", "glm", "inverse.gaussian", "mad",
                 "median", "prcomp", "predict", "sd")
    to your NAMESPACE file.
    ```

# CountClust

Version: 1.8.0

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  8.6Mb
      sub-directories of 1Mb or more:
        data   7.6Mb
    ```

# countyfloods

Version: 0.1.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘maps’
      All declared Imports should be used.
    ```

# countytimezones

Version: 1.0.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    Loading required package: stringr
    Loading required package: XML
    
    Attaching package: 'XML'
    
    The following object is masked from 'package:tools':
    
        toHTML
    
    
    Attaching package: 'acs'
    
    The following object is masked from 'package:base':
    
        apply
    
    Quitting from lines 111-123 (countytimezones.Rmd) 
    Error: processing vignette 'countytimezones.Rmd' failed with diagnostics:
    package or namespace load failed for 'choroplethr' in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]):
     there is no package called 'sf'
    Execution halted
    ```

# countyweather

Version: 0.1.0

## In both

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 1 marked UTF-8 string
    ```

# cowbell

Version: 0.1.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘ggplot2’ ‘misc3d’
      All declared Imports should be used.
    ```

# coxed

Version: 0.2.0

## In both

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘mediation’
    ```

# cplm

Version: 0.7-7

## In both

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    Error in re-building vignettes:
      ...
    Loading required package: coda
    Loading required package: Matrix
    Loading required package: splines
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'cplm.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `a4wide.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.20 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# cpvSNP

Version: 1.12.0

## In both

*   checking whether package ‘cpvSNP’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘GenomicFeatures’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘graph’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/cpvSNP/new/cpvSNP.Rcheck/00install.out’ for details.
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' call to ‘GSEABase’ which was already attached by Depends.
      Please remove these calls from your code.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    createArrayData: no visible global function definition for
      ‘elementMetadata<-’
    geneToSNPList: no visible global function definition for ‘findOverlaps’
    glossiMarginal: no visible global function definition for ‘pgamma’
    plotPvals: no visible binding for global variable ‘df’
    plotPvals: no visible binding for global variable ‘pval’
    simulate_chisq: no visible global function definition for ‘rnorm’
    vegasMarginal: no visible global function definition for ‘rbinom’
    vegasMarginal: no visible global function definition for ‘qchisq’
    vegasPrep: no visible global function definition for ‘elementMetadata’
    show,VEGASResult: no visible global function definition for ‘var’
    Undefined global functions or variables:
      GRanges IRanges Rle col2rgb density df elementMetadata
      elementMetadata<- findOverlaps legend lines pgamma plot points
      polygon pval qchisq rbinom rgb rnorm var
    Consider adding
      importFrom("grDevices", "col2rgb", "rgb")
      importFrom("graphics", "legend", "lines", "plot", "points", "polygon")
      importFrom("stats", "density", "df", "pgamma", "qchisq", "rbinom",
                 "rnorm", "var")
    to your NAMESPACE file.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    The following object is masked from ‘package:XML’:
    
        addNode
    
    making GRanges object
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'cpvSNP.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `beramono.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.87 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# cranly

Version: 0.2

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  6.7Mb
      sub-directories of 1Mb or more:
        doc   6.5Mb
    ```

# crawl

Version: 2.2.1

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘sf’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# CrispRVariants

Version: 1.8.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    'select()' returned many:1 mapping between keys and columns
    Classifying variants
    
    Looking up variant locations
    
    'select()' returned many:1 mapping between keys and columns
    'select()' returned many:1 mapping between keys and columns
    Classifying variants
    
    tlmgr search --file --global '/beramono.sty'
    Unknown directive ...l
    Warning in system2("tlmgr", args, ...) :
      running command ''tlmgr' search --file --global '/beramono.sty'' had status 25
    ! LaTeX Error: File `beramono.sty' not found.
    
    ! Emergency stop.
    <read *> 
    
    Error: processing vignette 'user_guide.Rmd' failed with diagnostics:
    Failed to compile user_guide.tex. See user_guide.log for more info.
    Execution halted
    ```

# crmPack

Version: 0.2.7

## In both

*   checking whether package ‘crmPack’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/crmPack/new/crmPack.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘crmPack’ ...
** package ‘crmPack’ successfully unpacked and MD5 sums checked
** R
** inst
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rjags', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/crmPack/rjags/libs/rjags.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/crmPack/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/crmPack/rjags/libs/rjags.so
  Reason: image not found
ERROR: lazy loading failed for package ‘crmPack’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/crmPack/new/crmPack.Rcheck/crmPack’

```
### CRAN

```
* installing *source* package ‘crmPack’ ...
** package ‘crmPack’ successfully unpacked and MD5 sums checked
** R
** inst
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rjags', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/crmPack/rjags/libs/rjags.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/crmPack/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/crmPack/rjags/libs/rjags.so
  Reason: image not found
ERROR: lazy loading failed for package ‘crmPack’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/crmPack/old/crmPack.Rcheck/crmPack’

```
# crossmeta

Version: 1.6.0

## In both

*   checking for code/documentation mismatches ... WARNING
    ```
    Data with usage in documentation object 'gs.names' but not in code:
      gs.names
    
    Data with usage in documentation object 'gslist' but not in code:
      gslist
    ```

*   checking dependencies in R code ... NOTE
    ```
    Missing or unexported objects:
      ‘crossmeta::gs.names’ ‘crossmeta::gslist’
    ```

*   checking R code for possible problems ... NOTE
    ```
    explore_paths : server: no visible binding for global variable ‘gslist’
    explore_paths : server: no visible binding for global variable
      ‘gs.names’
    Undefined global functions or variables:
      gs.names gslist
    ```

*   checking Rd cross-references ... NOTE
    ```
    Packages unavailable to check Rd xrefs: ‘PADOG’, ‘GeneMeta’
    ```

# Crossover

Version: 0.1-17

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    
    The following object is masked from 'package:MASS':
    
        geyser
    
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'Crossover.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `listings.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.64 \usepackage
                    {xspace}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# CrossScreening

Version: 0.1.1

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
      ...
    Warning in wilcox.test.default(d.lead, alternative = "greater") :
      cannot compute exact p-value with ties
    Warning in wilcox.test.default(d.lead, alternative = "greater") :
      cannot compute exact p-value with zeroes
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'CrossScreening-vignette.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `mathpazo.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.57 \usepackage
                    [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# csabounds

Version: 1.0.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘qte’
      All declared Imports should be used.
    ```

# csp

Version: 0.1.0

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 11.2Mb
      sub-directories of 1Mb or more:
        data  11.1Mb
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 59670 marked Latin-1 strings
      Note: found 16 marked UTF-8 strings
    ```

# CTDquerier

Version: 1.0.0

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 14.6Mb
      sub-directories of 1Mb or more:
        data      4.1Mb
        doc       6.1Mb
        extdata   4.1Mb
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 161 marked UTF-8 strings
    ```

# cummeRbund

Version: 2.22.0

## In both

*   checking whether package ‘cummeRbund’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘rtracklayer’ was built under R version 3.5.1
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/cummeRbund/new/cummeRbund.Rcheck/00install.out’ for details.
    ```

*   checking package dependencies ... NOTE
    ```
    Depends: includes the non-default packages:
      ‘BiocGenerics’ ‘RSQLite’ ‘ggplot2’ ‘reshape2’ ‘fastcluster’
      ‘rtracklayer’ ‘Gviz’
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  9.5Mb
      sub-directories of 1Mb or more:
        R         2.1Mb
        doc       1.6Mb
        extdata   5.5Mb
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Title field: should not end in a period.
    Packages listed in more than one of Depends, Imports, Suggests, Enhances:
      ‘BiocGenerics’ ‘plyr’
    A package should be listed in only one of these fields.
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' calls in package code:
      'NMFN' 'cluster' 'rjson' 'stringr'
      Please use :: or requireNamespace() instead.
      See section 'Suggested packages' in the 'Writing R Extensions' manual.
    Namespace in Imports field not imported from: 'S4Vectors'
      All declared Imports should be used.
    Packages in Depends field not imported from:
      'Gviz' 'RSQLite' 'fastcluster' 'ggplot2' 'reshape2' 'rtracklayer'
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      geom_point geom_rect geom_rug geom_segment geom_smooth geom_text
      geom_tile geom_vline ggplot guides hasAxis<- hclust ids import labs
      log2_fold_change make.db.names makeTxDb mean_cl_boot melt nnmf
      obsnames order.dendrogram p.adjust p_value pam plot plotIdeogram
      plotTracks position_dodge prcomp quant_status ranges read.delim
      read.table rowInd sample_1 sample_2 sample_name scale_color_gradient
      scale_color_hue scale_color_manual scale_colour_manual
      scale_fill_continuous scale_fill_gradient scale_fill_gradient2
      scale_fill_hue scale_x_continuous scale_x_discrete scale_x_log10
      scale_y_continuous scale_y_discrete scale_y_log10 sd seqnames
      significant start stat_density stat_smooth stat_sum stat_summary
      stdev str_split_fixed strand theme theme_bw toJSON tracking_id tracks
      unit v1 v2 value values variable varnames write.table x xlab xlim y
      ylab
    Consider adding
      importFrom("graphics", "plot")
      importFrom("stats", "as.dendrogram", "as.dist", "as.formula",
                 "cmdscale", "dist", "end", "hclust", "order.dendrogram",
                 "p.adjust", "prcomp", "sd", "start")
      importFrom("utils", "read.delim", "read.table", "write.table")
    to your NAMESPACE file.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    	distData
    Getting relCDS information:
    	distData
    Warning: Closing open result set, pending rows
    Warning: Closing open result set, pending rows
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'cummeRbund-example-workflow.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `float.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.23 \usepackage
                    {ifthen}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# curatedBreastData

Version: 2.8.0

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 283.1Mb
      sub-directories of 1Mb or more:
        data  282.7Mb
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
      That's OK, because we are immediately collapsing them into one feature.
    
    
    Starting with  286patients.
    found no multiple samples from the same patient(s)
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'curatedBreastData-manual.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `beramono.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.87 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# curatedMetagenomicData

Version: 1.10.2

## In both

*   checking whether package ‘curatedMetagenomicData’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘ExperimentHub’ was built under R version 3.5.1
      Warning: package ‘AnnotationHub’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/curatedMetagenomicData/new/curatedMetagenomicData.Rcheck/00install.out’ for details.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.6Mb
      sub-directories of 1Mb or more:
        doc    1.4Mb
        help   2.7Mb
    ```

# cutoffR

Version: 1.0

## In both

*   checking R code for possible problems ... NOTE
    ```
    Cut: no visible global function definition for ‘cor’
    MissSimulation: no visible global function definition for ‘rbinom’
    cutoff: no visible global function definition for ‘cor’
    Undefined global functions or variables:
      cor rbinom
    Consider adding
      importFrom("stats", "cor", "rbinom")
    to your NAMESPACE file.
    ```

# CVE

Version: 1.6.0

## In both

*   checking data for ASCII and uncompressed saves ... WARNING
    ```
      
      Note: significantly better compression could be obtained
            by using R CMD build --resave-data
                                    old_size new_size compress
      WGCNAmelanoma_extension.RData    2.4Mb    2.1Mb       xz
      crcCase.RData                    1.1Mb    675Kb       xz
      melanomaCase.RData               654Kb    472Kb       xz
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.6Mb
      sub-directories of 1Mb or more:
        data   4.1Mb
        doc    1.3Mb
    ```

# cvxclustr

Version: 1.1.1

## In both

*   checking R code for possible problems ... NOTE
    ```
    create_clustering_problem: no visible global function definition for
      ‘rnorm’
    Undefined global functions or variables:
      rnorm
    Consider adding
      importFrom("stats", "rnorm")
    to your NAMESPACE file.
    ```

*   checking line endings in Makefiles ... NOTE
    ```
    Found the following Makefile(s) without a final LF:
      src/Makevars
    Some ‘make’ programs ignore lines not ending in LF.
    ```

# cytofkit

Version: 1.12.0

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  8.5Mb
      sub-directories of 1Mb or more:
        doc       4.3Mb
        extdata   3.6Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    cytof_clusterPlot: no visible global function definition for
      ‘geom_text’
    cytof_progressionPlot: no visible global function definition for ‘aes’
    cytof_progressionPlot: no visible binding for global variable
      ‘Pseudotime’
    cytof_progressionPlot: no visible binding for global variable ‘cluster’
    cytofkitShinyAPP : <anonymous> : C_ScatterPlotInput: no visible global
      function definition for ‘scatterPlot’
    cytofkitShinyAPP : <anonymous>: no visible global function definition
      for ‘heatMap’
    cytofkitShinyAPP : <anonymous> : M_markerExpressionPlotInput: no
      visible global function definition for ‘scatterPlot’
    cytofkitShinyAPP : <anonymous> : M_stackDensityPlotInput: no visible
      global function definition for ‘stackDenistyPlot’
    cytofkitShinyAPP : <anonymous> : P_markerPlotInput: no visible global
      function definition for ‘cytof_expressionTrends’
    Undefined global functions or variables:
      Pseudotime aes cluster cytof_expressionTrends geom_text heatMap
      scatterPlot stackDenistyPlot
    ```

# cytometree

Version: 1.2.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘RColorBrewer’ ‘grDevices’ ‘robustbase’
      All declared Imports should be used.
    ```

# dada2

Version: 1.8.0

## In both

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Unexported object imported by a ':::' call: ‘ShortRead:::.set_omp_threads’
      See the note in ?`:::` about the use of this operator.
    ```

*   checking foreign function calls ... NOTE
    ```
    Foreign function call to a different package:
      .Call(ShortRead:::.set_omp_threads, ...)
    See chapter ‘System and foreign language interfaces’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    plotQualityProfile: no visible binding for global variable ‘Score’
    plotQualityProfile: no visible binding for global variable ‘Count’
    plotQualityProfile: no visible binding for global variable ‘Mean’
    plotQualityProfile: no visible binding for global variable ‘Q25’
    plotQualityProfile: no visible binding for global variable ‘Q50’
    plotQualityProfile: no visible binding for global variable ‘Q75’
    plotQualityProfile: no visible binding for global variable ‘Cum’
    samdf_to_qiime2: no visible global function definition for
      ‘write.table’
    seqtab_to_mothur: no visible global function definition for
      ‘write.table’
    seqtab_to_mothur: no visible binding for global variable ‘four’
    seqtab_to_qiime: no visible global function definition for
      ‘write.table’
    Undefined global functions or variables:
      := Count Cum Cycle Estimated Input Mean Nominal Observed Q25 Q50 Q75
      Qual Score abundance accept allMismatch als1 als2 biocLite four indel
      mismatch n0F n0R prefer seqF seqR write.table
    Consider adding
      importFrom("utils", "write.table")
    to your NAMESPACE file.
    ```

*   checking for GNU extensions in Makefiles ... NOTE
    ```
    GNU make is a SystemRequirements.
    ```

# dalmatian

Version: 0.3.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 36-71 (weights-1-simulate.Rmd) 
    Error: processing vignette 'weights-1-simulate.Rmd' failed with diagnostics:
    .onLoad failed in loadNamespace() for 'rjags', details:
      call: dyn.load(file, DLLpath = DLLpath, ...)
      error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/dalmatian/rjags/libs/rjags.so':
      dlopen(/Users/max/github/ggplot2/revdep/library.noindex/dalmatian/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
      Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/dalmatian/rjags/libs/rjags.so
      Reason: image not found
    Execution halted
    ```

# DaMiRseq

Version: 1.4.2

## In both

*   R CMD check timed out
    

*   checking whether package ‘DaMiRseq’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘DelayedArray’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/DaMiRseq/new/DaMiRseq.Rcheck/00install.out’ for details.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  6.9Mb
      sub-directories of 1Mb or more:
        data   6.0Mb
    ```

*   checking package subdirectories ... NOTE
    ```
    Found the following CITATION file in a non-standard place:
      CITATION
    Most likely ‘inst/CITATION’ should be used instead.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    DaMiR.EnsembleLearning2cl: no visible binding for global variable
      ‘Classifiers’
    DaMiR.EnsembleLearning2cl: no visible binding for global variable
      ‘Accuracy’
    DaMiR.EnsembleLearning2cl: no visible binding for global variable ‘MCC’
    DaMiR.EnsembleLearning2cl: no visible binding for global variable
      ‘Specificity’
    DaMiR.EnsembleLearning2cl: no visible binding for global variable
      ‘Sensitivity’
    DaMiR.EnsembleLearning2cl: no visible global function definition for
      ‘colSds’
    DaMiR.EnsembleLearningNcl: no visible binding for global variable
      ‘Classifiers’
    DaMiR.EnsembleLearningNcl: no visible binding for global variable
      ‘Accuracy’
    DaMiR.EnsembleLearningNcl: no visible global function definition for
      ‘colSds’
    DaMiR.MDSplot: no visible binding for global variable ‘X1’
    DaMiR.MDSplot: no visible binding for global variable ‘X2’
    Undefined global functions or variables:
      Accuracy Classifiers MCC Sensitivity Specificity X1 X2 colSds
    ```

# dams

Version: 0.2

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.3Mb
      sub-directories of 1Mb or more:
        data   5.1Mb
    ```

# DAPAR

Version: 1.12.11

## In both

*   checking R code for possible problems ... NOTE
    ```
    diffAnaVolcanoplot_rCharts: no visible binding for global variable ‘x’
    diffAnaVolcanoplot_rCharts: no visible binding for global variable ‘y’
    diffAnaVolcanoplot_rCharts: no visible binding for global variable ‘g’
    getTextForGOAnalysis: no visible binding for global variable
      ‘textGOParams’
    getTextForGOAnalysis: no visible binding for global variable ‘input’
    Undefined global functions or variables:
      g input textGOParams x y
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'Prostar_UserManual.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `soul.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.8 \usepackage
                   {url}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# dartR

Version: 1.0.5

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    tlmgr search --file --global '/tcolorbox.sty'
    Unknown directive ...l
    Warning in system2("tlmgr", args, ...) :
      running command ''tlmgr' search --file --global '/tcolorbox.sty'' had status 25
    ! LaTeX Error: File `tcolorbox.sty' not found.
    
    ! Emergency stop.
    <read *> 
    
    Error: processing vignette 'IntroTutorial_dartR.Rmd' failed with diagnostics:
    Failed to compile IntroTutorial_dartR.tex. See IntroTutorial_dartR.log for more info.
    Execution halted
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘DBI’ ‘Demerelate’ ‘misc3d’ ‘plotly’ ‘quadprog’ ‘rgl’
      All declared Imports should be used.
    ```

# dataMaid

Version: 1.2.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘DT’
      All declared Imports should be used.
    ```

# datarobot

Version: 2.9.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘curl’
      All declared Imports should be used.
    ```

# DChIPRep

Version: 1.10.0

## In both

*   checking whether package ‘DChIPRep’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package 'IRanges' was built under R version 3.5.1
      Warning: package 'GenomicRanges' was built under R version 3.5.1
      Warning: package 'DelayedArray' was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘DelayedArray’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/DChIPRep/new/DChIPRep.Rcheck/00install.out’ for details.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.1Mb
      sub-directories of 1Mb or more:
        doc       1.7Mb
        extdata   2.2Mb
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Description field: should contain one or more complete sentences.
    ```

# dcmr

Version: 1.0

## In both

*   checking R code for possible problems ... NOTE
    ```
    ...
      ‘combn’
    SampleParameterEstimates: no visible global function definition for
      ‘rnorm’
    plot,attribute.class-missing: no visible binding for global variable
      ‘attr.number’
    plot,attribute.class-missing: no visible binding for global variable
      ‘mean.attr’
    plot,attribute.profile.class-missing: no visible binding for global
      variable ‘attr.profile.number’
    plot,attribute.profile.class-missing: no visible binding for global
      variable ‘mean.attr.profile’
    summary,attribute.class: no visible binding for global variable ‘value’
    summary,attribute.profile.class: no visible binding for global variable
      ‘max.class’
    Undefined global functions or variables:
      attr.number attr.profile.number combn max.class mean.attr
      mean.attr.profile rmultinom rnorm runif value
    Consider adding
      importFrom("stats", "rmultinom", "rnorm", "runif")
      importFrom("utils", "combn")
    to your NAMESPACE file.
    ```

# ddPCRclust

Version: 1.0.1

## In both

*   checking R code for possible problems ... NOTE
    ```
    exportPlots: no visible binding for global variable ‘Cluster’
    Undefined global functions or variables:
      Cluster
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    
        Task B: find peaks...
    finished at 0.103 sec
    
    
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'ddPCRclust.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `beramono.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.87 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# debrowser

Version: 1.8.5

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘googleAuthR’
      All declared Imports should be used.
    ```

*   checking R code for possible problems ... NOTE
    ```
    deServer: no visible binding for global variable ‘debrowser’
    deServer: no visible global function definition for ‘getJsonObj’
    deServer: no visible global function definition for ‘access_token’
    debrowserdataload: no visible binding for global variable ‘demodata’
    getDensityPlot: no visible binding for global variable ‘samples’
    getGeneList: no visible global function definition for
      ‘showNotification’
    getMean: no visible binding for global variable ‘norm_data’
    installpack: no visible global function definition for
      ‘showNotification’
    plot_pca: no visible binding for global variable ‘x’
    plot_pca: no visible binding for global variable ‘y’
    plot_pca: no visible binding for global variable ‘samples’
    Undefined global functions or variables:
      access_token debrowser demodata getJsonObj norm_data samples
      showNotification x y
    ```

# DeclareDesign

Version: 0.10.0

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      > library(DeclareDesign)
      Loading required package: randomizr
      Loading required package: fabricatr
      Loading required package: estimatr
      > 
      > test_check("DeclareDesign")
      ── 1. Failure: error if you try to draw POs at a level using a variable that doe
      `my_potential_outcomes_formula(pop)` did not throw an error.
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 434 SKIPPED: 2 FAILED: 1
      1. Failure: error if you try to draw POs at a level using a variable that doesn't exist at that level (@test-potential-outcomes.R#160) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘sf’
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘methods’
      All declared Imports should be used.
    ```

# DeconRNASeq

Version: 1.22.0

## In both

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .BBSoptions
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  6.8Mb
      sub-directories of 1Mb or more:
        data   6.5Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' call to ‘grid’ which was already attached by Depends.
      Please remove these calls from your code.
    Packages in Depends field not imported from:
      ‘ggplot2’ ‘grid’ ‘pcaMethods’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    condplot: no visible global function definition for ‘plot’
    condplot: no visible global function definition for ‘rainbow’
    condplot: no visible global function definition for ‘lines’
    condplot: no visible global function definition for ‘axis’
    condplot: no visible global function definition for ‘title’
    decon.bootstrap: no visible global function definition for ‘t.test’
    multiplot: no visible global function definition for ‘grid.newpage’
    multiplot: no visible global function definition for ‘pushViewport’
    multiplot: no visible global function definition for ‘viewport’
    multiplot: no visible global function definition for ‘grid.layout’
    multiplot : vplayout: no visible global function definition for
      ‘viewport’
    Undefined global functions or variables:
      R2cum aes axis geom_abline geom_point ggplot grid.layout grid.newpage
      labs lines pca plot prep pushViewport rainbow t.test title viewport
      xlab ylab
    Consider adding
      importFrom("grDevices", "rainbow")
      importFrom("graphics", "axis", "lines", "plot", "title")
      importFrom("stats", "t.test")
    to your NAMESPACE file.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    The following object is masked from ‘package:limSolve’:
    
        resolution
    
    Loading required package: grid
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'DeconRNASeq.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.30 \ifthenelse
                    {\boolean{Sweave@inconsolata}}{%^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# decontam

Version: 1.0.0

## In both

*   checking R code for possible problems ... NOTE
    ```
    plot_frequency: no visible binding for global variable ‘DNA_conc’
    plot_frequency: no visible binding for global variable
      ‘taxon_abundance’
    plot_frequency: no visible binding for global variable ‘Type’
    plot_frequency: no visible binding for global variable ‘contam’
    plot_frequency: no visible binding for global variable ‘non.contam’
    Undefined global functions or variables:
      DNA_conc Type contam non.contam taxon_abundance
    ```

# DeepBlueR

Version: 1.6.0

## In both

*   R CMD check timed out
    

*   checking Rd files ... NOTE
    ```
    prepare_Rd: deepblue_enrich_regions_fast.Rd:35-38: Dropping empty section \examples
    ```

# DEGreport

Version: 1.16.0

## In both

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Package listed in more than one of Depends, Imports, Suggests, Enhances:
      ‘knitr’
    A package should be listed in only one of these fields.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      ‘enrichGO’
    .run_cluster_profiler: no visible global function definition for
      ‘simplify’
    .table_w_fc: no visible binding for global variable ‘comp’
    .table_w_fc: no visible binding for global variable ‘log2FoldChange’
    degCheckFactors: no visible binding for global variable ‘ratios’
    degCorCov: no visible binding for global variable ‘compare’
    degMV: no visible binding for global variable ‘min_median’
    degMV: no visible binding for global variable ‘max_sd’
    degPatterns: no visible global function definition for ‘rowMedians’
    degPatterns: no visible binding for global variable ‘genes’
    degPlotWide : <anonymous>: no visible binding for global variable
      ‘count’
    significants,TopTags: no visible binding for global variable ‘FDR’
    significants,TopTags: no visible binding for global variable ‘logFC’
    significants,list : <anonymous>: no visible binding for global variable
      ‘gene’
    Undefined global functions or variables:
      .x FDR base_mean comp compare count counts covar enrichGO gene genes
      keys log2FoldChange log2fc logFC max_sd min_median ratios rowMedians
      simplify
    ```

# DeLorean

Version: 1.5.0

## Newly broken

*   checking installed package size ... NOTE
    ```
      installed size is  7.5Mb
      sub-directories of 1Mb or more:
        libs   4.9Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘lattice’
      All declared Imports should be used.
    ```

*   checking for GNU extensions in Makefiles ... NOTE
    ```
    GNU make is a SystemRequirements.
    ```

## Newly fixed

*   checking whether package ‘DeLorean’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/DeLorean/old/DeLorean.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘DeLorean’ ...
** package ‘DeLorean’ successfully unpacked and MD5 sums checked
** libs
"/Library/Frameworks/R.framework/Resources/bin/Rscript" -e "source(file.path('..', 'tools', 'make_cc.R')); make_cc(commandArgs(TRUE))" stan_files/exact.stan
"/Library/Frameworks/R.framework/Resources/bin/Rscript" -e "source(file.path('..', 'tools', 'make_cc.R')); make_cc(commandArgs(TRUE))" stan_files/exactsizes.stan
"/Library/Frameworks/R.framework/Resources/bin/Rscript" -e "source(file.path('..', 'tools', 'make_cc.R')); make_cc(commandArgs(TRUE))" stan_files/lowrank.stan
"/Library/Frameworks/R.framework/Resources/bin/Rscript" -e "source(file.path('..', 'tools', 'make_cc.R')); make_cc(commandArgs(TRUE))" stan_files/lowranksizes.stan
DIAGNOSTIC(S) FROM PARSER:
Info: left-hand side variable (name=result) occurs on right-hand side of assignment, causing inefficient deep copy to avoid aliasing.

Wrote C++ file "stan_files/exactsizes.cc"
DIAGNOSTIC(S) FROM PARSER:
Info: left-hand side variable (name=result) occurs on right-hand side of assignment, causing inefficient deep copy to avoid aliasing.

Wrote C++ file "stan_files/exact.cc"
DIAGNOSTIC(S) FROM PARSER:
Info: left-hand side variable (name=result) occurs on right-hand side of assignment, causing inefficient deep copy to avoid aliasing.

DIAGNOSTIC(S) FROM PARSER:
Info: left-hand side variable (name=result) occurs on right-hand side of assignment, causing inefficient deep copy to avoid aliasing.

Wrote C++ file "stan_files/lowranksizes.cc"
Wrote C++ file "stan_files/lowrank.cc"
/usr/local/clang4/bin/clang++ -std=gnu++14 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I"../inst/include" -I"`"/Library/Frameworks/R.framework/Resources/bin/Rscript" --vanilla -e "cat(system.file('include', 'src', package = 'StanHeaders'))"`" -DBOOST_DISABLE_ASSERTS -DEIGEN_NO_DEBUG -DBOOST_MATH_OVERFLOW_ERROR_POLICY=errno_on_error -I"/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/BH/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include" -I/usr/local/include   -fPIC  -Wall -g -O2 -c init.cpp -o init.o
/usr/local/clang4/bin/clang++ -std=gnu++14 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I"../inst/include" -I"`"/Library/Frameworks/R.framework/Resources/bin/Rscript" --vanilla -e "cat(system.file('include', 'src', package = 'StanHeaders'))"`" -DBOOST_DISABLE_ASSERTS -DEIGEN_NO_DEBUG -DBOOST_MATH_OVERFLOW_ERROR_POLICY=errno_on_error -I"/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/BH/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include" -I/usr/local/include   -fPIC  -Wall -g -O2 -c stan_files/exact.cc -o stan_files/exact.o
/usr/local/clang4/bin/clang++ -std=gnu++14 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I"../inst/include" -I"`"/Library/Frameworks/R.framework/Resources/bin/Rscript" --vanilla -e "cat(system.file('include', 'src', package = 'StanHeaders'))"`" -DBOOST_DISABLE_ASSERTS -DEIGEN_NO_DEBUG -DBOOST_MATH_OVERFLOW_ERROR_POLICY=errno_on_error -I"/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/BH/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include" -I/usr/local/include   -fPIC  -Wall -g -O2 -c stan_files/exactsizes.cc -o stan_files/exactsizes.o
/usr/local/clang4/bin/clang++ -std=gnu++14 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I"../inst/include" -I"`"/Library/Frameworks/R.framework/Resources/bin/Rscript" --vanilla -e "cat(system.file('include', 'src', package = 'StanHeaders'))"`" -DBOOST_DISABLE_ASSERTS -DEIGEN_NO_DEBUG -DBOOST_MATH_OVERFLOW_ERROR_POLICY=errno_on_error -I"/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/BH/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include" -I/usr/local/include   -fPIC  -Wall -g -O2 -c stan_files/lowrank.cc -o stan_files/lowrank.o
/usr/local/clang4/bin/clang++ -std=gnu++14 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I"../inst/include" -I"`"/Library/Frameworks/R.framework/Resources/bin/Rscript" --vanilla -e "cat(system.file('include', 'src', package = 'StanHeaders'))"`" -DBOOST_DISABLE_ASSERTS -DEIGEN_NO_DEBUG -DBOOST_MATH_OVERFLOW_ERROR_POLICY=errno_on_error -I"/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/BH/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include" -I/usr/local/include   -fPIC  -Wall -g -O2 -c stan_files/lowranksizes.cc -o stan_files/lowranksizes.o
In file included from stan_files/exactsizes.cc:3:
In file included from stan_files/exactsizes.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Dense:1:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Core:531:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/lowrank.cc:3:
In file included from stan_files/lowrank.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Dense:1:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Core:531:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/exactsizes.cc:3:
In file included from stan_files/exactsizes.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Dense:2:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/LU:47:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/exactsizes.cc:3:
In file included from stan_files/exactsizes.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Dense:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Cholesky:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Jacobi:29:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/exact.cc:3:
In file included from stan_files/exact.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Dense:1:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Core:531:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/lowrank.cc:3:
In file included from stan_files/lowrank.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from stan_files/exactsizes.cc:3:
In file included from stan_files/exactsizes.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Dense:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Cholesky:43:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Dense:2:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/LU:47:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/lowrank.cc:3:
In file included from stan_files/lowrank.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Dense:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Cholesky:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Jacobi:29:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/exactsizes.cc:3:
In file included from stan_files/exactsizes.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Dense:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/QR:17:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Householder:27:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/lowrank.cc:3:
In file included from stan_files/lowrank.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Dense:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Cholesky:43:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/exact.cc:3:
In file included from stan_files/exact.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Dense:2:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/LU:47:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/exact.cc:3:
In file included from stan_files/exact.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Dense:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Cholesky:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Jacobi:29:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/lowrank.cc:3:
In file included from stan_files/lowrank.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Dense:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/QR:17:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Householder:27:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/exact.cc:3:
In file included from stan_files/exact.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Dense:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Cholesky:43:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/exact.cc:3:
In file included from stan_files/exact.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Dense:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/QR:17:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Householder:27:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/exactsizes.cc:3:
In file included from stan_files/exactsizes.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Dense:5:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/SVD:48:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/lowrank.cc:3:
In file included from stan_files/lowrank.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Dense:5:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/SVD:48:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/exact.cc:3:
In file included from stan_files/exact.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Dense:5:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/SVD:48:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/exactsizes.cc:3:
In file included from stan_files/exactsizes.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Dense:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Geometry:58:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/lowrank.cc:3:
In file included from stan_files/lowrank.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Dense:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Geometry:58:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/exactsizes.cc:3:
In file included from stan_files/exactsizes.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Dense:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Eigenvalues:58:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/exact.cc:3:
In file included from stan_files/exact.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Dense:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Geometry:58:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/lowrank.cc:3:
In file included from stan_files/lowrank.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Dense:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Eigenvalues:58:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/exact.cc:3:
In file included from stan_files/exact.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Dense:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Eigenvalues:58:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/lowranksizes.cc:3:
In file included from stan_files/lowranksizes.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Dense:1:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Core:531:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/lowranksizes.cc:3:
In file included from stan_files/lowranksizes.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Dense:2:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/LU:47:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/lowranksizes.cc:3:
In file included from stan_files/lowranksizes.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Dense:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Cholesky:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Jacobi:29:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/lowranksizes.cc:3:
In file included from stan_files/lowranksizes.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Dense:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Cholesky:43:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/lowranksizes.cc:3:
In file included from stan_files/lowranksizes.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Dense:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/QR:17:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Householder:27:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/lowranksizes.cc:3:
In file included from stan_files/lowranksizes.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Dense:5:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/SVD:48:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/lowranksizes.cc:3:
In file included from stan_files/lowranksizes.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Dense:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Geometry:58:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/lowranksizes.cc:3:
In file included from stan_files/lowranksizes.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Dense:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Eigenvalues:58:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/exactsizes.cc:3:
In file included from stan_files/exactsizes.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat.hpp:96:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/csr_extract_u.hpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Sparse:26:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/SparseCore:66:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/exactsizes.cc:3:
In file included from stan_files/exactsizes.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat.hpp:96:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/csr_extract_u.hpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Sparse:27:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/OrderingMethods:71:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/exactsizes.cc:3:
In file included from stan_files/exactsizes.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat.hpp:96:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/csr_extract_u.hpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Sparse:29:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/SparseCholesky:43:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/lowrank.cc:3:
In file included from stan_files/lowrank.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat.hpp:96:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/csr_extract_u.hpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Sparse:26:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/SparseCore:66:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/lowrank.cc:3:
In file included from stan_files/lowrank.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat.hpp:96:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/csr_extract_u.hpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Sparse:27:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/OrderingMethods:71:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/lowrank.cc:3:
In file included from stan_files/lowrank.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat.hpp:96:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/csr_extract_u.hpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Sparse:29:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/SparseCholesky:43:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/exact.cc:3:
In file included from stan_files/exact.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat.hpp:96:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/csr_extract_u.hpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Sparse:26:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/SparseCore:66:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/exactsizes.cc:3:
In file included from stan_files/exactsizes.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat.hpp:96:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/csr_extract_u.hpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Sparse:32:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/SparseQR:35:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/exact.cc:3:
In file included from stan_files/exact.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat.hpp:96:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/csr_extract_u.hpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Sparse:27:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/OrderingMethods:71:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/exact.cc:3:
In file included from stan_files/exact.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat.hpp:96:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/csr_extract_u.hpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Sparse:29:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/SparseCholesky:43:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/exactsizes.cc:3:
In file included from stan_files/exactsizes.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat.hpp:96:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/csr_extract_u.hpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Sparse:33:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/IterativeLinearSolvers:46:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/lowrank.cc:3:
In file included from stan_files/lowrank.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat.hpp:96:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/csr_extract_u.hpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Sparse:32:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/SparseQR:35:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/lowrank.cc:3:
In file included from stan_files/lowrank.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat.hpp:96:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/csr_extract_u.hpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Sparse:33:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/IterativeLinearSolvers:46:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/exact.cc:3:
In file included from stan_files/exact.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat.hpp:96:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/csr_extract_u.hpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Sparse:32:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/SparseQR:35:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/exact.cc:3:
In file included from stan_files/exact.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat.hpp:96:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/csr_extract_u.hpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Sparse:33:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/IterativeLinearSolvers:46:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/lowranksizes.cc:3:
In file included from stan_files/lowranksizes.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat.hpp:96:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/csr_extract_u.hpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Sparse:26:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/SparseCore:66:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/lowranksizes.cc:3:
In file included from stan_files/lowranksizes.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat.hpp:96:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/csr_extract_u.hpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Sparse:27:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/OrderingMethods:71:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/lowranksizes.cc:3:
In file included from stan_files/lowranksizes.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat.hpp:96:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/csr_extract_u.hpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Sparse:29:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/SparseCholesky:43:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/lowranksizes.cc:3:
In file included from stan_files/lowranksizes.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat.hpp:96:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/csr_extract_u.hpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Sparse:32:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/SparseQR:35:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/lowranksizes.cc:3:
In file included from stan_files/lowranksizes.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat.hpp:96:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/csr_extract_u.hpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Sparse:33:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/IterativeLinearSolvers:46:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/exact.cc:3:
In file included from stan_files/exactsizes.cc:3:
In file included from stan_files/exactsizes.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core.hpp:44:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core/set_zero_all_adjoints.hpp:14:13: warning: unused function 'set_zero_all_adjoints' [-Wunused-function]
In file included from stan_files/exact.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core.hpp:44:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core/set_zero_all_adjoints.hpp:14:13: static void set_zero_all_adjoints() {
            ^
warning: unused function 'set_zero_all_adjoints' [-Wunused-function]
In file included from stan_files/exactsizes.cc:3:
In file included from stan_files/exactsizes.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
static void set_zero_all_adjoints() {
            ^
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:In file included from stan_files/exact.cc:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat.hpp:70:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/autocorrelation.hpp:18:8: warning: function 'fft_next_good_size' is not needed and will not be emitted [-Wunneeded-internal-declaration]
size_t fft_next_good_size(size_t N) {
       ^
In file included from stan_files/exactsizes.cc:3:
stan_files/exactsizes.hpp:397:30: warning: unused typedef 'fun_return_scalar_t__' [-Wunused-local-typedef]
    typedef local_scalar_t__ fun_return_scalar_t__;
                             ^
3:
In file included from stan_files/exact.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat.hpp:70:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/autocorrelation.hpp:18:8: warning: function 'fft_next_good_size' is not needed and will not be emitted [-Wunneeded-internal-declaration]
size_t fft_next_good_size(size_t N) {
       ^
In file included from stan_files/exact.cc:3:
stan_files/exact.hpp:397:30: warning: unused typedef 'fun_return_scalar_t__' [-Wunused-local-typedef]
    typedef local_scalar_t__ fun_return_scalar_t__;
                             ^
In file included from stan_files/lowrank.cc:3:
In file included from stan_files/lowrank.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core.hpp:44:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core/set_zero_all_adjoints.hpp:14:13: warning: unused function 'set_zero_all_adjoints' [-Wunused-function]
static void set_zero_all_adjoints() {
            ^
In file included from stan_files/lowrank.cc:3:
In file included from stan_files/lowrank.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat.hpp:70:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/autocorrelation.hpp:18:8: warning: function 'fft_next_good_size' is not needed and will not be emitted [-Wunneeded-internal-declaration]
size_t fft_next_good_size(size_t N) {
       ^
In file included from stan_files/lowrank.cc:3:
stan_files/lowrank.hpp:495:30: warning: unused typedef 'fun_return_scalar_t__' [-Wunused-local-typedef]
    typedef local_scalar_t__ fun_return_scalar_t__;
                             ^
In file included from stan_files/lowranksizes.cc:3:
In file included from stan_files/lowranksizes.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core.hpp:44:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core/set_zero_all_adjoints.hpp:14:13: warning: unused function 'set_zero_all_adjoints' [-Wunused-function]
static void set_zero_all_adjoints() {
            ^
In file included from stan_files/lowranksizes.cc:3:
In file included from stan_files/lowranksizes.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat.hpp:70:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/autocorrelation.hpp:18:8: warning: function 'fft_next_good_size' is not needed and will not be emitted [-Wunneeded-internal-declaration]
size_t fft_next_good_size(size_t N) {
       ^
In file included from stan_files/lowranksizes.cc:3:
stan_files/lowranksizes.hpp:495:30: warning: unused typedef 'fun_return_scalar_t__' [-Wunused-local-typedef]
    typedef local_scalar_t__ fun_return_scalar_t__;
                             ^
16 warnings generated.
16 warnings generated.
16 warnings generated.
16 warnings generated.
/usr/local/clang4/bin/clang++ -std=gnu++14 -dynamiclib -Wl,-headerpad_max_install_names -undefined dynamic_lookup -single_module -multiply_defined suppress -L/Library/Frameworks/R.framework/Resources/lib -L/usr/local/clang4/lib -o DeLorean.so stan_files/exact.o stan_files/exactsizes.o stan_files/lowrank.o stan_files/lowranksizes.o init.o -F/Library/Frameworks/R.framework/.. -framework R -Wl,-framework -Wl,CoreFoundation
ld: warning: text-based stub file /System/Library/Frameworks//CoreFoundation.framework/CoreFoundation.tbd and library file /System/Library/Frameworks//CoreFoundation.framework/CoreFoundation are out of sync. Falling back to library file for linking.
rm stan_files/exact.cc stan_files/exactsizes.cc stan_files/lowranksizes.cc stan_files/lowrank.cc
installing to /Users/max/github/ggplot2/revdep/checks.noindex/DeLorean/new/DeLorean.Rcheck/DeLorean/libs
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
DIAGNOSTIC(S) FROM PARSER:
Info: left-hand side variable (name=result) occurs on right-hand side of assignment, causing inefficient deep copy to avoid aliasing.

DIAGNOSTIC(S) FROM PARSER:
Info: left-hand side variable (name=result) occurs on right-hand side of assignment, causing inefficient deep copy to avoid aliasing.

DIAGNOSTIC(S) FROM PARSER:
Info: left-hand side variable (name=result) occurs on right-hand side of assignment, causing inefficient deep copy to avoid aliasing.

DIAGNOSTIC(S) FROM PARSER:
Info: left-hand side variable (name=result) occurs on right-hand side of assignment, causing inefficient deep copy to avoid aliasing.

** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded
* DONE (DeLorean)

```
### CRAN

```
* installing *source* package ‘DeLorean’ ...
** package ‘DeLorean’ successfully unpacked and MD5 sums checked
** libs
"/Library/Frameworks/R.framework/Resources/bin/Rscript" -e "source(file.path('..', 'tools', 'make_cc.R')); make_cc(commandArgs(TRUE))" stan_files/exact.stan
"/Library/Frameworks/R.framework/Resources/bin/Rscript" -e "source(file.path('..', 'tools', 'make_cc.R')); make_cc(commandArgs(TRUE))" stan_files/exactsizes.stan
"/Library/Frameworks/R.framework/Resources/bin/Rscript" -e "source(file.path('..', 'tools', 'make_cc.R')); make_cc(commandArgs(TRUE))" stan_files/lowrank.stan
"/Library/Frameworks/R.framework/Resources/bin/Rscript" -e "source(file.path('..', 'tools', 'make_cc.R')); make_cc(commandArgs(TRUE))" stan_files/lowranksizes.stan
DIAGNOSTIC(S) FROM PARSER:
Info: left-hand side variable (name=result) occurs on right-hand side of assignment, causing inefficient deep copy to avoid aliasing.

DIAGNOSTIC(S) FROM PARSER:
Info: left-hand side variable (name=result) occurs on right-hand side of assignment, causing inefficient deep copy to avoid aliasing.

Wrote C++ file "stan_files/exact.cc"
Wrote C++ file "stan_files/exactsizes.cc"
DIAGNOSTIC(S) FROM PARSER:
Info: left-hand side variable (name=result) occurs on right-hand side of assignment, causing inefficient deep copy to avoid aliasing.

DIAGNOSTIC(S) FROM PARSER:
Info: left-hand side variable (name=result) occurs on right-hand side of assignment, causing inefficient deep copy to avoid aliasing.

Wrote C++ file "stan_files/lowrank.cc"
Wrote C++ file "stan_files/lowranksizes.cc"
Error in readRDS("/var/folders/lb/xhxqmcrd7gv302_b1pdfykh80000gn/T//Rtmp2BA6ai/file6fc232bb61ef") : 
  error reading from connection
Calls: .Last -> readRDS
Execution halted
/usr/local/clang4/bin/clang++ -std=gnu++14 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I"../inst/include" -I"`"/Library/Frameworks/R.framework/Resources/bin/Rscript" --vanilla -e "cat(system.file('include', 'src', package = 'StanHeaders'))"`" -DBOOST_DISABLE_ASSERTS -DEIGEN_NO_DEBUG -DBOOST_MATH_OVERFLOW_ERROR_POLICY=errno_on_error -I"/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/BH/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include" -I/usr/local/include   -fPIC  -Wall -g -O2 -c init.cpp -o init.o
/usr/local/clang4/bin/clang++ -std=gnu++14 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I"../inst/include" -I"`"/Library/Frameworks/R.framework/Resources/bin/Rscript" --vanilla -e "cat(system.file('include', 'src', package = 'StanHeaders'))"`" -DBOOST_DISABLE_ASSERTS -DEIGEN_NO_DEBUG -DBOOST_MATH_OVERFLOW_ERROR_POLICY=errno_on_error -I"/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/BH/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include" -I/usr/local/include   -fPIC  -Wall -g -O2 -c stan_files/exact.cc -o stan_files/exact.o
make: *** [stan_files/lowranksizes.cc] Error 1
make: *** Waiting for unfinished jobs....
In file included from stan_files/exact.cc:3:
In file included from stan_files/exact.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Dense:1:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Core:531:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/exact.cc:3:
In file included from stan_files/exact.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Dense:2:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/LU:47:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/exact.cc:3:
In file included from stan_files/exact.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Dense:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Cholesky:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Jacobi:29:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/exact.cc:3:
In file included from stan_files/exact.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Dense:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Cholesky:43:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/exact.cc:3:
In file included from stan_files/exact.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Dense:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/QR:17:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Householder:27:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/exact.cc:3:
In file included from stan_files/exact.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Dense:5:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/SVD:48:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/exact.cc:3:
In file included from stan_files/exact.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Dense:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Geometry:58:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/exact.cc:3:
In file included from stan_files/exact.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Dense:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Eigenvalues:58:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/exact.cc:3:
In file included from stan_files/exact.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat.hpp:96:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/csr_extract_u.hpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Sparse:26:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/SparseCore:66:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/exact.cc:3:
In file included from stan_files/exact.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat.hpp:96:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/csr_extract_u.hpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Sparse:27:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/OrderingMethods:71:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/exact.cc:3:
In file included from stan_files/exact.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat.hpp:96:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/csr_extract_u.hpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Sparse:29:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/SparseCholesky:43:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/exact.cc:3:
In file included from stan_files/exact.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat.hpp:96:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/csr_extract_u.hpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Sparse:32:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/SparseQR:35:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/exact.cc:3:
In file included from stan_files/exact.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat.hpp:96:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/csr_extract_u.hpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/Sparse:33:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/IterativeLinearSolvers:46:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/exact.cc:3:
In file included from stan_files/exact.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core.hpp:44:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/core/set_zero_all_adjoints.hpp:14:13: warning: unused function 'set_zero_all_adjoints' [-Wunused-function]
static void set_zero_all_adjoints() {
            ^
In file included from stan_files/exact.cc:3:
In file included from stan_files/exact.hpp:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat.hpp:70:
/Users/max/github/ggplot2/revdep/library.noindex/DeLorean/StanHeaders/include/stan/math/prim/mat/fun/autocorrelation.hpp:18:8: warning: function 'fft_next_good_size' is not needed and will not be emitted [-Wunneeded-internal-declaration]
size_t fft_next_good_size(size_t N) {
       ^
In file included from stan_files/exact.cc:3:
stan_files/exact.hpp:397:30: warning: unused typedef 'fun_return_scalar_t__' [-Wunused-local-typedef]
    typedef local_scalar_t__ fun_return_scalar_t__;
                             ^
16 warnings generated.
rm stan_files/exact.cc stan_files/exactsizes.cc stan_files/lowranksizes.cc stan_files/lowrank.cc
ERROR: compilation failed for package ‘DeLorean’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/DeLorean/old/DeLorean.Rcheck/DeLorean’

```
# deltaGseg

Version: 1.20.0

## In both

*   checking package dependencies ... NOTE
    ```
    Depends: includes the non-default packages:
      ‘ggplot2’ ‘changepoint’ ‘wavethresh’ ‘tseries’ ‘pvclust’ ‘fBasics’
      ‘grid’ ‘reshape’ ‘scales’
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .BBSoptions
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Description field: should contain one or more complete sentences.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    diagnosticPlots,SegSeriesTrajectories: no visible binding for global
      variable ‘residuals’
    diagnosticPlots,SegSeriesTrajectories: no visible global function
      definition for ‘acf’
    diagnosticPlots,SegSeriesTrajectories: no visible global function
      definition for ‘qnorm’
    diagnosticPlots,SegSeriesTrajectories: no visible binding for global
      variable ‘lag’
    Undefined global functions or variables:
      abline acf as.dist cor cutree dist hclust identify lag layout lines
      locator median mtext na.omit par points qnorm quantile read.table
      residuals setTxtProgressBar text txtProgressBar
    Consider adding
      importFrom("graphics", "abline", "identify", "layout", "lines",
                 "locator", "mtext", "par", "points", "text")
      importFrom("stats", "acf", "as.dist", "cor", "cutree", "dist",
                 "hclust", "lag", "median", "na.omit", "qnorm", "quantile",
                 "residuals")
      importFrom("utils", "read.table", "setTxtProgressBar",
                 "txtProgressBar")
    to your NAMESPACE file.
    ```

*   checking installed files from ‘inst/doc’ ... NOTE
    ```
    The following files should probably not be installed:
      ‘diagplots.png’, ‘simclust.png’, ‘simtraj.png’, ‘simtrajtr.png’,
      ‘simtrajtr2.png’, ‘traj1.png’, ‘traj1break.png’, ‘traj1ss0.png’,
      ‘traj1ss1.png’, ‘traj1tr.png’
    
    Consider the use of a .Rinstignore file: see ‘Writing R Extensions’,
    or move the vignette sources from ‘inst/doc’ to ‘vignettes’.
    ```

# demi

Version: 1.1.2

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'demi.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.30 \ifthenelse
                    {\boolean{Sweave@inconsolata}}{%^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    demi.wilcox.test.fast: no visible global function definition for
      ‘wilcox.test’
    demiequal: no visible global function definition for ‘wilcox.test’
    wprob: no visible global function definition for ‘combn’
    diffexp,DEMIDiff: no visible global function definition for ‘median’
    diffexp,DEMIDiff: no visible global function definition for ‘p.adjust’
    loadAnnotation,DEMIExperiment-environment: no visible global function
      definition for ‘data’
    loadBlat,DEMIExperiment-environment: no visible global function
      definition for ‘data’
    loadCytoband,DEMIExperiment-environment: no visible global function
      definition for ‘data’
    loadPathway,DEMIExperiment-environment: no visible global function
      definition for ‘data’
    Undefined global functions or variables:
      combn data dhyper median p.adjust t.test wilcox.test write.table
    Consider adding
      importFrom("stats", "dhyper", "median", "p.adjust", "t.test",
                 "wilcox.test")
      importFrom("utils", "combn", "data", "write.table")
    to your NAMESPACE file.
    ```

# dendextend

Version: 1.9.0

## In both

*   checking package dependencies ... NOTE
    ```
    Package which this enhances but not available for checking: ‘dendroextras’
    ```

*   checking Rd cross-references ... NOTE
    ```
    Packages unavailable to check Rd xrefs: ‘dendroextras’, ‘moduleColor’, ‘distory’, ‘phangorn’
    ```

# DEP

Version: 1.2.0

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.6Mb
      sub-directories of 1Mb or more:
        data   1.4Mb
        doc    3.1Mb
    ```

# derfinder

Version: 1.14.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Warning in citation("BiocStyle") :
      no date field in DESCRIPTION file of package 'BiocStyle'
    Quitting from lines 63-131 (derfinder-quickstart.Rmd) 
    Error: processing vignette 'derfinder-quickstart.Rmd' failed with diagnostics:
    package 'knitrBootstrap' not found
    Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  6.3Mb
      sub-directories of 1Mb or more:
        doc   5.0Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Unexported objects imported by ':::' calls:
      ‘GenomeInfoDb:::.guessSpeciesStyle’
      ‘GenomeInfoDb:::.supportedSeqnameMappings’
      See the note in ?`:::` about the use of this operator.
    There are ::: calls to the package's namespace in its code. A package
      almost never needs to use ::: for its own objects:
      ‘.smootherFstats’
    ```

# derfinderPlot

Version: 1.14.0

## In both

*   R CMD check timed out
    

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: 'RefManageR'
      All declared Imports should be used.
    ```

# DESeq2

Version: 1.20.0

## In both

*   checking whether package ‘DESeq2’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘DelayedArray’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/DESeq2/new/DESeq2.Rcheck/00install.out’ for details.
    ```

# destiny

Version: 2.10.2

## In both

*   checking running R code from vignettes ...
    ```
    ...
      When tangling ‘Diffusion-Map-recap.ipynbmeta’:
    Error: Either IPython 3+ or Jupyter has to be installed, but neither could be called.
    Execution halted
    when running code in ‘Diffusion-Maps.ipynbmeta’
      ...
    
      When tangling ‘Diffusion-Maps.ipynbmeta’:
    Error: Either IPython 3+ or Jupyter has to be installed, but neither could be called.
    Execution halted
    when running code in ‘Global-Sigma.ipynbmeta’
      ...
    
      When tangling ‘Global-Sigma.ipynbmeta’:
    Error: Either IPython 3+ or Jupyter has to be installed, but neither could be called.
    Execution halted
    when running code in ‘tidyverse.ipynbmeta’
      ...
    
      When tangling ‘tidyverse.ipynbmeta’:
    Error: Either IPython 3+ or Jupyter has to be installed, but neither could be called.
    Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.9Mb
      sub-directories of 1Mb or more:
        doc   4.3Mb
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Package listed in more than one of Depends, Imports, Suggests, Enhances:
      ‘SingleCellExperiment’
    A package should be listed in only one of these fields.
    'LinkingTo' for ‘grDevices’ is unused as it has no 'include' directory
    ```

*   checking Rd \usage sections ... NOTE
    ```
    S3 methods shown with full name in documentation object 'plot.DPT':
      ‘plot.DPT’
    
    S3 methods shown with full name in documentation object 'plot.DiffusionMap':
      ‘plot.DiffusionMap’
    
    The \usage entries for S3 methods should use the \method markup and not
    their full name.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking for unstated dependencies in vignettes ... NOTE
    ```
    '::' or ':::' imports not declared from:
      ‘gridExtra’ ‘viridis’
    'library' or 'require' calls not declared from:
      ‘IRdisplay’ ‘IRkernel’ ‘base64enc’ ‘forcats’ ‘readxl’ ‘repr’
      ‘tidyverse’
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Error: processing vignette 'DPT.ipynbmeta' failed with diagnostics:
    Either IPython 3+ or Jupyter has to be installed, but neither could be called.
    Execution halted
    ```

# detzrcr

Version: 0.2.3

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘utils’
      All declared Imports should be used.
    ```

# DEVis

Version: 1.0.0

## In both

*   checking whether package ‘DEVis’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘DelayedArray’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/DEVis/new/DEVis.Rcheck/00install.out’ for details.
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    
    factor levels were dropped which had no samples
    factor levels were dropped which had no samples
    using pre-existing size factors
    estimating dispersions
    gene-wise dispersion estimates
    mean-dispersion relationship
    final dispersion estimates
    fitting model and testing
    tlmgr search --file --global '/multirow.sty'
    Unknown directive ...l
    Warning in system2("tlmgr", args, ...) :
      running command ''tlmgr' search --file --global '/multirow.sty'' had status 25
    ! LaTeX Error: File `multirow.sty' not found.
    
    ! Emergency stop.
    <read *> 
    
    Error: processing vignette 'DEVis_vignette.Rmd' failed with diagnostics:
    Failed to compile DEVis_vignette.tex. See DEVis_vignette.log for more info.
    Execution halted
    ```

# dextergui

Version: 0.1.4

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    
    Attaching package: 'dplyr'
    
    The following objects are masked from 'package:stats':
    
        filter, lag
    
    The following objects are masked from 'package:base':
    
        intersect, setdiff, setequal, union
    
    Loading required package: RSQLite
    no column `person_id` provided, automatically generating unique person id's
    File img/main_bar.PNG not found in resource path
    Error: processing vignette 'dextergui.Rmd' failed with diagnostics:
    pandoc document conversion failed with error 99
    Execution halted
    ```

*   checking dependencies in R code ... NOTE
    ```
    Unexported objects imported by ':::' calls:
      ‘dexter:::get_resp_data’ ‘dexter:::qcolors’
      See the note in ?`:::` about the use of this operator.
    ```

# dfexplore

Version: 0.2.1

## In both

*   checking R code for possible problems ... NOTE
    ```
    simulate_dataframe : generate_dumb_answer: no visible global function
      definition for ‘rbinom’
    simulate_dataframe : generate_dumb_answer: no visible global function
      definition for ‘rnorm’
    simulate_dataframe : generate_initial: no visible global function
      definition for ‘rbinom’
    simulate_dataframe: no visible binding for global variable ‘randu’
    simulate_dataframe: no visible global function definition for ‘rbinom’
    simulate_dataframe: no visible global function definition for ‘rnorm’
    simulate_dataframe: no visible global function definition for ‘rpois’
    simulate_dataframe : add_NAs: no visible global function definition for
      ‘rbinom’
    Undefined global functions or variables:
      randu rbinom rnorm rpois
    Consider adding
      importFrom("datasets", "randu")
      importFrom("stats", "rbinom", "rnorm", "rpois")
    to your NAMESPACE file.
    ```

# dfpk

Version: 3.5.0

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.9Mb
      sub-directories of 1Mb or more:
        libs   5.4Mb
    ```

# DGVM3D

Version: 1.0.0

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.6Mb
      sub-directories of 1Mb or more:
        data   4.1Mb
        doc    1.3Mb
    ```

# dielectric

Version: 0.2.3

## In both

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Title field: should not end in a period.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Package in Depends field not imported from: ‘methods’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

*   checking R code for possible problems ... NOTE
    ```
    L2eV: no visible binding for global variable ‘constants’
    L2w: no visible binding for global variable ‘constants’
    dielectric: no visible global function definition for ‘new’
    dielectric2plot: no visible global function definition for ‘reshape’
    eV2L: no visible binding for global variable ‘constants’
    t2eV: no visible binding for global variable ‘constants’
    Undefined global functions or variables:
      constants new reshape
    Consider adding
      importFrom("methods", "new")
      importFrom("stats", "reshape")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

# DiffBind

Version: 2.8.0

## In both

*   R CMD check timed out
    

*   checking whether package ‘DiffBind’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘DelayedArray’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/DiffBind/new/DiffBind.Rcheck/00install.out’ for details.
    ```

*   checking package dependencies ... NOTE
    ```
    Package which this enhances but not available for checking: ‘XLConnect’
    ```

*   checking R code for possible problems ... NOTE
    ```
    pv.DBAplotVolcano: no visible binding for global variable ‘Fold’
    pv.DBAplotVolcano: no visible binding for global variable ‘Legend’
    Undefined global functions or variables:
      Fold Legend
    ```

# diffloop

Version: 1.8.0

## In both

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

# diffuStats

Version: 0.104.0

## In both

*   checking for GNU extensions in Makefiles ... NOTE
    ```
    GNU make is a SystemRequirements.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    All done
    Warning in wilcox.test.default(x = perf_mat[, met1], y = perf_mat[, met2],  :
      cannot compute exact p-value with zeroes
    Warning in wilcox.test.default(x = perf_mat[, met1], y = perf_mat[, met2],  :
      cannot compute exact confidence interval with zeroes
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'diffuStats.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `beramono.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.87 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# dirichletprocess

Version: 0.2.1

## In both

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'dirichletprocess.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.55 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# disto

Version: 0.2.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘dplyr’ ‘proxy’
      All declared Imports should be used.
    ```

# diveRsity

Version: 1.9.90

## In both

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'diveRsity.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `enumitem.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.7 \usepackage
                   {hyperref}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# DLMtool

Version: 5.2.3

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.5Mb
      sub-directories of 1Mb or more:
        R      2.0Mb
        data   2.1Mb
    ```

# dlookr

Version: 0.3.2

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘dbplyr’ ‘randomForest’
      All declared Imports should be used.
    ```

# dmrseq

Version: 1.0.14

## In both

*   checking examples ... ERROR
    ```
    ...
    > ### ** Examples
    > 
    > 
    > # get annotation information for hg19
    > annoTrack <- getAnnot('hg19')
    snapshotDate(): 2018-04-30
    Building CpG islands...
    downloading 0 resources
    loading from cache 
        ‘/Users/max//.AnnotationHub/5086’
    Building CpG shores...
    Building CpG shelves...
    Building inter-CpG-islands...
    Download of CpG annotation successful!
    Download of Gene annotation failed. 4 attempts left
    Download of Gene annotation failed. 3 attempts left
    Download of Gene annotation failed. 2 attempts left
    Download of Gene annotation failed. 1 attempts left
    Download of Gene annotation failed. 0 attempts left
    Error in getAnnot("hg19") : Annotation could not be retrieved.
    Execution halted
    ```

*   checking whether package ‘dmrseq’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘DelayedArray’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/dmrseq/new/dmrseq.Rcheck/00install.out’ for details.
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    
    Assuming file type is cov
    Genome hg18 will be built by lifting over hg19 annotations from annotatr
    snapshotDate(): 2018-04-30
    Building CpG islands...
    downloading 0 resources
    loading from cache 
        '/Users/max//.AnnotationHub/5086'
    Building CpG shores...
    Building CpG shelves...
    Building inter-CpG-islands...
    Download of CpG annotation successful!
    Download of Gene annotation failed. 4 attempts left
    Download of Gene annotation failed. 3 attempts left
    Download of Gene annotation failed. 2 attempts left
    Download of Gene annotation failed. 1 attempts left
    Download of Gene annotation failed. 0 attempts left
    Quitting from lines 541-546 (dmrseq.Rmd) 
    Error: processing vignette 'dmrseq.Rmd' failed with diagnostics:
    Annotation could not be retrieved.
    Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 10.7Mb
      sub-directories of 1Mb or more:
        data   9.0Mb
        doc    1.3Mb
    ```

# doBy

Version: 4.6-2

## In both

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'linest-lsmeans.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `a4wide.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.62 \usepackage
                    {boxedminipage,color}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# dodgr

Version: 0.0.3

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
      ...
    files ‘R/RcppExports.R’, ‘src/RcppExports.cpp’ have the wrong MD5 checksums
    ** libs
    /usr/local/clang4/bin/clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I. -I"/Users/max/github/ggplot2/revdep/library.noindex/dodgr/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2 -c RcppExports.cpp -o RcppExports.o
    /usr/local/clang4/bin/clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I. -I"/Users/max/github/ggplot2/revdep/library.noindex/dodgr/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2 -c dgraph.cpp -o dgraph.o
    /usr/local/clang4/bin/clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I. -I"/Users/max/github/ggplot2/revdep/library.noindex/dodgr/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2 -c dijkstra.cpp -o dijkstra.o
    /usr/local/clang4/bin/clang -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I. -I"/Users/max/github/ggplot2/revdep/library.noindex/dodgr/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c dodgr_init.c -o dodgr_init.o
    /usr/local/clang4/bin/clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I. -I"/Users/max/github/ggplot2/revdep/library.noindex/dodgr/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2 -c graph-contract.cpp -o graph-contract.o
    /usr/local/clang4/bin/clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I. -I"/Users/max/github/ggplot2/revdep/library.noindex/dodgr/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2 -c graph-sample.cpp -o graph-sample.o
    /usr/local/clang4/bin/clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I. -I"/Users/max/github/ggplot2/revdep/library.noindex/dodgr/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2 -c graph.cpp -o graph.o
    /usr/local/clang4/bin/clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I. -I"/Users/max/github/ggplot2/revdep/library.noindex/dodgr/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2 -c run_sp.cpp -o run_sp.o
    /usr/local/clang4/bin/clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I. -I"/Users/max/github/ggplot2/revdep/library.noindex/dodgr/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2 -c sf-as-network.cpp -o sf-as-network.o
    /usr/local/clang4/bin/clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I. -I"/Users/max/github/ggplot2/revdep/library.noindex/dodgr/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2 -c heaps/bheap.cpp -o heaps/bheap.o
    /usr/local/clang4/bin/clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I. -I"/Users/max/github/ggplot2/revdep/library.noindex/dodgr/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2 -c heaps/fheap.cpp -o heaps/fheap.o
    /usr/local/clang4/bin/clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I. -I"/Users/max/github/ggplot2/revdep/library.noindex/dodgr/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2 -c heaps/heap23.cpp -o heaps/heap23.o
    /usr/local/clang4/bin/clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I. -I"/Users/max/github/ggplot2/revdep/library.noindex/dodgr/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2 -c heaps/radixheap.cpp -o heaps/radixheap.o
    /usr/local/clang4/bin/clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I. -I"/Users/max/github/ggplot2/revdep/library.noindex/dodgr/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2 -c heaps/triheap.cpp -o heaps/triheap.o
    /usr/local/clang4/bin/clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I. -I"/Users/max/github/ggplot2/revdep/library.noindex/dodgr/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2 -c heaps/triheap_ext.cpp -o heaps/triheap_ext.o
    /usr/local/clang4/bin/clang++ -std=gnu++11 -dynamiclib -Wl,-headerpad_max_install_names -undefined dynamic_lookup -single_module -multiply_defined suppress -L/Library/Frameworks/R.framework/Resources/lib -L/usr/local/clang4/lib -o dodgr.so RcppExports.o dgraph.o dijkstra.o dodgr_init.o graph-contract.o graph-sample.o graph.o run_sp.o sf-as-network.o heaps/bheap.o heaps/fheap.o heaps/heap23.o heaps/radixheap.o heaps/triheap.o heaps/triheap_ext.o -F/Library/Frameworks/R.framework/.. -framework R -Wl,-framework -Wl,CoreFoundation
    ld: warning: text-based stub file /System/Library/Frameworks//CoreFoundation.framework/CoreFoundation.tbd and library file /System/Library/Frameworks//CoreFoundation.framework/CoreFoundation are out of sync. Falling back to library file for linking.
    installing to /private/var/folders/lb/xhxqmcrd7gv302_b1pdfykh80000gn/T/RtmppTvpjj/devtools_install_10ce8392d5ddd/dodgr/libs
    ```

# Doscheda

Version: 1.2.0

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  6.0Mb
      sub-directories of 1Mb or more:
        data             2.1Mb
        doc              1.6Mb
        shiny-examples   2.0Mb
    ```

# DOSE

Version: 3.6.1

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  6.1Mb
      sub-directories of 1Mb or more:
        data   3.0Mb
        doc    2.7Mb
    ```

# dpcR

Version: 0.4

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 284-293 (overview.Rmd) 
    Error: processing vignette 'overview.Rmd' failed with diagnostics:
    `data` must be uniquely named but has duplicate columns
    Execution halted
    ```

# dplyr

Version: 0.7.7

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      
      ── 2. Failure: funs() gives a clear error message (#3368) (@test-funs.R#36)  ───
      `funs(~mp[.])` threw an error with unexpected message.
      Expected match: "`~mp[.]` must be a function name (quoted or unquoted) or an unquoted call, not `~`"
      Actual message: "`~mp[.]` must be a function name (quoted or unquoted) or an unquoted call, not ``~``"
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 2734 SKIPPED: 6 FAILED: 2
      1. Failure: funs() gives a clear error message (#3368) (@test-funs.R#28) 
      2. Failure: funs() gives a clear error message (#3368) (@test-funs.R#36) 
      
      Error: testthat unit tests failed
      In addition: Warning message:
      call dbDisconnect() when finished working with a connection 
      Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    `geom_smooth()` using method = 'gam' and formula 'y ~ s(x, bs = "cs")'
    Warning: Removed 1 rows containing non-finite values (stat_smooth).
    Warning: Removed 1 rows containing missing values (geom_point).
    Quitting from lines 651-653 (programming.Rmd) 
    Error: processing vignette 'programming.Rmd' failed with diagnostics:
    `UQE()` is defunct. Please use `!!get_expr(x)`
    Execution halted
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 4 marked UTF-8 strings
    ```

# DRIMSeq

Version: 1.8.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    Warning in log(det(n * (-H))) : NaNs produced
    Warning in rowSums(lgamma(y + prec * prop) - lgamma(prop * prec)) :
      value out of range in 'lgamma'
    Warning in rowSums(lgamma(y + prec * prop) - lgamma(prop * prec)) :
      value out of range in 'lgamma'
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'DRIMSeq.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `beramono.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.87 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# drLumi

Version: 0.1.2

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'drLumi.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `float.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.61 \usepackage
                    {setspace}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# DSAIDE

Version: 0.7.0

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.2Mb
      sub-directories of 1Mb or more:
        media       2.2Mb
        shinyapps   2.6Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘knitr’ ‘rmarkdown’ ‘utils’
      All declared Imports should be used.
    ```

# DSAIRM

Version: 0.4.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘knitr’ ‘rmarkdown’
      All declared Imports should be used.
    ```

# dslabs

Version: 0.5.1

## In both

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 4891 marked UTF-8 strings
    ```

# dslice

Version: 1.1.5

## In both

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    
    
    Use ‘data(package = .packages(all.available = TRUE))’
    to list the data sets in all *available* packages.
    
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'dslice.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.30 \ifthenelse
                    {\boolean{Sweave@inconsolata}}{%^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# DstarM

Version: 0.3.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘Rcpp’
      All declared Imports should be used.
    ```

# dtwclust

Version: 5.5.1

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
      Distance matrix is not symmetric, and hierarchical clustering assumes it is (it ignores the upper triangular).
    Loading required package: doParallel
    Loading required package: foreach
    Loading required package: iterators
    Loading required package: clue
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'dtwclust.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.55 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking for GNU extensions in Makefiles ... NOTE
    ```
    GNU make is a SystemRequirements.
    ```

# dtwSat

Version: 0.2.4

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error: processing vignette 'applying_twdtw.Rmd' failed with diagnostics:
    pandoc version 2.2 or higher is required and was not found (see the help page ?rmarkdown::pandoc_available).
    Execution halted
    ```

# DVHmetrics

Version: 0.3.7

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
      max dose is less than requested dose
    Warning in getVolume(pm$valRefNum, type = "absolute", unitRef = pm$unitRef,  :
      max dose is less than requested dose
    Warning in getEQD2.default(D = D$dvh[, "dose"], fd = fd, ab = ab) :
      'D' must be > 0
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'DVHmetrics.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `scrartcl.cls' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: cls)
    
    ! Emergency stop.
    <read *> 
             
    l.5 DIV15]{scrartcl}\usepackage
                                   []{graphicx}\usepackage[]{color}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# dynr

Version: 0.1.13-4

## In both

*   checking whether package ‘dynr’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/dynr/new/dynr.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘dynr’ ...
** package ‘dynr’ successfully unpacked and MD5 sums checked
checking for gsl-config... no
configure: error: gsl-config not found, is GSL installed?
ERROR: configuration failed for package ‘dynr’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/dynr/new/dynr.Rcheck/dynr’

```
### CRAN

```
* installing *source* package ‘dynr’ ...
** package ‘dynr’ successfully unpacked and MD5 sums checked
checking for gsl-config... no
configure: error: gsl-config not found, is GSL installed?
ERROR: configuration failed for package ‘dynr’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/dynr/old/dynr.Rcheck/dynr’

```
# dynsim

Version: 1.2.1

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    Error in re-building vignettes:
      ...
    
    Lagging invest by 1 time units.
    
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'dynsim-overview.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.55 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# dynutils

Version: 1.0.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘Rcpp’ ‘processx’
      All declared Imports should be used.
    ```

# earlywarnings

Version: 1.0.59

## In both

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Description field: should contain one or more complete sentences.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    surrogates_RShiny: no visible global function definition for ‘par’
    surrogates_RShiny: no visible global function definition for ‘hist’
    surrogates_RShiny: no visible global function definition for ‘abline’
    surrogates_RShiny: no visible global function definition for ‘points’
    surrogates_RShiny: no visible global function definition for ‘title’
    surrogates_ews: no visible global function definition for ‘dev.new’
    surrogates_ews: no visible global function definition for ‘par’
    surrogates_ews: no visible global function definition for ‘hist’
    surrogates_ews: no visible global function definition for ‘abline’
    surrogates_ews: no visible global function definition for ‘points’
    surrogates_ews: no visible global function definition for ‘mtext’
    Undefined global functions or variables:
      abline aes contour dev.new geom_tile grid hist labs layout legend
      lines mtext par plot points rainbow scale_fill_gradient stat_contour
      text title topo.colors xlab ylab
    Consider adding
      importFrom("grDevices", "dev.new", "rainbow", "topo.colors")
      importFrom("graphics", "abline", "contour", "grid", "hist", "layout",
                 "legend", "lines", "mtext", "par", "plot", "points", "text",
                 "title")
    to your NAMESPACE file.
    ```

# earthtones

Version: 0.1.0

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      2: quasi_label(enquo(object), label)
      3: eval_bare(get_expr(quo), get_env(quo))
      4: get_earthtones()
      5: ggmap::get_map(location = c(longitude, latitude), maptype = "satellite", zoom = zoom, 
             ...)
      6: get_googlemap(center = location, zoom = zoom, maptype = maptype, scale = scale, messaging = messaging, 
             urlonly = urlonly, force = force, filename = filename, color = color, language = language)
      7: download.file(url, destfile = tmp, quiet = !messaging, mode = "wb")
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 3 SKIPPED: 0 FAILED: 1
      1. Error: returns normal (@test-earth.R#5) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 26-28 (earthtones.rmd) 
    Error: processing vignette 'earthtones.rmd' failed with diagnostics:
    cannot open URL 'http://maps.googleapis.com/maps/api/staticmap?center=36.094994,-111.837962&zoom=12&size=640x640&scale=2&maptype=satellite&language=en-EN&sensor=false'
    Execution halted
    ```

# EasyHTMLReport

Version: 0.1.1

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘ggplot2’ ‘reshape2’ ‘scales’ ‘xtable’
      All declared Imports should be used.
    Packages in Depends field not imported from:
      ‘base64enc’ ‘knitr’ ‘markdown’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    .file_attachment: no visible global function definition for
      ‘base64encode’
    .plot_attachment: no visible global function definition for ‘dev.off’
    .update_list: no visible global function definition for ‘modifyList’
    easyHtmlReport: no visible global function definition for ‘knit’
    easyHtmlReport: no visible global function definition for
      ‘markdownToHTML’
    mime_part.data.frame: no visible global function definition for
      ‘write.table’
    mime_part.ggplot: no visible binding for global variable ‘pdf’
    mime_part.matrix: no visible global function definition for
      ‘write.table’
    mime_part.trellis: no visible binding for global variable ‘pdf’
    simpleHtmlReport : <anonymous>: no visible global function definition
      for ‘write.table’
    Undefined global functions or variables:
      base64encode dev.off knit markdownToHTML modifyList pdf write.table
    Consider adding
      importFrom("grDevices", "dev.off", "pdf")
      importFrom("utils", "modifyList", "write.table")
    to your NAMESPACE file.
    ```

# echor

Version: 0.1.1

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘sf’
    ```

# ecoengine

Version: 1.11.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘magrittr’
      All declared Imports should be used.
    ```

# eda4treeR

Version: 0.2.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘dae’ ‘dplyr’
      All declared Imports should be used.
    ```

# edge

Version: 2.12.2

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    `geom_smooth()` using method = 'loess' and formula 'y ~ x'
    Warning: code chunks must not depend on the uncached chunk "endotoxin_sig"
    Warning: code chunks must not depend on the uncached chunk "endotoxin_extract"
    Warning: code chunks must not depend on the uncached chunk "sva"
    Warning: code chunks must not depend on the uncached chunk "kidneyModel"
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'edge.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `fullpage.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.60 \usepackage
                    {bibentry}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    fitFDist: no visible global function definition for ‘median’
    fitFDist: no visible global function definition for ‘lm.fit’
    fitFDist: no visible global function definition for ‘predict’
    fit_wmodels: no visible global function definition for ‘model.matrix’
    fit_wmodels: no visible global function definition for ‘lm.wfit’
    null: no visible global function definition for ‘model.matrix’
    apply_sva,deSet: no visible global function definition for ‘as.formula’
    apply_sva,deSet: no visible global function definition for ‘terms’
    fit_models,deSet: no visible global function definition for
      ‘model.matrix’
    fullModel<-,deSet: no visible global function definition for
      ‘model.matrix’
    lrt,deSet-deFit: no visible global function definition for ‘pf’
    nullModel<-,deSet: no visible global function definition for
      ‘model.matrix’
    Undefined global functions or variables:
      as.formula lm.fit lm.wfit median model.matrix pf predict terms
    Consider adding
      importFrom("stats", "as.formula", "lm.fit", "lm.wfit", "median",
                 "model.matrix", "pf", "predict", "terms")
    to your NAMESPACE file.
    ```

# eechidna

Version: 1.1

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  6.4Mb
      sub-directories of 1Mb or more:
        data   5.1Mb
        doc    1.2Mb
    ```

# eegc

Version: 1.6.1

## In both

*   R CMD check timed out
    

*   checking Rd cross-references ... WARNING
    ```
    Missing link or links in documentation object 'barplotEnrich.Rd':
      ‘[DOSE]{barplot.enrichResult}’
    
    See section 'Cross-references' in the 'Writing R Extensions' manual.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 11.7Mb
      sub-directories of 1Mb or more:
        data  11.0Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    grnPlot: no visible global function definition for ‘legend’
    markerScatter: no visible global function definition for
      ‘colorRampPalette’
    markerScatter: no visible global function definition for ‘plot’
    markerScatter: no visible global function definition for ‘points’
    markerScatter: no visible global function definition for ‘lm’
    markerScatter: no visible global function definition for ‘abline’
    markerScatter: no visible global function definition for ‘text’
    markerScatter: no visible global function definition for ‘legend’
    Undefined global functions or variables:
      abline adjustcolor axis colorRampPalette control density dev.copy2pdf
      legend lines lm model.matrix p.adjust par phyper plot points quantile
      results text title treat
    Consider adding
      importFrom("grDevices", "adjustcolor", "colorRampPalette",
                 "dev.copy2pdf")
      importFrom("graphics", "abline", "axis", "legend", "lines", "par",
                 "plot", "points", "text", "title")
      importFrom("stats", "density", "lm", "model.matrix", "p.adjust",
                 "phyper", "quantile")
    to your NAMESPACE file.
    ```

# EFDR

Version: 0.1.1

## In both

*   checking R code for possible problems ... NOTE
    ```
    .gdf : find_loss: no visible global function definition for ‘rnorm’
    .p.values : <anonymous>: no visible global function definition for
      ‘pnorm’
    .relist.dwt: no visible global function definition for ‘relist’
    .relist.dwt: no visible global function definition for ‘as’
    .std.wav.coeff : <anonymous>: no visible global function definition for
      ‘mad’
    regrid: no visible global function definition for ‘predict’
    regrid: no visible global function definition for ‘var’
    regrid: no visible global function definition for ‘medpolish’
    Undefined global functions or variables:
      as mad medpolish pnorm predict relist rnorm var
    Consider adding
      importFrom("methods", "as")
      importFrom("stats", "mad", "medpolish", "pnorm", "predict", "rnorm",
                 "var")
      importFrom("utils", "relist")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

# effectR

Version: 1.0.2

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘rmarkdown’
      All declared Imports should be used.
    ```

# EGSEA

Version: 1.8.0

## In both

*   R CMD check timed out
    

*   checking whether package ‘EGSEA’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘graph’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/EGSEA/new/EGSEA.Rcheck/00install.out’ for details.
    ```

*   checking R code for possible problems ... NOTE
    ```
    buildGeneSetDBIdx: no visible binding for global variable ‘GOTERM’
    buildMSigDBIdx: no visible binding for global variable ‘msigdb’
    combinePvalues : <anonymous>: no visible global function definition for
      ‘votep’
    generateSummaryPlots: no visible binding for global variable ‘x.data’
    generateSummaryPlots: no visible binding for global variable ‘y.data’
    generateSummaryPlots: no visible binding for global variable ‘gsSize’
    generateSummaryPlots: no visible binding for global variable ‘id’
    generateSummaryPlots: no visible binding for global variable ‘sig’
    loadKeggData: no visible binding for global variable ‘kegg.pathways’
    Undefined global functions or variables:
      GOTERM gsSize id kegg.pathways msigdb sig votep x.data y.data
    ```

# ELMER

Version: 2.4.4

## In both

*   R CMD check timed out
    

*   checking dependencies in R code ... WARNING
    ```
    '::' or ':::' import not declared from: 'data.table'
    ':::' call which should be '::': 'TCGAbiolinks:::TCGAVisualize_volcano'
      See the note in ?`:::` about the use of this operator.
    Unexported objects imported by ':::' calls:
      'TCGAbiolinks:::colDataPrepare' 'TCGAbiolinks:::get.GRCh.bioMart'
      See the note in ?`:::` about the use of this operator.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 44.6Mb
      sub-directories of 1Mb or more:
        doc  44.1Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    heatmapGene: no visible global function definition for
      'subsetByOverlaps'
    heatmapGene: no visible binding for global variable 'mae'
    motif.enrichment.plot: no visible binding for global variable 'y'
    motif.enrichment.plot: no visible binding for global variable 'x'
    motif.enrichment.plot: no visible binding for global variable 'z'
    motif.enrichment.plot: no visible binding for global variable 'upperOR'
    motif.enrichment.plot: no visible binding for global variable 'lowerOR'
    motif.enrichment.plot: no visible binding for global variable 'motif'
    motif.enrichment.plot: no visible binding for global variable 'OR'
    scatter: no visible binding for global variable 'value'
    scatter: no visible global function definition for 'cor.test'
    scatter: no visible binding for global variable 'mae'
    Undefined global functions or variables:
      Gene GeneID Hugo_Symbol OR Probe TF Target cor.test fisher.test gr
      hm450.hg38.manifest label lowerOR mae motif precede pvalue
      subsetByOverlaps upperOR value write.table x y z
    Consider adding
      importFrom("stats", "cor.test", "fisher.test")
      importFrom("utils", "write.table")
    to your NAMESPACE file.
    ```

# emdi

Version: 1.1.4

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘graphics’ ‘maptools’ ‘rgeos’
      All declared Imports should be used.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 24 marked Latin-1 strings
    ```

# EMDomics

Version: 2.10.0

## In both

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Authors@R field gives more than one person with maintainer role:
      Sadhika Malladi <contact@sadhikamalladi.com> [aut, cre]
      Daniel Schmolze <emd@schmolze.com> [aut, cre]
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    calculate_cvm : <anonymous>: no visible global function definition for
      ‘median’
    calculate_cvm_gene: no visible global function definition for ‘combn’
    calculate_emd: no visible global function definition for ‘combn’
    calculate_emd : <anonymous>: no visible global function definition for
      ‘median’
    calculate_emd_gene: no visible global function definition for ‘combn’
    calculate_ks: no visible global function definition for ‘combn’
    calculate_ks : <anonymous>: no visible global function definition for
      ‘p.adjust’
    calculate_ks : <anonymous>: no visible global function definition for
      ‘median’
    calculate_ks_gene: no visible global function definition for ‘combn’
    calculate_ks_gene: no visible global function definition for ‘ks.test’
    Undefined global functions or variables:
      combn hist ks.test median p.adjust
    Consider adding
      importFrom("graphics", "hist")
      importFrom("stats", "ks.test", "median", "p.adjust")
      importFrom("utils", "combn")
    to your NAMESPACE file.
    ```

# emmeans

Version: 1.2.4

## In both

*   checking package dependencies ... NOTE
    ```
    Packages which this enhances but not available for checking:
      ‘CARBayes’ ‘coxme’ ‘gee’ ‘glmmADMB’ ‘MCMCglmm’ ‘MCMCpack’
    ```

# emojifont

Version: 0.5.2

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  7.1Mb
      sub-directories of 1Mb or more:
        emoji_fonts   6.4Mb
    ```

# enrichplot

Version: 1.0.2

## In both

*   R CMD check timed out
    

# enrichwith

Version: 0.1.1

## In both

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘betareg’
    ```

# EnvStats

Version: 2.3.1

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 11.4Mb
      sub-directories of 1Mb or more:
        R      3.0Mb
        doc    3.3Mb
        help   4.4Mb
    ```

# epiflows

Version: 0.2.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘htmlwidgets’
      All declared Imports should be used.
    ```

# episensr

Version: 0.9.2

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘llogistic’ ‘logitnorm’
      All declared Imports should be used.
    ```

# epos

Version: 0.1.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘dplyr’ ‘testthat’ ‘tidyr’
      All declared Imports should be used.
    ```

# erccdashboard

Version: 1.14.0

## In both

*   checking R code for possible problems ... NOTE
    ```
    dynRangePlot: no visible binding for global variable ‘value’
    dynRangePlot: no visible binding for global variable ‘Rep’
    saveERCCPlots: no visible global function definition for ‘par’
    testDECount: no visible binding for global variable ‘Atrend’
    testDECount: no visible binding for global variable ‘Dispersion’
    testDECount: no visible binding for global variable ‘Ratio’
    Undefined global functions or variables:
      Atrend Dispersion Ratio Rep par value
    Consider adding
      importFrom("graphics", "par")
    to your NAMESPACE file.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    Loading required package: gridExtra
    Warning: Ignoring unknown aesthetics: y
    Warning: Transformation introduced infinite values in continuous y-axis
    Warning: Ignoring unknown aesthetics: y
    Warning: Transformation introduced infinite values in continuous y-axis
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'erccdashboard.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `fullpage.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.5 \usepackage
                   {Sweave}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# erma

Version: 0.12.0

## In both

*   checking whether package ‘erma’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘GenomicFeatures’ was built under R version 3.5.1
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘DelayedArray’ was built under R version 3.5.1
      Warning: package ‘Rsamtools’ was built under R version 3.5.1
      Warning: package ‘rtracklayer’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/erma/new/erma.Rcheck/00install.out’ for details.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 200.6Mb
      sub-directories of 1Mb or more:
        bed_tabix  161.3Mb
        data        37.1Mb
        doc          1.4Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    geneTxRange: no visible global function definition for ‘select’
    genemodelOLD: no visible binding for global variable ‘exonsBy’
    stateProfile: no visible binding for global variable ‘states_25’
    stateProfile: no visible binding for global variable ‘short_celltype’
    stateProfile: no visible binding for global variable ‘name’
    statesByRange: no visible binding for global variable ‘mod’
    statesByRange: no visible binding for global variable ‘upstream’
    statesByRange: no visible binding for global variable ‘downstream’
    statesByRange : <anonymous>: no visible binding for global variable
      ‘tss’
    statesByRange: no visible binding for global variable ‘states_25’
    Undefined global functions or variables:
      downstream exonsBy mod name select short_celltype states_25 tss
      upstream
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Failed with error:  'package 'DelayedArray' could not be loaded'
      Error in .requirePackage(package) : 
        unable to find required package 'SummarizedExperiment'
      Calls: <Anonymous> ... getClass -> getClassDef -> .classEnv -> .requirePackage
      Execution halted
    ```

# esATAC

Version: 1.2.3

## In both

*   checking whether package ‘esATAC’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘Rsamtools’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘DelayedArray’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/esATAC/new/esATAC.Rcheck/00install.out’ for details.
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .BBSoptions
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 10.0Mb
      sub-directories of 1Mb or more:
        doc       3.5Mb
        extdata   5.1Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Unexported objects imported by ':::' calls:
      ‘ShortRead:::.ShortReadQQA’ ‘ShortRead:::.qa_adapterContamination’
      See the note in ?`:::` about the use of this operator.
    ```

# esetVis

Version: 1.6.3

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.3Mb
      sub-directories of 1Mb or more:
        doc   4.9Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      ‘rowData’
    getMethodsInputObjectEsetVis: no visible binding for global variable
      ‘colData’
    getMethodsInputObjectEsetVis: no visible binding for global variable
      ‘assay’
    getMethodsInputObjectEsetVis : <anonymous>: no visible global function
      definition for ‘colData’
    getMethodsInputObjectEsetVis : <anonymous>: no visible global function
      definition for ‘rowData’
    ggvisPlotEset: no visible global function definition for ‘:=’
    ggvisPlotEset: no visible binding for global variable ‘fill’
    ggvisPlotEset: no visible binding for global variable ‘shape’
    ggvisPlotEset: no visible binding for global variable ‘size’
    ggvisPlotEset: no visible binding for global variable ‘opacity’
    ggvisPlotEset: no visible global function definition for ‘%>%’
    ggvisPlotEset: no visible global function definition for
      ‘scale_ordinal’
    ggvisPlotEset: no visible global function definition for ‘set_options’
    Undefined global functions or variables:
      %>% := assay colData fill opacity rowData scale_ordinal set_options
      shape size
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘a4Base’
    ```

# ESGtoolkit

Version: 0.1

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'ESG2toolkit_Intro_062014.tex' failed.
    LaTeX errors:
    ! Package babel Error: Unknown option `british'. Either you misspelled it
    (babel)                or the language definition file british.ldf was not foun
    d.
    
    See the babel package documentation for explanation.
    ! LaTeX Error: File `mathpazo.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.56 \usepackage
                    [utf8]{inputenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Title field: should not end in a period.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    esgplotts: no visible global function definition for ‘deltat’
    esgplotts: no visible global function definition for ‘xlab’
    esgplotts: no visible global function definition for ‘ylab’
    esgplotts: no visible global function definition for ‘theme’
    simdiff: no visible global function definition for ‘ts’
    simshocks: no visible global function definition for ‘ts’
    simshocks: no visible global function definition for ‘qnorm’
    simshocks : <anonymous>: no visible global function definition for ‘ts’
    Undefined global functions or variables:
      abline aes colorRampPalette coord_flip cor cor.test deltat
      element_blank end geom_density geom_point is.ts lines matplot par
      plot points polygon pt qnorm qt quantile scale_color_manual
      scale_fill_manual sd start t.test theme time ts tsp window xlab ylab
    Consider adding
      importFrom("grDevices", "colorRampPalette")
      importFrom("graphics", "abline", "lines", "matplot", "par", "plot",
                 "points", "polygon")
      importFrom("stats", "cor", "cor.test", "deltat", "end", "is.ts", "pt",
                 "qnorm", "qt", "quantile", "sd", "start", "t.test", "time",
                 "ts", "tsp", "window")
    to your NAMESPACE file.
    ```

# etm

Version: 1.0.4

## In both

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'etmCIF_tutorial.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.30 \ifthenelse
                    {\boolean{Sweave@inconsolata}}{%^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# eurostat

Version: 3.2.9

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘sf’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# EValue

Version: 1.1.5

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘devtools’ ‘msm’
      All declared Imports should be used.
    ```

# EventStudy

Version: 0.34

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  6.6Mb
      sub-directories of 1Mb or more:
        doc   5.8Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘curl’ ‘openxlsx’ ‘stringr’
      All declared Imports should be used.
    ```

# evolqg

Version: 0.2-5

## In both

*   checking examples ... ERROR
    ```
    ...
    > ### Aliases: LModularity
    > 
    > ### ** Examples
    > 
    > # A modular matrix:
    > modules = matrix(c(rep(c(1, 0, 0), each = 5),
    + rep(c(0, 1, 0), each = 5),
    + rep(c(0, 0, 1), each = 5)), 15)
    > cor.hypot = CreateHypotMatrix(modules)[[4]]
    > hypot.mask = matrix(as.logical(cor.hypot), 15, 15)
    > mod.cor = matrix(NA, 15, 15)
    > mod.cor[ hypot.mask] = runif(length(mod.cor[ hypot.mask]), 0.8, 0.9) # within-modules
    > mod.cor[!hypot.mask] = runif(length(mod.cor[!hypot.mask]), 0.3, 0.4) # between-modules
    > diag(mod.cor) = 1
    > mod.cor = (mod.cor + t(mod.cor))/2 # correlation matrices should be symmetric
    > 
    > LModularity(mod.cor)
    Error in method(g, ...) : 
      At optimal_modularity.c:85 : GLPK is not available, Unimplemented function call
    Calls: LModularity -> method
    Execution halted
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      Loading required package: plyr
      ── 1. Error: LModularity resturn resonable results (@test.LModularity.r#12)  ───
      At optimal_modularity.c:85 : GLPK is not available, Unimplemented function call
      1: expect_equal(modules, LModularity(mod.cor)[[2]]) at testthat/test.LModularity.r:12
      2: quasi_label(enquo(expected), expected.label)
      3: eval_bare(get_expr(quo), get_env(quo))
      4: LModularity(mod.cor)
      5: method(g, ...)
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 20953 SKIPPED: 1 FAILED: 1
      1. Error: LModularity resturn resonable results (@test.LModularity.r#12) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘Rcpp’
      All declared Imports should be used.
    ```

# ewoc

Version: 0.2.0

## In both

*   checking whether package ‘ewoc’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/ewoc/new/ewoc.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘ewoc’ ...
** package ‘ewoc’ successfully unpacked and MD5 sums checked
** R
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rjags', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/ewoc/rjags/libs/rjags.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/ewoc/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/ewoc/rjags/libs/rjags.so
  Reason: image not found
ERROR: lazy loading failed for package ‘ewoc’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/ewoc/new/ewoc.Rcheck/ewoc’

```
### CRAN

```
* installing *source* package ‘ewoc’ ...
** package ‘ewoc’ successfully unpacked and MD5 sums checked
** R
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rjags', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/ewoc/rjags/libs/rjags.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/ewoc/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/ewoc/rjags/libs/rjags.so
  Reason: image not found
ERROR: lazy loading failed for package ‘ewoc’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/ewoc/old/ewoc.Rcheck/ewoc’

```
# eyetrackingR

Version: 0.1.7

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
      ...
    Warning: Removed 37 rows containing non-finite values (stat_summary).
    Warning: Removed 37 rows containing non-finite values (stat_summary).
    Warning: Removed 37 rows containing non-finite values (stat_summary).
    Warning: Removed 37 rows containing non-finite values (stat_summary).
    Warning: Removed 37 rows containing non-finite values (stat_summary).
    Warning: Removed 37 rows containing non-finite values (stat_summary).
    Warning: Removed 37 rows containing non-finite values (stat_summary).
    Warning: Removed 37 rows containing non-finite values (stat_summary).
    Warning in make_onset_data(response_window_clean, onset_time = 15500, fixation_window_length = 1,  :
      Very few trials have a legitimate first AOI! Possible incorrect onset time?
    Warning in max(SwitchAOI) :
      no non-missing arguments to max; returning -Inf
    Warning in min(SwitchAOI) :
      no non-missing arguments to min; returning Inf
    Warning in max(df_plot$.Time) :
      no non-missing arguments to max; returning -Inf
    Quitting from lines 91-93 (onset_contingent_analysis_vignette.Rmd) 
    Error: processing vignette 'onset_contingent_analysis_vignette.Rmd' failed with diagnostics:
    replacement has 1 row, data has 0
    Execution halted
    ```

# ezsim

Version: 0.5.5

## In both

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Description field: should contain one or more complete sentences.
    BugReports field is not a suitable URL but contains an email address
      which will be used as from R 3.4.0
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    plot.ezsim: no visible global function definition for ‘tail’
    plot.ezsim: no visible global function definition for ‘plot’
    plot.ezsim: no visible binding for global variable ‘dnorm’
    plot.ezsim: no visible binding for global variable ‘pdf’
    plot.ezsim: no visible global function definition for ‘dev.off’
    plot.ezsim : <anonymous>: no visible global function definition for
      ‘dev.new’
    plot.summary.ezsim: no visible global function definition for ‘head’
    plot.summary.ezsim: no visible global function definition for ‘tail’
    plot.summary.ezsim: no visible binding for global variable ‘pdf’
    plot.summary.ezsim: no visible global function definition for ‘dev.off’
    plot.summary.ezsim : <anonymous>: no visible global function definition
      for ‘dev.new’
    Undefined global functions or variables:
      as.formula dev.new dev.off dnorm head pdf plot quantile runif tail
    Consider adding
      importFrom("grDevices", "dev.new", "dev.off", "pdf")
      importFrom("graphics", "plot")
      importFrom("stats", "as.formula", "dnorm", "quantile", "runif")
      importFrom("utils", "head", "tail")
    to your NAMESPACE file.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    Warning: The labeller API has been updated. Labellers taking `variable`and `value` arguments are now deprecated. See labellers documentation.
    Warning: The labeller API has been updated. Labellers taking `variable`and `value` arguments are now deprecated. See labellers documentation.
    Warning: The labeller API has been updated. Labellers taking `variable`and `value` arguments are now deprecated. See labellers documentation.
    Warning: The labeller API has been updated. Labellers taking `variable`and `value` arguments are now deprecated. See labellers documentation.
    Warning: The labeller API has been updated. Labellers taking `variable`and `value` arguments are now deprecated. See labellers documentation.
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'vignette.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `fullpage.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.5 \usepackage
                   {Sweave}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# facopy

Version: 1.13.0

## In both

*   checking whether package ‘facopy’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/facopy/new/facopy.Rcheck/00install.out’ for details.
    ```

*   checking package dependencies ... NOTE
    ```
    Depends: includes the non-default packages:
      ‘cgdsr’ ‘coin’ ‘ggplot2’ ‘gridExtra’ ‘facopy.annot’ ‘grid’
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

## Installation

### Devel

```
* installing *source* package ‘facopy’ ...
** R
** data
** inst
** byte-compile and prepare package for lazy loading
Error : object ‘setting.graph.attributes’ is not exported by 'namespace:DOSE'
ERROR: lazy loading failed for package ‘facopy’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/facopy/new/facopy.Rcheck/facopy’

```
### CRAN

```
* installing *source* package ‘facopy’ ...
** R
** data
** inst
** byte-compile and prepare package for lazy loading
Error : object ‘setting.graph.attributes’ is not exported by 'namespace:DOSE'
ERROR: lazy loading failed for package ‘facopy’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/facopy/old/facopy.Rcheck/facopy’

```
# factoextra

Version: 1.0.5

## In both

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘NbClust’
    ```

# factorMerger

Version: 0.3.6

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    LaTeX errors found:
    ! LaTeX Error: Command \k unavailable in encoding OT1.
    
    See the LaTeX manual or LaTeX Companion for explanation.
    Type  H <return>  for immediate help.
     ...                                              
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘forcats’ ‘formula.tools’
      All declared Imports should be used.
    ```

# FAOSTAT

Version: 2.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'FAOSTAT.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.55 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    Aggregation : foo: no visible global function definition for
      ‘weighted.mean’
    Aggregation: no visible global function definition for ‘txtProgressBar’
    Aggregation: no visible global function definition for
      ‘setTxtProgressBar’
    constructSYB: no visible global function definition for ‘str’
    fillCountryCode: no visible global function definition for ‘na.omit’
    getFAO: no visible global function definition for ‘read.csv’
    getWDImetaData: no visible global function definition for ‘write.csv’
    getWDItoSYB: no visible global function definition for ‘na.omit’
    lsgr: no visible global function definition for ‘na.omit’
    lsgr: no visible global function definition for ‘coef’
    lsgr: no visible global function definition for ‘lm’
    Undefined global functions or variables:
      coef lm na.omit read.csv setTxtProgressBar str txtProgressBar
      weighted.mean write.csv
    Consider adding
      importFrom("stats", "coef", "lm", "na.omit", "weighted.mean")
      importFrom("utils", "read.csv", "setTxtProgressBar", "str",
                 "txtProgressBar", "write.csv")
    to your NAMESPACE file.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 179 marked UTF-8 strings
    ```

# fCCAC

Version: 1.6.0

## In both

*   checking whether package ‘fCCAC’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/fCCAC/new/fCCAC.Rcheck/00install.out’ for details.
    ```

*   checking R code for possible problems ... NOTE
    ```
    fccac: no visible binding for global variable ‘variables’
    Undefined global functions or variables:
      variables
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    Warning: package ‘IRanges’ was built under R version 3.5.1
    Loading required package: GenomicRanges
    Warning: package ‘GenomicRanges’ was built under R version 3.5.1
    Loading required package: GenomeInfoDb
    Loading required package: grid
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'fCCAC.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `beramono.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.87 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# fdANOVA

Version: 0.1.2

## In both

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'fdANOVA.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.55 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# FDboost

Version: 0.3-2

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    
    Loading required package: maps
    See www.image.ucar.edu/~nychka/Fields for
     a vignette and other supplements. 
    Loading required package: mapdata
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'FLAM_canada.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `authblk.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.61 \usepackage
                    [left=25mm, right=25mm, top=20mm, bottom=20mm]{geometry}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# fdq

Version: 0.2

## In both

*   checking whether package ‘fdq’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/fdq/new/fdq.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘fdq’ ...
** package ‘fdq’ successfully unpacked and MD5 sums checked
** R
** byte-compile and prepare package for lazy loading
Warning in fun(libname, pkgname) : couldn't connect to display ""
Error: package or namespace load failed for ‘Fgmutils’ in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]):
 there is no package called ‘ReporteRsjars’
Error : package ‘Fgmutils’ could not be loaded
ERROR: lazy loading failed for package ‘fdq’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/fdq/new/fdq.Rcheck/fdq’

```
### CRAN

```
* installing *source* package ‘fdq’ ...
** package ‘fdq’ successfully unpacked and MD5 sums checked
** R
** byte-compile and prepare package for lazy loading
Warning in fun(libname, pkgname) : couldn't connect to display ""
Error: package or namespace load failed for ‘Fgmutils’ in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]):
 there is no package called ‘ReporteRsjars’
Error : package ‘Fgmutils’ could not be loaded
ERROR: lazy loading failed for package ‘fdq’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/fdq/old/fdq.Rcheck/fdq’

```
# FField

Version: 0.1.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' calls in package code:
      ‘ggplot2’ ‘gridExtra’
      Please use :: or requireNamespace() instead.
      See section 'Suggested packages' in the 'Writing R Extensions' manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    FFieldPtRepDemo: no visible global function definition for ‘ggplot’
    FFieldPtRepDemo: no visible binding for global variable ‘mtcars’
    FFieldPtRepDemo: no visible global function definition for ‘aes’
    FFieldPtRepDemo: no visible binding for global variable ‘mpg’
    FFieldPtRepDemo: no visible global function definition for ‘geom_point’
    FFieldPtRepDemo: no visible global function definition for ‘geom_text’
    FFieldPtRepDemo: no visible global function definition for ‘ggtitle’
    FFieldPtRepDemo: no visible global function definition for
      ‘geom_segment’
    FFieldPtRepDemo: no visible global function definition for
      ‘grid.arrange’
    Undefined global functions or variables:
      aes geom_point geom_segment geom_text ggplot ggtitle grid.arrange mpg
      mtcars
    Consider adding
      importFrom("datasets", "mtcars")
    to your NAMESPACE file.
    ```

# Fgmutils

Version: 0.9.4

## In both

*   checking whether package ‘Fgmutils’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/Fgmutils/new/Fgmutils.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘Fgmutils’ ...
** package ‘Fgmutils’ successfully unpacked and MD5 sums checked
** R
** byte-compile and prepare package for lazy loading
Warning in fun(libname, pkgname) : couldn't connect to display ""
Error in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]) : 
  there is no package called ‘ReporteRsjars’
ERROR: lazy loading failed for package ‘Fgmutils’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/Fgmutils/new/Fgmutils.Rcheck/Fgmutils’

```
### CRAN

```
* installing *source* package ‘Fgmutils’ ...
** package ‘Fgmutils’ successfully unpacked and MD5 sums checked
** R
** byte-compile and prepare package for lazy loading
Warning in fun(libname, pkgname) : couldn't connect to display ""
Error in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]) : 
  there is no package called ‘ReporteRsjars’
ERROR: lazy loading failed for package ‘Fgmutils’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/Fgmutils/old/Fgmutils.Rcheck/Fgmutils’

```
# fgsea

Version: 1.6.0

## Newly broken

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      fgsLbl> labels <- as.numeric(as.factor(gsub(" .*", "", es$title)))
      
      fgsLbl> fgseaRes <- fgseaLabel(pathways, mat, labels, nperm = 1000, minSize = 15, maxSize = 500)
      
      fgsLbl> ## End(No test)
      fgsLbl> 
      fgsLbl> 
      fgsLbl> 
      fgsLbl> 
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 59 SKIPPED: 1 FAILED: 1
      1. Error: fgsea works (@test_gsea_analysis.R#26) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

# fieldRS

Version: 0.1.1

## In both

*   checking whether package ‘fieldRS’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/fieldRS/new/fieldRS.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘fieldRS’ ...
** package ‘fieldRS’ successfully unpacked and MD5 sums checked
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
Error in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]) : 
  there is no package called ‘sf’
ERROR: lazy loading failed for package ‘fieldRS’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/fieldRS/new/fieldRS.Rcheck/fieldRS’

```
### CRAN

```
* installing *source* package ‘fieldRS’ ...
** package ‘fieldRS’ successfully unpacked and MD5 sums checked
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
Error in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]) : 
  there is no package called ‘sf’
ERROR: lazy loading failed for package ‘fieldRS’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/fieldRS/old/fieldRS.Rcheck/fieldRS’

```
# fiftystater

Version: 1.0.1

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 83-87 (fiftystater.Rmd) 
    Error: processing vignette 'fiftystater.Rmd' failed with diagnostics:
    unused argument (output)
    Execution halted
    ```

# FindMyFriends

Version: 1.10.0

## In both

*   R CMD check timed out
    

*   checking for code/documentation mismatches ... WARNING
    ```
    Functions or methods with usage in documentation object 'pgVirtual-class' but not in code:
      as
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.0Mb
      sub-directories of 1Mb or more:
        doc       1.5Mb
        extdata   1.8Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Unexported objects imported by ':::' calls:
      ‘gtable:::insert.unit’ ‘gtable:::z_arrange_gtables’
      See the note in ?`:::` about the use of this operator.
    ```

# fingerPro

Version: 1.1

## In both

*   checking whether package ‘fingerPro’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/fingerPro/new/fingerPro.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘fingerPro’ ...
** package ‘fingerPro’ successfully unpacked and MD5 sums checked
** libs
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/fingerPro/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/fingerPro/RcppGSL/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/fingerPro/RcppProgress/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c RcppExports.cpp -o RcppExports.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/fingerPro/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/fingerPro/RcppGSL/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/fingerPro/RcppProgress/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c fingerprinting.cpp -o fingerprinting.o
In file included from RcppExports.cpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/fingerPro/RcppGSL/include/RcppGSL.h:25:
/Users/max/github/ggplot2/revdep/library.noindex/fingerPro/RcppGSL/include/RcppGSLForward.h:26:10: fatal error: 'gsl/gsl_vector.h' file not found
#include <gsl/gsl_vector.h> 
         ^~~~~~~~~~~~~~~~~~
1 error generated.
make: *** [RcppExports.o] Error 1
make: *** Waiting for unfinished jobs....
In file included from fingerprinting.cpp:5:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/fingerPro/RcppGSL/include/RcppGSL.h:25:
/Users/max/github/ggplot2/revdep/library.noindex/fingerPro/RcppGSL/include/RcppGSLForward.h:26:10: fatal error: 'gsl/gsl_vector.h' file not found
#include <gsl/gsl_vector.h> 
         ^~~~~~~~~~~~~~~~~~
1 error generated.
make: *** [fingerprinting.o] Error 1
ERROR: compilation failed for package ‘fingerPro’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/fingerPro/new/fingerPro.Rcheck/fingerPro’

```
### CRAN

```
* installing *source* package ‘fingerPro’ ...
** package ‘fingerPro’ successfully unpacked and MD5 sums checked
** libs
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/fingerPro/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/fingerPro/RcppGSL/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/fingerPro/RcppProgress/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c RcppExports.cpp -o RcppExports.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/fingerPro/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/fingerPro/RcppGSL/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/fingerPro/RcppProgress/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c fingerprinting.cpp -o fingerprinting.o
In file included from RcppExports.cpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/fingerPro/RcppGSL/include/RcppGSL.h:25:
/Users/max/github/ggplot2/revdep/library.noindex/fingerPro/RcppGSL/include/RcppGSLForward.h:26:10: fatal error: 'gsl/gsl_vector.h' file not found
#include <gsl/gsl_vector.h> 
         ^~~~~~~~~~~~~~~~~~
1 error generated.
make: *** [RcppExports.o] Error 1
make: *** Waiting for unfinished jobs....
In file included from fingerprinting.cpp:5:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/fingerPro/RcppGSL/include/RcppGSL.h:25:
/Users/max/github/ggplot2/revdep/library.noindex/fingerPro/RcppGSL/include/RcppGSLForward.h:26:10: fatal error: 'gsl/gsl_vector.h' file not found
#include <gsl/gsl_vector.h> 
         ^~~~~~~~~~~~~~~~~~
1 error generated.
make: *** [fingerprinting.o] Error 1
ERROR: compilation failed for package ‘fingerPro’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/fingerPro/old/fingerPro.Rcheck/fingerPro’

```
# fingertipscharts

Version: 0.0.2

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘sf’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# fingertipsR

Version: 0.1.9

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘purrr’
      All declared Imports should be used.
    ```

# fishmove

Version: 0.3-3

## In both

*   checking R code for possible problems ... NOTE
    ```
    fishmove : coefs: no visible global function definition for ‘coef’
    fishmove : coefs: no visible global function definition for ‘pf’
    fishmove: no visible global function definition for ‘lm’
    fishmove: no visible global function definition for ‘predict.lm’
    fishmove.estimate: no visible global function definition for ‘quantile’
    fishmove.estimate : ddoublenorm: no visible global function definition
      for ‘dnorm’
    fishmove.query: no visible global function definition for ‘hasArg’
    fishmove.query : f: no visible global function definition for ‘pnorm’
    fishmove.query: no visible global function definition for ‘uniroot’
    pdk: no visible global function definition for ‘hasArg’
    pdk : eq: no visible global function definition for ‘dnorm’
    Undefined global functions or variables:
      coef dnorm hasArg lm pf pnorm predict.lm quantile uniroot
    Consider adding
      importFrom("methods", "hasArg")
      importFrom("stats", "coef", "dnorm", "lm", "pf", "pnorm", "predict.lm",
                 "quantile", "uniroot")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

# fitbitScraper

Version: 0.1.8

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 22-24 (fitbitScraper-examples.Rmd) 
    Error: processing vignette 'fitbitScraper-examples.Rmd' failed with diagnostics:
    login failed
    Execution halted
    ```

# fitdistrplus

Version: 1.0-11

## In both

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    Warning in plotdistcens(salinity, Turnbull = FALSE) :
      The argument Turnbull is deprecated and should note be used any more. 
                Now use the argument NPMLE to tell if you want to compute a nonparametric
                maximum likelihood estimation of the cumulative distribution, and the argument NPMLE.method
                to define the method chosen for the computation (Turnbull or Wang).
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'paper2JSS.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.30 \ifthenelse
                    {\boolean{Sweave@inconsolata}}{%^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# fitur

Version: 0.6.1

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘rstudioapi’
      All declared Imports should be used.
    ```

# FLightR

Version: 0.4.7

## Newly broken

*   checking examples ... ERROR
    ```
    ...
     checking dusk 13
     checking dusk 14
     checking dusk 15
    
    
    calibration method used: parametric.slope 
    > 
    > Grid<-make.grid(left=0, bottom=50, right=10, top=56,
    +   distance.from.land.allowed.to.use=c(-Inf, Inf),
    +   distance.from.land.allowed.to.stay=c(-Inf, Inf))
    > 
    > all.in<-make.prerun.object(Proc.data, Grid, start=c(5.43, 52.93),
    +                              Calibration=Calibration, threads=2)
    likelihood correction turned off as no correction found in the calibration
    making cluster
    Warning in socketConnection("localhost", port = port, server = TRUE, blocking = TRUE,  :
      port 11436 cannot be opened
    Error in socketConnection("localhost", port = port, server = TRUE, blocking = TRUE,  : 
      cannot open the connection
    Calls: make.prerun.object ... makePSOCKcluster -> newPSOCKnode -> socketConnection
    Execution halted
    ```

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘rgdal’
      All declared Imports should be used.
    Missing or unexported object: ‘ggmap::has_goog_key’
    ```

# flowAI

Version: 1.10.1

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.5Mb
      sub-directories of 1Mb or more:
        data   5.1Mb
    ```

# flowCHIC

Version: 1.14.0

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  6.2Mb
      sub-directories of 1Mb or more:
        extdata   5.7Mb
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Description field: should contain one or more complete sentences.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    plot_nmds,data.frame-data.frame: no visible global function definition
      for ‘as.dist’
    plot_nmds,data.frame-data.frame: no visible global function definition
      for ‘par’
    plot_nmds,data.frame-data.frame: no visible global function definition
      for ‘plot’
    plot_nmds,data.frame-data.frame: no visible global function definition
      for ‘hclust’
    plot_nmds,data.frame-data.frame: no visible global function definition
      for ‘points’
    plot_nmds,data.frame-data.frame: no visible global function definition
      for ‘text’
    plot_nmds,data.frame-data.frame: no visible global function definition
      for ‘legend’
    Undefined global functions or variables:
      as.dist count dev.off hclust legend par plot png points text
    Consider adding
      importFrom("grDevices", "dev.off", "png")
      importFrom("graphics", "legend", "par", "plot", "points", "text")
      importFrom("stats", "as.dist", "hclust")
    to your NAMESPACE file.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Packages unavailable to check Rd xrefs: ‘flowViz’, ‘flowCyBar’
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
      ...
    Warning in metaMDS(mat.dist, autotransform = FALSE) :
      stress is (nearly) zero: you may have insufficient data
    Warning in metaMDS(mat.dist, autotransform = FALSE) :
      stress is (nearly) zero: you may have insufficient data
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'flowCHICmanual.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `cite.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.10 \usepackage
                    {caption}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# flowDiv

Version: 2.0

## In both

*   checking whether package ‘flowDiv’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘flowWorkspace’ was built under R version 3.5.1
      Warning: package ‘flowCore’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/flowDiv/new/flowDiv.Rcheck/00install.out’ for details.
    ```

# flowTime

Version: 1.4.0

## In both

*   checking whether package ‘flowTime’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘flowCore’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/flowTime/new/flowTime.Rcheck/00install.out’ for details.
    ```

# fontHind

Version: 0.1.1

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 38-49 (Using_fontHind.Rmd) 
    Error: processing vignette 'Using_fontHind.Rmd' failed with diagnostics:
    polygon edge not found
    Execution halted
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘ggplot2’
      All declared Imports should be used.
    ```

# fontMPlus

Version: 0.1.1

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 56-67 (Using_fontMPlus.Rmd) 
    Error: processing vignette 'Using_fontMPlus.Rmd' failed with diagnostics:
    polygon edge not found
    Execution halted
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘ggplot2’
      All declared Imports should be used.
    ```

# forecast

Version: 8.4

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error: processing vignette 'JSS2008.Rmd' failed with diagnostics:
    pandoc version 2.2 or higher is required and was not found (see the help page ?rmarkdown::pandoc_available).
    Execution halted
    ```

*   checking Rd cross-references ... NOTE
    ```
    Packages unavailable to check Rd xrefs: ‘seasonal’, ‘forecTheta’
    ```

# ForecastComb

Version: 1.3.1

## In both

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘ForecastCombinations’
    ```

# ForecastFramework

Version: 0.9.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
      number of items to replace is not a multiple of replacement length
    Warning in private$.dnames[[1]][rows] = rownames(data) :
      number of items to replace is not a multiple of replacement length
    Warning in private$.dnames[[1]][rows] = rownames(data) :
      number of items to replace is not a multiple of replacement length
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'ClassDiagram.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `pbox.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.56 \usepackage
                    {adjustbox}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# FourCSeq

Version: 1.14.0

## In both

*   checking whether package ‘FourCSeq’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘DelayedArray’ was built under R version 3.5.1
      Note: break used in wrong context: no loop is visible 
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/FourCSeq/new/FourCSeq.Rcheck/00install.out’ for details.
    Information on the location(s) of code generating the ‘Note’s can be
    obtained by re-running with environment variable R_KEEP_PKG_SOURCE set
    to ‘yes’.
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    Coordinate system already present. Adding new coordinate system, which will replace the existing one.
    Coordinate system already present. Adding new coordinate system, which will replace the existing one.
    Coordinate system already present. Adding new coordinate system, which will replace the existing one.
    Coordinate system already present. Adding new coordinate system, which will replace the existing one.
    Coordinate system already present. Adding new coordinate system, which will replace the existing one.
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'FourCSeq.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `beramono.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.87 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Packages listed in more than one of Depends, Imports, Suggests, Enhances:
      ‘DESeq2’ ‘GenomicRanges’
    A package should be listed in only one of these fields.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Packages in Depends field not imported from:
      'ggplot2' 'methods'
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      'keepSeqlevels'
    updateObject,FourC: no visible global function definition for
      'callNextMethod'
    Undefined global functions or variables:
      DataFrame IRanges Rle Seqinfo SimpleList abline aes axis blues9
      callNextMethod change count differentialInteraction element_blank fit
      fitDown fitUp formula geom_path geom_point ggplot keepSeqlevels labs
      legend mad median metadata mid mtext new p.adjust par peak plot pnorm
      points predict rel relevel scale_fill_gradient2 scale_y_continuous
      seqlengths seqlevels seqlevels<- subjectHits subsetByOverlaps theme
      theme_bw theme_set write.table
    Consider adding
      importFrom("grDevices", "blues9")
      importFrom("graphics", "abline", "axis", "legend", "mtext", "par",
                 "plot", "points")
      importFrom("methods", "callNextMethod", "new")
      importFrom("stats", "formula", "mad", "median", "p.adjust", "pnorm",
                 "predict", "relevel")
      importFrom("utils", "write.table")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

# frailtyEM

Version: 0.8.8

## In both

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    Loading required package: survival
    Loading required package: gridExtra
    Warning: Removed 2 rows containing missing values (geom_path).
    Warning in emfrail(Surv(time, status) ~ age + sex + cluster(id), data = kidney,  :
      Hessian virtually 0; frailty variance might be at the edge of the parameter space.
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'frailtyEM_manual.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.55 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# frailtySurv

Version: 1.3.5

## In both

*   checking whether package ‘frailtySurv’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      hcubature.c:923:12: warning: comparison of unsigned enum expression < 0 is always false [-Wtautological-compare]
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/frailtySurv/new/frailtySurv.Rcheck/00install.out’ for details.
    ```

# FREddyPro

Version: 1.0

## In both

*   checking examples ... ERROR
    ```
    ...
    > 
    > ## Load the data
    > data(fluxes)
    > 
    > ## Clean fluxes
    > fluxes=cleanFluxes(fluxes,sdCor=TRUE,sdTimes=3,timesList=3,distCor=TRUE,
    +                    thresholdList=list(H=c(-100,1000),LE=c(-100,1000)))	
    > 
    > ## Remove some line as an example
    > fluxes=fluxes[-c(832,833,834,835,840,953),]
    > 
    > ## Create timestamp from date and time
    > fluxes=createTimestamp(fluxes)
    > 
    > ## Now fill the gap timestmaps with NA's
    > fluxes=fillTimestamp(fluxes,flux=TRUE)
    [1] Discontinuity in 3 instances
    [1] 2014-06-14 03:30:00 UTC 2014-06-14 06:00:00 UTC 2014-06-16 14:30:00 UTC
    Error in FUN(X[[i]], ...) : subscript out of bounds
    Calls: fillTimestamp ... shift_time -> [[ -> [[.POSIXlt -> .POSIXlt -> lapply
    Execution halted
    ```

# fredr

Version: 1.0.0

## In both

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 1 marked UTF-8 string
    ```

# FRK

Version: 0.2.2

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    The following objects are masked from 'package:base':
    
        intersect, setdiff, setequal, union
    
    Note: show_basis assumes spherical distance functions when plotting
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'FRK_intro.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `subfig.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.58 \usepackage
                    {graphicx}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking: ‘dggrids’ ‘INLA’
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  8.4Mb
      sub-directories of 1Mb or more:
        data   5.3Mb
        doc    2.1Mb
    ```

# frontiles

Version: 1.2

## In both

*   checking R code for possible problems ... NOTE
    ```
    ...
    alphafrontier.2d: no visible global function definition for ‘par’
    alphafrontier.2d: no visible global function definition for ‘polygon’
    alphafrontier.2d: no visible global function definition for ‘lines’
    alphafrontier.2d: no visible global function definition for ‘points’
    alphafrontier.3d: no visible global function definition for ‘layout’
    alphafrontier.3d: no visible global function definition for ‘barplot’
    alphafrontier.3d: no visible global function definition for ‘axis’
    alphafrontier.3d: no visible global function definition for ‘lines’
    alphafrontier.3d: no visible global function definition for ‘na.omit’
    ordermfrontier.2d: no visible global function definition for ‘par’
    ordermfrontier.2d: no visible global function definition for ‘polygon’
    ordermfrontier.2d: no visible global function definition for ‘lines’
    ordermfrontier.2d: no visible global function definition for ‘points’
    ordermscore.boot: no visible global function definition for ‘sd’
    Undefined global functions or variables:
      axis barplot layout legend lines mtext na.omit par points polygon sd
    Consider adding
      importFrom("graphics", "axis", "barplot", "layout", "legend", "lines",
                 "mtext", "par", "points", "polygon")
      importFrom("stats", "na.omit", "sd")
    to your NAMESPACE file.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    spDataLarge package with: `install.packages('spDataLarge',
    repos='https://nowosad.github.io/drat/', type='source'))`
    Loading required package: colorspace
    Loading required package: rgl
    Loading required package: sp
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'frontiles1.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.30 \ifthenelse
                    {\boolean{Sweave@inconsolata}}{%^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# FunciSNP

Version: 1.24.0

## In both

*   checking whether package ‘FunciSNP’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘GenomicFeatures’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/FunciSNP/new/FunciSNP.Rcheck/00install.out’ for details.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Package listed in more than one of Depends, Imports, Suggests, Enhances:
      ‘ggplot2’
    A package should be listed in only one of these fields.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    bedColors: no visible global function definition for 'col2rgb'
    bedColors: no visible global function definition for 'png'
    bedColors: no visible global function definition for 'box'
    bedColors: no visible global function definition for 'par'
    bedColors: no visible global function definition for 'axis'
    bedColors: no visible global function definition for 'dev.off'
    Undefined global functions or variables:
      R.squared TSS.human.GRCh37 abline as.dendrogram axis box col2rgb
      colorRampPalette data dev.off dist distance.from.tag fisher.test
      hclust lincRNA order.dendrogram p.adjust par pdf plot png r.2
      read.delim read.table refseqgenes sig text timestamp value variable
      write.table
    Consider adding
      importFrom("grDevices", "col2rgb", "colorRampPalette", "dev.off",
                 "pdf", "png")
      importFrom("graphics", "abline", "axis", "box", "par", "plot", "text")
      importFrom("stats", "as.dendrogram", "dist", "fisher.test", "hclust",
                 "order.dendrogram", "p.adjust")
      importFrom("utils", "data", "read.delim", "read.table", "timestamp",
                 "write.table")
    to your NAMESPACE file.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    To view results, submit bed file as a
    custom track in UCSC Genome Browser (genome.ucsc.edu), 
    
    Now have fun with your new YAFSNPss!!
    ####
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'FunciSNP_vignette.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `fullpage.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.10 \usepackage
                    {hyperref}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# furrowSeg

Version: 1.8.0

## In both

*   checking whether package ‘furrowSeg’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘EBImage’ was built under R version 3.5.1
      Warning: replacing previous import ‘EBImage::abind’ by ‘abind::abind’ when loading ‘furrowSeg’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/furrowSeg/new/furrowSeg.Rcheck/00install.out’ for details.
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Warning in has_utility("pdfcrop") :
      pdfcrop not installed or not in PATH
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'exampleFurrowSegmentation.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `float.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.30 \date
              {}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 349.5Mb
      sub-directories of 1Mb or more:
        data  348.5Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    identifyTimeMinArea: no visible global function definition for ‘points’
    identifyTimeMinArea: no visible global function definition for ‘abline’
    plotFeatureEvolution: no visible global function definition for ‘par’
    plotFeatureEvolution: no visible binding for global variable ‘median’
    plotFeatureEvolution: no visible binding for global variable ‘quantile’
    plotFeatureEvolution: no visible global function definition for ‘plot’
    plotFeatureEvolution: no visible global function definition for
      ‘polygon’
    plotFeatureEvolution: no visible global function definition for ‘rgb’
    plotFeatureEvolution: no visible global function definition for ‘axis’
    plotFeatureEvolution: no visible global function definition for ‘mtext’
    plotFeatureEvolution: no visible global function definition for ‘title’
    Undefined global functions or variables:
      abline axis median mtext par plot points polygon predict quantile rgb
      title
    Consider adding
      importFrom("grDevices", "rgb")
      importFrom("graphics", "abline", "axis", "mtext", "par", "plot",
                 "points", "polygon", "title")
      importFrom("stats", "median", "predict", "quantile")
    to your NAMESPACE file.
    ```

# gaiah

Version: 0.0.2

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘ggplot2’ ‘maptools’ ‘rgeos’ ‘stringr’ ‘tidyr’
      All declared Imports should be used.
    ```

# gapfill

Version: 0.9.6

## In both

*   checking package dependencies ... NOTE
    ```
    Package which this enhances but not available for checking: ‘doMPI’
    ```

# GARS

Version: 1.0.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    -- note: fitType='parametric', but the dispersion trend was not well captured by the
       function: y = a/x + b, and a local regression fit was automatically substituted.
       specify fitType='local' or 'mean' to avoid this message next time.
    Warning in has_utility("pdfcrop") :
      pdfcrop not installed or not in PATH
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'GARS.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `beramono.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.87 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# gastempt

Version: 0.4.01

## In both

*   checking whether package ‘gastempt’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/gastempt/new/gastempt.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘gastempt’ ...
** package ‘gastempt’ successfully unpacked and MD5 sums checked
** libs
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I"`"/Library/Frameworks/R.framework/Resources/bin/Rscript" --vanilla -e "cat(system.file('include', 'src', package = 'StanHeaders'))"`" -DBOOST_RESULT_OF_USE_TR1 -DBOOST_NO_DECLTYPE -DBOOST_DISABLE_ASSERTS -DEIGEN_NO_DEBUG -DBOOST_MATH_OVERFLOW_ERROR_POLICY=ignore_error -I"/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/gastempt/BH/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/gastempt/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c Modules.cpp -o Modules.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I"`"/Library/Frameworks/R.framework/Resources/bin/Rscript" --vanilla -e "cat(system.file('include', 'src', package = 'StanHeaders'))"`" -DBOOST_RESULT_OF_USE_TR1 -DBOOST_NO_DECLTYPE -DBOOST_DISABLE_ASSERTS -DEIGEN_NO_DEBUG -DBOOST_MATH_OVERFLOW_ERROR_POLICY=ignore_error -I"/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/gastempt/BH/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/gastempt/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c init.cpp -o init.o
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/stan_fit.hpp:20:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/stan_args.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/BH/include/boost/lexical_cast.hpp:32:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/BH/include/boost/lexical_cast/try_lexical_convert.hpp:42:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/BH/include/boost/lexical_cast/detail/converter_lexical.hpp:52:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/BH/include/boost/container/container_fwd.hpp:61:
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/BH/include/boost/container/detail/std_fwd.hpp:27:1: warning: inline namespaces are a C++11 feature [-Wc++11-inline-namespace]
BOOST_MOVE_STD_NS_BEG
^
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/BH/include/boost/move/detail/std_ns_begin.hpp:18:34: note: expanded from macro 'BOOST_MOVE_STD_NS_BEG'
   #define BOOST_MOVE_STD_NS_BEG _LIBCPP_BEGIN_NAMESPACE_STD
                                 ^
/usr/local/clang4/bin/../include/c++/v1/__config:388:52: note: expanded from macro '_LIBCPP_BEGIN_NAMESPACE_STD'
#define _LIBCPP_BEGIN_NAMESPACE_STD namespace std {inline namespace _LIBCPP_NAMESPACE {
                                                   ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/core.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/core/autodiffstackstorage.hpp:4:
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/memory/stack_alloc.hpp:145:10: warning: 'auto' type specifier is a C++11 extension [-Wc++11-extensions]
    for (auto& block : blocks_)
         ^
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/memory/stack_alloc.hpp:145:22: warning: range-based for loop is a C++11 extension [-Wc++11-extensions]
    for (auto& block : blocks_)
                     ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/core.hpp:4:
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/core/autodiffstackstorage.hpp:47:69: warning: deleted function definitions are a C++11 extension [-Wc++11-extensions]
    AutodiffStackStorage &operator=(const AutodiffStackStorage &) = delete;
                                                                    ^
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/core/autodiffstackstorage.hpp:60:30: warning: deleted function definitions are a C++11 extension [-Wc++11-extensions]
  AutodiffStackSingleton() = delete;
                             ^
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/core/autodiffstackstorage.hpp:61:71: warning: deleted function definitions are a C++11 extension [-Wc++11-extensions]
  explicit AutodiffStackSingleton(AutodiffStackSingleton_t const &) = delete;
                                                                      ^
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/core/autodiffstackstorage.hpp:62:73: warning: deleted function definitions are a C++11 extension [-Wc++11-extensions]
  AutodiffStackSingleton &operator=(const AutodiffStackSingleton_t &) = delete;
                                                                        ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/Dense:1:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/Core:531:
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/Dense:2:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/LU:47:
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/Dense:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/Cholesky:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/Jacobi:29:
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/Dense:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/Cholesky:43:
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/Dense:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/QR:17:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/Householder:27:
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/Dense:5:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/SVD:48:
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/Dense:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/Geometry:58:
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/Dense:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/Eigenvalues:58:
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/core.hpp:42:
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/core/recover_memory.hpp:25:8: warning: 'auto' type specifier is a C++11 extension [-Wc++11-extensions]
  for (auto &x : ChainableStack::instance().var_alloc_stack_) {
       ^
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/core/recover_memory.hpp:25:16: warning: range-based for loop is a C++11 extension [-Wc++11-extensions]
  for (auto &x : ChainableStack::instance().var_alloc_stack_) {
               ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/core.hpp:44:
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/core/set_zero_all_adjoints.hpp:15:8: warning: 'auto' type specifier is a C++11 extension [-Wc++11-extensions]
  for (auto &x : ChainableStack::instance().var_stack_)
       ^
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/core/set_zero_all_adjoints.hpp:15:16: warning: range-based for loop is a C++11 extension [-Wc++11-extensions]
  for (auto &x : ChainableStack::instance().var_stack_)
               ^
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/core/set_zero_all_adjoints.hpp:17:8: warning: 'auto' type specifier is a C++11 extension [-Wc++11-extensions]
  for (auto &x : ChainableStack::instance().var_nochain_stack_)
       ^
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/core/set_zero_all_adjoints.hpp:17:16: warning: range-based for loop is a C++11 extension [-Wc++11-extensions]
  for (auto &x : ChainableStack::instance().var_nochain_stack_)
               ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat.hpp:22:
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:19:59: error: expected expression
class ops_partials_edge<ViewElt, Eigen::Matrix<Op, R, C>> {
                                                          ^
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:35:2: error: expected a type
};
 ^
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:38:70: error: expected expression
class ops_partials_edge<ViewElt, std::vector<Eigen::Matrix<Op, R, C>>> {
                                                                     ^
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:38:72: error: expected a type
class ops_partials_edge<ViewElt, std::vector<Eigen::Matrix<Op, R, C>>> {
                                                                       ^
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:53:2: error: expected a type
};
 ^
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:56:60: error: a space is required between consecutive right angle brackets (use '> >')
class ops_partials_edge<ViewElt, std::vector<std::vector<Op>>> {
                                                           ^~
                                                           > > 
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:58:68: error: a space is required between consecutive right angle brackets (use '> >')
  typedef empty_broadcast_array<ViewElt, std::vector<std::vector<Op>>>
                                                                   ^~
                                                                   > > 
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:61:63: error: a space is required between consecutive right angle brackets (use '> >')
  empty_broadcast_array<partials_t, std::vector<std::vector<Op>>> partials_vec_;
                                                              ^~
                                                              > > 
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:63:62: error: a space is required between consecutive right angle brackets (use '> >')
  explicit ops_partials_edge(const std::vector<std::vector<Op>> ops) {}
                                                             ^~
                                                             > >
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat.hpp:96:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/fun/csr_extract_u.hpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/Sparse:26:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/SparseCore:66:
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat.hpp:96:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/fun/csr_extract_u.hpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/Sparse:27:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/OrderingMethods:71:
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat.hpp:96:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/fun/csr_extract_u.hpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/Sparse:29:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/SparseCholesky:43:
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat.hpp:96:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/fun/csr_extract_u.hpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/Sparse:32:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/SparseQR:35:
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat.hpp:96:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/fun/csr_extract_u.hpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/Sparse:33:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/IterativeLinearSolvers:46:
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat.hpp:99:
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/fun/csr_matrix_times_vector.hpp:91:14: warning: range-based for loop is a C++11 extension [-Wc++11-extensions]
  for (int i : v)
             ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat.hpp:100:
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/fun/csr_to_dense_matrix.hpp:48:14: warning: range-based for loop is a C++11 extension [-Wc++11-extensions]
  for (int i : v)
             ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat.hpp:156:
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/fun/log_mix.hpp:178:26: error: expected expression
              .unaryExpr([](T_partials_return x) { return exp(x); });
                         ^
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/fun/log_mix.hpp:262:24: error: expected expression
            .unaryExpr([](T_partials_return x) { return exp(x); });
                       ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat.hpp:161:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp.hpp:5:
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_2x2.hpp:21:13: warning: alias declarations are a C++11 extension [-Wc++11-extensions]
  using T = typename Mtype::Scalar;
            ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat.hpp:162:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_multiply.hpp:5:
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_action_handler.hpp:20:10: error: unknown type name 'constexpr'
  static constexpr int p_max = 8;
         ^
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_action_handler.hpp:20:20: error: expected member name or ';' after declaration specifiers
  static constexpr int p_max = 8;
  ~~~~~~~~~~~~~~~~ ^
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_action_handler.hpp:21:10: error: unknown type name 'constexpr'
  static constexpr int m_max = 55;
         ^
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_action_handler.hpp:21:20: error: expected member name or ';' after declaration specifiers
  static constexpr int m_max = 55;
  ~~~~~~~~~~~~~~~~ ^
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_action_handler.hpp:22:10: error: unknown type name 'constexpr'
  static constexpr double tol = 1.1e-16;
         ^
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_action_handler.hpp:22:20: error: expected member name or ';' after declaration specifiers
  static constexpr double tol = 1.1e-16;
  ~~~~~~~~~~~~~~~~ ^
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_action_handler.hpp:25:29: error: function definition does not declare parameters
  const std::vector<double> theta_m_single_precision{
                            ^
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_action_handler.hpp:28:29: error: function definition does not declare parameters
  const std::vector<double> theta_m_double_precision{
                            ^
fatal error: too many errors emitted, stopping now [-ferror-limit=]
29 warnings and 20 errors generated.
make: *** [Modules.o] Error 1
ERROR: compilation failed for package ‘gastempt’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/gastempt/new/gastempt.Rcheck/gastempt’

```
### CRAN

```
* installing *source* package ‘gastempt’ ...
** package ‘gastempt’ successfully unpacked and MD5 sums checked
** libs
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I"`"/Library/Frameworks/R.framework/Resources/bin/Rscript" --vanilla -e "cat(system.file('include', 'src', package = 'StanHeaders'))"`" -DBOOST_RESULT_OF_USE_TR1 -DBOOST_NO_DECLTYPE -DBOOST_DISABLE_ASSERTS -DEIGEN_NO_DEBUG -DBOOST_MATH_OVERFLOW_ERROR_POLICY=ignore_error -I"/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/gastempt/BH/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/gastempt/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c Modules.cpp -o Modules.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I"`"/Library/Frameworks/R.framework/Resources/bin/Rscript" --vanilla -e "cat(system.file('include', 'src', package = 'StanHeaders'))"`" -DBOOST_RESULT_OF_USE_TR1 -DBOOST_NO_DECLTYPE -DBOOST_DISABLE_ASSERTS -DEIGEN_NO_DEBUG -DBOOST_MATH_OVERFLOW_ERROR_POLICY=ignore_error -I"/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/gastempt/BH/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/gastempt/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c init.cpp -o init.o
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/stan_fit.hpp:20:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/stan_args.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/BH/include/boost/lexical_cast.hpp:32:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/BH/include/boost/lexical_cast/try_lexical_convert.hpp:42:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/BH/include/boost/lexical_cast/detail/converter_lexical.hpp:52:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/BH/include/boost/container/container_fwd.hpp:61:
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/BH/include/boost/container/detail/std_fwd.hpp:27:1: warning: inline namespaces are a C++11 feature [-Wc++11-inline-namespace]
BOOST_MOVE_STD_NS_BEG
^
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/BH/include/boost/move/detail/std_ns_begin.hpp:18:34: note: expanded from macro 'BOOST_MOVE_STD_NS_BEG'
   #define BOOST_MOVE_STD_NS_BEG _LIBCPP_BEGIN_NAMESPACE_STD
                                 ^
/usr/local/clang4/bin/../include/c++/v1/__config:388:52: note: expanded from macro '_LIBCPP_BEGIN_NAMESPACE_STD'
#define _LIBCPP_BEGIN_NAMESPACE_STD namespace std {inline namespace _LIBCPP_NAMESPACE {
                                                   ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/core.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/core/autodiffstackstorage.hpp:4:
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/memory/stack_alloc.hpp:145:10: warning: 'auto' type specifier is a C++11 extension [-Wc++11-extensions]
    for (auto& block : blocks_)
         ^
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/memory/stack_alloc.hpp:145:22: warning: range-based for loop is a C++11 extension [-Wc++11-extensions]
    for (auto& block : blocks_)
                     ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/core.hpp:4:
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/core/autodiffstackstorage.hpp:47:69: warning: deleted function definitions are a C++11 extension [-Wc++11-extensions]
    AutodiffStackStorage &operator=(const AutodiffStackStorage &) = delete;
                                                                    ^
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/core/autodiffstackstorage.hpp:60:30: warning: deleted function definitions are a C++11 extension [-Wc++11-extensions]
  AutodiffStackSingleton() = delete;
                             ^
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/core/autodiffstackstorage.hpp:61:71: warning: deleted function definitions are a C++11 extension [-Wc++11-extensions]
  explicit AutodiffStackSingleton(AutodiffStackSingleton_t const &) = delete;
                                                                      ^
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/core/autodiffstackstorage.hpp:62:73: warning: deleted function definitions are a C++11 extension [-Wc++11-extensions]
  AutodiffStackSingleton &operator=(const AutodiffStackSingleton_t &) = delete;
                                                                        ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/Dense:1:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/Core:531:
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/Dense:2:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/LU:47:
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/Dense:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/Cholesky:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/Jacobi:29:
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/Dense:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/Cholesky:43:
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/Dense:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/QR:17:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/Householder:27:
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/Dense:5:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/SVD:48:
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/Dense:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/Geometry:58:
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/Dense:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/Eigenvalues:58:
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/core.hpp:42:
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/core/recover_memory.hpp:25:8: warning: 'auto' type specifier is a C++11 extension [-Wc++11-extensions]
  for (auto &x : ChainableStack::instance().var_alloc_stack_) {
       ^
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/core/recover_memory.hpp:25:16: warning: range-based for loop is a C++11 extension [-Wc++11-extensions]
  for (auto &x : ChainableStack::instance().var_alloc_stack_) {
               ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/core.hpp:44:
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/core/set_zero_all_adjoints.hpp:15:8: warning: 'auto' type specifier is a C++11 extension [-Wc++11-extensions]
  for (auto &x : ChainableStack::instance().var_stack_)
       ^
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/core/set_zero_all_adjoints.hpp:15:16: warning: range-based for loop is a C++11 extension [-Wc++11-extensions]
  for (auto &x : ChainableStack::instance().var_stack_)
               ^
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/core/set_zero_all_adjoints.hpp:17:8: warning: 'auto' type specifier is a C++11 extension [-Wc++11-extensions]
  for (auto &x : ChainableStack::instance().var_nochain_stack_)
       ^
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/core/set_zero_all_adjoints.hpp:17:16: warning: range-based for loop is a C++11 extension [-Wc++11-extensions]
  for (auto &x : ChainableStack::instance().var_nochain_stack_)
               ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat.hpp:22:
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:19:59: error: expected expression
class ops_partials_edge<ViewElt, Eigen::Matrix<Op, R, C>> {
                                                          ^
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:35:2: error: expected a type
};
 ^
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:38:70: error: expected expression
class ops_partials_edge<ViewElt, std::vector<Eigen::Matrix<Op, R, C>>> {
                                                                     ^
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:38:72: error: expected a type
class ops_partials_edge<ViewElt, std::vector<Eigen::Matrix<Op, R, C>>> {
                                                                       ^
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:53:2: error: expected a type
};
 ^
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:56:60: error: a space is required between consecutive right angle brackets (use '> >')
class ops_partials_edge<ViewElt, std::vector<std::vector<Op>>> {
                                                           ^~
                                                           > > 
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:58:68: error: a space is required between consecutive right angle brackets (use '> >')
  typedef empty_broadcast_array<ViewElt, std::vector<std::vector<Op>>>
                                                                   ^~
                                                                   > > 
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:61:63: error: a space is required between consecutive right angle brackets (use '> >')
  empty_broadcast_array<partials_t, std::vector<std::vector<Op>>> partials_vec_;
                                                              ^~
                                                              > > 
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:63:62: error: a space is required between consecutive right angle brackets (use '> >')
  explicit ops_partials_edge(const std::vector<std::vector<Op>> ops) {}
                                                             ^~
                                                             > >
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat.hpp:96:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/fun/csr_extract_u.hpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/Sparse:26:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/SparseCore:66:
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat.hpp:96:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/fun/csr_extract_u.hpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/Sparse:27:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/OrderingMethods:71:
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat.hpp:96:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/fun/csr_extract_u.hpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/Sparse:29:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/SparseCholesky:43:
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat.hpp:96:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/fun/csr_extract_u.hpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/Sparse:32:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/SparseQR:35:
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat.hpp:96:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/fun/csr_extract_u.hpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/Sparse:33:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/IterativeLinearSolvers:46:
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat.hpp:99:
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/fun/csr_matrix_times_vector.hpp:91:14: warning: range-based for loop is a C++11 extension [-Wc++11-extensions]
  for (int i : v)
             ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat.hpp:100:
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/fun/csr_to_dense_matrix.hpp:48:14: warning: range-based for loop is a C++11 extension [-Wc++11-extensions]
  for (int i : v)
             ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat.hpp:156:
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/fun/log_mix.hpp:178:26: error: expected expression
              .unaryExpr([](T_partials_return x) { return exp(x); });
                         ^
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/fun/log_mix.hpp:262:24: error: expected expression
            .unaryExpr([](T_partials_return x) { return exp(x); });
                       ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat.hpp:161:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp.hpp:5:
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_2x2.hpp:21:13: warning: alias declarations are a C++11 extension [-Wc++11-extensions]
  using T = typename Mtype::Scalar;
            ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat.hpp:162:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_multiply.hpp:5:
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_action_handler.hpp:20:10: error: unknown type name 'constexpr'
  static constexpr int p_max = 8;
         ^
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_action_handler.hpp:20:20: error: expected member name or ';' after declaration specifiers
  static constexpr int p_max = 8;
  ~~~~~~~~~~~~~~~~ ^
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_action_handler.hpp:21:10: error: unknown type name 'constexpr'
  static constexpr int m_max = 55;
         ^
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_action_handler.hpp:21:20: error: expected member name or ';' after declaration specifiers
  static constexpr int m_max = 55;
  ~~~~~~~~~~~~~~~~ ^
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_action_handler.hpp:22:10: error: unknown type name 'constexpr'
  static constexpr double tol = 1.1e-16;
         ^
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_action_handler.hpp:22:20: error: expected member name or ';' after declaration specifiers
  static constexpr double tol = 1.1e-16;
  ~~~~~~~~~~~~~~~~ ^
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_action_handler.hpp:25:29: error: function definition does not declare parameters
  const std::vector<double> theta_m_single_precision{
                            ^
/Users/max/github/ggplot2/revdep/library.noindex/gastempt/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_action_handler.hpp:28:29: error: function definition does not declare parameters
  const std::vector<double> theta_m_double_precision{
                            ^
fatal error: too many errors emitted, stopping now [-ferror-limit=]
29 warnings and 20 errors generated.
make: *** [Modules.o] Error 1
ERROR: compilation failed for package ‘gastempt’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/gastempt/old/gastempt.Rcheck/gastempt’

```
# gcatest

Version: 1.10.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'gcatest.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `float.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.58 \usepackage
                    {gensymb}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking R code for possible problems ... NOTE
    ```
    gcat: no visible global function definition for ‘pchisq’
    gcat.stat: no visible global function definition for ‘complete.cases’
    gcatest: no visible global function definition for ‘pchisq’
    Undefined global functions or variables:
      complete.cases pchisq
    Consider adding
      importFrom("stats", "complete.cases", "pchisq")
    to your NAMESPACE file.
    ```

# gCrisprTools

Version: 1.8.0

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  9.2Mb
      sub-directories of 1Mb or more:
        data   2.3Mb
        doc    1.1Mb
    ```

# GDCRNATools

Version: 1.1.1

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error: processing vignette 'GDCRNATools.Rmd' failed with diagnostics:
    there is no package called ‘prettydoc’
    Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  6.7Mb
      sub-directories of 1Mb or more:
        R      3.2Mb
        data   1.1Mb
        doc    2.2Mb
    ```

*   checking top-level files ... NOTE
    ```
    File
      LICENSE
    is not mentioned in the DESCRIPTION file.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    gdcRNAMerge: no visible global function definition for ‘read.table’
    gdcRNAMerge : <anonymous>: no visible global function definition for
      ‘read.delim’
    gdcRNAMerge: no visible global function definition for ‘read.delim’
    hyperTestFun: no visible global function definition for ‘phyper’
    kmTestFun: no visible global function definition for ‘pchisq’
    kmTestFun: no visible global function definition for ‘qnorm’
    manifestDownloadFun: no visible global function definition for
      ‘read.table’
    mirCorTestFun: no visible global function definition for ‘cor.test’
    multiRegFun: no visible global function definition for ‘cor.test’
    Undefined global functions or variables:
      Category Counts FDR Regulation Terms URLencode aggregate cor.test
      download.file foldEnrichment model.matrix p.adjust pchisq phyper
      qnorm read.delim read.table unzip write.table
    Consider adding
      importFrom("stats", "aggregate", "cor.test", "model.matrix",
                 "p.adjust", "pchisq", "phyper", "qnorm")
      importFrom("utils", "URLencode", "download.file", "read.delim",
                 "read.table", "unzip", "write.table")
    to your NAMESPACE file.
    ```

# genBart

Version: 1.0.1

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘NMF’ ‘RColorBrewer’ ‘VennDiagram’ ‘clValid’ ‘data.table’ ‘grid’
      ‘gtools’ ‘pca3d’ ‘rmarkdown’ ‘scales’ ‘shinydashboard’ ‘shinyjs’
      ‘statmod’ ‘stringr’
      All declared Imports should be used.
    ```

# gender

Version: 0.5.2

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘genderdata’
    ```

# GENEAsphere

Version: 1.4

## In both

*   checking whether package ‘GENEAsphere’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/GENEAsphere/new/GENEAsphere.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘GENEAsphere’ ...
** package ‘GENEAsphere’ successfully unpacked and MD5 sums checked
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
Error : object ‘epoch.apply’ is not exported by 'namespace:GENEAread'
ERROR: lazy loading failed for package ‘GENEAsphere’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/GENEAsphere/new/GENEAsphere.Rcheck/GENEAsphere’

```
### CRAN

```
* installing *source* package ‘GENEAsphere’ ...
** package ‘GENEAsphere’ successfully unpacked and MD5 sums checked
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
Error : object ‘epoch.apply’ is not exported by 'namespace:GENEAread'
ERROR: lazy loading failed for package ‘GENEAsphere’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/GENEAsphere/old/GENEAsphere.Rcheck/GENEAsphere’

```
# GeneralizedUmatrix

Version: 1.1.2

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.0Mb
      sub-directories of 1Mb or more:
        doc   4.7Mb
    ```

# genlogis

Version: 1.0.0

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘genlogis-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: genlog_simu
    > ### Title: Simulating the Generalized logistic distribution
    > ### Aliases: genlog_simu
    > ### Keywords: genlogis
    > 
    > ### ** Examples
    > 
    >  
    > genlog_simu(real.par = c(0.3, 0.9, 1.5, 0.0), init.par = c(0.9, 0.3, 0.2, 0.0), 
    +             sample.size = 100, k = 50, threads = 2, seed = 200) 
    Warning in socketConnection("localhost", port = port, server = TRUE, blocking = TRUE,  :
      port 11756 cannot be opened
    Error in socketConnection("localhost", port = port, server = TRUE, blocking = TRUE,  : 
      cannot open the connection
    Calls: genlog_simu ... makePSOCKcluster -> newPSOCKnode -> socketConnection
    Execution halted
    ```

# GenoGAM

Version: 1.8.0

## In both

*   checking whether package ‘GenoGAM’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘Rsamtools’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘DelayedArray’ was built under R version 3.5.1
      Note: break used in wrong context: no loop is visible 
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/GenoGAM/new/GenoGAM.Rcheck/00install.out’ for details.
    Information on the location(s) of code generating the ‘Note’s can be
    obtained by re-running with environment variable R_KEEP_PKG_SOURCE set
    to ‘yes’.
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    3: package ‘GenomicRanges’ was built under R version 3.5.1 
    4: package ‘DelayedArray’ was built under R version 3.5.1 
    Warning: Removed 4 rows containing missing values (geom_point).
    Warning in has_utility("pdfcrop") :
      pdfcrop not installed or not in PATH
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'GenoGAM.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `beramono.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.87 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.0Mb
      sub-directories of 1Mb or more:
        R         2.0Mb
        doc       1.2Mb
        extdata   1.7Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    plot_base: no visible global function definition for ‘lines’
    plot_base: no visible global function definition for ‘abline’
    plot_base: no visible global function definition for ‘mtext’
    writeToBroadPeaks: no visible global function definition for
      ‘write.table’
    writeToNarrowPeaks: no visible global function definition for
      ‘write.table’
    xsd : <anonymous>: no visible binding for global variable ‘position’
    xsd: no visible binding for global variable ‘position’
    Undefined global functions or variables:
      abline as.formula axis coefficients dev.off dnbinom estimate fdr fit
      gene hist id lines mtext optim p.adjust par plot png pnorm position
      pval pvalue region runif write.table zscore
    Consider adding
      importFrom("grDevices", "dev.off", "png")
      importFrom("graphics", "abline", "axis", "hist", "lines", "mtext",
                 "par", "plot")
      importFrom("stats", "as.formula", "coefficients", "dnbinom", "optim",
                 "p.adjust", "pnorm", "runif")
      importFrom("utils", "write.table")
    to your NAMESPACE file.
    ```

*   checking Rd \usage sections ... NOTE
    ```
    S3 methods shown with full name in documentation object 'plot.GenoGAM':
      ‘plot.GenoGAM’
    
    The \usage entries for S3 methods should use the \method markup and not
    their full name.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

# genomation

Version: 1.12.0

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  6.0Mb
      sub-directories of 1Mb or more:
        doc       3.5Mb
        extdata   1.2Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘RUnit’
      All declared Imports should be used.
    Unexported object imported by a ':::' call: ‘BiocGenerics:::testPackage’
      See the note in ?`:::` about the use of this operator.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ScoreMatrixBin,RleList-GRangesList: no visible binding for global
      variable ‘id’
    ScoreMatrixBin,RleList-GRangesList: no visible global function
      definition for ‘:=’
    Undefined global functions or variables:
      := id
    ```

*   checking files in ‘vignettes’ ... NOTE
    ```
    The following directory looks like a leftover from 'knitr':
      ‘cache’
    Please remove from your package.
    ```

# GenomicDataCommons

Version: 1.4.3

## Newly broken

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    Loading required package: magrittr
    
    Attaching package: 'GenomicDataCommons'
    
    The following object is masked from 'package:stats':
    
        filter
    
    Quitting from lines 145-146 (overview.Rmd) 
    Error: processing vignette 'overview.Rmd' failed with diagnostics:
    failed to rename downloaded file:
    
      from: '/Users/max/Library/Caches/GenomicDataCommons/1c07df73-e470-4755-a183-30eed2d08ea3/.partial_download'
      to: '/Users/max/Library/Caches/GenomicDataCommons/1c07df73-e470-4755-a183-30eed2d08ea3/538a1fe4-8e69-4a29-a789-da52c075e75f.htseq.counts.gz'
      reason:
        cannot rename file
        '/Users/max/Library/Caches/GenomicDataCommons/1c07df73-e470-4755-a183-30eed2d08ea3/.partial_download'
        to
        '/Users/max/Library/Caches/GenomicDataCommons/1c07df73-e470-4755-a183-30eed2d08ea3/538a1fe4-8e69-4a29-a789-da52c075e75f.htseq.counts.gz',
        reason 'No such file or directory'
    Execution halted
    ```

## In both

*   checking Rd \usage sections ... WARNING
    ```
    Undocumented arguments in documentation object '.htseq_importer'
      ‘fnames’
    
    Functions with \usage entries need to have the appropriate \alias
    entries, and all their arguments documented.
    The \usage entries must correspond to syntactically valid R code.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    default_fields.character: no visible binding for global variable
      ‘defaults’
    gdc_rnaseq: no visible binding for global variable ‘case_id’
    gdc_rnaseq: no visible binding for global variable ‘file_id’
    Undefined global functions or variables:
      case_id defaults file_id
    ```

# GenomicInteractions

Version: 1.14.0

## In both

*   checking whether package ‘GenomicInteractions’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘DelayedArray’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/GenomicInteractions/new/GenomicInteractions.Rcheck/00install.out’ for details.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 11.1Mb
      sub-directories of 1Mb or more:
        doc       1.8Mb
        extdata   7.9Mb
    ```

# genotypeeval

Version: 1.12.0

## In both

*   checking whether package ‘genotypeeval’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘DelayedArray’ was built under R version 3.5.1
      Warning: package ‘Rsamtools’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/genotypeeval/new/genotypeeval.Rcheck/00install.out’ for details.
    ```

*   checking for missing documentation entries ... WARNING
    ```
    Undocumented code objects:
      ‘makeHistogram’ ‘pcaPlot’ ‘tsnePlot’
    All user-level objects in a package should have documentation entries.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Missing or unexported object: ‘GenomicRanges::subset’
    ```

# GenVisR

Version: 1.12.1

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      NULL
      NULL
      NULL
      NULL
      NULL
      NULL
      NULL
      NULL
      NULL
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 557 SKIPPED: 45 FAILED: 1
      1. Failure: setDomainHeights domains are nested correctly (@test-Lolliplot-class.R#525) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   R CMD check timed out
    

*   checking installed package size ... NOTE
    ```
      installed size is 14.0Mb
      sub-directories of 1Mb or more:
        doc      11.5Mb
        extdata   1.0Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘reshape2’
      All declared Imports should be used.
    ```

*   checking R code for possible problems ... NOTE
    ```
    setTierTwo,data.table : a: no visible binding for global variable ‘tmp’
    toLolliplot,GMS: no visible binding for global variable ‘missingINdex’
    toLolliplot,VEP: no visible binding for global variable ‘missingINdex’
    Undefined global functions or variables:
      missingINdex tmp
    ```

# geofacet

Version: 0.1.9

## In both

*   checking whether package ‘geofacet’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/geofacet/new/geofacet.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘geofacet’ ...
** package ‘geofacet’ successfully unpacked and MD5 sums checked
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
Error in loadNamespace(j <- i[[1L]], c(lib.loc, .libPaths()), versionCheck = vI[[j]]) : 
  there is no package called ‘sf’
ERROR: lazy loading failed for package ‘geofacet’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/geofacet/new/geofacet.Rcheck/geofacet’

```
### CRAN

```
* installing *source* package ‘geofacet’ ...
** package ‘geofacet’ successfully unpacked and MD5 sums checked
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
Error in loadNamespace(j <- i[[1L]], c(lib.loc, .libPaths()), versionCheck = vI[[j]]) : 
  there is no package called ‘sf’
ERROR: lazy loading failed for package ‘geofacet’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/geofacet/old/geofacet.Rcheck/geofacet’

```
# geozoning

Version: 1.0.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘ggplot2’ ‘utils’
      All declared Imports should be used.
    ```

# GerminaR

Version: 1.2

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘DT’ ‘shinydashboard’
      All declared Imports should be used.
    ```

# germinationmetrics

Version: 0.1.1.1

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    tlmgr search --file --global '/colortbl.sty'
    Unknown directive ...l
    Warning in system2("tlmgr", args, ...) :
      running command ''tlmgr' search --file --global '/colortbl.sty'' had status 25
    ! LaTeX Error: File `colortbl.sty' not found.
    
    ! Emergency stop.
    <read *> 
    
    Error: processing vignette 'Introduction.Rmd' failed with diagnostics:
    Failed to compile Introduction.tex. See Introduction.log for more info.
    Execution halted
    ```

# gespeR

Version: 1.12.0

## In both

*   checking for code/documentation mismatches ... WARNING
    ```
    Codoc mismatches from documentation object 'c,Phenotypes-method':
    \S4method{c}{Phenotypes}
      Code: function(x, ...)
      Docs: function(x, ..., recursive = FALSE)
      Argument names in docs not in code:
        recursive
    ```

*   checking R code for possible problems ... NOTE
    ```
    .gespeR.cv: no visible global function definition for ‘coef’
    .select.model: no visible global function definition for ‘predict’
    concordance: no visible global function definition for ‘cor’
    lasso.rand: no visible global function definition for ‘runif’
    plot.gespeR: no visible global function definition for ‘hist’
    stability.selection: no visible global function definition for ‘lm’
    Phenotypes,character: no visible global function definition for
      ‘read.delim’
    Undefined global functions or variables:
      coef cor hist lm predict read.delim runif
    Consider adding
      importFrom("graphics", "hist")
      importFrom("stats", "coef", "cor", "lm", "predict", "runif")
      importFrom("utils", "read.delim")
    to your NAMESPACE file.
    ```

# getTBinR

Version: 0.5.5

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      > library(getTBinR)
      > 
      > test_check("getTBinR")
      ── 1. Failure: map_tb_burden can have a custom legend specified. (@test-map_tb_b
      plot$labels$fill not equal to `test_label`.
      1/1 mismatches
      x[1]: "`test (test - test)`"
      y[1]: "test (test - test)"
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 75 SKIPPED: 29 FAILED: 1
      1. Failure: map_tb_burden can have a custom legend specified. (@test-map_tb_burden.R#62) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

# gfer

Version: 0.1.10

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘tidyr’
      All declared Imports should be used.
    ```

# ggalt

Version: 0.4.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘plotly’
      All declared Imports should be used.
    ```

# ggbeeswarm

Version: 0.6.0

## In both

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Loading required package: ggplot2
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'usageExamples.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.55 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# ggbio

Version: 1.28.5

## In both

*   R CMD check timed out
    

*   checking dependencies in R code ... NOTE
    ```
    ':::' call which should be '::': 'ggplot2:::set_last_plot'
      See the note in ?`:::` about the use of this operator.
    Unexported objects imported by ':::' calls:
      'S4Vectors:::top_prenv' 'ggplot2:::add_ggplot' 'ggplot2:::cunion'
      'ggplot2:::rename_aes' 'ggplot2:::rescale01'
      See the note in ?`:::` about the use of this operator.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    layout_karyogram,GRanges: no visible binding for global variable 'xend'
    layout_karyogram,GRanges: no visible binding for global variable 'yend'
    layout_karyogram,GRanges: no visible binding for global variable 'y2'
    layout_karyogram,GRanges: no visible binding for global variable
      'yend2'
    layout_karyogram,GRanges: no visible binding for global variable 'name'
    plotFragLength,character-GRanges: no visible binding for global
      variable '.fragLength'
    plotSpliceSum,character-EnsDb: possible error in GRangesFilter(which,
      condition = "overlapping"): unused argument (condition =
      "overlapping")
    stat_mismatch,GRanges: no visible binding for global variable 'sts'
    stat_mismatch,GRanges: no visible binding for global variable 'eds'
    stat_mismatch,GRanges: no visible binding for global variable 'read'
    Undefined global functions or variables:
      .fragLength .layout_circle.stats .x breaks coefs data eds fe fl
      gieStain ideoCyto indexProbesProcessed midpoint mt name read se
      stepping sts value variable x xend y y.text y2 yend yend2
    Consider adding
      importFrom("utils", "data")
    to your NAMESPACE file.
    ```

# ggcyto

Version: 1.8.2

## In both

*   checking whether package ‘ggcyto’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘flowCore’ was built under R version 3.5.1
      Warning: package ‘flowWorkspace’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/ggcyto/new/ggcyto.Rcheck/00install.out’ for details.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  6.0Mb
      sub-directories of 1Mb or more:
        doc   5.5Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘scales’
      All declared Imports should be used.
    Unexported objects imported by ':::' calls:
      ‘flowWorkspace:::.mergeGates’ ‘flowWorkspace:::compact’
      ‘flowWorkspace:::fix_y_axis’ ‘flowWorkspace:::isNegated’
      ‘ggplot2:::+.gg’ ‘ggplot2:::add_group’ ‘ggplot2:::as_gg_data_frame’
      ‘ggplot2:::check_aesthetics’ ‘ggplot2:::is.waive’
      ‘ggplot2:::is_calculated_aes’ ‘ggplot2:::make_labels’
      ‘ggplot2:::make_scale’ ‘ggplot2:::plot_clone’
      ‘ggplot2:::print.ggplot’ ‘ggplot2:::scales_add_defaults’
      ‘ggplot2:::scales_list’ ‘ggplot2:::update_theme’
      See the note in ?`:::` about the use of this operator.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    fortify_fs.GatingSetList: no visible global function definition for
      ‘getS3method’
    getFlowFrame.GatingSetList: no visible global function definition for
      ‘getS3method’
    getFlowFrame.ncdfFlowList: no visible global function definition for
      ‘getS3method’
    ggcyto.GatingSetList: no visible global function definition for
      ‘getS3method’
    ggcyto.flowSet: no visible binding for global variable ‘name’
    ggcyto.flowSet: no visible binding for global variable ‘axis’
    ggcyto.ncdfFlowList: no visible global function definition for
      ‘getS3method’
    ggcyto_arrange: no visible binding for global variable ‘name’
    Undefined global functions or variables:
      approx axis density desc dist getS3method gray modifyList name
    Consider adding
      importFrom("grDevices", "gray")
      importFrom("graphics", "axis")
      importFrom("stats", "approx", "density", "dist")
      importFrom("utils", "getS3method", "modifyList")
    to your NAMESPACE file.
    ```

# ggdag

Version: 0.1.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘ggforce’ ‘plyr’
      All declared Imports should be used.
    ```

# ggdistribute

Version: 1.0.2

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘dplyr’
      All declared Imports should be used.
    ```

# ggdmc

Version: 0.2.5.2

## In both

*   checking examples ... ERROR
    ```
    ...
      ..- attr(*, "names")= chr  "a" "v.f1" "v.f2" "z" ...
    > ## 'data.frame':	400 obs. of  4 variables:
    > ## $ S : Factor w/ 2 levels "s1","s2": 1 1 1 1 1 1 1 1 1 1 ...
    > ## $ F : Factor w/ 2 levels "f1","f2": 1 1 1 1 1 1 1 1 1 1 ...
    > ## $ R : Factor w/ 2 levels "r1","r2": 1 1 1 2 1 1 1 1 2 1 ...
    > ## $ RT: num  0.26 0.255 0.572 0.25 0.518 ...
    > 
    > dmi1 <- BuildDMI(dat, m1)
    > npar <- length(GetPNames(m1))
    > 
    > p.prior <- BuildPrior(
    +    dists = rep("tnorm", npar),
    +    p1    = c(a=2,  v.f1=2.5, v.f2=1.25, z=.5, sz=.3, sv=1,  t0=.3),
    +    p2    = c(a=.5, v.f1=.5,  v.f2=.35,  z=.1, sz=.1, sv=.3, t0=.05),
    +    lower = c(0,-5, -5, 0, 0, 0, 0),
    +    upper = c(5, 7,  7, 2, 2, 2, 2))
    > 
    > ## Set up a new DMC sample with 16 iteration. The default thin is 1
    > sam0 <- StartNewsamples(nmc = 16, data = dmi1, prior = p.prior)
    OMP: Error #15: Initializing libomp.dylib, but found libomp.dylib already initialized.
    OMP: Hint: This means that multiple copies of the OpenMP runtime have been linked into the program. That is dangerous, since it can degrade performance or cause incorrect results. The best thing to do is to ensure that only a single OpenMP runtime is linked into the process, e.g. by avoiding static linking of the OpenMP runtime in any library. As an unsafe, unsupported, undocumented workaround you can set the environment variable KMP_DUPLICATE_LIB_OK=TRUE to allow the program to continue to execute, but that may cause crashes or silently produce incorrect results. For more information, please see http://www.intel.com/software/products/support/.
    ```

# GGEBiplots

Version: 0.1.1

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘GGEBiplotGUI’ ‘gge’ ‘utils’
      All declared Imports should be used.
    ```

# ggedit

Version: 0.3.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘magrittr’
      All declared Imports should be used.
    ```

# ggeffects

Version: 0.6.0

## In both

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘ordinal’
    ```

# ggenealogy

Version: 0.3.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘tibble’
      All declared Imports should be used.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 2356 marked UTF-8 strings
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
      conversion failure on 'Ondřej Chochola' in 'mbcsToSbcs': dot substituted for <99>
    Warning in grid.Call.graphics(C_text, as.graphicsAnnot(x$label), x$x, x$y,  :
      conversion failure on 'Ondřej Chochola' in 'mbcsToSbcs': dot substituted for <c5>
    Warning in grid.Call.graphics(C_text, as.graphicsAnnot(x$label), x$x, x$y,  :
      conversion failure on 'Ondřej Chochola' in 'mbcsToSbcs': dot substituted for <99>
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'ggenealogy.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `media9.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.4 ^^M
           
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# ggExtra

Version: 0.8

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘grDevices’
      All declared Imports should be used.
    ```

# ggFacetSample

Version: 1.0.0

## In both

*   checking examples ... ERROR
    ```
    ...
    > 
    >   testdata <- data.frame(
    +     rowcat = paste0("1", sample(LETTERS[1:10], sampn, replace = TRUE)),
    +     rowcat2 = paste0("2", sample(LETTERS[11:15], sampn, replace = TRUE)),
    +     colcat = paste0("2", sample(letters[1:10], sampn, replace = TRUE)),
    +     x = runif(sampn),
    +     y = runif(sampn)
    +   )
    > 
    >   p <- ggplot2::ggplot(testdata) + ggplot2::aes(x=x, y=y, color=rowcat2) +
    +     ggplot2::geom_point()
    > 
    >   # plot the works (not actually recommended)
    >   ## Not run: p + ggplot2::facet_grid(rowcat ~ colcat)
    > 
    >   # plot a sample of the row-facets:
    >   p + facet_sample(rowcat ~ colcat, sample_n = c(3, NA))
    Error in UseMethod("as.quoted") : 
      no applicable method for 'as.quoted' applied to an object of class "quosures"
    Calls: <Anonymous> ... <Anonymous> -> f -> <Anonymous> -> f -> <Anonymous>
    Execution halted
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘plyr’
      All declared Imports should be used.
    ```

# ggfan

Version: 0.1.2

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘colorspace’ ‘grid’ ‘rstan’
      All declared Imports should be used.
    ```

# ggformula

Version: 0.9.0

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘sf’
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘tidyr’
      All declared Imports should be used.
    ```

# gggenes

Version: 0.3.1

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘rlang’
      All declared Imports should be used.
    ```

# ggguitar

Version: 0.1.1

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘gridExtra’ ‘lazyeval’ ‘readr’
      All declared Imports should be used.
    ```

# ggimage

Version: 0.1.7

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘gridGraphics’ ‘scales’
      All declared Imports should be used.
    ```

# gginnards

Version: 0.0.1

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘grid’ ‘tibble’
      All declared Imports should be used.
    ```

# ggiraph

Version: 0.5.0

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘sf’
    ```

# ggiraphExtra

Version: 0.2.9

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘ggforce’ ‘webshot’ ‘ztable’
      All declared Imports should be used.
    ```

# gglogo

Version: 0.1.3

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    
    Attaching package: 'gglogo'
    
    The following object is masked from 'package:ggplot2':
    
        fortify
    
    Warning in grid.Call.graphics(C_text, as.graphicsAnnot(x$label), x$x, x$y,  :
      no font could be found for family "Garamond"
    Warning in grid.Call.graphics(C_text, as.graphicsAnnot(x$label), x$x, x$y,  :
      no font could be found for family "Garamond"
    Warning in grid.Call.graphics(C_text, as.graphicsAnnot(x$label), x$x, x$y,  :
      no font could be found for family "Garamond"
    Quitting from lines 46-52 (gglogo-alphabet.Rmd) 
    Error: processing vignette 'gglogo-alphabet.Rmd' failed with diagnostics:
    replacement has 1 row, data has 0
    Execution halted
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘knitr’
      All declared Imports should be used.
    ```

# ggmap

Version: 2.6.1

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘ggmap-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: get_map
    > ### Title: Grab a map.
    > ### Aliases: get_map
    > 
    > ### ** Examples
    > 
    > map <- get_map()
    Warning in download.file(url, destfile = tmp, quiet = !messaging, mode = "wb") :
      cannot open URL 'http://maps.googleapis.com/maps/api/staticmap?center=29.763284,-95.363271&zoom=10&size=640x640&scale=2&maptype=terrain&language=en-EN&sensor=false': HTTP status was '403 Forbidden'
    Error in download.file(url, destfile = tmp, quiet = !messaging, mode = "wb") : 
      cannot open URL 'http://maps.googleapis.com/maps/api/staticmap?center=29.763284,-95.363271&zoom=10&size=640x640&scale=2&maptype=terrain&language=en-EN&sensor=false'
    Calls: get_map -> get_googlemap -> download.file
    Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.2Mb
      sub-directories of 1Mb or more:
        data   4.8Mb
    ```

# ggmcmc

Version: 1.1

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
      no font could be found for family "Source Sans Pro"
    Warning in grid.Call.graphics(C_text, as.graphicsAnnot(x$label), x$x, x$y,  :
      no font could be found for family "Source Sans Pro"
    Warning in grid.Call.graphics(C_text, as.graphicsAnnot(x$label), x$x, x$y,  :
      no font could be found for family "Source Sans Pro"
    Warning in grid.Call.graphics(C_text, as.graphicsAnnot(x$label), x$x, x$y,  :
      no font could be found for family "Source Sans Pro"
    Warning in grid.Call.graphics(C_text, as.graphicsAnnot(x$label), x$x, x$y,  :
      no font could be found for family "Source Sans Pro"
    Warning in grid.Call.graphics(C_text, as.graphicsAnnot(x$label), x$x, x$y,  :
      no font could be found for family "Source Sans Pro"
    Warning in grid.Call.graphics(C_text, as.graphicsAnnot(x$label), x$x, x$y,  :
      no font could be found for family "Source Sans Pro"
    Warning in grid.Call.graphics(C_text, as.graphicsAnnot(x$label), x$x, x$y,  :
      no font could be found for family "Source Sans Pro"
    Warning in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y,  :
      no font could be found for family "Source Sans Pro"
    Quitting from lines 142-143 (using_ggmcmc.Rmd) 
    Error: processing vignette 'using_ggmcmc.Rmd' failed with diagnostics:
    polygon edge not found
    Execution halted
    ```

# ggplotAssist

Version: 0.1.3

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘gcookbook’ ‘ggthemes’ ‘moonBook’ ‘tidyverse’
      All declared Imports should be used.
    ```

# ggpmisc

Version: 0.3.0

## In both

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘gginnards’
    ```

# ggpol

Version: 0.0.4

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘dplyr’ ‘grDevices’
      All declared Imports should be used.
    ```

# ggQQunif

Version: 0.1.5

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘dplyr’
      All declared Imports should be used.
    ```

# ggquickeda

Version: 0.1.1

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘DT’ ‘Formula’ ‘Hmisc’ ‘colourpicker’ ‘dplyr’ ‘ggpmisc’ ‘ggrepel’
      ‘grDevices’ ‘gridExtra’ ‘lazyeval’ ‘markdown’ ‘plotly’ ‘quantreg’
      ‘rlang’ ‘shinyjs’ ‘table1’ ‘tidyr’
      All declared Imports should be used.
    ```

# ggRandomForests

Version: 2.0.1

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘randomForest’
      All declared Imports should be used.
    ```

# ggrasp

Version: 1.0

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  9.3Mb
      sub-directories of 1Mb or more:
        extdata   9.0Mb
    ```

# ggridges

Version: 0.5.1

## In both

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 6242 marked UTF-8 strings
    ```

# ggROC

Version: 1.0

## In both

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Description field: should contain one or more complete sentences.
    Package listed in more than one of Depends, Imports, Suggests, Enhances:
      ‘ggplot2’
    A package should be listed in only one of these fields.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Package in Depends field not imported from: ‘ggplot2’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ggroc: no visible global function definition for ‘ggplot’
    ggroc: no visible global function definition for ‘aes’
    ggroc: no visible global function definition for ‘geom_point’
    ggroc: no visible global function definition for ‘geom_line’
    ggroc: no visible global function definition for ‘theme’
    ggroc: no visible global function definition for ‘element_text’
    ggroc: no visible global function definition for ‘labs’
    ggroc: no visible global function definition for ‘ggsave’
    Undefined global functions or variables:
      aes element_text geom_line geom_point ggplot ggsave labs theme
    ```

# ggspatial

Version: 1.0.1

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘sf’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# ggstatsplot

Version: 0.0.6

## In both

*   R CMD check timed out
    

*   checking installed package size ... NOTE
    ```
      installed size is  5.3Mb
      sub-directories of 1Mb or more:
        doc    2.6Mb
        help   2.3Mb
    ```

# ggtern

Version: 3.0.0

## In both

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘chemometrics’
    ```

# ggthemes

Version: 4.0.1

## Newly fixed

*   checking examples ... ERROR
    ```
    ...
    
    > require("dplyr")
    Loading required package: dplyr
    
    Attaching package: ‘dplyr’
    
    The following objects are masked from ‘package:stats’:
    
        filter, lag
    
    The following objects are masked from ‘package:base’:
    
        intersect, setdiff, setequal, union
    
    > canva_df <- map2_df(canva_palettes, names(canva_palettes),
    +                     ~ tibble(colors = .x, .id = seq_along(colors),
    +                              palette = .y))
    Error in eval_tidy(xs[[i]], unique_output) : 
      first argument ('table') not of type VECSXP, R_HashSizeCheck
    Calls: map2_df ... map2 -> .f -> tibble -> as_tibble -> lst_quos -> eval_tidy
    Execution halted
    ```

## In both

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 138 marked UTF-8 strings
    ```

# GGtools

Version: 5.16.0

## In both

*   R CMD check timed out
    

*   checking whether package ‘GGtools’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘GenomicFeatures’ was built under R version 3.5.1
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/GGtools/new/GGtools.Rcheck/00install.out’ for details.
    ```

*   checking package dependencies ... NOTE
    ```
    Packages which this enhances but not available for checking:
      ‘MatrixEQTL’ ‘gwascat’
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 74.5Mb
      sub-directories of 1Mb or more:
        R       2.0Mb
        data   27.0Mb
        doc     1.6Mb
        parts   2.0Mb
        pup     2.0Mb
        rdas   10.3Mb
        vcf    28.8Mb
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Description field: should contain one or more complete sentences.
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' call to 'Homo.sapiens' which was already attached by Depends.
      Please remove these calls from your code.
    Packages in Depends field not imported from:
      'Homo.sapiens' 'parallel'
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    topKfeats: no visible binding for global variable 'i2'
    tscan2df: no visible global function definition for '%dopar%'
    tscan2df: no visible global function definition for 'foreach'
    tscan2df: no visible binding for global variable 'i'
    tscan2gr: no visible global function definition for '%dopar%'
    tscan2gr: no visible global function definition for 'foreach'
    tscan2gr: no visible binding for global variable 'i'
    waldtests : <anonymous>: no visible global function definition for
      'wald.test'
    plot,gwSnpScreenResult-character: no visible global function definition
      for 'snpcount'
    plot,gwSnpScreenResult-character: no visible global function definition
      for 'snpsBySeqname'
    Undefined global functions or variables:
      %dopar% .N FDR Homo.sapiens Matrix_eQTL_engine SlicedData
      as.data.table assay assays bindcadd bindmaf chi.squared colData curp
      detectCores excl export.gff3 firstHalf firstThird foreach forestplot
      gwastagger gwrngs hg19.si.df hmm878 i i1 i2 lastThird maf mafs
      mclapply midThird modelLINEAR npc overlapsAny pl radiusUsed ranges<-
      rowRanges runOneSplit select setkey setkeyv setnames snp snpcount
      snpsBySeqname target tileGenome value wald.test x
    ```

*   checking Rd cross-references ... NOTE
    ```
    Packages unavailable to check Rd xrefs: ‘MatrixEQTL’, ‘gwascat’
    ```

# ggtree

Version: 1.12.7

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  9.2Mb
      sub-directories of 1Mb or more:
        doc        4.9Mb
        examples   3.7Mb
    ```

# ghibli

Version: 0.1.2

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘ggplot2’
      All declared Imports should be used.
    ```

# gitter

Version: 1.1.1

## In both

*   checking S3 generic/method consistency ... WARNING
    ```
    xdpyinfo:  unable to open display "".
    See section ‘Generic functions and methods’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking replacement functions ... WARNING
    ```
    xdpyinfo:  unable to open display "".
    The argument of a replacement function which corresponds to the right
    hand side must be named ‘value’.
    ```

*   checking for missing documentation entries ... WARNING
    ```
    xdpyinfo:  unable to open display "".
    All user-level objects in a package should have documentation entries.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking for code/documentation mismatches ... WARNING
    ```
    xdpyinfo:  unable to open display "".
    xdpyinfo:  unable to open display "".
    xdpyinfo:  unable to open display "".
    ```

*   checking dependencies in R code ... NOTE
    ```
    xdpyinfo:  unable to open display "".
    ```

*   checking foreign function calls ... NOTE
    ```
    xdpyinfo:  unable to open display "".
    See chapter ‘System and foreign language interfaces’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    xdpyinfo:  unable to open display "".
    ```

*   checking Rd \usage sections ... NOTE
    ```
    xdpyinfo:  unable to open display "".
    The \usage entries for S3 methods should use the \method markup and not
    their full name.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

# glmmfields

Version: 0.1.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    Error in re-building vignettes:
      ...
    Loading required package: Rcpp
    
    Attaching package: 'dplyr'
    
    The following objects are masked from 'package:stats':
    
        filter, lag
    
    The following objects are masked from 'package:base':
    
        intersect, setdiff, setequal, union
    
    Waiting for profiling to be done...
    trying deprecated constructor; please alert package maintainer
    error occurred during calling the sampler; sampling not done
    Quitting from lines 134-135 (spatial-glms.Rmd) 
    Error: processing vignette 'spatial-glms.Rmd' failed with diagnostics:
    incorrect number of dimensions
    Execution halted
    ```

# glmmTMB

Version: 0.2.2.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    Warning in f(x, order = 1) : value out of range in 'lgamma'
    Warning in f(x, order = 1) : value out of range in 'lgamma'
    Warning in f(x, order = 1) : value out of range in 'lgamma'
    Warning in f(x, order = 1) : value out of range in 'lgamma'
    Warning in f(x, order = 1) : value out of range in 'lgamma'
    Warning in f(x, order = 1) : value out of range in 'lgamma'
    Warning in f(x, order = 1) : value out of range in 'lgamma'
    Loading required package: stats4
    Warning in qt((1 - level)/2, df) : NaNs produced
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'glmmTMB.tex' failed.
    BibTeX errors:
    The top-level auxiliary file: glmmTMB.aux
    I couldn't open style file chicago.bst
    ---line 19 of file glmmTMB.aux
     : \bibstyle{chicago
     :                  }
    I'm skipping whatever remains of this command
    I found no style file---while reading file glmmTMB.aux
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 15.0Mb
      sub-directories of 1Mb or more:
        libs  13.8Mb
    ```

# GmAMisc

Version: 1.0.0

## In both

*   checking whether package ‘GmAMisc’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/GmAMisc/new/GmAMisc.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘GmAMisc’ ...
** package ‘GmAMisc’ successfully unpacked and MD5 sums checked
** R
** data
*** moving datasets to lazyload DB
** byte-compile and prepare package for lazy loading
Error in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]) : 
  there is no package called ‘sf’
ERROR: lazy loading failed for package ‘GmAMisc’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/GmAMisc/new/GmAMisc.Rcheck/GmAMisc’

```
### CRAN

```
* installing *source* package ‘GmAMisc’ ...
** package ‘GmAMisc’ successfully unpacked and MD5 sums checked
** R
** data
*** moving datasets to lazyload DB
** byte-compile and prepare package for lazy loading
Error in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]) : 
  there is no package called ‘sf’
ERROR: lazy loading failed for package ‘GmAMisc’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/GmAMisc/old/GmAMisc.Rcheck/GmAMisc’

```
# GOexpress

Version: 1.14.0

## In both

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.2Mb
      sub-directories of 1Mb or more:
        data   4.3Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    GO_analyse: no visible binding for global variable ‘microarray2dataset’
    GO_analyse: no visible binding for global variable ‘prefix2dataset’
    mart_from_ensembl: no visible binding for global variable
      ‘prefix2dataset’
    Undefined global functions or variables:
      microarray2dataset prefix2dataset
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘VennDiagram’
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
      neighborhood radius 22.23
    Warning in predLoess(object$y, object$x, newx = if (is.null(newdata)) object$x else if (is.data.frame(newdata)) as.matrix(model.frame(delete.response(terms(object)),  :
      reciprocal condition number  1.3784e-16
    Warning in predLoess(object$y, object$x, newx = if (is.null(newdata)) object$x else if (is.data.frame(newdata)) as.matrix(model.frame(delete.response(terms(object)),  :
      There are other near singularities as well. 1783.4
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'GOexpress-UsersGuide.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `beramono.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.87 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# gogamer

Version: 0.4.3

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Complete output:
      > library(testthat)
      > library(gogamer)
      > 
      > test_check("gogamer")
      Error: segfault from C stack overflow
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 36 SKIPPED: 1 FAILED: 0
      Execution halted
    ```

# GoogleGenomics

Version: 2.2.0

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘GoogleGenomics-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: getReads
    > ### Title: Get reads from Google Genomics.
    > ### Aliases: getReads
    > 
    > ### ** Examples
    > 
    > # Authenticated on package load from the env variable GOOGLE_API_KEY.
    > reads <- getReads()
    Error in getSearchPage("reads", body, fields, pageToken) : 
      You are not authenticated; see ?GoogleGenomics::authenticate.
    Calls: getReads -> getReadsPage -> getSearchPage
    Execution halted
    ```

*   checking whether package ‘GoogleGenomics’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘DelayedArray’ was built under R version 3.5.1
      Warning: package ‘Rsamtools’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/GoogleGenomics/new/GoogleGenomics.Rcheck/00install.out’ for details.
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘RProtoBuf’
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘RProtoBuf’
    ```

*   checking for GNU extensions in Makefiles ... NOTE
    ```
    GNU make is a SystemRequirements.
    ```

# googlesheets

Version: 0.3.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘tibble’
      All declared Imports should be used.
    ```

# GOsummaries

Version: 2.16.1

## In both

*   checking R code for possible problems ... NOTE
    ```
    ...
    panel_boxplot: no visible binding for global variable ‘x’
    panel_boxplot: no visible binding for global variable ‘y’
    panel_boxplot: no visible binding for global variable ‘Class’
    panel_dummy: no visible binding for global variable ‘y’
    panel_dummy: no visible binding for global variable ‘x’
    plotWordcloud: no visible global function definition for
      ‘colorRampPalette’
    plotWordcloud: no visible global function definition for ‘runif’
    plot_motor: no visible global function definition for ‘dev.off’
    print.gosummaries: no visible global function definition for ‘head’
    pspearman: no visible global function definition for ‘pt’
    spearman_mds: no visible global function definition for ‘cor’
    Undefined global functions or variables:
      Class bmp boxplot.stats colorRampPalette cor dev.off head jpeg
      packageVersion pdf png pt runif tiff x y
    Consider adding
      importFrom("grDevices", "bmp", "boxplot.stats", "colorRampPalette",
                 "dev.off", "jpeg", "pdf", "png", "tiff")
      importFrom("stats", "cor", "pt", "runif")
      importFrom("utils", "head", "packageVersion")
    to your NAMESPACE file.
    ```

*   checking for unstated dependencies in vignettes ... NOTE
    ```
    '::' or ':::' import not declared from: ‘BiocStyle’
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    Error in re-building vignettes:
      ...
    This is vegan 2.5-2
    No id variables; using all as measure variables
    No id variables; using all as measure variables
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'GOsummaries-basics.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `beramono.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.87 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# GOTHiC

Version: 1.16.0

## In both

*   checking whether package ‘GOTHiC’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘rtracklayer’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/GOTHiC/new/GOTHiC.Rcheck/00install.out’ for details.
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'package_vignettes.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `datetime.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.6 \newdate
                {date}{19}{08}{2013}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking dependencies in R code ... NOTE
    ```
    ':::' call which should be '::': ‘S4Vectors:::orderIntegerPairs’
      See the note in ?`:::` about the use of this operator.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    .getRestrictionSitesFromBSgenome: no visible global function definition
      for ‘seqlengths’
    .importHicup: no visible global function definition for ‘read.table’
    .onlyPairing: no visible global function definition for ‘read.table’
    .onlyPairing: no visible global function definition for ‘ScanBamParam’
    GOTHiC: no visible binding for global variable
      ‘BSgenome.Hsapiens.UCSC.hg19’
    GOTHiC: no visible binding for global variable ‘filtered’
    GOTHiC: no visible binding for global variable ‘interactingLoci’
    mapReadsToRestrictionSites: no visible binding for global variable
      ‘resGR’
    Undefined global functions or variables:
      BSgenome.Hsapiens.UCSC.hg19 IntervalTree ScanBamParam V1 binom.test
      biocLite chr1 chr2 dev.off filtered frequencies int1 int2
      interactingLoci isCircular locus1 locus2 p.adjust pdf pvalue
      read.table resGR seqlengths x11
    Consider adding
      importFrom("grDevices", "dev.off", "pdf", "x11")
      importFrom("stats", "binom.test", "p.adjust")
      importFrom("utils", "read.table")
    to your NAMESPACE file.
    ```

# gpmap

Version: 0.1.1

## In both

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Description field: should contain one or more complete sentences.
    Checking should be performed on sources prepared by ‘R CMD build’.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Packages in Depends field not imported from:
      ‘foreach’ ‘ggplot2’ ‘isotone’ ‘plyr’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    plot3_dec: no visible global function definition for ‘foreach’
    plot3_dec: no visible global function definition for ‘ggplot’
    plot3_dec: no visible global function definition for ‘aes_string’
    plot3_dec: no visible global function definition for ‘geom_line’
    plot3_dec: no visible global function definition for ‘facet_grid’
    plot3_dec: no visible global function definition for ‘as.formula’
    plot3_dec: no visible global function definition for ‘labs’
    plot3_orig: no visible global function definition for ‘%do%’
    plot3_orig: no visible global function definition for ‘foreach’
    plot3_orig: no visible global function definition for ‘ggplot’
    plot3_orig: no visible global function definition for ‘aes_string’
    plot3_orig: no visible global function definition for ‘geom_line’
    plot3_orig: no visible global function definition for ‘facet_wrap’
    plot3_orig: no visible global function definition for ‘as.formula’
    plot3_orig: no visible global function definition for ‘labs’
    Undefined global functions or variables:
      %do% %dopar% aaply activeSet aes_string as.formula facet_grid
      facet_wrap foreach geom_line geom_point ggplot labs laply var
    Consider adding
      importFrom("stats", "as.formula", "var")
    to your NAMESPACE file.
    ```

# gQTLstats

Version: 1.12.0

## In both

*   R CMD check timed out
    

*   checking whether package ‘gQTLstats’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘GenomicFeatures’ was built under R version 3.5.1
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/gQTLstats/new/gQTLstats.Rcheck/00install.out’ for details.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 65.7Mb
      sub-directories of 1Mb or more:
        data        11.0Mb
        doc          1.1Mb
        registries  18.8Mb
        vcf         33.8Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      ‘permScore_3’
    storeToMaxAssocBySNP: no visible global function definition for ‘nth’
    storeToMaxAssocBySNP: no visible binding for global variable ‘MAF’
    storeToMaxAssocBySNP: no visible binding for global variable ‘probeid’
    storeToMaxAssocBySNP: no visible binding for global variable ‘mindist’
    tqbrowser: no visible global function definition for ‘experiments’
    tqbrowser : server: no visible global function definition for
      ‘experiments’
    tqbrowser : server: no visible global function definition for
      ‘TabixFile’
    tqbrowser : server: no visible binding for global variable ‘assoc’
    tqbrowser : server: no visible binding for global variable ‘stateid’
    tqbrowser : server: no visible binding for global variable ‘state’
    transTable: no visible binding for global variable ‘i’
    tsByRank_sing: no visible binding for global variable ‘i’
    tsByRank_sing : <anonymous>: no visible binding for global variable ‘i’
    boxswarm,SnpToGeneQTL: no visible binding for global variable ‘g1’
    Undefined global functions or variables:
      DNAStringSetList MAF TabixFile TxDb assoc calls ch chisq criterion ex
      exonsBy experiments g1 geom_boxplot gt i id maf mindist ml10fdr nth
      permScore_1 permScore_2 permScore_3 probeid snp state stateid value x
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘MultiAssayExperiment’
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 8 marked Latin-1 strings
      Note: found 12 marked UTF-8 strings
    ```

# grapherator

Version: 1.0.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘reshape2’
      All declared Imports should be used.
    ```

# grattan

Version: 1.7.0.0

## In both

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking:
      ‘taxstats’ ‘taxstats1516’
    ```

# Greg

Version: 1.2.2

## In both

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘rmeta’
    ```

# GRENITS

Version: 1.32.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Packages in Depends field not imported from:
      ‘Rcpp’ ‘RcppArmadillo’ ‘ggplot2’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      ‘scale_fill_gradient’
    .plotDistribParents.LargeMat: no visible global function definition for
      ‘theme’
    .plotDistribParents.LargeMat: no visible global function definition for
      ‘element_blank’
    .plotDistribParents.LargeMat: no visible global function definition for
      ‘element_text’
    .plotDistribParents.LargeMat: no visible global function definition for
      ‘ggtitle’
    .plotDistribParents.LargeMat: no visible global function definition for
      ‘labs’
    .plotDistribParents.LargeMat: no visible global function definition for
      ‘scale_x_discrete’
    .plotDistribParents.LargeMat: no visible global function definition for
      ‘scale_y_discrete’
    .plotDistribParents.LargeMat: no visible global function definition for
      ‘facet_wrap’
    Undefined global functions or variables:
      GeneNames Var1 Var2 aes element_blank element_text facet_wrap
      geom_tile ggplot ggtitle labs scale_fill_gradient scale_x_discrete
      scale_y_discrete theme value variable
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
                        Martina Morris, University of Washington
                        Skye Bender-deMoll, University of Washington
     For citation information, type citation("network").
     Type help("network-package") to get started.
    
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'GRENITS_package.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.30 \ifthenelse
                    {\boolean{Sweave@inconsolata}}{%^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# greport

Version: 0.7-1

## In both

*   checking R code for possible problems ... NOTE
    ```
    accrualReport: multiple local function definitions for ‘gg’ with
      different formal arguments
    ```

# gridsampler

Version: 0.6

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘BiasedUrn’
      All declared Imports should be used.
    ```

# GRmetrics

Version: 1.6.1

## In both

*   checking whether package ‘GRmetrics’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘DelayedArray’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/GRmetrics/new/GRmetrics.Rcheck/00install.out’ for details.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.3Mb
      sub-directories of 1Mb or more:
        doc   5.1Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    .GRcalculate: no visible binding for global variable
      ‘treatment_duration’
    .GRlogisticFit: no visible binding for global variable ‘experiment’
    GRdrawDRC: no visible binding for global variable ‘GRvalue’
    Undefined global functions or variables:
      GRvalue experiment treatment_duration
    ```

# GSCA

Version: 2.10.0

## In both

*   checking package dependencies ... NOTE
    ```
    Depends: includes the non-default packages:
      ‘shiny’ ‘sp’ ‘gplots’ ‘ggplot2’ ‘reshape2’ ‘RColorBrewer’ ‘rhdf5’
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' calls in package code:
      ‘Affyhgu133A2Expr’ ‘Affyhgu133Plus2Expr’ ‘Affyhgu133aExpr’
      ‘Affymoe4302Expr’
      Please use :: or requireNamespace() instead.
      See section 'Suggested packages' in the 'Writing R Extensions' manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    GSCAplot: no visible global function definition for ‘title’
    GSCAplot: no visible global function definition for ‘dev.off’
    annotatePeaks: no visible binding for global variable ‘allreffile’
    tabSearch: no visible global function definition for ‘data’
    tabSearch: no visible binding for global variable ‘Affyhgu133aExprtab’
    tabSearch: no visible binding for global variable ‘Affymoe4302Exprtab’
    tabSearch: no visible binding for global variable ‘Affyhgu133A2Exprtab’
    tabSearch: no visible binding for global variable
      ‘Affyhgu133Plus2Exprtab’
    Undefined global functions or variables:
      Affyhgu133A2Exprtab Affyhgu133Plus2Exprtab Affyhgu133aExprtab
      Affymoe4302Exprtab P.value SampleType Var1 Var2 allreffile
      colorRampPalette data dev.off fisher.test geneid hist par pdf qnorm
      quantile sd str t.stat t.test title value variable write.csv
      write.table
    Consider adding
      importFrom("grDevices", "colorRampPalette", "dev.off", "pdf")
      importFrom("graphics", "hist", "par", "title")
      importFrom("stats", "fisher.test", "qnorm", "quantile", "sd", "t.test")
      importFrom("utils", "data", "str", "write.csv", "write.table")
    to your NAMESPACE file.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    Loading required package: reshape2
    Loading required package: RColorBrewer
    Loading required package: rhdf5
    Loading required package: Affymoe4302Expr
    Loading required package: Affyhgu133aExpr
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'GSCA.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.30 \ifthenelse
                    {\boolean{Sweave@inconsolata}}{%^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# gsDesign

Version: 3.0-1

## In both

*   checking R code for possible problems ... NOTE
    ```
    plotgsCP: no visible global function definition for ‘opts’
    plotgsPower: no visible global function definition for ‘opts’
    qplotit: no visible global function definition for ‘opts’
    Undefined global functions or variables:
      opts
    ```

# GSE

Version: 4.1

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘rrcov’
      All declared Imports should be used.
    ```

# GSODR

Version: 1.2.3

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘sf’
    ```

# GUIgems

Version: 0.1

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘rpanel’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# Guitar

Version: 1.18.0

## In both

*   R CMD check timed out
    

*   checking whether package ‘Guitar’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘Rsamtools’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘GenomicFeatures’ was built under R version 3.5.1
      Warning: package ‘rtracklayer’ was built under R version 3.5.1
      Warning: package ‘DelayedArray’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/Guitar/new/Guitar.Rcheck/00install.out’ for details.
    ```

*   checking package dependencies ... NOTE
    ```
    Depends: includes the non-default packages:
      ‘Rsamtools’ ‘GenomicFeatures’ ‘rtracklayer’ ‘GenomicAlignments’
      ‘GenomicRanges’ ‘ggplot2’ ‘grid’ ‘IRanges’
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.2Mb
      sub-directories of 1Mb or more:
        extdata   4.7Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    .makeFigure_fill: no visible global function definition for ‘pdf’
    .makeFigure_fill: no visible global function definition for ‘dev.off’
    .makeFigure_nofill: no visible global function definition for ‘pdf’
    .makeFigure_nofill: no visible global function definition for ‘dev.off’
    BED12toGRangesList: no visible global function definition for
      ‘read.table’
    BED12toGRangesList: no visible global function definition for ‘head’
    narrowPeaktoGRanges: no visible global function definition for
      ‘read.table’
    Undefined global functions or variables:
      dev.off head pdf read.table
    Consider adding
      importFrom("grDevices", "dev.off", "pdf")
      importFrom("utils", "head", "read.table")
    to your NAMESPACE file.
    ```

# gutenbergr

Version: 0.1.4

## In both

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 13617 marked UTF-8 strings
    ```

# gwascat

Version: 2.12.0

## In both

*   R CMD check timed out
    

*   checking whether package ‘gwascat’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘GenomicFeatures’ was built under R version 3.5.1
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/gwascat/new/gwascat.Rcheck/00install.out’ for details.
    ```

*   checking package dependencies ... NOTE
    ```
    Package which this enhances but not available for checking: ‘SNPlocs.Hsapiens.dbSNP144.GRCh37’
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 37.7Mb
      sub-directories of 1Mb or more:
        data     28.7Mb
        obo       3.0Mb
        olddata   2.2Mb
        tab       1.1Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    chklocs: no visible binding for global variable 'gwrngs19'
    chklocs: no visible global function definition for 'snpsBySeqname'
    lo38to19: no visible binding for global variable 'si.hs.37'
    makeCurrentGwascat: no visible binding for global variable 'si.hs.38'
    makeCurrentGwascat.legacy: no visible binding for global variable
      'si.hs.38'
    snpGenos: no visible global function definition for 'getSNPlocs'
    tfilt: no visible binding for global variable 'phr'
    tpad: no visible binding for global variable 'phr'
    traitsManh: no visible binding for global variable 'PVALUE_MLOG'
    variantProps: no visible binding for global variable 'gwrngs'
    Undefined global functions or variables:
      PVALUE_MLOG getSNPlocs gwrngs gwrngs19 phr si.hs.37 si.hs.38
      snpsBySeqname
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 6 marked Latin-1 strings
      Note: found 1 marked UTF-8 string
    ```

*   checking for unstated dependencies in vignettes ... NOTE
    ```
    '::' or ':::' import not declared from: ‘BiocStyle’
    ```

# h2o

Version: 3.20.0.8

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 146.3Mb
      sub-directories of 1Mb or more:
        java  144.4Mb
    ```

# h5vc

Version: 2.14.0

## In both

*   checking R code for possible problems ... NOTE
    ```
    ...
    callVariantsSingle : <anonymous> : <anonymous>: no visible binding for
      global variable ‘BlockID’
    mergeTallyFiles : <anonymous>: no visible binding for global variable
      ‘group’
    mergeTallyFiles: no visible binding for global variable ‘SourceFile’
    mismatchPlot: no visible binding for global variable ‘Sample’
    plotMutationSpectrum: no visible binding for global variable
      ‘altAllele’
    plotMutationSpectrum: no visible binding for global variable ‘tmp’
    rerunBatchTallies: no visible binding for global variable ‘regID’
    resizeCohort: no visible binding for global variable ‘newSamples’
    tallyRangesBatch : <anonymous>: no visible binding for global variable
      ‘bamFiles’
    tallyRangesBatch: no visible binding for global variable ‘verbose’
    Undefined global functions or variables:
      AF BlockID Sample SourceFile SupFwd SupRev Support altAllele bamFiles
      binom.test fisher.test group hist newSamples pValue regID tmp verbose
    Consider adding
      importFrom("graphics", "hist")
      importFrom("stats", "binom.test", "fisher.test")
    to your NAMESPACE file.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘deepSNV’
    ```

# Hapi

Version: 0.0.3

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error: processing vignette 'Hapi.Rmd' failed with diagnostics:
    there is no package called ‘prettydoc’
    Execution halted
    ```

# Haplin

Version: 7.0.0

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘Rmpi’
    ```

# harrypotter

Version: 0.1.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘MASS’
      All declared Imports should be used.
    ```

# hazus

Version: 0.1

## In both

*   checking R code for possible problems ... NOTE
    ```
    extract_hazus_functions: no visible global function definition for
      ‘data’
    Undefined global functions or variables:
      data
    Consider adding
      importFrom("utils", "data")
    to your NAMESPACE file.
    ```

# hBayesDM

Version: 0.6.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘rstantools’
      All declared Imports should be used.
    ```

# HDCytoData

Version: 1.0.0

## In both

*   checking whether package ‘HDCytoData’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘ExperimentHub’ was built under R version 3.5.1
      Warning: package ‘AnnotationHub’ was built under R version 3.5.1
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘DelayedArray’ was built under R version 3.5.1
      Warning: package ‘flowCore’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/HDCytoData/new/HDCytoData.Rcheck/00install.out’ for details.
    ```

# hdm

Version: 0.2.3

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
      ...
    Warning in grep(cn[I.c], colnames(X)) :
      argument 'pattern' has length > 1 and only the first element will be used
    Warning in grep(cn[I.c], colnames(X)) :
      argument 'pattern' has length > 1 and only the first element will be used
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'hdm_introduction.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `soul.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.69 \usepackage
                    {natbib}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# heatmaply

Version: 0.15.2

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.0Mb
      sub-directories of 1Mb or more:
        doc   4.6Mb
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘d3heatmap’
    ```

# hei

Version: 0.1.0

## In both

*   checking files in ‘vignettes’ ... NOTE
    ```
    Files named as vignettes but with no recognized vignette engine:
       ‘vignettes/Analysis.Rmd’
    (Is a VignetteBuilder field missing?)
    ```

# helminthR

Version: 1.0.6

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘knitr’ ‘rmarkdown’
      All declared Imports should be used.
    ```

# hiAnnotator

Version: 1.14.0

## In both

*   checking whether package ‘hiAnnotator’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/hiAnnotator/new/hiAnnotator.Rcheck/00install.out’ for details.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    makeChunks: no visible global function definition for ‘breakInChunks’
    makeChunks: no visible global function definition for ‘detectCores’
    makeChunks : <anonymous>: no visible global function definition for
      ‘keepSeqlevels’
    makeChunks : <anonymous>: no visible global function definition for
      ‘seqlevelsInUse’
    makeGRanges: no visible global function definition for ‘IRanges’
    makeGRanges: no visible global function definition for ‘seqlengths’
    makeGRanges: no visible global function definition for ‘seqlevels<-’
    makeGRanges: no visible global function definition for ‘sortSeqlevels’
    makeGRanges: no visible global function definition for ‘seqlevelsInUse’
    makeGRanges: no visible global function definition for ‘seqlengths<-’
    makeGRanges: no visible global function definition for ‘seqlevels’
    Undefined global functions or variables:
      IRanges breakInChunks countQueryHits detectCores dist featureName
      keepSeqlevels mid n overlapsAny qStrand queryHits seqlengths
      seqlengths<- seqlevels seqlevels<- seqlevelsInUse sortSeqlevels
      subjectHits
    Consider adding
      importFrom("stats", "dist")
    to your NAMESPACE file.
    ```

*   checking files in ‘vignettes’ ... NOTE
    ```
    The following directory looks like a leftover from 'knitr':
      ‘figure’
    Please remove from your package.
    ```

# HiCcompare

Version: 1.2.0

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  6.7Mb
      sub-directories of 1Mb or more:
        data   5.8Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    sim.other.methods: no visible binding for global variable ‘IF2’
    sim.other.methods: no visible binding for global variable ‘adj.M’
    sim.other.methods: no visible binding for global variable ‘M’
    sim_matrix: no visible binding for global variable ‘bias.slope’
    total_sum: no visible binding for global variable ‘IF2’
    total_sum: no visible binding for global variable ‘M’
    total_sum: no visible binding for global variable ‘IF1’
    total_sum: no visible binding for global variable ‘chr1’
    volcano: no visible binding for global variable ‘A’
    volcano: no visible binding for global variable ‘adj.IF1’
    volcano: no visible binding for global variable ‘adj.IF2’
    volcano: no visible binding for global variable ‘p.value’
    volcano: no visible binding for global variable ‘D’
    Undefined global functions or variables:
      A D IF IF1 IF2 M Z adj.IF1 adj.IF2 adj.M axis bias.slope bp
      centromere_locations chr1 chr2 count fold.change i j p.adj p.value
      pnorm region1 region2 start1 start2
    Consider adding
      importFrom("graphics", "axis")
      importFrom("stats", "D", "pnorm")
    to your NAMESPACE file.
    ```

# hierarchicalDS

Version: 2.9

## In both

*   checking R code for possible problems ... NOTE
    ```
    ...
    rrw: no visible global function definition for ‘rnorm’
    simulate_data: no visible global function definition for ‘pnorm’
    simulate_data: no visible global function definition for ‘rpois’
    switch_pdf: no visible global function definition for ‘dpois’
    switch_pdf: no visible global function definition for ‘dnorm’
    switch_sample: no visible global function definition for ‘rpois’
    switch_sample: no visible global function definition for ‘rnorm’
    switch_sample: no visible global function definition for ‘runif’
    switch_sample_prior: no visible global function definition for ‘rgamma’
    switch_sample_prior: no visible global function definition for ‘rnorm’
    table.mcmc: no visible global function definition for ‘write.csv’
    Undefined global functions or variables:
      abline dnorm dpois legend median model.matrix optimize par plot pnorm
      points rgamma rnorm rpois runif var write.csv
    Consider adding
      importFrom("graphics", "abline", "legend", "par", "plot", "points")
      importFrom("stats", "dnorm", "dpois", "median", "model.matrix",
                 "optimize", "pnorm", "rgamma", "rnorm", "rpois", "runif",
                 "var")
      importFrom("utils", "write.csv")
    to your NAMESPACE file.
    ```

# hierarchicalSets

Version: 1.0.2

## In both

*   checking dependencies in R code ... NOTE
    ```
    Unexported objects imported by ':::' calls:
      ‘ggplot2:::build_guides’ ‘ggplot2:::guide_axis’
      ‘ggplot2:::scales_list’
      See the note in ?`:::` about the use of this operator.
    ```

# highcharter

Version: 0.5.0

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 16.5Mb
      sub-directories of 1Mb or more:
        doc          13.7Mb
        htmlwidgets   1.8Mb
    ```

# HighDimOut

Version: 1.0.0

## In both

*   checking R code for possible problems ... NOTE
    ```
    Func.ABOD: no visible global function definition for ‘combn’
    Func.ABOD: no visible global function definition for ‘var’
    Func.SOD: no visible global function definition for ‘var’
    Func.trans : erf: no visible global function definition for ‘pnorm’
    Func.trans: no visible global function definition for ‘sd’
    Undefined global functions or variables:
      combn pnorm sd var
    Consider adding
      importFrom("stats", "pnorm", "sd", "var")
      importFrom("utils", "combn")
    to your NAMESPACE file.
    ```

# HistData

Version: 0.8-4

## In both

*   checking Rd cross-references ... NOTE
    ```
    Packages unavailable to check Rd xrefs: ‘Guerry’, ‘alr3’, ‘agridat’
    ```

# HMP

Version: 1.5.1

## In both

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘genalg’
    ```

# HMP16SData

Version: 1.0.1

## In both

*   checking whether package ‘HMP16SData’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘DelayedArray’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/HMP16SData/new/HMP16SData.Rcheck/00install.out’ for details.
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    Attaching package: 'dendextend'
    
    The following object is masked from 'package:stats':
    
        cutree
    
    ========================================
    circlize version 0.4.4
    CRAN page: https://cran.r-project.org/package=circlize
    Github page: https://github.com/jokergoo/circlize
    Documentation: http://jokergoo.github.io/circlize_book/book/
    
    If you use it in published research, please cite:
    Gu, Z. circlize implements and enhances circular visualization 
      in R. Bioinformatics 2014.
    ========================================
    
    Quitting from lines 58-71 (HMP16SData.Rmd) 
    Error: processing vignette 'HMP16SData.Rmd' failed with diagnostics:
    there is no package called 'curatedMetagenomicData'
    Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘curatedMetagenomicData’
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 19.1Mb
      sub-directories of 1Mb or more:
        doc       1.5Mb
        extdata  17.4Mb
    ```

# horserule

Version: 1.0.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘MASS’ ‘Rdpack’
      All declared Imports should be used.
    ```

# hpiR

Version: 0.2.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘knitr’
      All declared Imports should be used.
    ```

# HRM

Version: 1.0.2

## In both

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking:
      ‘RGtk2’ ‘cairoDevice’ ‘RGtk2Extras’
    ```

# HTSSIP

Version: 1.4.0

## In both

*   R CMD check timed out
    

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘igraph’
      All declared Imports should be used.
    ```

# httk

Version: 1.8

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 15.1Mb
      sub-directories of 1Mb or more:
        data  13.1Mb
        doc    1.3Mb
    ```

# hurricaneexposure

Version: 0.0.1

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘hurricaneexposuredata’
    ```

# huxtable

Version: 4.2.0

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      12: stop(msg, domain = NA)
      
      [WARNING] This document format requires a nonempty <title> element.
        Please specify either 'title' or 'pagetitle' in the metadata.
        Falling back to 'fileefc93746299a.utf8'
      [WARNING] This document format requires a nonempty <title> element.
        Please specify either 'title' or 'pagetitle' in the metadata.
        Falling back to 'fileefc9675bbc96.utf8'
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 738 SKIPPED: 57 FAILED: 2
      1. Error: quick_pdf works (@test-quick-output.R#41) 
      2. Error: quick_pdf works with height and width options (@test-quick-output.R#53) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

# hydrolinks

Version: 0.10.0

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘sf’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# hydroscoper

Version: 1.1.1

## In both

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 3031 marked UTF-8 strings
    ```

# hyperSpec

Version: 0.99-20180627

## In both

*   checking PDF version of manual ... WARNING
    ```
    ...
    LaTeX errors found:
    ! Please use \mathaccent for accents in math mode.
    \add@accent ...@spacefactor \spacefactor }\accent 
                                                      #1 #2\egroup \spacefactor ...
    l.3047 ...n{R\textasciicircum{}2}{}}{pearson.dist}
                                                      
    ! Missing { inserted.
    <to be read again> 
                       \egroup 
    l.3047 ...n{R\textasciicircum{}2}{}}{pearson.dist}
                                                      
    ! You can't use `\spacefactor' in math mode.
    \add@accent ...}\accent #1 #2\egroup \spacefactor 
                                                      \accent@spacefactor 
    l.3047 ...n{R\textasciicircum{}2}{}}{pearson.dist}
                                                      
    ! Missing } inserted.
    <inserted text> 
                    }
    l.3047 ...n{R\textasciicircum{}2}{}}{pearson.dist}
                                                      
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.1Mb
      sub-directories of 1Mb or more:
        doc   3.8Mb
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    Iteration 4: 38 support points
    Iteration 5: 32 support points
    Iteration 6: 29 support points
    Iteration 7: 29 support points
    spectrum      1:    29 support points, noise = 15.0,   7 iterations
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'baseline.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `scrartcl.cls' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: cls)
    
    ! Emergency stop.
    <read *> 
             
    l.3 \usepackage
                   [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# IAPWS95

Version: 1.1.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘Rcpp’ ‘ggplot2’ ‘pander’
      All declared Imports should be used.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 3 marked UTF-8 strings
    ```

# ibmdbR

Version: 1.50.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'ibmdbR.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.30 \ifthenelse
                    {\boolean{Sweave@inconsolata}}{%^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘SparkR’
    ```

# iBMQ

Version: 1.20.0

## In both

*   checking whether package ‘iBMQ’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/iBMQ/new/iBMQ.Rcheck/00install.out’ for details.
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .BBSoptions
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

## Installation

### Devel

```
* installing *source* package ‘iBMQ’ ...
checking for gcc... /usr/local/clang4/bin/clang
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether /usr/local/clang4/bin/clang accepts -g... yes
checking for /usr/local/clang4/bin/clang option to accept ISO C89... none needed
checking how to run the C preprocessor... /usr/local/clang4/bin/clang -E
checking for pkg-config... /usr/local/bin/pkg-config
checking pkg-config is at least version 0.9.0... yes
checking for GSL... no
checking for gsl-config... no
checking for GSL - version >= 1.2... no
*** The gsl-config script installed by GSL could not be found
*** If GSL was installed in PREFIX, make sure PREFIX/bin is in
*** your path, or set the GSL_CONFIG environment variable to the
*** full path to gsl-config.
configure: error: Cannot find Gnu Scientific Library >=1.6
ERROR: configuration failed for package ‘iBMQ’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/iBMQ/new/iBMQ.Rcheck/iBMQ’

```
### CRAN

```
* installing *source* package ‘iBMQ’ ...
checking for gcc... /usr/local/clang4/bin/clang
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether /usr/local/clang4/bin/clang accepts -g... yes
checking for /usr/local/clang4/bin/clang option to accept ISO C89... none needed
checking how to run the C preprocessor... /usr/local/clang4/bin/clang -E
checking for pkg-config... /usr/local/bin/pkg-config
checking pkg-config is at least version 0.9.0... yes
checking for GSL... no
checking for gsl-config... no
checking for GSL - version >= 1.2... no
*** The gsl-config script installed by GSL could not be found
*** If GSL was installed in PREFIX, make sure PREFIX/bin is in
*** your path, or set the GSL_CONFIG environment variable to the
*** full path to gsl-config.
configure: error: Cannot find Gnu Scientific Library >=1.6
ERROR: configuration failed for package ‘iBMQ’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/iBMQ/old/iBMQ.Rcheck/iBMQ’

```
# iCNV

Version: 1.0.0

## In both

*   checking whether package ‘iCNV’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘Rsamtools’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘rtracklayer’ was built under R version 3.5.1
      Note: break used in wrong context: no loop is visible 
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/iCNV/new/iCNV.Rcheck/00install.out’ for details.
    Information on the location(s) of code generating the ‘Note’s can be
    obtained by re-running with environment variable R_KEEP_PKG_SOURCE set
    to ‘yes’.
    ```

# ICtest

Version: 0.3-1

## In both

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘fICA’
    ```

# IDE

Version: 0.2.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'IDE_intro.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `algorithm.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.58 \usepackage
                    {algpseudocode}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# ideal

Version: 1.4.0

## In both

*   R CMD check timed out
    

*   checking whether package ‘ideal’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘graph’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/ideal/new/ideal.Rcheck/00install.out’ for details.
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ideal: no visible binding for '<<-' assignment to ‘ideal_env’
    ideal : <anonymous>: no visible binding for global variable ‘airway’
    ideal : <anonymous>: no visible binding for global variable ‘ideal_env’
    Undefined global functions or variables:
      airway ideal_env
    ```

# idealstan

Version: 0.2.7

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Warning in .local(object, ...) :
      some chains had errors; consider specifying chains = 1 to debug
    Quitting from lines 72-73 (Package_Introduction.Rmd) 
    Error: processing vignette 'Package_Introduction.Rmd' failed with diagnostics:
    object 'all_resid' not found
    Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  8.0Mb
      sub-directories of 1Mb or more:
        data   3.5Mb
        libs   3.6Mb
    ```

# ifaTools

Version: 0.17

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'pritikin-schmidt.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `microtype.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.43 ^^M
            
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# IHW

Version: 1.8.0

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  6.1Mb
      sub-directories of 1Mb or more:
        doc   5.8Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    ihw.default: no visible global function definition for ‘p.adjust’
    ihw_convex: no visible global function definition for ‘gurobi’
    ihw_internal: no visible global function definition for ‘p.adjust’
    ihw_milp: no visible global function definition for ‘str’
    ihw_milp: no visible global function definition for ‘gurobi’
    plot_decisionboundary: no visible binding for global variable ‘stratum’
    plot_decisionboundary: no visible binding for global variable
      ‘covariate’
    plot_decisionboundary: no visible binding for global variable ‘pvalue’
    plot_decisionboundary: no visible binding for global variable ‘fold’
    thresholds_ihwResult: no visible global function definition for
      ‘na.exclude’
    thresholds,ihwResult: no visible global function definition for
      ‘na.exclude’
    Undefined global functions or variables:
      covariate fold gurobi mcols mcols<- metadata metadata<- na.exclude
      p.adjust pvalue runif str stratum
    Consider adding
      importFrom("stats", "na.exclude", "p.adjust", "runif")
      importFrom("utils", "str")
    to your NAMESPACE file.
    ```

# IHWpaper

Version: 1.7.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 29-65 (BH-explanation.Rmd) 
    Error: processing vignette 'BH-explanation.Rmd' failed with diagnostics:
    Palette not found.
    Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 23.9Mb
      sub-directories of 1Mb or more:
        doc      13.1Mb
        extdata   9.8Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    scott_fdrreg: no visible global function definition for ‘FDRreg’
    scott_fdrreg: no visible global function definition for ‘getFDR’
    sim_fun_eval: no visible binding for global variable ‘fdr_method’
    sim_fun_eval: no visible binding for global variable ‘fdr_pars’
    sim_fun_eval: no visible binding for global variable ‘FDP’
    sim_fun_eval: no visible binding for global variable ‘rj_ratio’
    sim_fun_eval: no visible binding for global variable ‘FPR’
    sim_fun_eval: no visible binding for global variable ‘FWER’
    Undefined global functions or variables:
      FDP FDRreg FPR FWER fdr_method fdr_pars getFDR rj_ratio
    ```

# imager

Version: 0.41.1

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 10.2Mb
      sub-directories of 1Mb or more:
        data      1.4Mb
        doc       1.1Mb
        include   2.8Mb
        libs      3.1Mb
    ```

# IMAS

Version: 1.4.0

## In both

*   checking whether package ‘IMAS’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘GenomicFeatures’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/IMAS/new/IMAS.Rcheck/00install.out’ for details.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    Loading required package: ggplot2
    Loading required package: IVAS
    'select()' returned 1:1 mapping between keys and columns
    'select()' returned 1:1 mapping between keys and columns
    'select()' returned 1:1 mapping between keys and columns
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'IMAS.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `pdfpages.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.8 ^^M
           
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# Imetagene

Version: 1.10.0

## In both

*   checking whether package ‘Imetagene’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘metagene’ was built under R version 3.5.1
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/Imetagene/new/Imetagene.Rcheck/00install.out’ for details.
    ```

# ImmuneSpaceR

Version: 1.8.2

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 2 SKIPPED: 0 FAILED: 18
      1. Failure: all studies can loaded with empty string (@test-CreateConnection.R#33) 
      2. Failure: Nonexistent study fails to load (@test-CreateConnection.R#39) 
      3. Failure: Existing study can be loaded (@test-CreateConnection.R#44) 
      4. Failure: ImmuneSignatures study can be loaded (@test-CreateConnection.R#49) 
      5. Failure: ImmuneSignatures study can be loaded (@test-CreateConnection.R#50) 
      6. Failure: Lyoplate study cannot be loaded (@test-CreateConnection.R#56) 
      7. Error: (unknown) (@test-dataIntegration.R#11) 
      8. Error: (unknown) (@test-getDataset.R#9) 
      9. Error: (unknown) (@test-getGEAnalysis.R#11) 
      1. ...
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    A .netrc file is required to connect to ImmuneSpace. For more information on how to create one, refer to the Configuration section of the introduction vignette.
    Quitting from lines 63-66 (Intro_to_ImmuneSpaceR.Rmd) 
    Error: processing vignette 'Intro_to_ImmuneSpaceR.Rmd' failed with diagnostics:
    Invalid credential or deactivated account. Check your account in the portal.
    Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  7.4Mb
      sub-directories of 1Mb or more:
        doc   6.8Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Unexported objects imported by ':::' calls:
      ‘Rlabkey:::.lkdefaults’ ‘Rlabkey:::labkey.getRequestOptions’
      See the note in ?`:::` about the use of this operator.
    ```

*   checking R code for possible problems ... NOTE
    ```
    .getDataToPlot: no visible binding for global variable ‘value_reported’
    Undefined global functions or variables:
      value_reported
    ```

# ImpulseDE2

Version: 1.4.0

## In both

*   checking R code for possible problems ... NOTE
    ```
    plotGenes: no visible global function definition for ‘error’
    plotGenes: no visible binding for global variable ‘normCounts’
    plotGenes: no visible binding for global variable ‘Condition’
    plotGenes: no visible binding for global variable ‘Batch’
    plotGenes: no visible binding for global variable ‘value’
    plotGenes: no visible binding for global variable ‘BatchFit’
    Undefined global functions or variables:
      Batch BatchFit Condition error normCounts value
    ```

# IncDTW

Version: 1.0.4

## In both

*   checking for GNU extensions in Makefiles ... NOTE
    ```
    GNU make is a SystemRequirements.
    ```

# incR

Version: 1.1.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘dplyr’ ‘rgeos’
      All declared Imports should be used.
    ```

# InformationValue

Version: 1.2.3

## In both

*   checking DESCRIPTION meta-information ... NOTE
    ```
    BugReports field should be the URL of a single webpage
    ```

# inlabru

Version: 2.1.9

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘INLA’
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘INLA’
    ```

# InTAD

Version: 1.0.0

## In both

*   checking whether package ‘InTAD’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘DelayedArray’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/InTAD/new/InTAD.Rcheck/00install.out’ for details.
    ```

# IntClust

Version: 0.1.0

## In both

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'IntClust_Vignette.tex' failed.
    LaTeX errors:
    ! Package babel Error: Unknown option `english'. Either you misspelled it
    (babel)                or the language definition file english.ldf was not foun
    d.
    
    See the babel package documentation for explanation.
    ! LaTeX Error: File `tcolorbox.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.8 \usepackage
                   {Sweave}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# interactiveDisplay

Version: 1.18.0

## In both

*   checking dependencies in R code ... WARNING
    ```
    '::' or ':::' import not declared from: ‘.interactiveDisplayBase’
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'interactiveDisplay.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `cite.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.57 \usepackage
                    [nottoc,numbib]{tocbibind}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Package which this enhances but not available for checking: ‘rstudio’
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      for global variable ‘Var1’
    display,RangedSummarizedExperiment : <anonymous>: no visible binding
      for global variable ‘Var2’
    display,RangedSummarizedExperiment : <anonymous>: no visible binding
      for global variable ‘value’
    Undefined global functions or variables:
      AnnotationTrack GO.db GRanges GRangesList GenomeAxisTrack IRanges
      IdeogramTrack MRcounts Var1 Var2 as.dendrogram assays cim
      colorRampPalette coord_flip cutree dendrapply dev.off dist
      elementMetadata experimentData exprs fData hclust installed.packages
      is.leaf layout_circle legend libSize mcols pData plot plotFeature
      plotOrd plotTracks png rainbow ranges rowRanges seqlengths
      seqlengths<- seqlevels<- seqnames ucscGenomes value
    Consider adding
      importFrom("grDevices", "colorRampPalette", "dev.off", "png",
                 "rainbow")
      importFrom("graphics", "legend", "plot")
      importFrom("stats", "as.dendrogram", "cutree", "dendrapply", "dist",
                 "hclust", "is.leaf")
      importFrom("utils", "installed.packages")
    to your NAMESPACE file.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Failed with error:  'package 'DelayedArray' could not be loaded'
      Error in .requirePackage(package) : 
        unable to find required package 'SummarizedExperiment'
      Calls: <Anonymous> ... getClass -> getClassDef -> .classEnv -> .requirePackage
      Execution halted
    ```

# interplot

Version: 0.2.1

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘gridExtra’
      All declared Imports should be used.
    ```

# IONiseR

Version: 2.4.0

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.7Mb
      sub-directories of 1Mb or more:
        doc       3.7Mb
        extdata   1.5Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      ‘start_time’
    readFast5Summary.mc: no visible binding for global variable ‘duration’
    readFast5Summary.mc: no visible binding for global variable
      ‘num_events’
    [,Fast5Summary-ANY-ANY-ANY: no visible binding for global variable
      ‘baseCalledTemplate’
    [,Fast5Summary-ANY-ANY-ANY: no visible binding for global variable
      ‘baseCalledComplement’
    [,Fast5Summary-ANY-ANY-ANY: no visible binding for global variable
      ‘component’
    [,Fast5Summary-ANY-ANY-ANY: no visible binding for global variable
      ‘idx’
    show,Fast5Summary: no visible binding for global variable ‘full_2D’
    show,Fast5Summary: no visible binding for global variable ‘pass’
    Undefined global functions or variables:
      := AAAAA TTTTT accumulation baseCalledComplement baseCalledTemplate
      bases_called category channel circleFun component duration error freq
      full_2D group hour idx matrixCol matrixRow meanZValue mean_value
      median_signal minute mux name nbases new_reads num_events oddEven
      pass pentamer rbindlist readIDs seq_length start_time time_bin
      time_group x y zvalue
    ```

# IPSUR

Version: 3.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘Hmisc’ ‘RcmdrPlugin.IPSUR’ ‘TeachingDemos’ ‘actuar’ ‘aplpack’
      ‘binom’ ‘boot’ ‘coin’ ‘distrEx’ ‘e1071’ ‘lmtest’ ‘mvtnorm’ ‘prob’
      ‘qcc’ ‘reshape’ ‘scatterplot3d’ ‘vcd’
      All declared Imports should be used.
    ```

# ipumsr

Version: 0.3.0

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘sf’
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘R6’
      All declared Imports should be used.
    ```

# IrisSpatialFeatures

Version: 1.3.0

## In both

*   R CMD check timed out
    

# iSEE

Version: 1.0.1

## In both

*   checking whether package ‘iSEE’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘DelayedArray’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/iSEE/new/iSEE.Rcheck/00install.out’ for details.
    ```

*   checking package dependencies ... NOTE
    ```
    Package which this enhances but not available for checking: ‘ExperimentHub’
    ```

*   checking dependencies in R code ... NOTE
    ```
    Unexported object imported by a ':::' call: ‘S4Vectors:::selectSome’
      See the note in ?`:::` about the use of this operator.
    ```

# isobar

Version: 1.26.0

## In both

*   checking Rd cross-references ... WARNING
    ```
    Missing link or links in documentation object 'IBSpectra-class.Rd':
      ‘[MSnbase]{MSnbase}’
    
    See section 'Cross-references' in the 'Writing R Extensions' manual.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.4Mb
      sub-directories of 1Mb or more:
        R      2.0Mb
        data   1.9Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    There are ::: calls to the package's namespace in its code. A package
      almost never needs to use ::: for its own objects:
      ‘.as.matrix’ ‘.as.vect’ ‘.convertPeptideModif’
      ‘.proteinGroupAsConciseDataFrame’ ‘.read.idfile’ ‘.sum.bool’
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    coerce,IBSpectra-MSnSet: no visible binding for global variable ‘o’
    coerce,MSnSet-IBSpectra: no visible global function definition for
      ‘qual’
    df,Tlsd: no visible global function definition for ‘param’
    estimateRatio,IBSpectra-ANY-missing-missing-character-missing: no
      visible binding for global variable ‘i’
    estimateRatio,IBSpectra-ANY-missing-missing-missing-character: no
      visible binding for global variable ‘i’
    estimateRatioNumeric,numeric-numeric-NoiseModel: no visible binding for
      global variable ‘center.var’
    location,Tlsd: no visible global function definition for ‘param’
    plotRatio,IBSpectra-character-character-character: no visible binding
      for global variable ‘pch’
    plotRatio,IBSpectra-character-character-character: no visible binding
      for global variable ‘noise.model.col’
    plotRatio,IBSpectra-character-character-character: no visible binding
      for global variable ‘pch.p’
    scale,Tlsd: no visible global function definition for ‘param’
    Undefined global functions or variables:
      center.var d g i mz name noise.model.col o param pch pch.p peptide
      qual ratio type
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    
    The following object is masked from ‘package:base’:
    
        paste0
    
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'isobar-devel.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `times.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.8 \usepackage
                   {hyperref}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# IsoformSwitchAnalyzeR

Version: 1.2.0

## In both

*   checking whether package ‘IsoformSwitchAnalyzeR’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘rtracklayer’ was built under R version 3.5.1
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/IsoformSwitchAnalyzeR/new/IsoformSwitchAnalyzeR.Rcheck/00install.out’ for details.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.3Mb
      sub-directories of 1Mb or more:
        doc   3.5Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    switchPlotTranscript: no visible global function definition for
      'queryHits'
    switchPlotTranscript: no visible binding for global variable 'y'
    switchPlotTranscript: no visible binding for global variable 'yend'
    switchPlotTranscript: no visible binding for global variable 'x'
    switchPlotTranscript: no visible binding for global variable 'ymin'
    switchPlotTranscript: no visible binding for global variable 'xmin'
    switchPlotTranscript: no visible binding for global variable 'ymax'
    switchPlotTranscript: no visible binding for global variable 'xmax'
    switchPlotTranscript: no visible binding for global variable 'Domain'
    Undefined global functions or variables:
      AStype CI_down CI_hi CI_low CI_up Comparison Condition Domain IF Rle
      Significant feature2 fisher.test geneFraction gene_expression
      grid.draw idNr isoFraction isoform_feature nrGenesWithConsequences
      nrIsoWithConsequences plotViewport popViewport propCiHi propCiLo
      propOfRelevantEvents propUp propUpCiHi propUpCiLo queryHits sigEval
      sigLevel sigLevelPos significance splicingResult switchConsequence
      value variable x xmax xmin y yend ymax ymin
    Consider adding
      importFrom("stats", "fisher.test")
    to your NAMESPACE file.
    ```

*   checking for unstated dependencies in vignettes ... NOTE
    ```
    'library' or 'require' call not declared from: ‘pheatmap’
    ```

# isomiRs

Version: 1.8.0

## In both

*   checking whether package ‘isomiRs’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘DelayedArray’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/isomiRs/new/isomiRs.Rcheck/00install.out’ for details.
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  8.7Mb
      sub-directories of 1Mb or more:
        data   7.1Mb
        doc    1.2Mb
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Package listed in more than one of Depends, Imports, Suggests, Enhances:
      ‘tidyr’
    A package should be listed in only one of these fields.
    ```

*   checking R code for possible problems ... NOTE
    ```
    .predict_correlate_mirna_targets: no visible binding for global
      variable ‘mrna_de’
    .predict_correlate_mirna_targets: no visible binding for global
      variable ‘mirna_se’
    .predict_correlate_mirna_targets: no visible binding for global
      variable ‘mrna_se’
    .run_enricher: no visible binding for global variable ‘sel_genes’
    .run_enricher: no visible global function definition for ‘enrichGO’
    .run_enricher: no visible binding for global variable ‘Count’
    .viz_mirna_gene_enrichment: no visible binding for global variable ‘X1’
    isoNetwork: no visible binding for global variable ‘Description’
    isoNetwork: no visible binding for global variable ‘geneID’
    isoPlotNet: no visible binding for global variable ‘ngene’
    Undefined global functions or variables:
      Count Description X1 enrichGO geneID mirna_se mrna_de mrna_se ngene
      sel_genes
    ```

# ITGM

Version: 0.6

## In both

*   checking whether package ‘ITGM’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/ITGM/new/ITGM.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘ITGM’ ...
** package ‘ITGM’ successfully unpacked and MD5 sums checked
** R
** byte-compile and prepare package for lazy loading
Warning in fun(libname, pkgname) : couldn't connect to display ""
Error: package or namespace load failed for ‘Fgmutils’ in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]):
 there is no package called ‘ReporteRsjars’
Error : package ‘Fgmutils’ could not be loaded
ERROR: lazy loading failed for package ‘ITGM’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/ITGM/new/ITGM.Rcheck/ITGM’

```
### CRAN

```
* installing *source* package ‘ITGM’ ...
** package ‘ITGM’ successfully unpacked and MD5 sums checked
** R
** byte-compile and prepare package for lazy loading
Warning in fun(libname, pkgname) : couldn't connect to display ""
Error: package or namespace load failed for ‘Fgmutils’ in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]):
 there is no package called ‘ReporteRsjars’
Error : package ‘Fgmutils’ could not be loaded
ERROR: lazy loading failed for package ‘ITGM’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/ITGM/old/ITGM.Rcheck/ITGM’

```
# ITNr

Version: 0.3.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘comtradr’
      All declared Imports should be used.
    ```

# IVAS

Version: 2.0.0

## In both

*   checking whether package ‘IVAS’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘GenomicFeatures’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Note: next used in wrong context: no loop is visible 
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/IVAS/new/IVAS.Rcheck/00install.out’ for details.
    Information on the location(s) of code generating the ‘Note’s can be
    obtained by re-running with environment variable R_KEEP_PKG_SOURCE set
    to ‘yes’.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
        'citation("Biobase")', and for packages 'citation("pkgname")'.
    
    Loading required package: ggplot2
    'select()' returned 1:1 mapping between keys and columns
    'select()' returned 1:1 mapping between keys and columns
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'IVAS.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `a4wide.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.7 \graphicspath
                     {{./result/}}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# ivygapSE

Version: 1.2.0

## In both

*   checking whether package ‘ivygapSE’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘DelayedArray’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/ivygapSE/new/ivygapSE.Rcheck/00install.out’ for details.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 16.6Mb
      sub-directories of 1Mb or more:
        data  13.9Mb
        doc    2.4Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    exprByType: no visible binding for global variable ‘ivySE’
    ivyGlimpse: no visible binding for global variable ‘ivySE’
    ivyGlimpse : server: no visible binding for global variable ‘ivySE’
    Undefined global functions or variables:
      ivySE
    ```

# jarbes

Version: 1.7.0

## In both

*   checking whether package ‘jarbes’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/jarbes/new/jarbes.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘jarbes’ ...
** package ‘jarbes’ successfully unpacked and MD5 sums checked
** R
** data
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rjags', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/jarbes/rjags/libs/rjags.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/jarbes/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/jarbes/rjags/libs/rjags.so
  Reason: image not found
ERROR: lazy loading failed for package ‘jarbes’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/jarbes/new/jarbes.Rcheck/jarbes’

```
### CRAN

```
* installing *source* package ‘jarbes’ ...
** package ‘jarbes’ successfully unpacked and MD5 sums checked
** R
** data
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rjags', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/jarbes/rjags/libs/rjags.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/jarbes/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/jarbes/rjags/libs/rjags.so
  Reason: image not found
ERROR: lazy loading failed for package ‘jarbes’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/jarbes/old/jarbes.Rcheck/jarbes’

```
# jmv

Version: 0.9.4

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘BayesFactor’ ‘GGally’ ‘GPArotation’ ‘MASS’ ‘PMCMR’ ‘R6’ ‘ROCR’
      ‘afex’ ‘car’ ‘emmeans’ ‘ggridges’ ‘lavaan’ ‘multcomp’ ‘mvnormtest’
      ‘nnet’ ‘psych’ ‘vcd’ ‘vcdExtra’
      All declared Imports should be used.
    ```

# jmvcore

Version: 0.9.4

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘RProtoBuf’
    ```

# joineRmeta

Version: 0.1.1

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘nlme’
      All declared Imports should be used.
    ```

# joineRML

Version: 0.4.2

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'technical.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `setspace.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.15 \usepackage
                    {hyperref}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# JointAI

Version: 0.3.0

## In both

*   checking whether package ‘JointAI’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/JointAI/new/JointAI.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘JointAI’ ...
** package ‘JointAI’ successfully unpacked and MD5 sums checked
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
Error: package or namespace load failed for ‘rjags’:
 .onLoad failed in loadNamespace() for 'rjags', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/JointAI/rjags/libs/rjags.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/JointAI/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/JointAI/rjags/libs/rjags.so
  Reason: image not found
Error : package ‘rjags’ could not be loaded
ERROR: lazy loading failed for package ‘JointAI’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/JointAI/new/JointAI.Rcheck/JointAI’

```
### CRAN

```
* installing *source* package ‘JointAI’ ...
** package ‘JointAI’ successfully unpacked and MD5 sums checked
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
Error: package or namespace load failed for ‘rjags’:
 .onLoad failed in loadNamespace() for 'rjags', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/JointAI/rjags/libs/rjags.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/JointAI/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/JointAI/rjags/libs/rjags.so
  Reason: image not found
Error : package ‘rjags’ could not be loaded
ERROR: lazy loading failed for package ‘JointAI’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/JointAI/old/JointAI.Rcheck/JointAI’

```
# jtools

Version: 1.1.1

## In both

*   checking Rd cross-references ... NOTE
    ```
    Packages unavailable to check Rd xrefs: ‘wec’, ‘rockchalk’, ‘pequod’, ‘piecewiseSEM’
    ```

# kamila

Version: 0.1.1.2

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘mclust’
      All declared Imports should be used.
    ```

# kdetrees

Version: 0.1.5

## In both

*   checking R code for possible problems ... NOTE
    ```
    ...
    as.matrix.multiPhylo : tip2tip: no visible global function definition
      for ‘cophenetic’
    bw.nn : <anonymous>: no visible global function definition for
      ‘quantile’
    dist.diss: no visible binding for global variable ‘median’
    dist.diss: no visible global function definition for ‘dist’
    hist.kdetrees: no visible binding for global variable ‘density’
    kdetrees : cutoff: no visible global function definition for ‘quantile’
    kdetrees.complete: no visible global function definition for ‘plot’
    kdetrees.complete: no visible global function definition for ‘hist’
    kdetrees.complete: no visible global function definition for
      ‘write.csv’
    plot.kdetrees: no visible binding for global variable ‘density’
    Undefined global functions or variables:
      cophenetic density dist hist median plot quantile write.csv
    Consider adding
      importFrom("graphics", "hist", "plot")
      importFrom("stats", "cophenetic", "density", "dist", "median",
                 "quantile")
      importFrom("utils", "write.csv")
    to your NAMESPACE file.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'kdetrees.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.30 \ifthenelse
                    {\boolean{Sweave@inconsolata}}{%^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# keras

Version: 2.2.0

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.7Mb
      sub-directories of 1Mb or more:
        doc   4.3Mb
    ```

# kissDE

Version: 1.0.0

## In both

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    Trying to fit models on data...
    This can be a time-consuming step, so do not hesitate to have 
        a look at the very well-written vignette !
    Computing pvalues...
    Computing size of the effect and last cutoffs...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'kissDE.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `beramono.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.87 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# KraljicMatrix

Version: 0.2.1

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘dplyr’ ‘tibble’
      All declared Imports should be used.
    ```

# Lahman

Version: 6.0-0

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  7.6Mb
      sub-directories of 1Mb or more:
        data   7.4Mb
    ```

# LambertW

Version: 0.6.4

## In both

*   checking whether package ‘LambertW’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      W_gamma_Cpp.cpp:14:39: warning: explicitly assigning value of variable of type 'int' to itself [-Wself-assign]
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/LambertW/new/LambertW.Rcheck/00install.out’ for details.
    ```

# LANDD

Version: 1.1.0

## In both

*   checking R code for possible problems ... NOTE
    ```
    getGO: no visible global function definition for ‘new’
    getGO: no visible global function definition for ‘getGeneric’
    graph.kd: no visible global function definition for ‘as’
    Undefined global functions or variables:
      as getGeneric new
    Consider adding
      importFrom("methods", "as", "getGeneric", "new")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

# landscapemetrics

Version: 0.2

## In both

*   checking package dependencies ... NOTE
    ```
    Packages which this enhances but not available for checking: ‘stars’ ‘sf’
    ```

# largeVis

Version: 0.2.1.1

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  6.6Mb
      sub-directories of 1Mb or more:
        libs           1.6Mb
        testdata       1.8Mb
        vignettedata   2.0Mb
    ```

# lavaSearch2

Version: 1.4

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error: processing vignette 'overview.ltx' failed with diagnostics:
    Running 'texi2dvi' on 'overview.ltx' failed.
    LaTeX errors:
    ! LaTeX Error: File `listings.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.8 \lstset
               {^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Execution halted
    ```

# LDheatmap

Version: 0.99-5

## In both

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Loading required package: grid
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'LDheatmap.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.55 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# LEAP

Version: 0.2

## In both

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'LEAP_Vignette.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.30 \ifthenelse
                    {\boolean{Sweave@inconsolata}}{%^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# learningCurve

Version: 1.1.2

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘gridExtra’ ‘scales’ ‘tibble’
      All declared Imports should be used.
    ```

# learnstats

Version: 0.1.1

## In both

*   checking R code for possible problems ... NOTE
    ```
    ...
    qqsim: no visible global function definition for ‘mtext’
    qqsim: no visible global function definition for ‘jpeg’
    qqsim: no visible global function definition for ‘dev.off’
    timeseressim: no visible global function definition for ‘rnorm’
    timeseressim: no visible global function definition for ‘par’
    timeseressim: no visible global function definition for ‘plot’
    timeseressim: no visible global function definition for ‘mtext’
    timeseressim: no visible global function definition for ‘jpeg’
    timeseressim: no visible global function definition for ‘dev.off’
    Undefined global functions or variables:
      axis dchisq dev.off dnorm dt hist jpeg legend mtext par pchisq pf
      plot pnorm polygon pt qchisq qf qqline qqnorm qt rchisq rnorm rt
      runif segments title
    Consider adding
      importFrom("grDevices", "dev.off", "jpeg")
      importFrom("graphics", "axis", "hist", "legend", "mtext", "par",
                 "plot", "polygon", "segments", "title")
      importFrom("stats", "dchisq", "dnorm", "dt", "pchisq", "pf", "pnorm",
                 "pt", "qchisq", "qf", "qqline", "qqnorm", "qt", "rchisq",
                 "rnorm", "rt", "runif")
    to your NAMESPACE file.
    ```

# LedPred

Version: 1.14.1

## In both

*   checking DESCRIPTION meta-information ... NOTE
    ```
    License components which are templates and need '+ file LICENSE':
      MIT
    ```

*   checking R code for possible problems ... NOTE
    ```
    .plotCostGamma: no visible global function definition for ‘title’
    .plotCostGamma: no visible global function definition for ‘points’
    mapFeaturesToCRMs: no visible global function definition for
      ‘write.table’
    mapFeaturesToCRMs: no visible global function definition for
      ‘read.table’
    mapFeaturesToCRMs: no visible global function definition for
      ‘download.file’
    scoreData: no visible global function definition for ‘write.table’
    Undefined global functions or variables:
      download.file points read.table title write.table
    Consider adding
      importFrom("graphics", "points", "title")
      importFrom("utils", "download.file", "read.table", "write.table")
    to your NAMESPACE file.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Loading required package: e1071
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'LedPred.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `datetime.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.4 \usepackage
                   [utf8]{inputenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# lfa

Version: 1.10.0

## In both

*   checking for code/documentation mismatches ... WARNING
    ```
    Codoc mismatches from documentation object 'lfa':
    lfa
      Code: function(X, d, adjustments = NULL, override = FALSE, safety =
                     FALSE, ploidy = 2)
      Docs: function(X, d, adjustments = NULL, override = FALSE, safety =
                     FALSE)
      Argument names in code not in docs:
        ploidy
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'lfa.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `float.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.58 \usepackage
                    {gensymb}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking R code for possible problems ... NOTE
    ```
    compute.nulls: no visible global function definition for ‘rbinom’
    lfa: no visible global function definition for ‘complete.cases’
    lfa: no visible global function definition for ‘residuals’
    lfa: no visible global function definition for ‘lm’
    lreg: no visible global function definition for ‘glm’
    read.bed: no visible global function definition for ‘read.table’
    trunc.svd: no visible global function definition for ‘rnorm’
    Undefined global functions or variables:
      complete.cases glm lm rbinom read.table residuals rnorm
    Consider adding
      importFrom("stats", "complete.cases", "glm", "lm", "rbinom",
                 "residuals", "rnorm")
      importFrom("utils", "read.table")
    to your NAMESPACE file.
    ```

# likeLTD

Version: 6.3.0

## In both

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    Authors: D. Ardia, K. Mullen, B. Peterson and J. Ulrich
    
    Loading required package: ggplot2
    Loading required package: gtools
    Loading required package: rtf
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'usage.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `fullpage.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.5 ^^M
           
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# likert

Version: 1.3.5

## In both

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 7 marked UTF-8 strings
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'likert-xtable.tex' failed.
    LaTeX errors:
    ! Package babel Error: Unknown option `english'. Either you misspelled it
    (babel)                or the language definition file english.ldf was not foun
    d.
    
    See the babel package documentation for explanation.
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.30 \ifthenelse
                    {\boolean{Sweave@inconsolata}}{%^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# lilikoi

Version: 0.1.0

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.3Mb
      sub-directories of 1Mb or more:
        data      4.1Mb
        extdata   1.1Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘Matrix’ ‘devtools’ ‘e1071’ ‘glmnet’ ‘hash’ ‘pamr’ ‘randomForest’
      All declared Imports should be used.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 3837 marked UTF-8 strings
    ```

# lime

Version: 0.4.0

## Newly broken

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      4: .h2o.__checkConnectionHealth()
      5: stop(sprintf("H2O connection has been severed. Cannot connect to instance at %s\n", 
             h2o.getBaseURL(conn)), rv$curlErrorMessage)
      
      Error in .h2o.__checkConnectionHealth() : 
        H2O connection has been severed. Cannot connect to instance at http://localhost:54321/
      Failed to connect to localhost port 54321: Connection refused
      Calls: <Anonymous> -> .h2o.__remoteSend -> .h2o.__checkConnectionHealth
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 25 SKIPPED: 0 FAILED: 2
      1. Error: H2OMultinomialClassification: lime explanation only produces one entry per case and feature (@test-h2o.R#45) 
      2. Error: H2ORegression: lime explanation only produces one entry per case and feature (@test-h2o.R#72) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

# LINC

Version: 1.8.0

## In both

*   R CMD check timed out
    

*   checking installed package size ... NOTE
    ```
      installed size is  6.1Mb
      sub-directories of 1Mb or more:
        doc   3.6Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    justlinc,matrix: no visible global function definition for ‘plot’
    Undefined global functions or variables:
      plot
    Consider adding
      importFrom("graphics", "plot")
    to your NAMESPACE file.
    ```

*   checking R/sysdata.rda ... NOTE
    ```
      
      Note: significantly better compression could be obtained
            by using R CMD build --resave-data
                  old_size new_size compress
      sysdata.rda    449Kb    307Kb       xz
    ```

# LineagePulse

Version: 1.0.0

## In both

*   checking whether package ‘LineagePulse’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Note: possible error in 'evalLogLikGeneMM(vecCounts = vecCounts, ': unused arguments (matDispParam = matDispParam, matDropParam = matDropParam, matWeights = matWeights) 
      Note: possible error in 'evalLogLikGeneMM(vecCounts = vecCounts, ': unused arguments (matDispParam = do.call(cbind, lapply(seq_len(ncol(matWeights)), function(m) vecDispParam)), matDropParam = do.call(cbind, lapply(seq_len(ncol(matWeights)), function(m) vecPiParam)), matWeights = matWeights) 
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/LineagePulse/new/LineagePulse.Rcheck/00install.out’ for details.
    Information on the location(s) of code generating the ‘Note’s can be
    obtained by re-running with environment variable R_KEEP_PKG_SOURCE set
    to ‘yes’.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      function(m) vecPiParam)), matWeights = matWeights)
    evalLogLikMatrix : <anonymous>: possible error in scaNCells =
      length(vecCounts)): unused arguments (matDispParam = do.call(cbind,
      lapply(seq_len(ncol(matWeights)), function(m) vecDispParam)),
      matDropParam = do.call(cbind, lapply(seq_len(ncol(matWeights)),
      function(m) vecPiParam)), matWeights = matWeights)
    plotCellDensity: no visible binding for global variable ‘continuous’
    plotGene: no visible binding for global variable ‘x’
    plotGene: no visible binding for global variable ‘dropout_posterior’
    plotGene: no visible binding for global variable ‘groups’
    plotGene: no visible binding for global variable ‘dfAnnot’
    plotGene: no visible binding for global variable ‘mean_count’
    plotGene: no visible binding for global variable ‘quantile_25’
    plotGene: no visible binding for global variable ‘quantile_75’
    plotGene: no visible binding for global variable ‘model’
    plotGene: no visible binding for global variable ‘continuous’
    plotGene: no visible binding for global variable ‘trajectory_contour’
    plotGene: no visible binding for global variable ‘ncells’
    Undefined global functions or variables:
      continuous dfAnnot dropout_posterior groups lsDispModel mean_count
      model ncells quantile_25 quantile_75 trajectory_contour x
    ```

# Linnorm

Version: 2.4.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Warning in has_utility("pdfcrop") :
      pdfcrop not installed or not in PATH
    tlmgr search --file --global '/beramono.sty'
    Unknown directive ...l
    ! LaTeX Error: File `beramono.sty' not found.
    
    ! Emergency stop.
    <read *> 
    
    Error: processing vignette 'Linnorm_User_Manual.Rmd' failed with diagnostics:
    Failed to compile Linnorm_User_Manual.tex. See Linnorm_User_Manual.log for more info.
    Execution halted
    ```

*   checking R code for possible problems ... NOTE
    ```
    Linnorm.HClust: no visible binding for global variable ‘x’
    Linnorm.HClust: no visible binding for global variable ‘y’
    Linnorm.HClust: no visible binding for global variable ‘xend’
    Linnorm.HClust: no visible binding for global variable ‘yend’
    Linnorm.HClust: no visible binding for global variable ‘cluster’
    Linnorm.HClust: no visible binding for global variable ‘X1’
    Linnorm.HClust: no visible binding for global variable ‘X2’
    Linnorm.HVar: no visible binding for global variable ‘SD’
    Linnorm.HVar: no visible binding for global variable ‘group’
    Undefined global functions or variables:
      SD X1 X2 cluster group x xend y yend
    ```

# lme4

Version: 1.1-18-1

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  7.1Mb
      sub-directories of 1Mb or more:
        R          1.5Mb
        doc        2.4Mb
        testdata   1.6Mb
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
        lmList
    
    This is mgcv 1.8-24. For overview type 'help("mgcv-package")'.
    This is gamm4 0.2-5
    
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'lmer.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.55 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# LocalControl

Version: 1.1.1

## In both

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    
    The following object is masked from 'package:dendextend':
    
        set
    
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'LocalControl-jss-2018.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.55 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# LocFDRPois

Version: 1.0.0

## In both

*   checking R code for possible problems ... NOTE
    ```
    AnalyticalOptim: no visible global function definition for ‘optim’
    LLConstructor : LL: no visible global function definition for ‘dpois’
    MixtureDensity: no visible global function definition for ‘glm’
    MixtureDensity : f_hat: no visible global function definition for
      ‘predict’
    NullDensity : f0: no visible global function definition for ‘dpois’
    Undefined global functions or variables:
      dpois glm optim predict
    Consider adding
      importFrom("stats", "dpois", "glm", "optim", "predict")
    to your NAMESPACE file.
    ```

# LOGIT

Version: 1.3

## In both

*   checking examples ... ERROR
    ```
    ...
    > ### Title: Display confusion or classification matrix following logistic
    > ###   regression using glm with binomial family.
    > ### Aliases: confusion_stat
    > ### Keywords: models
    > 
    > ### ** Examples
    > 
    > library(MASS)
    > library(LOGIT)
    > data(R84)
    > R84$cage <- R84$age - mean(R84$age)
    > R84$cdoc <- R84$docvis - mean(R84$docvis)
    > mylogit <- glm(outwork ~ cdoc + female + kids + cage + factor(edlevel),
    + family=binomial, data=R84)
    > mu <- predict(mylogit, type="response")
    > cutpoint<-ROCtest(mylogit, fold=10, type="Sensitivity")$cut
    > mu[mu>=cutpoint]<-1
    > mu[mu<cutpoint]<-0
    > confusion_stat(mu, R84$outwork)
    Error: `data` and `reference` should be factors with the same levels.
    Execution halted
    ```

# lolR

Version: 2.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘ggplot2’
      All declared Imports should be used.
    ```

# loose.rock

Version: 1.0.9

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘futile.options’ ‘ggfortify’ ‘grDevices’ ‘stats’
      All declared Imports should be used.
    ```

# LPWC

Version: 0.99.3

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘ggplot2’ ‘knitr’
      All declared Imports should be used.
    ```

# lslx

Version: 0.6.3

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘ggplot2’ ‘lavaan’
      All declared Imports should be used.
    ```

# LymphoSeq

Version: 1.8.0

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  8.2Mb
      sub-directories of 1Mb or more:
        doc       2.5Mb
        extdata   5.5Mb
    ```

# m2b

Version: 1.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘e1071’ ‘utils’
      All declared Imports should be used.
    ```

# M3C

Version: 1.2.0

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  7.6Mb
      sub-directories of 1Mb or more:
        doc   7.1Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    M3Cdendcomputations: no visible global function definition for
      ‘dev.off’
    M3Creal: no visible global function definition for ‘colorRampPalette’
    M3Creal: no visible global function definition for ‘png’
    M3Creal: no visible global function definition for ‘dev.off’
    clustersim: no visible binding for global variable ‘X3’
    clustersim: no visible binding for global variable ‘PC1’
    clustersim: no visible binding for global variable ‘PC2’
    clustersim: no visible global function definition for ‘png’
    clustersim: no visible global function definition for ‘dev.off’
    myPal: no visible global function definition for ‘rgb’
    pca: no visible binding for global variable ‘PC1’
    pca: no visible binding for global variable ‘PC2’
    pca: no visible global function definition for ‘png’
    pca: no visible global function definition for ‘dev.off’
    Undefined global functions or variables:
      K PAC_SCORE PC1 PC2 P_SCORE RCSI X3 colorRampPalette consensusindex
      dev.off i k png rgb
    Consider adding
      importFrom("grDevices", "colorRampPalette", "dev.off", "png", "rgb")
    to your NAMESPACE file.
    ```

*   checking for unstated dependencies in vignettes ... NOTE
    ```
    '::' or ':::' imports not declared from:
      ‘ggsci’ ‘gplots’
    'library' or 'require' calls not declared from:
      ‘ggsci’ ‘gplots’
    ```

# MAc

Version: 1.1.1

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘R2wd’
    
    Packages which this enhances but not available for checking:
      ‘compute.es’ ‘irr’
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' calls in package code:
      ‘R2wd’ ‘ggplot2’ ‘metafor’
      Please use :: or requireNamespace() instead.
      See section 'Suggested packages' in the 'Writing R Extensions' manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    wd.default: no visible global function definition for ‘wdGet’
    wd.default: no visible global function definition for ‘wdNewDoc’
    wd.default: no visible global function definition for ‘wdHeading’
    wd.default: no visible global function definition for ‘wdTable’
    wd.macat: no visible global function definition for ‘wdGet’
    wd.macat: no visible global function definition for ‘wdNewDoc’
    wd.macat: no visible global function definition for ‘wdHeading’
    wd.macat: no visible global function definition for ‘wdTable’
    wd.mareg: no visible global function definition for ‘wdGet’
    wd.mareg: no visible global function definition for ‘wdNewDoc’
    wd.mareg: no visible global function definition for ‘wdHeading’
    wd.mareg: no visible global function definition for ‘wdTable’
    wd.omni: no visible global function definition for ‘wdGet’
    wd.omni: no visible global function definition for ‘wdNewDoc’
    wd.omni: no visible global function definition for ‘wdHeading’
    wd.omni: no visible global function definition for ‘wdTable’
    Undefined global functions or variables:
      aes facet_wrap g geom_boxplot geom_errorbarh geom_jitter geom_point
      geom_smooth geom_vline ggplot id l.ci95 opts r rma scale_y_continuous
      se.z se.z.tau stat_abline stat_summary u.ci95 var.g wdGet wdHeading
      wdNewDoc wdTable wi wi.tau xlab xlim ylab ylim z
    ```

# MACPET

Version: 1.0.0

## In both

*   checking whether package ‘MACPET’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘DelayedArray’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/MACPET/new/MACPET.Rcheck/00install.out’ for details.
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Warning in has_utility("pdfcrop") :
      pdfcrop not installed or not in PATH
    tlmgr search --file --global '/beramono.sty'
    Unknown directive ...l
    Warning in system2("tlmgr", args, ...) :
      running command ''tlmgr' search --file --global '/beramono.sty'' had status 25
    ! LaTeX Error: File `beramono.sty' not found.
    
    ! Emergency stop.
    <read *> 
    
    Error: processing vignette 'MACPET.Rmd' failed with diagnostics:
    Failed to compile MACPET.tex. See MACPET.log for more info.
    Execution halted
    ```

# MAd

Version: 0.8-2.1

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘R2wd’
    
    Packages which this enhances but not available for checking:
      ‘compute.es’ ‘irr’
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' calls in package code:
      ‘R2wd’ ‘ggplot2’ ‘metafor’
      Please use :: or requireNamespace() instead.
      See section 'Suggested packages' in the 'Writing R Extensions' manual.
    ```

# mafs

Version: 0.0.3

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘Rcpp’ ‘cmprsk’ ‘colorspace’ ‘etm’ ‘fracdiff’ ‘gtable’ ‘munsell’
      ‘numDeriv’ ‘plyr’ ‘quadprog’ ‘scales’ ‘timeDate’ ‘tseries’ ‘zoo’
      All declared Imports should be used.
    ```

# maftools

Version: 1.6.15

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 12.0Mb
      sub-directories of 1Mb or more:
        doc       6.0Mb
        extdata   5.1Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      assembly_version bg boxStat cf chromosome chromosome_end
      chromosome_start ci.low ci.up clusters cohort con con.class
      consequence_type conv count count2 cytoband distance domain_lenght
      downstream dp endDist ens_id fdr fisher_pvalue fract
      fract_muts_in_clusters fraction fraction_APOBEC_mutations fs g1_muts
      g1_title g1_tot g2_muts g2_title g2_tot gene gene1 gene2
      gene_affected gg.fp hgnc_symbol i.End_Position i.Start_Position
      icgc_sample_id id idx lab labPos labThis label loc log10OR
      log10OR_high log10OR_low minDiff mut.load.p mutated_from_allele
      mutated_to_allele mutations muts_in_clusters nGenes nMut nMuts
      nSamples nVars n_A n_C n_C>G_and_C>T n_G n_T n_mutated_Feature
      n_mutated_Feature1 n_mutated_Feature2 n_mutated_group1
      n_mutated_group2 n_mutations nonApobec non_APOBEC_mutations numMat or
      pValue p_value pair peakID pfam poissonFdr pos2 posRounded protein.ID
      pval qvalues reference_genome_allele refseq.ID sampleId
      sequencing_strategy significant site startDist statFontSize survLower
      survProb survUp tCw tCw_to_A tCw_to_G tCw_to_G+tCw_to_T tCw_to_T tFdr
      t_alt_count t_ref_count t_vaf tcw th tot total trinucleotide uid
      updown upstream value variable variantId verification_platform
      verification_status wGa wGa_to_A wGa_to_C wGa_to_T wga ybig ysmall
      ystart
    ```

*   checking for unstated dependencies in vignettes ... NOTE
    ```
    '::' or ':::' import not declared from: ‘pheatmap’
    'library' or 'require' call not declared from: ‘pheatmap’
    ```

# MAGeCKFlute

Version: 1.0.1

## In both

*   R CMD check timed out
    

*   checking whether package ‘MAGeCKFlute’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘IRanges’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/MAGeCKFlute/new/MAGeCKFlute.Rcheck/00install.out’ for details.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  9.7Mb
      sub-directories of 1Mb or more:
        doc       4.1Mb
        extdata   4.7Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    RankView: no visible binding for global variable ‘Gene’
    ScatterView: no visible binding for global variable ‘Control’
    ScatterView: no visible binding for global variable ‘Treatment’
    ScatterView: no visible binding for global variable ‘group’
    Selector: no visible binding for global variable ‘GeneNum’
    Selector: no visible binding for global variable ‘x’
    Selector: no visible binding for global variable ‘y’
    SquareView: no visible binding for global variable ‘Control’
    SquareView: no visible binding for global variable ‘Treatment’
    SquareView: no visible binding for global variable ‘group’
    SquareView: no visible binding for global variable ‘Gene’
    TransGeneID: no visible binding for global variable ‘bods’
    ViolinView: no visible binding for global variable ‘variable’
    ViolinView: no visible binding for global variable ‘value’
    enrich.DAVID: no visible global function definition for ‘qvalue’
    getOrg: no visible binding for global variable ‘bods’
    Undefined global functions or variables:
      ..density.. Control Count Diff Gene GeneNum NES Name PC1 PC2 Rank
      Treatment Zuber_Essential bods category count group keggConv logP
      parseKGML2Graph2 percent pos qvalue r rn.list shape value variable x
      y
    ```

# magick

Version: 2.0

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  7.9Mb
      sub-directories of 1Mb or more:
        doc   6.8Mb
    ```

# malariaAtlas

Version: 0.0.2

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘grid’
      All declared Imports should be used.
    ```

# manhattanly

Version: 0.2.0

## In both

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘qqman’
    ```

# mapr

Version: 0.4.0

## In both

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 164 marked UTF-8 strings
    ```

# margins

Version: 0.3.23

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    margins(m)
    
    Number of messages:
    message 
          6 
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'TechnicalDetails.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `setspace.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.56 \usepackage
                    {booktabs}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Packages which this enhances but not available for checking:
      ‘AER’ ‘betareg’ ‘ordinal’
    ```

*   checking Rd cross-references ... NOTE
    ```
    Packages unavailable to check Rd xrefs: ‘betareg’, ‘AER’
    ```

# marked

Version: 1.2.1

## In both

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    
    probitCJS MCMC beginning...
    p model = ~time + sex
    phi model = ~Flood
    MCMC complete, processing output...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'markedVignette.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `psfrag.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.24 ...kage{epsfig}\usepackage{psfrag}\usepackage
                                                      {lineno}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# MARSS

Version: 3.10.8

## In both

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'EMDerivation.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.30 \ifthenelse
                    {\boolean{Sweave@inconsolata}}{%^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# MAST

Version: 1.6.1

## In both

*   checking whether package ‘MAST’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘DelayedArray’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/MAST/new/MAST.Rcheck/00install.out’ for details.
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  9.4Mb
      sub-directories of 1Mb or more:
        R      1.0Mb
        data   4.4Mb
        doc    3.7Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    assay_idx: no visible global function definition for ‘assayNames’
    collectResiduals: no visible global function definition for
      ‘assayNames’
    collectResiduals: no visible global function definition for ‘assay<-’
    collectResiduals: no visible global function definition for
      ‘assayNames<-’
    primerAverage: no visible global function definition for ‘assay<-’
    primerAverage: no visible global function definition for ‘rowData<-’
    assay<-,SingleCellAssay-missing: no visible global function definition
      for ‘assay<-’
    assayNames<-,SingleCellAssay-character: no visible global function
      definition for ‘assayNames’
    assayNames<-,SingleCellAssay-character: no visible global function
      definition for ‘assays<-’
    exprs<-,SingleCellAssay-ANY: no visible global function definition for
      ‘assay<-’
    Undefined global functions or variables:
      assay<- assayNames assayNames<- assays<- rowData<-
    ```

# matchingMarkets

Version: 0.3-6

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    0%   10   20   30   40   50   60   70   80   90   100%
    [----|----|----|----|----|----|----|----|----|----|
    **************************************************|
    Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
    Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'matching.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.55 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.2Mb
      sub-directories of 1Mb or more:
        java   3.3Mb
    ```

# MCbiclust

Version: 1.4.0

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  9.9Mb
      sub-directories of 1Mb or more:
        data   3.1Mb
        doc    5.9Mb
    ```

# MCMC.OTU

Version: 1.0.10

## In both

*   checking examples ... WARNING
    ```
    Found the following significant warnings:
    
      Warning: 'cBind' is deprecated.
    Deprecated functions may be defunct as soon as of the next release of
    R.
    See ?Deprecated.
    ```

# MCMC.qpcr

Version: 1.2.3

## In both

*   checking examples ... WARNING
    ```
    Found the following significant warnings:
    
      Warning: 'cBind' is deprecated.
    Deprecated functions may be defunct as soon as of the next release of
    R.
    See ?Deprecated.
    ```

# mcMST

Version: 1.0.1

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘parallelMap’ ‘reshape2’
      All declared Imports should be used.
    ```

# mCSEA

Version: 1.0.1

## In both

*   checking whether package ‘mCSEA’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘GenomicFeatures’ was built under R version 3.5.1
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/mCSEA/new/mCSEA.Rcheck/00install.out’ for details.
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
        exprTest
    
    Warning in fgsea::fgsea(genes, rank, minSize = minCpGs, nperm = nperm, nproc = nproc) :
      There are ties in the preranked stats (7.59% of the list).
    The order of those tied genes will be arbitrary, which may produce unexpected results.
    Warning in fgsea::fgsea(genes, rank, minSize = minCpGs, nperm = nperm, nproc = nproc) :
      There are duplicate gene names, fgsea may produce unexpected results
    Warning in has_utility("pdfcrop") :
      pdfcrop not installed or not in PATH
    tlmgr search --file --global '/beramono.sty'
    Unknown directive ...l
    Warning in system2("tlmgr", args, ...) :
      running command ''tlmgr' search --file --global '/beramono.sty'' had status 25
    ! LaTeX Error: File `beramono.sty' not found.
    
    ! Emergency stop.
    <read *> 
    
    Error: processing vignette 'mCSEA.Rmd' failed with diagnostics:
    Failed to compile mCSEA.tex. See mCSEA.log for more info.
    Execution halted
    ```

# mdpeer

Version: 1.0.1

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘glmnet’
      All declared Imports should be used.
    ```

# mdsr

Version: 0.1.6

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.6Mb
      sub-directories of 1Mb or more:
        data   5.5Mb
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 2694 marked UTF-8 strings
    ```

# MEAL

Version: 1.10.1

## In both

*   R CMD check timed out
    

*   checking for missing documentation entries ... WARNING
    ```
    Undocumented code objects:
      'DAPipeline' 'DAProbe' 'DARegion' 'RDAset' 'filterSet'
      'plotBestFeatures' 'preparePhenotype'
    All user-level objects in a package should have documentation entries.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.9Mb
      sub-directories of 1Mb or more:
        doc   5.6Mb
    ```

*   checking top-level files ... NOTE
    ```
    File
      LICENSE
    is not mentioned in the DESCRIPTION file.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Missing object imported by a ':::' call: 'MultiDataSet:::nrows'
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      'betas'
    correlationMethSNPs: no visible binding for global variable 'num_cores'
    plotRegion: no visible global function definition for 'data'
    plotRegion: no visible binding for global variable 'dmrcatedata'
    plotRegion: no visible binding for global variable 'tx.hg19'
    plotRegion: no visible binding for global variable 'tx.hg38'
    plotRegion: no visible binding for global variable 'tx.mm10'
    plotRegion: no visible global function definition for
      'subsetByOverlaps'
    plotRegion: no visible global function definition for 'mcols<-'
    runBlockFinder: no visible global function definition for 'assays'
    runBlockFinder: no visible global function definition for 'colData'
    runRDA: no visible global function definition for 'rowData'
    runSVA: no visible global function definition for 'resid'
    Undefined global functions or variables:
      assays betas colData data dmrcatedata mcols<- num_cores resid rowData
      subsetByOverlaps tx.hg19 tx.hg38 tx.mm10
    Consider adding
      importFrom("stats", "resid")
      importFrom("utils", "data")
    to your NAMESPACE file.
    ```

# medmod

Version: 1.0.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘R6’ ‘ggplot2’ ‘lavaan’
      All declared Imports should be used.
    ```

# memapp

Version: 2.10

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘DT’ ‘RColorBrewer’ ‘RODBC’ ‘dplyr’ ‘foreign’ ‘formattable’ ‘ggplot2’
      ‘haven’ ‘magrittr’ ‘mem’ ‘openxlsx’ ‘plotly’ ‘readxl’ ‘shinyBS’
      ‘shinydashboard’ ‘shinyjs’ ‘shinythemes’ ‘stringi’ ‘stringr’ ‘tidyr’
      All declared Imports should be used.
    ```

# memery

Version: 0.5.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘colourpicker’ ‘shinyBS’ ‘shinycssloaders’
      All declared Imports should be used.
    ```

# memor

Version: 0.1.1

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    tlmgr search --file --global '/xltxtra.sty'
    Unknown directive ...l
    Warning in system2("tlmgr", args, ...) :
      running command ''tlmgr' search --file --global '/xltxtra.sty'' had status 25
    ! LaTeX Error: File `xltxtra.sty' not found.
    
    ! Emergency stop.
    <read *> 
    
    Error: processing vignette 'memor.Rmd' failed with diagnostics:
    Failed to compile memor.tex. See memor.log for more info.
    Execution halted
    ```

# MergeGUI

Version: 0.2-1

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: ‘gWidgetsRGtk2’ ‘cairoDevice’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# messina

Version: 1.16.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
        'browseVignettes()'. To cite Bioconductor, see
        'citation("Biobase")', and for packages 'citation("pkgname")'.
    
    Warning in has_utility("pdfcrop") :
      pdfcrop not installed or not in PATH
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'messina.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `beramono.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.87 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Title field: should not end in a period.
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' call to ‘Biobase’ in package code.
      Please use :: or requireNamespace() instead.
      See section 'Suggested packages' in the 'Writing R Extensions' manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      ‘quantile’
    messinaSurvKMplot : <anonymous>: no visible binding for global variable
      ‘sd’
    messinaSurvKMplotSingleGroup: no visible binding for global variable
      ‘median’
    messinaSurvKMplotSingleGroup: no visible binding for global variable
      ‘quantile’
    messinaSurvKMplotSingleGroup: no visible binding for global variable
      ‘sd’
    messinaSurvObjPlot: no visible global function definition for
      ‘quantile’
    messinaSurvObjectiveFunc: no visible global function definition for
      ‘coef’
    messinaSurvPlot: no visible global function definition for ‘str’
    Undefined global functions or variables:
      approx coef median quantile reorder sd str
    Consider adding
      importFrom("stats", "approx", "coef", "median", "quantile", "reorder",
                 "sd")
      importFrom("utils", "str")
    to your NAMESPACE file.
    ```

# metabomxtr

Version: 1.14.0

## In both

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    
        Vignettes contain introductory material; view with
        'browseVignettes()'. To cite Bioconductor, see
        'citation("Biobase")', and for packages 'citation("pkgname")'.
    
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'Metabomxtr_Vignette.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.30 \ifthenelse
                    {\boolean{Sweave@inconsolata}}{%^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# metacoder

Version: 0.3.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘ggrepel’ ‘reshape’ ‘svglite’
      All declared Imports should be used.
    ```

# MetaCyto

Version: 1.2.1

## In both

*   checking R code for possible problems ... NOTE
    ```
    collectData: no visible binding for global variable ‘parameter_name’
    collectData: no visible binding for global variable ‘value’
    panelSummary: no visible binding for global variable ‘antibodies’
    panelSummary: no visible binding for global variable ‘value’
    plotGA: no visible binding for global variable ‘lower’
    plotGA: no visible binding for global variable ‘upper’
    searchCluster : <anonymous>: no visible binding for global variable
      ‘triS’
    Undefined global functions or variables:
      antibodies lower parameter_name triS upper value
    ```

# metagen

Version: 1.0

## In both

*   checking R code for possible problems ... NOTE
    ```
    ...
    pivotalStream: no visible global function definition for ‘rchisq’
    pivotalStream : <anonymous>: no visible global function definition for
      ‘rchisq’
    pivotalStream: no visible global function definition for ‘rnorm’
    rBinomGauss: no visible global function definition for ‘rnorm’
    rBinomGauss: no visible global function definition for ‘rbinom’
    rD: no visible global function definition for ‘rchisq’
    rY: no visible global function definition for ‘rnorm’
    render: no visible global function definition for ‘pdf’
    render: no visible global function definition for ‘dev.off’
    renderSVG: no visible global function definition for ‘svg’
    renderSVG: no visible global function definition for ‘dev.off’
    Undefined global functions or variables:
      data dev.off pdf qchisq qnorm qt quantile rbinom rchisq rnorm svg
      uniroot var
    Consider adding
      importFrom("grDevices", "dev.off", "pdf", "svg")
      importFrom("stats", "qchisq", "qnorm", "qt", "quantile", "rbinom",
                 "rchisq", "rnorm", "uniroot", "var")
      importFrom("utils", "data")
    to your NAMESPACE file.
    ```

# metagene

Version: 2.12.2

## In both

*   checking whether package ‘metagene’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package 'GenomicRanges' was built under R version 3.5.1
      Warning: package 'IRanges' was built under R version 3.5.1
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/metagene/new/metagene.Rcheck/00install.out’ for details.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 10.1Mb
      sub-directories of 1Mb or more:
        doc       3.2Mb
        extdata   5.4Mb
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Package listed in more than one of Depends, Imports, Suggests, Enhances:
      ‘similaRpeak’
    A package should be listed in only one of these fields.
    ```

*   checking R code for possible problems ... NOTE
    ```
    avoid_gaps_update: no visible binding for global variable 'value'
    avoid_gaps_update: no visible binding for global variable 'bam'
    avoid_gaps_update: no visible binding for global variable 'tab'
    avoid_gaps_update: no visible binding for global variable 'nuc'
    permutation_test: no visible global function definition for '.'
    permutation_test: no visible binding for global variable 'value'
    plot_metagene: no visible binding for global variable 'bin'
    plot_metagene: no visible binding for global variable 'value'
    plot_metagene: no visible binding for global variable 'qinf'
    plot_metagene: no visible binding for global variable 'qsup'
    plot_metagene: no visible binding for global variable 'group'
    plot_metagene: no visible binding for global variable 'nuc'
    plot_metagene: no visible binding for global variable 'design'
    plot_metagene: no visible binding for global variable 'nuctot'
    Undefined global functions or variables:
      . bam bin design group nuc nuctot qinf qsup tab value
    ```

# MetaIntegrator

Version: 2.0.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘DT’ ‘GEOmetadb’ ‘RMySQL’ ‘RSQLite’ ‘gplots’ ‘pheatmap’ ‘readr’
      All declared Imports should be used.
    ```

# MetamapsDB

Version: 0.0.2

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘Matrix’ ‘shiny’
      All declared Imports should be used.
    ```

# metaMix

Version: 0.2

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘Rmpi’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# MetaStan

Version: 0.0.1

## In both

*   checking whether package ‘MetaStan’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/MetaStan/new/MetaStan.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘MetaStan’ ...
** package ‘MetaStan’ successfully unpacked and MD5 sums checked
** libs
"/Library/Frameworks/R.framework/Resources/bin/Rscript" -e "source(file.path('..', 'tools', 'make_cc.R')); make_cc(commandArgs(TRUE))" stan_files/BNHM_Higgins.stan
"/Library/Frameworks/R.framework/Resources/bin/Rscript" -e "source(file.path('..', 'tools', 'make_cc.R')); make_cc(commandArgs(TRUE))" stan_files/BNHM_Smith.stan
"/Library/Frameworks/R.framework/Resources/bin/Rscript" -e "source(file.path('..', 'tools', 'make_cc.R')); make_cc(commandArgs(TRUE))" stan_files/Beta_binomial.stan
"/Library/Frameworks/R.framework/Resources/bin/Rscript" -e "source(file.path('..', 'tools', 'make_cc.R')); make_cc(commandArgs(TRUE))" stan_files/Fix_Eff.stan
Wrote C++ file "stan_files/BNHM_Higgins.cc"
Wrote C++ file "stan_files/Beta_binomial.cc"
Wrote C++ file "stan_files/Fix_Eff.cc"
Wrote C++ file "stan_files/BNHM_Smith.cc"
Error in readRDS("/var/folders/lb/xhxqmcrd7gv302_b1pdfykh80000gn/T//Rtmp2BA6ai/file6fc23cdde308") : 
  error reading from connection
Calls: .Last -> readRDS
Execution halted
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I"../inst/include" -I"`"/Library/Frameworks/R.framework/Resources/bin/Rscript" --vanilla -e "cat(system.file('include', 'src', package = 'StanHeaders'))"`" -DBOOST_RESULT_OF_USE_TR1 -DBOOST_NO_DECLTYPE -DBOOST_DISABLE_ASSERTS -DEIGEN_NO_DEBUG -DBOOST_MATH_OVERFLOW_ERROR_POLICY=errno_on_error -I"/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/BH/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c init.cpp -o init.o
make: *** [stan_files/Fix_Eff.cc] Error 1
make: *** Waiting for unfinished jobs....
rm stan_files/BNHM_Higgins.cc stan_files/Fix_Eff.cc stan_files/Beta_binomial.cc stan_files/BNHM_Smith.cc
ERROR: compilation failed for package ‘MetaStan’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/MetaStan/new/MetaStan.Rcheck/MetaStan’

```
### CRAN

```
* installing *source* package ‘MetaStan’ ...
** package ‘MetaStan’ successfully unpacked and MD5 sums checked
** libs
"/Library/Frameworks/R.framework/Resources/bin/Rscript" -e "source(file.path('..', 'tools', 'make_cc.R')); make_cc(commandArgs(TRUE))" stan_files/BNHM_Higgins.stan
"/Library/Frameworks/R.framework/Resources/bin/Rscript" -e "source(file.path('..', 'tools', 'make_cc.R')); make_cc(commandArgs(TRUE))" stan_files/BNHM_Smith.stan
"/Library/Frameworks/R.framework/Resources/bin/Rscript" -e "source(file.path('..', 'tools', 'make_cc.R')); make_cc(commandArgs(TRUE))" stan_files/Beta_binomial.stan
"/Library/Frameworks/R.framework/Resources/bin/Rscript" -e "source(file.path('..', 'tools', 'make_cc.R')); make_cc(commandArgs(TRUE))" stan_files/Fix_Eff.stan
Wrote C++ file "stan_files/Fix_Eff.cc"
Wrote C++ file "stan_files/Beta_binomial.cc"
Wrote C++ file "stan_files/BNHM_Smith.cc"
Wrote C++ file "stan_files/BNHM_Higgins.cc"
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I"../inst/include" -I"`"/Library/Frameworks/R.framework/Resources/bin/Rscript" --vanilla -e "cat(system.file('include', 'src', package = 'StanHeaders'))"`" -DBOOST_RESULT_OF_USE_TR1 -DBOOST_NO_DECLTYPE -DBOOST_DISABLE_ASSERTS -DEIGEN_NO_DEBUG -DBOOST_MATH_OVERFLOW_ERROR_POLICY=errno_on_error -I"/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/BH/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c init.cpp -o init.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I"../inst/include" -I"`"/Library/Frameworks/R.framework/Resources/bin/Rscript" --vanilla -e "cat(system.file('include', 'src', package = 'StanHeaders'))"`" -DBOOST_RESULT_OF_USE_TR1 -DBOOST_NO_DECLTYPE -DBOOST_DISABLE_ASSERTS -DEIGEN_NO_DEBUG -DBOOST_MATH_OVERFLOW_ERROR_POLICY=errno_on_error -I"/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/BH/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c stan_files/Beta_binomial.cc -o stan_files/Beta_binomial.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I"../inst/include" -I"`"/Library/Frameworks/R.framework/Resources/bin/Rscript" --vanilla -e "cat(system.file('include', 'src', package = 'StanHeaders'))"`" -DBOOST_RESULT_OF_USE_TR1 -DBOOST_NO_DECLTYPE -DBOOST_DISABLE_ASSERTS -DEIGEN_NO_DEBUG -DBOOST_MATH_OVERFLOW_ERROR_POLICY=errno_on_error -I"/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/BH/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c stan_files/Fix_Eff.cc -o stan_files/Fix_Eff.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I"../inst/include" -I"`"/Library/Frameworks/R.framework/Resources/bin/Rscript" --vanilla -e "cat(system.file('include', 'src', package = 'StanHeaders'))"`" -DBOOST_RESULT_OF_USE_TR1 -DBOOST_NO_DECLTYPE -DBOOST_DISABLE_ASSERTS -DEIGEN_NO_DEBUG -DBOOST_MATH_OVERFLOW_ERROR_POLICY=errno_on_error -I"/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/BH/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c stan_files/BNHM_Higgins.cc -o stan_files/BNHM_Higgins.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I"../inst/include" -I"`"/Library/Frameworks/R.framework/Resources/bin/Rscript" --vanilla -e "cat(system.file('include', 'src', package = 'StanHeaders'))"`" -DBOOST_RESULT_OF_USE_TR1 -DBOOST_NO_DECLTYPE -DBOOST_DISABLE_ASSERTS -DEIGEN_NO_DEBUG -DBOOST_MATH_OVERFLOW_ERROR_POLICY=errno_on_error -I"/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/BH/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c stan_files/BNHM_Smith.cc -o stan_files/BNHM_Smith.o
In file included from stan_files/Fix_Eff.cc:3:
In file included from stan_files/Fix_Eff.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:20:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_args.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/BH/include/boost/lexical_cast.hpp:32:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/BH/include/boost/lexical_cast/try_lexical_convert.hpp:42:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/BH/include/boost/lexical_cast/detail/converter_lexical.hpp:52:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/BH/include/boost/container/container_fwd.hpp:61:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/BH/include/boost/container/detail/std_fwd.hpp:27:1: warning: inline namespaces are a C++11 feature [-Wc++11-inline-namespace]
BOOST_MOVE_STD_NS_BEG
^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/BH/include/boost/move/detail/std_ns_begin.hpp:18:34: note: expanded from macro 'BOOST_MOVE_STD_NS_BEG'
   #define BOOST_MOVE_STD_NS_BEG _LIBCPP_BEGIN_NAMESPACE_STD
                                 ^
/usr/local/clang4/bin/../include/c++/v1/__config:388:52: note: expanded from macro '_LIBCPP_BEGIN_NAMESPACE_STD'
#define _LIBCPP_BEGIN_NAMESPACE_STD namespace std {inline namespace _LIBCPP_NAMESPACE {
                                                   ^
In file included from stan_files/Beta_binomial.cc:3:
In file included from stan_files/Beta_binomial.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:20:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_args.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/BH/include/boost/lexical_cast.hpp:32:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/BH/include/boost/lexical_cast/try_lexical_convert.hpp:42:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/BH/include/boost/lexical_cast/detail/converter_lexical.hpp:52:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/BH/include/boost/container/container_fwd.hpp:61:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/BH/include/boost/container/detail/std_fwd.hpp:27:1: warning: inline namespaces are a C++11 feature [-Wc++11-inline-namespace]
BOOST_MOVE_STD_NS_BEG
^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/BH/include/boost/move/detail/std_ns_begin.hpp:18:34: note: expanded from macro 'BOOST_MOVE_STD_NS_BEG'
   #define BOOST_MOVE_STD_NS_BEG _LIBCPP_BEGIN_NAMESPACE_STD
                                 ^
/usr/local/clang4/bin/../include/c++/v1/__config:388:52: note: expanded from macro '_LIBCPP_BEGIN_NAMESPACE_STD'
#define _LIBCPP_BEGIN_NAMESPACE_STD namespace std {inline namespace _LIBCPP_NAMESPACE {
                                                   ^
In file included from stan_files/BNHM_Higgins.cc:3:
In file included from stan_files/BNHM_Higgins.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:20:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_args.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/BH/include/boost/lexical_cast.hpp:32:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/BH/include/boost/lexical_cast/try_lexical_convert.hpp:42:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/BH/include/boost/lexical_cast/detail/converter_lexical.hpp:52:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/BH/include/boost/container/container_fwd.hpp:61:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/BH/include/boost/container/detail/std_fwd.hpp:27:1: warning: inline namespaces are a C++11 feature [-Wc++11-inline-namespace]
BOOST_MOVE_STD_NS_BEG
^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/BH/include/boost/move/detail/std_ns_begin.hpp:18:34: note: expanded from macro 'BOOST_MOVE_STD_NS_BEG'
   #define BOOST_MOVE_STD_NS_BEG _LIBCPP_BEGIN_NAMESPACE_STD
                                 ^
/usr/local/clang4/bin/../include/c++/v1/__config:388:52: note: expanded from macro '_LIBCPP_BEGIN_NAMESPACE_STD'
#define _LIBCPP_BEGIN_NAMESPACE_STD namespace std {inline namespace _LIBCPP_NAMESPACE {
                                                   ^
In file included from stan_files/Fix_Eff.cc:3:
In file included from stan_files/Fix_Eff.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/autodiffstackstorage.hpp:4:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/memory/stack_alloc.hpp:145:10: warning: 'auto' type specifier is a C++11 extension [-Wc++11-extensions]
    for (auto& block : blocks_)
         ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/memory/stack_alloc.hpp:145:22: warning: range-based for loop is a C++11 extension [-Wc++11-extensions]
    for (auto& block : blocks_)
                     ^
In file included from stan_files/Fix_Eff.cc:3:
In file included from stan_files/Fix_Eff.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core.hpp:4:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/autodiffstackstorage.hpp:47:69: warning: deleted function definitions are a C++11 extension [-Wc++11-extensions]
    AutodiffStackStorage &operator=(const AutodiffStackStorage &) = delete;
                                                                    ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/autodiffstackstorage.hpp:60:30: warning: deleted function definitions are a C++11 extension [-Wc++11-extensions]
  AutodiffStackSingleton() = delete;
                             ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/autodiffstackstorage.hpp:61:71: warning: deleted function definitions are a C++11 extension [-Wc++11-extensions]
  explicit AutodiffStackSingleton(AutodiffStackSingleton_t const &) = delete;
                                                                      ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/autodiffstackstorage.hpp:62:73: warning: deleted function definitions are a C++11 extension [-Wc++11-extensions]
  AutodiffStackSingleton &operator=(const AutodiffStackSingleton_t &) = delete;
                                                                        ^
In file included from stan_files/Beta_binomial.cc:3:
In file included from stan_files/Beta_binomial.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/autodiffstackstorage.hpp:4:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/memory/stack_alloc.hpp:145:10: warning: 'auto' type specifier is a C++11 extension [-Wc++11-extensions]
    for (auto& block : blocks_)
         ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/memory/stack_alloc.hpp:145:22: warning: range-based for loop is a C++11 extension [-Wc++11-extensions]
    for (auto& block : blocks_)
                     ^
In file included from stan_files/Beta_binomial.cc:3:
In file included from stan_files/Beta_binomial.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core.hpp:4:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/autodiffstackstorage.hpp:47:69: warning: deleted function definitions are a C++11 extension [-Wc++11-extensions]
    AutodiffStackStorage &operator=(const AutodiffStackStorage &) = delete;
                                                                    ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/autodiffstackstorage.hpp:60:30: warning: deleted function definitions are a C++11 extension [-Wc++11-extensions]
  AutodiffStackSingleton() = delete;
                             ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/autodiffstackstorage.hpp:61:71: warning: deleted function definitions are a C++11 extension [-Wc++11-extensions]
  explicit AutodiffStackSingleton(AutodiffStackSingleton_t const &) = delete;
                                                                      ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/autodiffstackstorage.hpp:62:73: warning: deleted function definitions are a C++11 extension [-Wc++11-extensions]
  AutodiffStackSingleton &operator=(const AutodiffStackSingleton_t &) = delete;
                                                                        ^
In file included from stan_files/BNHM_Higgins.cc:3:
In file included from stan_files/BNHM_Higgins.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/autodiffstackstorage.hpp:4:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/memory/stack_alloc.hpp:145:10: warning: 'auto' type specifier is a C++11 extension [-Wc++11-extensions]
    for (auto& block : blocks_)
         ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/memory/stack_alloc.hpp:145:22: warning: range-based for loop is a C++11 extension [-Wc++11-extensions]
    for (auto& block : blocks_)
                     ^
In file included from stan_files/BNHM_Higgins.cc:3:
In file included from stan_files/BNHM_Higgins.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core.hpp:4:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/autodiffstackstorage.hpp:47:69: warning: deleted function definitions are a C++11 extension [-Wc++11-extensions]
    AutodiffStackStorage &operator=(const AutodiffStackStorage &) = delete;
                                                                    ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/autodiffstackstorage.hpp:60:30: warning: deleted function definitions are a C++11 extension [-Wc++11-extensions]
  AutodiffStackSingleton() = delete;
                             ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/autodiffstackstorage.hpp:61:71: warning: deleted function definitions are a C++11 extension [-Wc++11-extensions]
  explicit AutodiffStackSingleton(AutodiffStackSingleton_t const &) = delete;
                                                                      ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/autodiffstackstorage.hpp:62:73: warning: deleted function definitions are a C++11 extension [-Wc++11-extensions]
  AutodiffStackSingleton &operator=(const AutodiffStackSingleton_t &) = delete;
                                                                        ^
In file included from stan_files/BNHM_Smith.cc:3:
In file included from stan_files/BNHM_Smith.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:20:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_args.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/BH/include/boost/lexical_cast.hpp:32:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/BH/include/boost/lexical_cast/try_lexical_convert.hpp:42:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/BH/include/boost/lexical_cast/detail/converter_lexical.hpp:52:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/BH/include/boost/container/container_fwd.hpp:61:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/BH/include/boost/container/detail/std_fwd.hpp:27:1: warning: inline namespaces are a C++11 feature [-Wc++11-inline-namespace]
BOOST_MOVE_STD_NS_BEG
^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/BH/include/boost/move/detail/std_ns_begin.hpp:18:34: note: expanded from macro 'BOOST_MOVE_STD_NS_BEG'
   #define BOOST_MOVE_STD_NS_BEG _LIBCPP_BEGIN_NAMESPACE_STD
                                 ^
/usr/local/clang4/bin/../include/c++/v1/__config:388:52: note: expanded from macro '_LIBCPP_BEGIN_NAMESPACE_STD'
#define _LIBCPP_BEGIN_NAMESPACE_STD namespace std {inline namespace _LIBCPP_NAMESPACE {
                                                   ^
In file included from stan_files/BNHM_Smith.cc:3:
In file included from stan_files/BNHM_Smith.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/autodiffstackstorage.hpp:4:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/memory/stack_alloc.hpp:145:10: warning: 'auto' type specifier is a C++11 extension [-Wc++11-extensions]
    for (auto& block : blocks_)
         ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/memory/stack_alloc.hpp:145:22: warning: range-based for loop is a C++11 extension [-Wc++11-extensions]
    for (auto& block : blocks_)
                     ^
In file included from stan_files/BNHM_Smith.cc:3:
In file included from stan_files/BNHM_Smith.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core.hpp:4:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/autodiffstackstorage.hpp:47:69: warning: deleted function definitions are a C++11 extension [-Wc++11-extensions]
    AutodiffStackStorage &operator=(const AutodiffStackStorage &) = delete;
                                                                    ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/autodiffstackstorage.hpp:60:30: warning: deleted function definitions are a C++11 extension [-Wc++11-extensions]
  AutodiffStackSingleton() = delete;
                             ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/autodiffstackstorage.hpp:61:71: warning: deleted function definitions are a C++11 extension [-Wc++11-extensions]
  explicit AutodiffStackSingleton(AutodiffStackSingleton_t const &) = delete;
                                                                      ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/autodiffstackstorage.hpp:62:73: warning: deleted function definitions are a C++11 extension [-Wc++11-extensions]
  AutodiffStackSingleton &operator=(const AutodiffStackSingleton_t &) = delete;
                                                                        ^
In file included from stan_files/Fix_Eff.cc:3:
In file included from stan_files/Fix_Eff.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Dense:1:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Core:531:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/Beta_binomial.cc:3:
In file included from stan_files/Beta_binomial.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Dense:1:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Core:531:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/Fix_Eff.cc:3:
In file included from stan_files/Fix_Eff.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Dense:2:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/LU:47:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/Fix_Eff.cc:3:
In file included from stan_files/Fix_Eff.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Dense:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Cholesky:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Jacobi:29:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/BNHM_Higgins.cc:3:
In file included from stan_files/BNHM_Higgins.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Dense:1:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Core:531:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/Fix_Eff.cc:3:
In file included from stan_files/Fix_Eff.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Dense:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Cholesky:43:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/Beta_binomial.cc:3:
In file included from stan_files/Beta_binomial.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Dense:2:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/LU:47:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/Fix_Eff.cc:3:
In file included from stan_files/Fix_Eff.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Dense:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/QR:17:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Householder:27:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/Beta_binomial.cc:3:
In file included from stan_files/Beta_binomial.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Dense:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Cholesky:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Jacobi:29:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/BNHM_Higgins.cc:3:
In file included from stan_files/BNHM_Higgins.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Dense:2:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/LU:47:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/BNHM_Higgins.cc:3:
In file included from stan_files/BNHM_Higgins.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Dense:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Cholesky:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Jacobi:29:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/Beta_binomial.cc:3:
In file included from stan_files/Beta_binomial.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Dense:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Cholesky:43:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/Beta_binomial.cc:3:
In file included from stan_files/Beta_binomial.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Dense:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/QR:17:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Householder:27:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/BNHM_Higgins.cc:3:
In file included from stan_files/BNHM_Higgins.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Dense:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Cholesky:43:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/BNHM_Higgins.cc:3:
In file included from stan_files/BNHM_Higgins.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Dense:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/QR:17:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Householder:27:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/Fix_Eff.cc:3:
In file included from stan_files/Fix_Eff.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Dense:5:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/SVD:48:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/Beta_binomial.cc:3:
In file included from stan_files/Beta_binomial.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Dense:5:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/SVD:48:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/BNHM_Higgins.cc:3:
In file included from stan_files/BNHM_Higgins.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Dense:5:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/SVD:48:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/Fix_Eff.cc:3:
In file included from stan_files/Fix_Eff.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Dense:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Geometry:58:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/Beta_binomial.cc:3:
In file included from stan_files/Beta_binomial.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Dense:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Geometry:58:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/BNHM_Higgins.cc:3:
In file included from stan_files/BNHM_Higgins.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Dense:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Geometry:58:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/Fix_Eff.cc:3:
In file included from stan_files/Fix_Eff.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Dense:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Eigenvalues:58:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/Beta_binomial.cc:3:
In file included from stan_files/Beta_binomial.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Dense:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Eigenvalues:58:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/BNHM_Higgins.cc:3:
In file included from stan_files/BNHM_Higgins.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Dense:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Eigenvalues:58:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/Fix_Eff.cc:3:
In file included from stan_files/Fix_Eff.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core.hpp:42:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/recover_memory.hpp:25:8: warning: 'auto' type specifier is a C++11 extension [-Wc++11-extensions]
  for (auto &x : ChainableStack::instance().var_alloc_stack_) {
       ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/recover_memory.hpp:25:16: warning: range-based for loop is a C++11 extension [-Wc++11-extensions]
  for (auto &x : ChainableStack::instance().var_alloc_stack_) {
               ^
In file included from stan_files/Fix_Eff.cc:3:
In file included from stan_files/Fix_Eff.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core.hpp:44:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/set_zero_all_adjoints.hpp:15:8: warning: 'auto' type specifier is a C++11 extension [-Wc++11-extensions]
  for (auto &x : ChainableStack::instance().var_stack_)
       ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/set_zero_all_adjoints.hpp:15:16: warning: range-based for loop is a C++11 extension [-Wc++11-extensions]
  for (auto &x : ChainableStack::instance().var_stack_)
               ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/set_zero_all_adjoints.hpp:17:8: warning: 'auto' type specifier is a C++11 extension [-Wc++11-extensions]
  for (auto &x : ChainableStack::instance().var_nochain_stack_)
       ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/set_zero_all_adjoints.hpp:17:16: warning: range-based for loop is a C++11 extension [-Wc++11-extensions]
  for (auto &x : ChainableStack::instance().var_nochain_stack_)
               ^
In file included from stan_files/Beta_binomial.cc:3:
In file included from stan_files/Beta_binomial.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core.hpp:42:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/recover_memory.hpp:25:8: warning: 'auto' type specifier is a C++11 extension [-Wc++11-extensions]
  for (auto &x : ChainableStack::instance().var_alloc_stack_) {
       ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/recover_memory.hpp:25:16: warning: range-based for loop is a C++11 extension [-Wc++11-extensions]
  for (auto &x : ChainableStack::instance().var_alloc_stack_) {
               ^
In file included from stan_files/Beta_binomial.cc:3:
In file included from stan_files/Beta_binomial.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core.hpp:44:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/set_zero_all_adjoints.hpp:15:8: warning: 'auto' type specifier is a C++11 extension [-Wc++11-extensions]
  for (auto &x : ChainableStack::instance().var_stack_)
       ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/set_zero_all_adjoints.hpp:15:16: warning: range-based for loop is a C++11 extension [-Wc++11-extensions]
  for (auto &x : ChainableStack::instance().var_stack_)
               ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/set_zero_all_adjoints.hpp:17:8: warning: 'auto' type specifier is a C++11 extension [-Wc++11-extensions]
  for (auto &x : ChainableStack::instance().var_nochain_stack_)
       ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/set_zero_all_adjoints.hpp:17:16: warning: range-based for loop is a C++11 extension [-Wc++11-extensions]
  for (auto &x : ChainableStack::instance().var_nochain_stack_)
               ^
In file included from stan_files/BNHM_Smith.cc:3:
In file included from stan_files/BNHM_Smith.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Dense:1:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Core:531:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/BNHM_Higgins.cc:3:
In file included from stan_files/BNHM_Higgins.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core.hpp:42:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/recover_memory.hpp:25:8: warning: 'auto' type specifier is a C++11 extension [-Wc++11-extensions]
  for (auto &x : ChainableStack::instance().var_alloc_stack_) {
       ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/recover_memory.hpp:25:16: warning: range-based for loop is a C++11 extension [-Wc++11-extensions]
  for (auto &x : ChainableStack::instance().var_alloc_stack_) {
               ^
In file included from stan_files/BNHM_Higgins.cc:3:
In file included from stan_files/BNHM_Higgins.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core.hpp:44:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/set_zero_all_adjoints.hpp:15:8: warning: 'auto' type specifier is a C++11 extension [-Wc++11-extensions]
  for (auto &x : ChainableStack::instance().var_stack_)
       ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/set_zero_all_adjoints.hpp:15:16: warning: range-based for loop is a C++11 extension [-Wc++11-extensions]
  for (auto &x : ChainableStack::instance().var_stack_)
               ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/set_zero_all_adjoints.hpp:17:8: warning: 'auto' type specifier is a C++11 extension [-Wc++11-extensions]
  for (auto &x : ChainableStack::instance().var_nochain_stack_)
       ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/set_zero_all_adjoints.hpp:17:16: warning: range-based for loop is a C++11 extension [-Wc++11-extensions]
  for (auto &x : ChainableStack::instance().var_nochain_stack_)
               ^
In file included from stan_files/BNHM_Smith.cc:3:
In file included from stan_files/BNHM_Smith.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Dense:2:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/LU:47:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/BNHM_Smith.cc:3:
In file included from stan_files/BNHM_Smith.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Dense:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Cholesky:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Jacobi:29:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/Fix_Eff.cc:3:
In file included from stan_files/Fix_Eff.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat.hpp:22:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:19:59: error: expected expression
class ops_partials_edge<ViewElt, Eigen::Matrix<Op, R, C>> {
                                                          ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:35:2: error: expected a type
};
 ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:38:70: error: expected expression
class ops_partials_edge<ViewElt, std::vector<Eigen::Matrix<Op, R, C>>> {
                                                                     ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:38:72: error: expected a type
class ops_partials_edge<ViewElt, std::vector<Eigen::Matrix<Op, R, C>>> {
                                                                       ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:53:2: error: expected a type
};
 ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:56:60: error: a space is required between consecutive right angle brackets (use '> >')
class ops_partials_edge<ViewElt, std::vector<std::vector<Op>>> {
                                                           ^~
                                                           > > 
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:58:68: error: a space is required between consecutive right angle brackets (use '> >')
  typedef empty_broadcast_array<ViewElt, std::vector<std::vector<Op>>>
                                                                   ^~
                                                                   > > 
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:61:63: error: a space is required between consecutive right angle brackets (use '> >')
  empty_broadcast_array<partials_t, std::vector<std::vector<Op>>> partials_vec_;
                                                              ^~
                                                              > > 
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:63:62: error: a space is required between consecutive right angle brackets (use '> >')
  explicit ops_partials_edge(const std::vector<std::vector<Op>> ops) {}
                                                             ^~
                                                             > >
In file included from stan_files/BNHM_Smith.cc:3:
In file included from stan_files/BNHM_Smith.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Dense:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Cholesky:43:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/BNHM_Smith.cc:3:
In file included from stan_files/BNHM_Smith.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Dense:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/QR:17:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Householder:27:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/Beta_binomial.cc:3:
In file included from stan_files/Beta_binomial.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat.hpp:22:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:19:59: error: expected expression
class ops_partials_edge<ViewElt, Eigen::Matrix<Op, R, C>> {
                                                          ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:35:2: error: expected a type
};
 ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:38:70: error: expected expression
class ops_partials_edge<ViewElt, std::vector<Eigen::Matrix<Op, R, C>>> {
                                                                     ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:38:72: error: expected a type
class ops_partials_edge<ViewElt, std::vector<Eigen::Matrix<Op, R, C>>> {
                                                                       ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:53:2: error: expected a type
};
 ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:56:60: error: a space is required between consecutive right angle brackets (use '> >')
class ops_partials_edge<ViewElt, std::vector<std::vector<Op>>> {
                                                           ^~
                                                           > > 
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:58:68: error: a space is required between consecutive right angle brackets (use '> >')
  typedef empty_broadcast_array<ViewElt, std::vector<std::vector<Op>>>
                                                                   ^~
                                                                   > > 
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:61:63: error: a space is required between consecutive right angle brackets (use '> >')
  empty_broadcast_array<partials_t, std::vector<std::vector<Op>>> partials_vec_;
                                                              ^~
                                                              > > 
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:63:62: error: a space is required between consecutive right angle brackets (use '> >')
  explicit ops_partials_edge(const std::vector<std::vector<Op>> ops) {}
                                                             ^~
                                                             > >
In file included from stan_files/BNHM_Higgins.cc:3:
In file included from stan_files/BNHM_Higgins.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat.hpp:22:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:19:59: error: expected expression
class ops_partials_edge<ViewElt, Eigen::Matrix<Op, R, C>> {
                                                          ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:35:2: error: expected a type
};
 ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:38:70: error: expected expression
class ops_partials_edge<ViewElt, std::vector<Eigen::Matrix<Op, R, C>>> {
                                                                     ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:38:72: error: expected a type
class ops_partials_edge<ViewElt, std::vector<Eigen::Matrix<Op, R, C>>> {
                                                                       ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:53:2: error: expected a type
};
 ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:56:60: error: a space is required between consecutive right angle brackets (use '> >')
class ops_partials_edge<ViewElt, std::vector<std::vector<Op>>> {
                                                           ^~
                                                           > > 
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:58:68: error: a space is required between consecutive right angle brackets (use '> >')
  typedef empty_broadcast_array<ViewElt, std::vector<std::vector<Op>>>
                                                                   ^~
                                                                   > > 
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:61:63: error: a space is required between consecutive right angle brackets (use '> >')
  empty_broadcast_array<partials_t, std::vector<std::vector<Op>>> partials_vec_;
                                                              ^~
                                                              > > 
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:63:62: error: a space is required between consecutive right angle brackets (use '> >')
  explicit ops_partials_edge(const std::vector<std::vector<Op>> ops) {}
                                                             ^~
                                                             > >
In file included from stan_files/BNHM_Smith.cc:3:
In file included from stan_files/BNHM_Smith.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Dense:5:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/SVD:48:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/BNHM_Smith.cc:3:
In file included from stan_files/BNHM_Smith.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Dense:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Geometry:58:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/BNHM_Smith.cc:3:
In file included from stan_files/BNHM_Smith.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Dense:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Eigenvalues:58:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/BNHM_Smith.cc:3:
In file included from stan_files/BNHM_Smith.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core.hpp:42:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/recover_memory.hpp:25:8: warning: 'auto' type specifier is a C++11 extension [-Wc++11-extensions]
  for (auto &x : ChainableStack::instance().var_alloc_stack_) {
       ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/recover_memory.hpp:25:16: warning: range-based for loop is a C++11 extension [-Wc++11-extensions]
  for (auto &x : ChainableStack::instance().var_alloc_stack_) {
               ^
In file included from stan_files/BNHM_Smith.cc:3:
In file included from stan_files/BNHM_Smith.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core.hpp:44:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/set_zero_all_adjoints.hpp:15:8: warning: 'auto' type specifier is a C++11 extension [-Wc++11-extensions]
  for (auto &x : ChainableStack::instance().var_stack_)
       ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/set_zero_all_adjoints.hpp:15:16: warning: range-based for loop is a C++11 extension [-Wc++11-extensions]
  for (auto &x : ChainableStack::instance().var_stack_)
               ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/set_zero_all_adjoints.hpp:17:8: warning: 'auto' type specifier is a C++11 extension [-Wc++11-extensions]
  for (auto &x : ChainableStack::instance().var_nochain_stack_)
       ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/core/set_zero_all_adjoints.hpp:17:16: warning: range-based for loop is a C++11 extension [-Wc++11-extensions]
  for (auto &x : ChainableStack::instance().var_nochain_stack_)
               ^
In file included from stan_files/BNHM_Smith.cc:3:
In file included from stan_files/BNHM_Smith.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat.hpp:22:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:19:59: error: expected expression
class ops_partials_edge<ViewElt, Eigen::Matrix<Op, R, C>> {
                                                          ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:35:2: error: expected a type
};
 ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:38:70: error: expected expression
class ops_partials_edge<ViewElt, std::vector<Eigen::Matrix<Op, R, C>>> {
                                                                     ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:38:72: error: expected a type
class ops_partials_edge<ViewElt, std::vector<Eigen::Matrix<Op, R, C>>> {
                                                                       ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:53:2: error: expected a type
};
 ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:56:60: error: a space is required between consecutive right angle brackets (use '> >')
class ops_partials_edge<ViewElt, std::vector<std::vector<Op>>> {
                                                           ^~
                                                           > > 
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:58:68: error: a space is required between consecutive right angle brackets (use '> >')
  typedef empty_broadcast_array<ViewElt, std::vector<std::vector<Op>>>
                                                                   ^~
                                                                   > > 
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:61:63: error: a space is required between consecutive right angle brackets (use '> >')
  empty_broadcast_array<partials_t, std::vector<std::vector<Op>>> partials_vec_;
                                                              ^~
                                                              > > 
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:63:62: error: a space is required between consecutive right angle brackets (use '> >')
  explicit ops_partials_edge(const std::vector<std::vector<Op>> ops) {}
                                                             ^~
                                                             > >
In file included from stan_files/Fix_Eff.cc:3:
In file included from stan_files/Fix_Eff.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat.hpp:96:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/csr_extract_u.hpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Sparse:26:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/SparseCore:66:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/Fix_Eff.cc:3:
In file included from stan_files/Fix_Eff.hppIn file included from stan_files/Beta_binomial.cc:3:
In file included from stan_files/Beta_binomial.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat.hpp:96:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/csr_extract_u.hpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Sparse:26:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/SparseCore:66:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
:    #pragma clang diagnostic pop18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from 
                             ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat.hpp:96:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/csr_extract_u.hpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Sparse:27:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/OrderingMethods:71:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/Fix_Eff.cc:3:
In file included from stan_files/Fix_Eff.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat.hpp:96:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/csr_extract_u.hpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Sparse:29:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/SparseCholesky:43:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/BNHM_Higgins.cc:3:
In file included from stan_files/BNHM_Higgins.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat.hpp:96:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/csr_extract_u.hpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Sparse:26:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/SparseCore:66:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/Beta_binomial.cc:3:
In file included from stan_files/Beta_binomial.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat.hpp:96:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/csr_extract_u.hpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Sparse:27:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/OrderingMethods:71:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/Beta_binomial.cc:3:
In file included from stan_files/Beta_binomial.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat.hpp:96:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/csr_extract_u.hpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Sparse:29:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/SparseCholesky:43:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/BNHM_Higgins.cc:3:
In file included from stan_files/BNHM_Higgins.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat.hpp:96:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/csr_extract_u.hpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Sparse:27:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/OrderingMethods:71:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/BNHM_Higgins.cc:3:
In file included from stan_files/BNHM_Higgins.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat.hpp:96:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/csr_extract_u.hpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Sparse:29:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/SparseCholesky:43:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/Fix_Eff.cc:3:
In file included from stan_files/Fix_Eff.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat.hpp:96:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/csr_extract_u.hpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Sparse:32:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/SparseQR:35:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/Beta_binomial.cc:3:
In file included from stan_files/Beta_binomial.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat.hpp:96:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/csr_extract_u.hpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Sparse:32:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/SparseQR:35:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/Fix_Eff.cc:3:
In file included from stan_files/Fix_Eff.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat.hpp:96:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/csr_extract_u.hpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Sparse:33:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/IterativeLinearSolvers:46:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/Fix_Eff.cc:3:
In file included from stan_files/Fix_Eff.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat.hpp:99:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/csr_matrix_times_vector.hpp:91:14: warning: range-based for loop is a C++11 extension [-Wc++11-extensions]
  for (int i : v)
             ^
In file included from stan_files/Fix_Eff.cc:3:
In file included from stan_files/Fix_Eff.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat.hpp:100:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/csr_to_dense_matrix.hpp:48:14: warning: range-based for loop is a C++11 extension [-Wc++11-extensions]
  for (int i : v)
             ^
In file included from stan_files/BNHM_Higgins.cc:3:
In file included from stan_files/BNHM_Higgins.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat.hpp:96:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/csr_extract_u.hpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Sparse:32:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/SparseQR:35:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/Beta_binomial.cc:3:
In file included from stan_files/Beta_binomial.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat.hpp:96:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/csr_extract_u.hpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Sparse:33:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/IterativeLinearSolvers:46:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/Beta_binomial.cc:3:
In file included from stan_files/Beta_binomial.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat.hpp:99:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/csr_matrix_times_vector.hpp:91:14: warning: range-based for loop is a C++11 extension [-Wc++11-extensions]
  for (int i : v)
             ^
In file included from stan_files/Beta_binomial.cc:3:
In file included from stan_files/Beta_binomial.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat.hpp:100:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/csr_to_dense_matrix.hpp:48:14: warning: range-based for loop is a C++11 extension [-Wc++11-extensions]
  for (int i : v)
             ^
In file included from stan_files/BNHM_Higgins.cc:3:
In file included from stan_files/BNHM_Higgins.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat.hpp:96:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/csr_extract_u.hpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Sparse:33:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/IterativeLinearSolvers:46:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/BNHM_Higgins.cc:3:
In file included from stan_files/BNHM_Higgins.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat.hpp:99:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/csr_matrix_times_vector.hpp:91:14: warning: range-based for loop is a C++11 extension [-Wc++11-extensions]
  for (int i : v)
             ^
In file included from stan_files/BNHM_Higgins.cc:3:
In file included from stan_files/BNHM_Higgins.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat.hpp:100:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/csr_to_dense_matrix.hpp:48:14: warning: range-based for loop is a C++11 extension [-Wc++11-extensions]
  for (int i : v)
             ^
In file included from stan_files/Fix_Eff.cc:3:
In file included from stan_files/Fix_Eff.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat.hpp:156:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/log_mix.hpp:178:26: error: expected expression
              .unaryExpr([](T_partials_return x) { return exp(x); });
                         ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/log_mix.hpp:262:24: error: expected expression
            .unaryExpr([](T_partials_return x) { return exp(x); });
                       ^
In file included from stan_files/Fix_Eff.cc:3:
In file included from stan_files/Fix_Eff.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat.hpp:161:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp.hpp:5:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_2x2.hpp:21:13: warning: alias declarations are a C++11 extension [-Wc++11-extensions]
  using T = typename Mtype::Scalar;
            ^
In file included from stan_files/Beta_binomial.cc:3:
In file included from stan_files/Beta_binomial.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat.hpp:156:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/log_mix.hpp:178:26: error: expected expression
              .unaryExpr([](T_partials_return x) { return exp(x); });
                         ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/log_mix.hpp:262:24: error: expected expression
            .unaryExpr([](T_partials_return x) { return exp(x); });
                       ^
In file included from stan_files/Beta_binomial.cc:3:
In file included from stan_files/Beta_binomial.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat.hpp:161:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp.hpp:5:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_2x2.hpp:21:13: warning: alias declarations are a C++11 extension [-Wc++11-extensions]
  using T = typename Mtype::Scalar;
            ^
In file included from stan_files/BNHM_Higgins.cc:3:
In file included from stan_files/BNHM_Higgins.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat.hpp:156:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/log_mix.hpp:178:26: error: expected expression
              .unaryExpr([](T_partials_return x) { return exp(x); });
                         ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/log_mix.hpp:262:24: error: expected expression
            .unaryExpr([](T_partials_return x) { return exp(x); });
                       ^
In file included from stan_files/BNHM_Higgins.cc:3:
In file included from stan_files/BNHM_Higgins.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat.hpp:161:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp.hpp:5:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_2x2.hpp:21:13: warning: alias declarations are a C++11 extension [-Wc++11-extensions]
  using T = typename Mtype::Scalar;
            ^
In file included from stan_files/Fix_Eff.cc:3:
In file included from stan_files/Fix_Eff.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat.hpp:162:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_multiply.hpp:5:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_action_handler.hpp:20:10: error: unknown type name 'constexpr'
  static constexpr int p_max = 8;
         ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_action_handler.hpp:20:20: error: expected member name or ';' after declaration specifiers
  static constexpr int p_max = 8;
  ~~~~~~~~~~~~~~~~ ^
In file included from stan_files/Beta_binomial.cc:3:
In file included from stan_files/Beta_binomial.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat.hpp:162:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_multiply.hpp:5:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_action_handler.hpp:20:10: error: unknown type name 'constexpr'
  static constexpr int p_max = 8;
         ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_action_handler.hpp:20:20: error: expected member name or ';' after declaration specifiers
  static constexpr int p_max = 8;
  ~~~~~~~~~~~~~~~~ ^
In file included from stan_files/BNHM_Higgins.cc:3:
In file included from stan_files/BNHM_Higgins.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat.hpp:162:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_multiply.hpp:5:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_action_handler.hpp:20:10: error: unknown type name 'constexpr'
  static constexpr int p_max = 8;
         ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_action_handler.hpp:20:20: error: expected member name or ';' after declaration specifiers
  static constexpr int p_max = 8;
  ~~~~~~~~~~~~~~~~ ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_action_handler.hpp:21:10: error: unknown type name 'constexpr'
  static constexpr int m_max = 55;
         ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_action_handler.hpp:21:20: error: expected member name or ';' after declaration specifiers
  static constexpr int m_max = 55;
  ~~~~~~~~~~~~~~~~ ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_action_handler.hpp:21:10: error: unknown type name 'constexpr'
  static constexpr int m_max = 55;
         ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_action_handler.hpp:21:20: error: expected member name or ';' after declaration specifiers
  static constexpr int m_max = 55;
  ~~~~~~~~~~~~~~~~ ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_action_handler.hpp:21:10: error: unknown type name 'constexpr'
  static constexpr int m_max = 55;
         ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_action_handler.hpp:21:20: error: expected member name or ';' after declaration specifiers
  static constexpr int m_max = 55;
  ~~~~~~~~~~~~~~~~ ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_action_handler.hpp:22:10: error: unknown type name 'constexpr'
  static constexpr double tol = 1.1e-16;
         ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_action_handler.hpp:22:20: error: expected member name or ';' after declaration specifiers
  static constexpr double tol = 1.1e-16;
  ~~~~~~~~~~~~~~~~ ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_action_handler.hpp:25:29: error: function definition does not declare parameters
  const std::vector<double> theta_m_single_precision{
                            ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_action_handler.hpp:28:29: error: function definition does not declare parameters
  const std::vector<double> theta_m_double_precision{
                            ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_action_handler.hpp:22:10: error: unknown type name 'constexpr'
  static constexpr double tol = 1.1e-16;
         ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_action_handler.hpp:22:20: error: expected member name or ';' after declaration specifiers
  static constexpr double tol = 1.1e-16;
  ~~~~~~~~~~~~~~~~ ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_action_handler.hpp:25:29: error: function definition does not declare parameters
  const std::vector<double> theta_m_single_precision{
                            ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_action_handler.hpp:28:29: error: function definition does not declare parameters
  const std::vector<double> theta_m_double_precision{
                            ^
fatal error: too many errors emitted, stopping now [-ferror-limit=]
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_action_handler.hpp:22:10: error: unknown type name 'constexpr'
  static constexpr double tol = 1.1e-16;
         ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_action_handler.hpp:22:20: error: expected member name or ';' after declaration specifiers
  static constexpr double tol = 1.1e-16;
  ~~~~~~~~~~~~~~~~ ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_action_handler.hpp:25:29: error: function definition does not declare parameters
  const std::vector<double> theta_m_single_precision{
                            ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_action_handler.hpp:28:29: error: function definition does not declare parameters
  const std::vector<double> theta_m_double_precision{
                            ^
fatal error: too many errors emitted, stopping now [-ferror-limit=]
fatal error: too many errors emitted, stopping now [-ferror-limit=]
In file included from stan_files/BNHM_Smith.cc:3:
In file included from stan_files/BNHM_Smith.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat.hpp:96:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/csr_extract_u.hpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Sparse:26:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/SparseCore:66:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/BNHM_Smith.cc:3:
In file included from stan_files/BNHM_Smith.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat.hpp:96:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/csr_extract_u.hpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Sparse:27:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/OrderingMethods:71:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/BNHM_Smith.cc:3:
In file included from stan_files/BNHM_Smith.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat.hpp:96:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/csr_extract_u.hpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Sparse:29:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/SparseCholesky:43:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/BNHM_Smith.cc:3:
In file included from stan_files/BNHM_Smith.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat.hpp:96:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/csr_extract_u.hpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Sparse:32:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/SparseQR:35:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/BNHM_Smith.cc:3:
In file included from stan_files/BNHM_Smith.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat.hpp:96:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/csr_extract_u.hpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/Sparse:33:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/IterativeLinearSolvers:46:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from stan_files/BNHM_Smith.cc:3:
In file included from stan_files/BNHM_Smith.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat.hpp:99:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/csr_matrix_times_vector.hpp:91:14: warning: range-based for loop is a C++11 extension [-Wc++11-extensions]
  for (int i : v)
             ^
In file included from stan_files/BNHM_Smith.cc:3:
In file included from stan_files/BNHM_Smith.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat.hpp:100:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/csr_to_dense_matrix.hpp:48:14: warning: range-based for loop is a C++11 extension [-Wc++11-extensions]
  for (int i : v)
             ^
In file included from stan_files/BNHM_Smith.cc:3:
In file included from stan_files/BNHM_Smith.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat.hpp:156:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/log_mix.hpp:178:26: error: expected expression
              .unaryExpr([](T_partials_return x) { return exp(x); });
                         ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/log_mix.hpp:262:24: error: expected expression
            .unaryExpr([](T_partials_return x) { return exp(x); });
                       ^
In file included from stan_files/BNHM_Smith.cc:3:
In file included from stan_files/BNHM_Smith.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat.hpp:161:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp.hpp:5:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_2x2.hpp:21:13: warning: alias declarations are a C++11 extension [-Wc++11-extensions]
  using T = typename Mtype::Scalar;
            ^
In file included from stan_files/BNHM_Smith.cc:3:
In file included from stan_files/BNHM_Smith.hpp:18:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat.hpp:162:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_multiply.hpp:5:
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_action_handler.hpp:20:10: error: unknown type name 'constexpr'
  static constexpr int p_max = 8;
         ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_action_handler.hpp:20:20: error: expected member name or ';' after declaration specifiers
  static constexpr int p_max = 8;
  ~~~~~~~~~~~~~~~~ ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_action_handler.hpp:21:10: error: unknown type name 'constexpr'
  static constexpr int m_max = 55;
         ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_action_handler.hpp:21:20: error: expected member name or ';' after declaration specifiers
  static constexpr int m_max = 55;
  ~~~~~~~~~~~~~~~~ ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_action_handler.hpp:22:10: error: unknown type name 'constexpr'
  static constexpr double tol = 1.1e-16;
         ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_action_handler.hpp:22:20: error: expected member name or ';' after declaration specifiers
  static constexpr double tol = 1.1e-16;
  ~~~~~~~~~~~~~~~~ ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_action_handler.hpp:25:29: error: function definition does not declare parameters
  const std::vector<double> theta_m_single_precision{
                            ^
/Users/max/github/ggplot2/revdep/library.noindex/MetaStan/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_action_handler.hpp:28:29: error: function definition does not declare parameters
  const std::vector<double> theta_m_double_precision{
                            ^
fatal error: too many errors emitted, stopping now [-ferror-limit=]
29 warnings and 20 errors generated.
29 warnings and 20 errors generated.
29 warnings and 20 errors generated.
make: *** [stan_files/Fix_Eff.o] Error 1
make: *** Waiting for unfinished jobs....
make: *** [stan_files/Beta_binomial.o] Error 1
make: *** [stan_files/BNHM_Higgins.o] Error 1
29 warnings and 20 errors generated.
make: *** [stan_files/BNHM_Smith.o] Error 1
rm stan_files/BNHM_Higgins.cc stan_files/Fix_Eff.cc stan_files/Beta_binomial.cc stan_files/BNHM_Smith.cc
ERROR: compilation failed for package ‘MetaStan’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/MetaStan/old/MetaStan.Rcheck/MetaStan’

```
# meteogRam

Version: 1.0

## In both

*   checking R code for possible problems ... NOTE
    ```
    crosssection: no visible global function definition for
      ‘colorRampPalette’
    crosssection: no visible global function definition for
      ‘filled.contour’
    crosssection: no visible global function definition for ‘axis’
    crosssection: no visible global function definition for ‘par’
    crosssection: no visible global function definition for ‘contour’
    temperatures: no visible binding for global variable ‘time’
    Undefined global functions or variables:
      axis colorRampPalette contour filled.contour par time
    Consider adding
      importFrom("grDevices", "colorRampPalette")
      importFrom("graphics", "axis", "contour", "filled.contour", "par")
      importFrom("stats", "time")
    to your NAMESPACE file.
    ```

# methimpute

Version: 1.2.0

## In both

*   checking whether package ‘methimpute’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/methimpute/new/methimpute.Rcheck/00install.out’ for details.
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
      category covered
    Inside counts for context CHH
      category missing
      category covered
    Writing to file /var/folders/lb/xhxqmcrd7gv302_b1pdfykh80000gn/T//RtmpA9OC7e/filebf84433c85d3 ... 0.13s
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'methimpute.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `beramono.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.87 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# Methplot

Version: 1.0

## In both

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Description field: should contain one or more complete sentences.
    ```

*   checking R code for possible problems ... NOTE
    ```
    getdata: no visible global function definition for ‘read.csv’
    plotdata: no visible global function definition for ‘head’
    Undefined global functions or variables:
      head read.csv
    Consider adding
      importFrom("utils", "head", "read.csv")
    to your NAMESPACE file.
    ```

# MethylAid

Version: 1.14.0

## In both

*   R CMD check timed out
    

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘MethylAidData’
    ```

# MethylMix

Version: 2.10.2

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘digest’
      All declared Imports should be used.
    ```

# methylumi

Version: 2.26.0

## In both

*   checking whether package ‘methylumi’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘GenomicFeatures’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘DelayedArray’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/methylumi/new/methylumi.Rcheck/00install.out’ for details.
    ```

*   checking for missing documentation entries ... WARNING
    ```
    Undocumented code objects:
      ‘IDATsToMatrices’ ‘IDATtoMatrix’ ‘tcgaPipeline’
    Undocumented S4 methods:
      generic '[' and siglist 'MethyLumiM,ANY,ANY,ANY'
      generic '[' and siglist 'MethyLumiSet,ANY,ANY,ANY'
    All user-level objects in a package (including S4 classes and methods)
    should have documentation entries.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking contents of ‘data’ directory ... WARNING
    ```
    Files not of a type allowed in a ‘data’ directory:
      ‘5318317007_A_Grn.idat’ ‘5318317007_A_Red.idat’
      ‘5318317007_B_Grn.idat’ ‘5318317007_B_Red.idat’
      ‘5318317007_C_Grn.idat’ ‘5318317007_C_Red.idat’
    Please use e.g. ‘inst/extdata’ for non-R data files
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'methylumi.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `fullpage.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.58 \usepackage
                    {hyperref}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Depends: includes the non-default packages:
      ‘Biobase’ ‘scales’ ‘reshape2’ ‘ggplot2’ ‘matrixStats’
      ‘FDb.InfiniumMethylation.hg19’ ‘minfi’
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 11.1Mb
      sub-directories of 1Mb or more:
        R         2.0Mb
        data      6.8Mb
        extdata   1.7Mb
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Packages listed in more than one of Depends, Imports, Suggests, Enhances:
      ‘Biobase’ ‘minfi’ ‘lattice’ ‘matrixStats’
    A package should be listed in only one of these fields.
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' calls to packages already attached by Depends:
      ‘FDb.InfiniumMethylation.hg19’ ‘ggplot2’ ‘matrixStats’ ‘minfi’
      ‘reshape2’ ‘scales’
      Please remove these calls from your code.
    'library' or 'require' calls in package code:
      ‘Biostrings’ ‘MASS’ ‘lumi’ ‘parallel’ ‘rtracklayer’
      Please use :: or requireNamespace() instead.
      See section 'Suggested packages' in the 'Writing R Extensions' manual.
    Namespace in Imports field not imported from: ‘graphics’
      All declared Imports should be used.
    Packages in Depends field not imported from:
      ‘FDb.InfiniumMethylation.hg19’ ‘ggplot2’ ‘matrixStats’ ‘methods’
      ‘reshape2’ ‘scales’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ':::' call which should be '::': ‘lumi:::produceMethylationGEOSubmissionFile’
      See the note in ?`:::` about the use of this operator.
    Unexported objects imported by ':::' calls:
      ‘Biobase:::unsafeSetSlot’ ‘genefilter:::.findCentralMap’
      See the note in ?`:::` about the use of this operator.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      IlluminaHumanMethylation450kCOLORCHANNEL MethylSet RGChannelSet
      abline addColorChannelInfo aes allele as.dist axis box capture.output
      channel.probes colMedians colQuantiles colSds colorRampPalette
      coord_flip data dnorm dropouts drops ecdf facet_grid features
      gamma.integral gamma.mle gamma.mode geom_histogram ggplot hclust
      hm27.controls hm27.ordering hm450.controls hm450.ordering huber index
      intensity lines log_trans mclapply melt mu offset opts p.adjust
      packageDescription packageVersion par plot.density pnorm points
      position_identity read.csv read.delim read.table rect rowMins
      scale_colour_manual scale_fill_manual scale_shape_manual
      scale_x_continuous scale_y_continuous scale_y_discrete
      subsetCommonProbes text theme_bw title value variable weighted.mean
    Consider adding
      importFrom("grDevices", "colorRampPalette")
      importFrom("graphics", "abline", "axis", "box", "lines", "par",
                 "points", "rect", "text", "title")
      importFrom("stats", "as.dist", "dnorm", "ecdf", "hclust", "offset",
                 "p.adjust", "pnorm", "weighted.mean")
      importFrom("utils", "capture.output", "data", "packageDescription",
                 "packageVersion", "read.csv", "read.delim", "read.table")
    to your NAMESPACE file.
    ```

*   checking Rd files ... NOTE
    ```
    prepare_Rd: estimateM.Rd:34-36: Dropping empty section \seealso
    prepare_Rd: estimateM.Rd:37-39: Dropping empty section \examples
    ```

*   checking for unstated dependencies in vignettes ... NOTE
    ```
    'library' or 'require' call not declared from: ‘TxDb.Hsapiens.UCSC.hg19.knownGene’
    ```

# mhtboot

Version: 1.3.3

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'vignette1.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `setspace.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.63 \usepackage
                    {verbatim}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# micromap

Version: 1.9.3

## In both

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    Coordinate system already present. Adding new coordinate system, which will replace the existing one.
    Warning: Ignoring unknown aesthetics: x
    Warning: Ignoring unknown aesthetics: x
    Warning: Ignoring unknown aesthetics: x
    Warning: Ignoring unknown aesthetics: x
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'Introduction_Guide.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.30 \ifthenelse
                    {\boolean{Sweave@inconsolata}}{%^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# microplot

Version: 1.0-42

## In both

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Loading required package: RColorBrewer
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'rmhPoster.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.30 \ifthenelse
                    {\boolean{Sweave@inconsolata}}{%^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# MIGSA

Version: 1.4.0

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/runTests.R’ failed.
    Last 13 lines of output:
      
          Filter, Find, Map, Position, Reduce, anyDuplicated, append,
          as.data.frame, basename, cbind, colMeans, colSums, colnames,
          dirname, do.call, duplicated, eval, evalq, get, grep, grepl,
          intersect, is.unsorted, lapply, lengths, mapply, match, mget,
          order, paste, pmax, pmax.int, pmin, pmin.int, rank, rbind,
          rowMeans, rowSums, rownames, sapply, setdiff, sort, table, tapply,
          union, unique, unsplit, which, which.max, which.min
      
      
      
      Error in library("RUnit", quietly = TRUE) : 
        there is no package called 'RUnit'
      Calls: <Anonymous> -> library
      Execution halted
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    replace the existing scale.
    Scale for 'x' is already present. Adding another scale for 'x', which will
    replace the existing scale.
    Scale for 'y' is already present. Adding another scale for 'y', which will
    replace the existing scale.
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'MIGSA.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.55 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# milr

Version: 0.3.0

## In both

*   R CMD check timed out
    

# MIMOSA

Version: 1.18.0

## In both

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    Undefined global functions or variables:
      AnnotatedDataFrame ExpressionSet PTID Proportion Proportion_REF
      RefTreat aes as.formula callNextMethod contour coord_trans dbeta dexp
      ecdf facet_grid facet_wrap fisher.test geom_boxplot geom_jitter
      geom_line ggtitle gray.colors huber image is.formula ldply legend
      lines mclapply model.frame na.omit optim p.adjust pData<- par pbeta
      plot points position_jitter qbeta quantile rbeta rbinom read.table
      rmultinom rnorm runif scale_color_brewer scale_fill_brewer show terms
      title var
    Consider adding
      importFrom("grDevices", "gray.colors")
      importFrom("graphics", "contour", "image", "legend", "lines", "par",
                 "plot", "points", "title")
      importFrom("methods", "callNextMethod", "show")
      importFrom("stats", "as.formula", "dbeta", "dexp", "ecdf",
                 "fisher.test", "model.frame", "na.omit", "optim",
                 "p.adjust", "pbeta", "qbeta", "quantile", "rbeta", "rbinom",
                 "rmultinom", "rnorm", "runif", "terms", "var")
      importFrom("utils", "read.table")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

# mineCitrus

Version: 1.0.0

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.1Mb
      sub-directories of 1Mb or more:
        data   5.0Mb
    ```

# MineICA

Version: 1.20.0

## In both

*   checking whether package ‘MineICA’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘graph’ was built under R version 3.5.1
      Warning: package ‘limma’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/MineICA/new/MineICA.Rcheck/00install.out’ for details.
    ```

*   checking package dependencies ... NOTE
    ```
    Depends: includes the non-default packages:
      ‘BiocGenerics’ ‘Biobase’ ‘plyr’ ‘ggplot2’ ‘scales’ ‘foreach’ ‘xtable’
      ‘biomaRt’ ‘gtools’ ‘GOstats’ ‘cluster’ ‘marray’ ‘mclust’
      ‘RColorBrewer’ ‘colorspace’ ‘igraph’ ‘Rgraphviz’ ‘graph’ ‘annotate’
      ‘Hmisc’ ‘fastICA’ ‘JADE’
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Packages listed in more than one of Depends, Imports, Suggests, Enhances:
      ‘biomaRt’ ‘GOstats’ ‘cluster’ ‘mclust’ ‘igraph’
    A package should be listed in only one of these fields.
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' call to ‘GOstats’ which was already attached by Depends.
      Please remove these calls from your code.
    Namespace in Imports field not imported from: ‘lumiHumanAll.db’
      All declared Imports should be used.
    Packages in Depends field not imported from:
      ‘GOstats’ ‘Hmisc’ ‘JADE’ ‘RColorBrewer’ ‘Rgraphviz’ ‘annotate’
      ‘biomaRt’ ‘cluster’ ‘colorspace’ ‘fastICA’ ‘foreach’ ‘ggplot2’
      ‘graph’ ‘gtools’ ‘igraph’ ‘marray’ ‘mclust’ ‘methods’ ‘plyr’ ‘scales’
      ‘xtable’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ':::' calls which should be '::':
      ‘Biobase:::annotation’ ‘Biobase:::validMsg’ ‘fpc:::pamk’
      ‘lumi:::getChipInfo’ ‘mclust:::adjustedRandIndex’
      See the note in ?`:::` about the use of this operator.
    Unexported object imported by a ':::' call: ‘Biobase:::isValidVersion’
      See the note in ?`:::` about the use of this operator.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      reorder scale_colour_gradientn scale_colour_manual scale_fill_manual
      scale_linetype_manual scale_shape_manual scale_x_continuous
      scale_x_discrete scale_y_continuous shapiro.test sigCategories
      terrain_hcl theme theme_bw title tkplot.fit.to.screen unit useMart
      validObject vcount viewport wilcox.test write.table xlab xtable
    Consider adding
      importFrom("grDevices", "cm.colors", "dev.off", "graphics.off",
                 "heat.colors", "pdf")
      importFrom("graphics", "abline", "axis", "frame", "hist", "layout",
                 "legend", "mtext", "par", "plot", "plot.new", "points",
                 "title")
      importFrom("methods", "callNextMethod", "new", "validObject")
      importFrom("stats", "aggregate", "as.dendrogram", "as.dist",
                 "as.hclust", "chisq.test", "cor", "cor.test", "cutree",
                 "dist", "hclust", "kmeans", "kruskal.test", "lm", "median",
                 "na.omit", "order.dendrogram", "p.adjust", "quantile",
                 "reorder", "shapiro.test", "wilcox.test")
      importFrom("utils", "capture.output", "combn", "read.table",
                 "write.table")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    replace the existing scale.
    Warning: `axis.ticks.margin` is deprecated. Please set `margin` property  of `axis.text` instead
    Scale for 'x' is already present. Adding another scale for 'x', which will
    replace the existing scale.
    Warning: `axis.ticks.margin` is deprecated. Please set `margin` property  of `axis.text` instead
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'MineICA.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `times.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.7 \usepackage
                   {hyperref}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# MiRAnorm

Version: 1.0.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘MASS’ ‘cluster’ ‘dendextend’ ‘npmv’ ‘utils’
      All declared Imports should be used.
    ```

# mirIntegrator

Version: 1.10.0

## In both

*   checking R code for possible problems ... NOTE
    ```
    integrate_mir : isGraphNEL: no visible global function definition for
      ‘is’
    pathways2pdf: no visible global function definition for ‘pdf’
    pathways2pdf: no visible global function definition for ‘par’
    pathways2pdf: no visible global function definition for ‘graphics.off’
    plotLines: no visible binding for global variable ‘x’
    plotLines: no visible binding for global variable ‘y’
    plotLines: no visible binding for global variable ‘pathways_set’
    plotPathway2Colors: no visible global function definition for ‘legend’
    plot_change: no visible global function definition for ‘complete.cases’
    Undefined global functions or variables:
      complete.cases graphics.off is legend par pathways_set pdf x y
    Consider adding
      importFrom("grDevices", "graphics.off", "pdf")
      importFrom("graphics", "legend", "par")
      importFrom("methods", "is")
      importFrom("stats", "complete.cases")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    Loading required package: Rgraphviz
    Loading required package: grid
    There are  881 miRNAs meassured and 15 of them were included in the analysis.
    There are  19611 mRNAs meassured and 0 of them were included in the analysis.
    The over-representaion p-value is not defined for cut-off free analysis and will not be computed!
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'mirIntegrator.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.30 \ifthenelse
                    {\boolean{Sweave@inconsolata}}{%^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# Mirsynergy

Version: 1.16.0

## In both

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    Loading required package: foreach
    Loaded glmnet 2.0-16
    
    Discard row/column of all zeros in W
    `stat_bin()` using `bins = 30`. Pick better value with `binwidth`.
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'Mirsynergy.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.30 \ifthenelse
                    {\boolean{Sweave@inconsolata}}{%^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# MissingDataGUI

Version: 0.2-5

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: ‘gWidgetsRGtk2’ ‘cairoDevice’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# missRows

Version: 1.0.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
      ...
    Loading required package: ggplot2
    Loading required package: MultiAssayExperiment
    Warning in has_utility("pdfcrop") :
      pdfcrop not installed or not in PATH
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'missRows.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `float.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.36 \usepackage
                    {lmodern}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Error in .requirePackage(package) : 
        unable to find required package 'MultiAssayExperiment'
      Calls: <Anonymous> ... getClass -> getClassDef -> .classEnv -> .requirePackage
      Execution halted
    ```

# mistat

Version: 1.0-5

## In both

*   checking PDF version of manual ... WARNING
    ```
    ...
                                                      
    ! Please use \mathaccent for accents in math mode.
    \add@accent ...@spacefactor \spacefactor }\accent 
                                                      #1 #2\egroup \spacefactor ...
    l.2831 ...icircum{}2}{} values of PLACE data}{TSQ}
                                                      
    ! Missing { inserted.
    <to be read again> 
                       \egroup 
    l.2831 ...icircum{}2}{} values of PLACE data}{TSQ}
                                                      
    ! You can't use `\spacefactor' in math mode.
    \add@accent ...}\accent #1 #2\egroup \spacefactor 
                                                      \accent@spacefactor 
    l.2831 ...icircum{}2}{} values of PLACE data}{TSQ}
                                                      
    ! Missing } inserted.
    <inserted text> 
                    }
    l.2831 ...icircum{}2}{} values of PLACE data}{TSQ}
                                                      
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘grDevices’
      All declared Imports should be used.
    ```

# mixOmics

Version: 6.3.2

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  6.6Mb
      sub-directories of 1Mb or more:
        R      2.0Mb
        data   4.0Mb
    ```

# MixSIAR

Version: 3.1.10

## In both

*   checking examples ... ERROR
    ```
    ...
    Running examples in ‘MixSIAR-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: mixsiar_gui
    > ### Title: Run the GUI version of MixSIAR
    > ### Aliases: mixsiar_gui
    > 
    > ### ** Examples
    > 
    > mixsiar_gui()
    Error in mixsiar_gui() : 
      *** gWidgetsRGtk2 package not able to be loaded. ***
            If 'library('gWidgetsRGtk2')' does not work, MixSIAR GUI will not run.
            On Windows/Linux, try 'install.packages('gWidgetsRGtk2')'.
            On Mac, close R, download and install GTK+ from:
            http://r.research.att.com/#other. Then install latest X11 application
            (xQuartz) from http://xquartz.macosforge.org/landing/.
    
            If installing GTK+ continues to be problematic, consider using the
            script version of MixSIAR. See manual for help and examples.
    Execution halted
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 25 SKIPPED: 0 FAILED: 9
      1. Error: Wolves ex works (@test_ex1_wolves.R#33) 
      2. Error: Geese ex works (@test_ex2_geese.R#27) 
      3. Error: Lake ex works (@test_ex3_lake.R#27) 
      4. Error: Palmyra ex works (@test_ex4_palmyra.R#27) 
      5. Error: Killer whale ex works (@test_ex5_killerwhale.R#28) 
      6. Error: Snail ex works (@test_ex6_snail.R#27) 
      7. Error: Storm-petrel ex works (@test_ex7_stormpetrel.R#27) 
      8. Error: Cladocera ex works (@test_ex8_cladocera.R#31) 
      9. Error: Isopod ex works (@test_ex9_isopod.R#27) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘gWidgetsRGtk2’
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘rjags’
      All declared Imports should be used.
    ```

# mizer

Version: 1.0

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Complete output:
      > library(testthat)
      > library(mizer)
      > 
      > test_check("mizer")
      ── 1. Failure: retune_abundance reproduces scaling model (@test-wrapper_function
      max(abs(initial_n - pr@initial_n)) is not strictly less than 2e-11. Difference: 2.47e-11
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 508 SKIPPED: 1 FAILED: 1
      1. Failure: retune_abundance reproduces scaling model (@test-wrapper_functions.R#17) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'mizer_vignette.tex' failed.
    LaTeX errors:
    ! Package babel Error: Unknown option `english'. Either you misspelled it
    (babel)                or the language definition file english.ldf was not foun
    d.
    
    See the babel package documentation for explanation.
    ! LaTeX Error: File `float.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.65 \usepackage
                    {subfig}	% To add subfigures within figures, with labels (se...
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘deSolve’ ‘progress’
      All declared Imports should be used.
    ```

# mlbgameday

Version: 0.1.2

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘doParallel’ ‘iterators’ ‘parallel’
      All declared Imports should be used.
    ```

# mlogit

Version: 0.3-0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    Please cite the 'maxLik' package as:
    Henningsen, Arne and Toomet, Ott (2011). maxLik: A package for maximum likelihood estimation in R. Computational Statistics 26(3), 443-458. DOI 10.1007/s00180-010-0217-1.
    
    If you have questions, suggestions, or comments regarding the 'maxLik' package, please use a forum or 'tracker' at maxLik's R-Forge site:
    https://r-forge.r-project.org/projects/maxlik/
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'Exercises.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.55 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# mlr

Version: 2.13

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘sf’
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.5Mb
      sub-directories of 1Mb or more:
        R      2.0Mb
        data   2.3Mb
    ```

# MLSeq

Version: 1.20.3

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    variance of dispersion residuals not estimated (necessary only for differential expression calling)
    variance of dispersion residuals not estimated (necessary only for differential expression calling)
    variance of dispersion residuals not estimated (necessary only for differential expression calling)
    Error in .local(object, ...) : 
      Incorrect elements in 'control' argument. It should be defined using 'trainControl(...)' function.
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'MLSeq.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `beramono.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.87 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘xtable’
      All declared Imports should be used.
    ```

# MLZ

Version: 0.1.1

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 13.9Mb
      sub-directories of 1Mb or more:
        libs  13.4Mb
    ```

# MMDiff2

Version: 1.8.0

## In both

*   checking whether package ‘MMDiff2’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘Rsamtools’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/MMDiff2/new/MMDiff2.Rcheck/00install.out’ for details.
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    computing 3 pair-wise distances...
    computing distances for WT.AB2 vs Null.AB2
    computing distances for WT.AB2 vs Resc.AB2
    computing distances for Null.AB2 vs Resc.AB2
    preparing Output...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'MMDiff2.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `beramono.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.87 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .BBSoptions
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘methods’
      All declared Imports should be used.
    ```

*   checking R code for possible problems ... NOTE
    ```
    DBAmmd: no visible global function definition for ‘new’
    getReads: no visible global function definition for ‘IRangesList’
    Undefined global functions or variables:
      IRangesList new
    Consider adding
      importFrom("methods", "new")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Error in .requirePackage(package) : 
        unable to find required package 'MMDiff2'
      Calls: <Anonymous> ... getClass -> getClassDef -> .classEnv -> .requirePackage
      Execution halted
    ```

# Mobilize

Version: 2.16-4

## In both

*   checking R code for possible problems ... NOTE
    ```
    ...
      ‘theme_text’
    mapmobility: no visible global function definition for ‘lines’
    responseplot: no visible global function definition for ‘na.omit’
    scatterplot: no visible global function definition for ‘na.omit’
    scatterplot: no visible global function definition for ‘opts’
    scatterplot: no visible global function definition for ‘theme_text’
    sharedtimeplot: no visible global function definition for ‘na.omit’
    timeplot: no visible global function definition for ‘na.omit’
    timeplot: no visible global function definition for ‘opts’
    timeplot.character: no visible global function definition for ‘runif’
    timeplot.character: no visible global function definition for ‘rnorm’
    timeplot.character: no visible binding for global variable ‘text’
    userplot: no visible global function definition for ‘na.omit’
    Undefined global functions or variables:
      head lines na.omit opts quantile rnorm runif text theme_blank
      theme_text
    Consider adding
      importFrom("graphics", "lines", "text")
      importFrom("stats", "na.omit", "quantile", "rnorm", "runif")
      importFrom("utils", "head")
    to your NAMESPACE file.
    ```

# modelgrid

Version: 1.1.0.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘ggplot2’ ‘lattice’
      All declared Imports should be used.
    ```

# momentuHMM

Version: 1.4.2

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    1: DM$angle = list(mean = ~state2(angleFormula(d, strength = w)), 
    2:                 concentration= ~1))
                                         ^
    Warning in highr::hilight(x, format, prompt = options$prompt, markup = opts$markup) :
      the syntax of the source code is invalid; the fallback mode is used
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'momentuHMM.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `setspace.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.58 \usepackage
                    {natbib}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.1Mb
      sub-directories of 1Mb or more:
        data   2.1Mb
        doc    1.7Mb
    ```

# monocle

Version: 2.8.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'monocle-vignette.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `helvet.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.31 \renewcommand
                      {\familydefault}{\sfdefault}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘Rcpp’ ‘biocViews’
      All declared Imports should be used.
    Missing or unexported objects:
      ‘VGAM::gaussianff’ ‘scater::newSCESet’
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    cth_classifier_cds: no visible global function definition for ‘nei’
    cth_classifier_cell: no visible global function definition for ‘nei’
    diff_test_helper: no visible binding for global variable ‘Size_Factor’
    exportCDS: no visible binding for global variable ‘use_for_ordering’
    extract_good_ordering: no visible global function definition for ‘nei’
    fit_model_helper: no visible binding for global variable ‘Size_Factor’
    get_next_node_id: no visible binding for '<<-' assignment to
      ‘next_node’
    get_next_node_id: no visible binding for global variable ‘next_node’
    importCDS: no visible global function definition for ‘gaussianff’
    make_canonical: no visible global function definition for ‘nei’
    measure_diameter_path: no visible global function definition for ‘nei’
    orderCells: no visible binding for '<<-' assignment to ‘next_node’
    plot_multiple_branches_pseudotime: no visible binding for global
      variable ‘pseudocount’
    plot_multiple_branches_pseudotime: no visible binding for global
      variable ‘Branch’
    project2MST: no visible global function definition for ‘nei’
    Undefined global functions or variables:
      Branch Size_Factor gaussianff nei next_node pseudocount
      use_for_ordering
    ```

*   checking files in ‘vignettes’ ... NOTE
    ```
    The following directory looks like a leftover from 'knitr':
      ‘figure’
    Please remove from your package.
    ```

# moonBook

Version: 0.2.3

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘purrr’
      All declared Imports should be used.
    ```

# morse

Version: 3.1.1

## In both

*   checking whether package ‘morse’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/morse/new/morse.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘morse’ ...
** package ‘morse’ successfully unpacked and MD5 sums checked
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rjags', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/morse/rjags/libs/rjags.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/morse/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/morse/rjags/libs/rjags.so
  Reason: image not found
ERROR: lazy loading failed for package ‘morse’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/morse/new/morse.Rcheck/morse’

```
### CRAN

```
* installing *source* package ‘morse’ ...
** package ‘morse’ successfully unpacked and MD5 sums checked
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rjags', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/morse/rjags/libs/rjags.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/morse/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/morse/rjags/libs/rjags.so
  Reason: image not found
ERROR: lazy loading failed for package ‘morse’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/morse/old/morse.Rcheck/morse’

```
# MortalityTables

Version: 1.0

## In both

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘pracma’
    ```

# mosaic

Version: 1.4.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    
    Using parallel package.
      * Set seed with set.rseed().
      * Disable this message with options(`mosaic:parallelMessage` = FALSE)
    
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'Resampling.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `xstring.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.5 \RequirePackage
                       {xcolor}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Package which this enhances but not available for checking: ‘manipulate’
    ```

# mosaicData

Version: 0.17.0

## In both

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 7 marked UTF-8 strings
    ```

# mosaicModel

Version: 0.3.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘MASS’ ‘caret’ ‘ggformula’ ‘knitr’ ‘testthat’ ‘tidyverse’
      All declared Imports should be used.
    ```

# moveHMM

Version: 1.6

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    Warning: 'mode(width)' and 'mode(height)' differ between new and previous
    	 ==> NOT changing 'width' & 'height'
    Note: Some angles are equal to pi, and the corresponding pseudo-residuals are not included
    Warning: 'mode(width)' and 'mode(height)' differ between new and previous
    	 ==> NOT changing 'width' & 'height'
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'moveHMM-guide.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `listings.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.62 \usepackage
                    {color} % for lstlisting background color^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# moveVis

Version: 0.9.8

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
             extent_factor = 2e-04, out_format = "gif")) at testthat/test-animate_move.R:4
      2: quasi_label(enquo(object), label)
      3: eval_bare(get_expr(quo), get_env(quo))
      4: animate_move(vars.global$m, vars.global$out_dir, vars.global$conv_dir, tail_elements = 10, 
             paths_mode = "true_data", frames_nmax = 20, log_level = 3, extent_factor = 2e-04, 
             out_format = "gif")
      5: out("Download from Google failed. Please check your internet connection.", type = 3)
      6: stop(input, call. = FALSE)
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 10 SKIPPED: 1 FAILED: 1
      1. Error: animate_move GIF (@test-animate_move.R#4) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

# MplusAutomation

Version: 0.7-2

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'Vignette.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.30 \ifthenelse
                    {\boolean{Sweave@inconsolata}}{%^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# mppR

Version: 1.1.10

## In both

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'mppR_vignette.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.55 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# msgbsR

Version: 1.4.0

## In both

*   checking whether package ‘msgbsR’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/msgbsR/new/msgbsR.Rcheck/00install.out’ for details.
    ```

*   checking data for ASCII and uncompressed saves ... WARNING
    ```
      
      Note: significantly better compression could be obtained
            by using R CMD build --resave-data
                   old_size new_size compress
      ratdata.rda     318Kb    127Kb       xz
      ratdata2.rda    287Kb    116Kb       xz
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 16.1Mb
      sub-directories of 1Mb or more:
        extdata  15.2Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: 'utils'
      All declared Imports should be used.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    
        strsplit
    
    Loading required package: rtracklayer
    Warning: package 'rtracklayer' was built under R version 3.5.1
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'msgbsR_Vignette.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.30 \ifthenelse
                    {\boolean{Sweave@inconsolata}}{%^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# MSnbase

Version: 2.6.4

## In both

*   R CMD check timed out
    

*   checking installed package size ... NOTE
    ```
      installed size is 13.1Mb
      sub-directories of 1Mb or more:
        R      2.0Mb
        data   1.9Mb
        doc    7.8Mb
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Authors@R field gives more than one person with maintainer role:
      Laurent Gatto <lg390@cam.ac.uk> [aut, cre]
      Johannes Rainer <Johannes.Rainer@eurac.edu> [aut, cre]
      Sebastian Gibb <mail@sebastiangibb.de> [aut, cre]
    ```

*   checking dependencies in R code ... NOTE
    ```
    Unexported objects imported by ':::' calls:
      ‘Biobase:::.showAnnotatedDataFrame’ ‘MALDIquant:::.estimateNoise’
      ‘MALDIquant:::.localMaxima’ ‘MALDIquant:::.movingAverage’
      ‘MALDIquant:::.savitzkyGolay’
      See the note in ?`:::` about the use of this operator.
    ```

# MSnID

Version: 1.14.0

## In both

*   checking R code for possible problems ... NOTE
    ```
    ...
    .read_mzIDs.mzR.engine.single.file: no visible binding for global
      variable ‘DatabaseAccess’
    .read_mzIDs.mzR.engine.single.file: no visible binding for global
      variable ‘DatabaseDescription’
    .read_mzIDs.mzR.engine.single.file: no visible binding for global
      variable ‘DBseqLength’
    infer_parsimonious_accessions,MSnID : infer_acc: no visible binding for
      global variable ‘accession’
    infer_parsimonious_accessions,MSnID : infer_acc: no visible binding for
      global variable ‘N’
    infer_parsimonious_accessions,MSnID : infer_acc: no visible binding for
      global variable ‘pepSeq’
    recalibrate,MSnID: no visible global function definition for ‘median’
    recalibrate,MSnID: no visible global function definition for ‘density’
    Undefined global functions or variables:
      DBseqLength DatabaseAccess DatabaseDescription N accession density i
      location mass median modification name optim pepSeq quantile rnorm
      spectrumID
    Consider adding
      importFrom("stats", "density", "median", "optim", "quantile", "rnorm")
    to your NAMESPACE file.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    
    The following object is masked from ‘package:base’:
    
        trimws
    
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'msnid_vignette.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `beramono.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.87 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# msPurity

Version: 1.6.2

## In both

*   R CMD check timed out
    

*   checking dependencies in R code ... NOTE
    ```
    Package in Depends field not imported from: ‘Rcpp’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    package 'methods' is used but not declared
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    stde: no visible global function definition for ‘sd’
    stderror: no visible global function definition for ‘sd’
    xcmsGroupPurity: no visible binding for global variable ‘median’
    xcmsGroupPurity: no visible global function definition for ‘median’
    averageSpectra,purityD: no visible binding for global variable ‘i’
    subtract,purityD: no visible binding for global variable ‘i’
    validate,purityA: no visible global function definition for ‘head’
    writeOut,purityD: no visible global function definition for ‘write.csv’
    Undefined global functions or variables:
      abline alli approxfun as.dist dev.off dist dnorm fix head i idx
      legend lines median mtch mtchi na.omit parallel plot png points
      purity read.csv read.table scanid sd text topn variable write.csv
    Consider adding
      importFrom("grDevices", "dev.off", "png")
      importFrom("graphics", "abline", "legend", "lines", "plot", "points",
                 "text")
      importFrom("stats", "approxfun", "as.dist", "dist", "dnorm", "median",
                 "na.omit", "sd")
      importFrom("utils", "fix", "head", "read.csv", "read.table",
                 "write.csv")
    to your NAMESPACE file.
    ```

# MSstats

Version: 3.12.3

## In both

*   R CMD check timed out
    

*   checking R code for possible problems ... NOTE
    ```
    ...
    designSampleSizeClassificationPlots: no visible binding for global
      variable ‘Train_size’
    designSampleSizeClassificationPlots: no visible binding for global
      variable ‘Protein_number’
    groupComparisonPlots: no visible binding for global variable ‘Protein’
    groupComparisonPlots: no visible binding for global variable ‘logFC’
    groupComparisonPlots: no visible binding for global variable ‘ciw’
    linear_quantlim: no visible binding for global variable ‘label’
    modelBasedQCPlots: no visible binding for global variable ‘residual’
    nonlinear_quantlim: no visible binding for global variable ‘label’
    plot_quantlim: no visible binding for global variable ‘x’
    plot_quantlim: no visible binding for global variable ‘y’
    plot_quantlim: no visible binding for global variable ‘ymin’
    plot_quantlim: no visible binding for global variable ‘ymax’
    plot_quantlim: no visible binding for global variable ‘shape’
    Undefined global functions or variables:
      ABUNDANCE FEATURE FRACTION Intensity LABEL Mean Name PeptideSequence
      Protein ProteinName Protein_number RUN Selected_fragments
      Selected_peptides Train_size aggr_Fragment_Annotation aggr_Peak_Area
      analysis ciw datafeature fea label logFC missing.col ncount ount
      residual shape weight x y ymax ymin
    ```

# MSstatsQC

Version: 1.2.0

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘RforProteomics’
    ```

# mudata2

Version: 1.0.3

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/test-all.R’ failed.
    Last 13 lines of output:
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 930 SKIPPED: 0 FAILED: 12
      1. Error: mudata_prepare_column and mudata_parse_column are opposites (@test_mudata.io.R#244) 
      2. Error: mudata_prepare_tbl works as intended (@test_mudata.io.R#288) 
      3. Error: mudata_prepare_tbl and mudata_parse_tbl are opposites (@test_mudata.io.R#322) 
      4. Failure: as_* functions produce the expected output type (@test_types.R#210) 
      5. Failure: wkt parsing returns an sf::sfc (@test_types.R#252) 
      6. Failure: wkt parsing works when there are parsing errors/NA values (@test_types.R#259) 
      7. Error: wkt parsing works when there are parsing errors/NA values (@test_types.R#261) 
      8. Failure: wkt parsing works with zero-length input (@test_types.R#285) 
      9. Failure: objects generate the correct type strings (@test_types.R#306) 
      1. ...
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘sf’
    ```

# MultiBD

Version: 0.2.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'SIRtrans.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `mathpazo.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.53 \usepackage
                    [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# MultiDataSet

Version: 1.8.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Unexported objects imported by ':::' calls:
      ‘Biobase:::assayDataEnvLock’ ‘Biobase:::assayDataStorageMode’
      See the note in ?`:::` about the use of this operator.
    ```

*   checking R code for possible problems ... NOTE
    ```
    downloadGEO : <anonymous>: no visible global function definition for
      ‘methylationSet’
    lambdaClayton: no visible global function definition for ‘qnorm’
    lambdaClayton: no visible global function definition for ‘qchisq’
    qq_plot: no visible global function definition for ‘qbeta’
    Undefined global functions or variables:
      methylationSet qbeta qchisq qnorm
    Consider adding
      importFrom("stats", "qbeta", "qchisq", "qnorm")
    to your NAMESPACE file.
    ```

*   checking for unstated dependencies in vignettes ... NOTE
    ```
    'library' or 'require' calls not declared from:
      ‘gplots’ ‘lattice’
    ```

# MultiMeta

Version: 0.1

## In both

*   checking R code for possible problems ... NOTE
    ```
    ...
    multi_meta: no visible global function definition for ‘write.table’
    multi_meta: no visible global function definition for ‘read.table’
    multi_meta: no visible global function definition for ‘pchisq’
    qqplotter: no visible global function definition for ‘read.table’
    qqplotter: no visible global function definition for ‘qchisq’
    qqplotter: no visible global function definition for ‘ppoints’
    qqplotter: no visible global function definition for ‘median’
    qqplotter: no visible global function definition for ‘pchisq’
    qqplotter: no visible global function definition for ‘pdf’
    qqplotter: no visible global function definition for ‘plot’
    qqplotter: no visible global function definition for ‘abline’
    qqplotter: no visible global function definition for ‘dev.off’
    Undefined global functions or variables:
      abline axis colorRampPalette cov2cor dev.off median pchisq pdf plot
      points ppoints qchisq read.table write.table
    Consider adding
      importFrom("grDevices", "colorRampPalette", "dev.off", "pdf")
      importFrom("graphics", "abline", "axis", "plot", "points")
      importFrom("stats", "cov2cor", "median", "pchisq", "ppoints", "qchisq")
      importFrom("utils", "read.table", "write.table")
    to your NAMESPACE file.
    ```

# multipanelfigure

Version: 2.0.0

## In both

*   checking Rd cross-references ... NOTE
    ```
    Packages unavailable to check Rd xrefs: ‘tiff’, ‘rsvg’
    ```

# MutationalPatterns

Version: 1.6.1

## In both

*   R CMD check timed out
    

*   checking whether package ‘MutationalPatterns’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/MutationalPatterns/new/MutationalPatterns.Rcheck/00install.out’ for details.
    ```

# MWASTools

Version: 1.4.0

## In both

*   checking R code for possible problems ... NOTE
    ```
    JBA_binning: no visible binding for global variable ‘median’
    JBA_corDistribution: no visible binding for global variable ‘r.coeff’
    JBA_mergeClusters: no visible binding for global variable ‘median’
    JBA_plotBins: no visible binding for global variable ‘limits’
    JBA_plotBins: no visible binding for global variable ‘value’
    JBA_plotBins: no visible binding for global variable ‘intensity’
    JBA_plotBins: no visible binding for global variable ‘mean_cor’
    JBA_plotBins: no visible binding for global variable ‘clean_mean’
    MWAS_heatmap: no visible binding for '<<-' assignment to ‘hr’
    MWAS_heatmap: no visible binding for global variable ‘hr’
    MWAS_scatterplotMS: no visible binding for global variable ‘logpval’
    QC_CV_scatterplot: no visible binding for global variable ‘abs.CV’
    Undefined global functions or variables:
      abs.CV clean_mean hr intensity limits logpval mean_cor median r.coeff
      value
    Consider adding
      importFrom("stats", "median")
    to your NAMESPACE file.
    ```

# mwaved

Version: 1.1.5

## In both

*   checking whether package ‘mwaved’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/mwaved/new/mwaved.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘mwaved’ ...
** package ‘mwaved’ successfully unpacked and MD5 sums checked
checking for Rscript... yes
configure: Checking if FFTW3 library is available using pkg-config
checking for pkg-config... /usr/local/bin/pkg-config
checking pkg-config is at least version 0.9.0... yes
checking for FFTW... no
FFTW library not found, please install fftw-3-3-3 or greater
configure: creating ./config.status
config.status: creating src/Makevars
** libs
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/mwaved/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c RcppExports.cpp -o RcppExports.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/mwaved/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c mwaved.cpp -o mwaved.o
mwaved.cpp:1:10: fatal error: 'fftw3.h' file not found
#include <fftw3.h>
         ^~~~~~~~~
1 error generated.
make: *** [mwaved.o] Error 1
make: *** Waiting for unfinished jobs....
ERROR: compilation failed for package ‘mwaved’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/mwaved/new/mwaved.Rcheck/mwaved’

```
### CRAN

```
* installing *source* package ‘mwaved’ ...
** package ‘mwaved’ successfully unpacked and MD5 sums checked
checking for Rscript... yes
configure: Checking if FFTW3 library is available using pkg-config
checking for pkg-config... /usr/local/bin/pkg-config
checking pkg-config is at least version 0.9.0... yes
checking for FFTW... no
FFTW library not found, please install fftw-3-3-3 or greater
configure: creating ./config.status
config.status: creating src/Makevars
** libs
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/mwaved/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c RcppExports.cpp -o RcppExports.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/mwaved/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c mwaved.cpp -o mwaved.o
mwaved.cpp:1:10: fatal error: 'fftw3.h' file not found
#include <fftw3.h>
         ^~~~~~~~~
1 error generated.
make: *** [mwaved.o] Error 1
make: *** Waiting for unfinished jobs....
ERROR: compilation failed for package ‘mwaved’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/mwaved/old/mwaved.Rcheck/mwaved’

```
# myTAI

Version: 0.8.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘biomartr’
      All declared Imports should be used.
    ```

# naniar

Version: 0.4.0.0

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      8: withVisible(function_list[[k]](value))
      9: function_list[[k]](value)
      10: dplyr::mutate(., any_missing = label_shadow(., !!!quo_vars, missing = missing, complete = complete))
      11: mutate.tbl_df(., any_missing = label_shadow(., !!!quo_vars, missing = missing, complete = complete))
      12: mutate_impl(.data, dots)
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 456 SKIPPED: 23 FAILED: 4
      1. Error: add_label_shadow returns a tibble (@test-add-label-shadow.R#7) 
      2. Error: add_label_shadow adds the right number of columns (@test-add-label-shadow.R#13) 
      3. Error: add_label_shadow adds a column with suffix 'any_missing' (@test-add-label-shadow.R#20) 
      4. Error: (unknown) (@test-add-label-shadow.R#45) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

# NBLDA

Version: 0.99.0

## In both

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘PoiClaClu’
    ```

# ncappc

Version: 0.3.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘bookdown’
      All declared Imports should be used.
    ```

# NeatMap

Version: 0.3.6.2

## In both

*   checking R code for possible problems ... NOTE
    ```
    data.reduction: no visible global function definition for ‘prcomp’
    data.reduction: no visible global function definition for ‘hclust’
    data.reduction: no visible global function definition for ‘as.dist’
    data.reduction: no visible global function definition for ‘cor’
    data.reduction: no visible global function definition for ‘dist’
    heatmap1: no visible global function definition for ‘sd’
    lineplot: no visible global function definition for ‘sd’
    normalize: no visible global function definition for ‘sd’
    profileplot3d : ColorFunction: no visible global function definition
      for ‘rgb’
    profileplot3d : ColorFunction: no visible global function definition
      for ‘colorRamp’
    profileplot3d: no visible global function definition for ‘sd’
    profileplot3d: no visible global function definition for ‘gray’
    Undefined global functions or variables:
      as.dist colorRamp cor dist gray hclust prcomp rgb sd
    Consider adding
      importFrom("grDevices", "colorRamp", "gray", "rgb")
      importFrom("stats", "as.dist", "cor", "dist", "hclust", "prcomp", "sd")
    to your NAMESPACE file.
    ```

*   checking compiled code ... NOTE
    ```
    File ‘NeatMap/libs/NeatMap.so’:
      Found ‘_rand’, possibly from ‘rand’ (C)
        Object: ‘nMDS_R.o’
      Found ‘_srand’, possibly from ‘srand’ (C)
        Object: ‘nMDS_R.o’
    
    Compiled code should not call entry points which might terminate R nor
    write to stdout/stderr instead of to the console, nor use Fortran I/O
    nor system RNGs.
    
    See ‘Writing portable packages’ in the ‘Writing R Extensions’ manual.
    ```

# nethet

Version: 1.12.0

## In both

*   checking whether package ‘nethet’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘limma’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/nethet/new/nethet.Rcheck/00install.out’ for details.
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Warning in has_utility("pdfcrop") :
      pdfcrop not installed or not in PATH
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'nethet.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `beramono.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.87 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    t2cov.lr: no visible global function definition for ‘var’
    t2cov.lr: no visible global function definition for ‘pchisq’
    t2diagcov.lr: no visible global function definition for ‘var’
    t2diagcov.lr: no visible global function definition for ‘pchisq’
    test.sd: no visible global function definition for ‘var’
    test.sd: no visible global function definition for ‘pnorm’
    twosample_single_regr: no visible global function definition for ‘coef’
    twosample_single_regr: no visible global function definition for ‘lm’
    Undefined global functions or variables:
      abline boxplot coef cor cov.wt dev.off dnorm grey hist kmeans legend
      lines lm median optimize p.adjust par pchisq pdf pnorm quantile rbeta
      rnorm segments shapiro.test var write.csv
    Consider adding
      importFrom("grDevices", "dev.off", "grey", "pdf")
      importFrom("graphics", "abline", "boxplot", "hist", "legend", "lines",
                 "par", "segments")
      importFrom("stats", "coef", "cor", "cov.wt", "dnorm", "kmeans", "lm",
                 "median", "optimize", "p.adjust", "pchisq", "pnorm",
                 "quantile", "rbeta", "rnorm", "shapiro.test", "var")
      importFrom("utils", "write.csv")
    to your NAMESPACE file.
    ```

# netjack

Version: 1.1.1

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘Rdpack’
      All declared Imports should be used.
    ```

# netresponse

Version: 1.40.0

## In both

*   checking whether package ‘netresponse’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘graph’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/netresponse/new/netresponse.Rcheck/00install.out’ for details.
    ```

*   checking files in ‘vignettes’ ... WARNING
    ```
    Files in the 'vignettes' directory but no files in 'inst/doc':
      ‘NetResponse.Rmd’, ‘NetResponse.md’, ‘TODO/TODO.Rmd’,
        ‘fig/NetResponse2-1.png’, ‘fig/NetResponse2b-1.png’,
        ‘fig/NetResponse3-1.png’, ‘fig/NetResponse4-1.png’,
        ‘fig/NetResponse5-1.png’, ‘fig/NetResponse7-1.png’,
        ‘fig/vdp-1.png’, ‘main.R’, ‘netresponse.bib’, ‘netresponse.pdf’
    Package has no Sweave vignette sources and no VignetteBuilder field.
    ```

*   checking compiled code ... NOTE
    ```
    File ‘netresponse/libs/netresponse.so’:
      Found ‘_rand’, possibly from ‘rand’ (C)
        Object: ‘netresponse.o’
      Found ‘_srand’, possibly from ‘srand’ (C)
        Object: ‘netresponse.o’
    
    Compiled code should not call entry points which might terminate R nor
    write to stdout/stderr instead of to the console, nor use Fortran I/O
    nor system RNGs.
    
    See ‘Writing portable packages’ in the ‘Writing R Extensions’ manual.
    ```

# netSmooth

Version: 1.0.2

## In both

*   checking whether package ‘netSmooth’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘scater’ was built under R version 3.5.1
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘DelayedArray’ was built under R version 3.5.1
      Warning: namespace ‘bigmemory’ is not available and has been replaced
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/netSmooth/new/netSmooth.Rcheck/00install.out’ for details.
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  6.6Mb
      sub-directories of 1Mb or more:
        data   4.1Mb
        doc    2.3Mb
    ```

# NetworkChange

Version: 0.4

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘plyr’
      All declared Imports should be used.
    ```

# NetworkInference

Version: 1.2.2

## In both

*   checking examples ... ERROR
    ```
    ...
    > ### ** Examples
    > 
    > 
    > data(cascades)
    > result <- netinf(cascades, n_edges = 6, params = 1)
    Initializing trees...
    Getting possible edges...
    0%   10   20   30   40   50   60   70   80   90   100%
    [----|----|----|----|----|----|----|----|----|----|
    **************************************************|
    Inferring 6 edges...
    0%   10   20   30   40   50   60   70   80   90   100%
    [----|----|----|----|----|----|----|----|----|----|
    ********
     *** caught segfault ***
    address 0x1700000000, cause 'memory not mapped'
    
    Traceback:
     1: netinf_(cascade_nodes = cascade_nodes, cascade_times = cascades$cascade_times,     model = model, params = params, n_edges = n_edges, quiet = quiet,     auto_edges = auto_edges, cutoff = cutoff)
     2: netinf(cascades, n_edges = 6, params = 1)
    An irrecoverable exception occurred. R is aborting now ...
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      23: tryCatchList(expr, classes, parentenv, handlers)
      24: tryCatch(withCallingHandlers({    eval(code, test_env)    if (!handled && !is.null(test)) {        skip_empty()    }}, expectation = handle_expectation, skip = handle_skip, warning = handle_warning,     message = handle_message, error = handle_error), error = handle_fatal,     skip = function(e) {    })
      25: test_code(NULL, exprs, env)
      26: source_file(path, new.env(parent = env), chdir = TRUE, wrap = wrap)
      27: force(code)
      28: with_reporter(reporter = reporter, start_end_reporter = start_end_reporter,     {        lister$start_file(basename(path))        source_file(path, new.env(parent = env), chdir = TRUE,             wrap = wrap)        end_context()    })
      29: FUN(X[[i]], ...)
      30: lapply(paths, test_file, env = env, reporter = current_reporter,     start_end_reporter = FALSE, load_helpers = FALSE, wrap = wrap)
      31: force(code)
      32: with_reporter(reporter = current_reporter, results <- lapply(paths,     test_file, env = env, reporter = current_reporter, start_end_reporter = FALSE,     load_helpers = FALSE, wrap = wrap))
      33: test_files(paths, reporter = reporter, env = env, stop_on_failure = stop_on_failure,     stop_on_warning = stop_on_warning, wrap = wrap)
      34: test_dir(path = test_path, reporter = reporter, env = env, filter = filter,     ..., stop_on_failure = stop_on_failure, stop_on_warning = stop_on_warning,     wrap = wrap)
      35: test_package_dir(package = package, test_path = test_path, filter = filter,     reporter = reporter, ..., stop_on_failure = stop_on_failure,     stop_on_warning = stop_on_warning, wrap = wrap)
      36: test_check("NetworkInference")
      An irrecoverable exception occurred. R is aborting now ...
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
     8: timing_fn(handle(ev <- withCallingHandlers(withVisible(eval(expr,     envir, enclos)), warning = wHandler, error = eHandler, message = mHandler)))
     9: evaluate_call(expr, parsed$src[[i]], envir = envir, enclos = enclos,     debug = debug, last = i == length(out), use_try = stop_on_error !=         2L, keep_warning = keep_warning, keep_message = keep_message,     output_handler = output_handler, include_timing = include_timing)
    10: evaluate::evaluate(...)
    11: evaluate(code, envir = env, new_device = FALSE, keep_warning = !isFALSE(options$warning),     keep_message = !isFALSE(options$message), stop_on_error = if (options$error &&         options$include) 0L else 2L, output_handler = knit_handlers(options$render,         options))
    12: in_dir(input_dir(), evaluate(code, envir = env, new_device = FALSE,     keep_warning = !isFALSE(options$warning), keep_message = !isFALSE(options$message),     stop_on_error = if (options$error && options$include) 0L else 2L,     output_handler = knit_handlers(options$render, options)))
    13: block_exec(params)
    14: call_block(x)
    15: process_group.block(group)
    16: process_group(group)
    17: withCallingHandlers(if (tangle) process_tangle(group) else process_group(group),     error = function(e) {        setwd(wd)        cat(res, sep = "\n", file = output %n% "")        message("Quitting from lines ", paste(current_lines(i),             collapse = "-"), " (", knit_concord$get("infile"),             ") ")    })
    18: process_file(text, output)
    19: knitr::knit(knit_input, knit_output, envir = envir, quiet = quiet,     encoding = encoding)
    20: rmarkdown::render(file, encoding = encoding, quiet = quiet, envir = globalenv())
    21: vweave_rmarkdown(...)
    22: engine$weave(file, quiet = quiet, encoding = enc)
    23: doTryCatch(return(expr), name, parentenv, handler)
    24: tryCatchOne(expr, names, parentenv, handlers[[1L]])
    25: tryCatchList(expr, classes, parentenv, handlers)
    26: tryCatch({    engine$weave(file, quiet = quiet, encoding = enc)    setwd(startdir)    find_vignette_product(name, by = "weave", engine = engine)}, error = function(e) {    stop(gettextf("processing vignette '%s' failed with diagnostics:\n%s",         file, conditionMessage(e)), domain = NA, call. = FALSE)})
    27: buildVignettes(dir = "/Users/max/github/ggplot2/revdep/checks.noindex/NetworkInference/new/NetworkInference.Rcheck/vign_test/NetworkInference")
    An irrecoverable exception occurred. R is aborting now ...
    ```

# networktools

Version: 1.2.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘devtools’
      All declared Imports should be used.
    ```

# neuropsychology

Version: 0.5.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘htmlTable’ ‘lme4’ ‘stringi’
      All declared Imports should be used.
    ```

# NFP

Version: 0.99.2

## In both

*   checking whether package ‘NFP’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘graph’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/NFP/new/NFP.Rcheck/00install.out’ for details.
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
        rowMeans, rowSums, rownames, sapply, setdiff, sort, table, tapply,
        union, unique, unsplit, which, which.max, which.min
    
    Loading required package: graphite
    Warning: package 'graphite' was built under R version 3.5.1
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'NFP.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `beramono.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.87 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘NFPdata’
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  8.5Mb
      sub-directories of 1Mb or more:
        data   8.0Mb
    ```

# nima

Version: 0.5.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘ProjectTemplate’ ‘devtools’ ‘plyr’ ‘survival’
      All declared Imports should be used.
    ```

# nimble

Version: 0.6-12

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 17.7Mb
      sub-directories of 1Mb or more:
        CppCode        1.2Mb
        R              7.1Mb
        classic-bugs   1.5Mb
        include        5.6Mb
        tests          1.3Mb
    ```

# nlmixr

Version: 1.0.0-7

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘PreciseSums’ ‘numDeriv’
      All declared Imports should be used.
    ```

# nlstimedist

Version: 1.1.1

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      > library(testthat)
      > library(nlstimedist)
      > 
      > test_check("nlstimedist")
      ── 1. Failure: Ensure the glance method is returning expected values (@test-glan
      `out` not equal to `expect`.
      Rows in x but not y: 1. Rows in y but not x: 1. 
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 6 SKIPPED: 0 FAILED: 1
      1. Failure: Ensure the glance method is returning expected values (@test-glance.timedist.R#19) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

# NlsyLinks

Version: 2.0.6

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'Faq.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `parskip.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.60 \usepackage
                    {amssymb,amsmath} %ftp://ftp.ams.org/ams/doc/amsmath/short-m...
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  8.7Mb
      sub-directories of 1Mb or more:
        data      4.4Mb
        reports   2.2Mb
    ```

# nLTT

Version: 1.4

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘deSolve’
      All declared Imports should be used.
    ```

# NMF

Version: 0.21.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
      ...
    Error in citation(x) : package 'doMPI' not found
    Converted 9 of 10 package citations to BibTeX
    Writing 11 Bibtex entries ... OK
    Results written to file 'Rpackages.bib'
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'NMF-vignette.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `a4wide.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.62 \usepackage
                    {xspace}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘doMPI’
    ```

# noaastormevents

Version: 0.1.0

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘hurricaneexposuredata’
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘RColorBrewer’ ‘XML’ ‘choroplethr’ ‘choroplethrMaps’ ‘data.table’
      ‘forcats’ ‘hurricaneexposure’ ‘plyr’
      All declared Imports should be used.
    ```

# NOAAWeather

Version: 0.1.0

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.5Mb
      sub-directories of 1Mb or more:
        data   5.1Mb
    ```

# NormalizeMets

Version: 0.25

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  9.3Mb
      sub-directories of 1Mb or more:
        doc   8.6Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘knitr’
      All declared Imports should be used.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘DiffCorr’
    ```

# NORRRM

Version: 1.0.0

## In both

*   checking R code for possible problems ... NOTE
    ```
    CIPW: no visible global function definition for ‘data’
    CIPW: no visible binding for global variable ‘AtomWeight’
    CIPW: no visible binding for global variable ‘MinWeight’
    CIPW: no visible binding for global variable ‘OxiWeight’
    CIPW.trace: no visible global function definition for ‘data’
    CIPW.trace: no visible binding for global variable ‘AtomWeight’
    CIPW.trace: no visible binding for global variable ‘MinWeight’
    CIPW.trace: no visible binding for global variable ‘OxiWeight’
    Undefined global functions or variables:
      AtomWeight MinWeight OxiWeight data
    Consider adding
      importFrom("utils", "data")
    to your NAMESPACE file.
    ```

# nucleR

Version: 2.12.1

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    reading file /Users/max/github/ggplot2/revdep/checks.noindex/nucleR/new/nucleR.Rcheck/nucleR/extdata/cellCycleM_chrII_5000-25000.bam
    processing flags
    processing strand +
    processing strand -
    Warning in has_utility("pdfcrop") :
      pdfcrop not installed or not in PATH
    Warning in exp - (ctr - mean(ctr[ctr != 0])) :
      longer object length is not a multiple of shorter object length
    Warning: Removed 1001 rows containing missing values (position_stack).
    tlmgr search --file --global '/beramono.sty'
    Unknown directive ...l
    Warning in system2("tlmgr", args, ...) :
      running command ''tlmgr' search --file --global '/beramono.sty'' had status 25
    ! LaTeX Error: File `beramono.sty' not found.
    
    ! Emergency stop.
    <read *> 
    
    Error: processing vignette 'nucleR.Rmd' failed with diagnostics:
    Failed to compile nucleR.tex. See nucleR.log for more info.
    Execution halted
    ```

# nullabor

Version: 0.3.5

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘forecast’ ‘rlang’ ‘tidyverse’ ‘tsibble’
      All declared Imports should be used.
    ```

# nzelect

Version: 0.4.0

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.3Mb
      sub-directories of 1Mb or more:
        data   5.0Mb
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 6409 marked UTF-8 strings
    ```

# oaxaca

Version: 0.1.4

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'oaxaca.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.55 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# observer

Version: 0.1.2

## In both

*   checking Rd cross-references ... NOTE
    ```
    Packages unavailable to check Rd xrefs: ‘ensurer’, ‘validate’
    ```

# omicRexposome

Version: 1.2.0

## In both

*   R CMD check timed out
    

*   checking whether package ‘omicRexposome’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: namespace ‘omicRexposome’ is not available and has been replaced
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/omicRexposome/new/omicRexposome.Rcheck/00install.out’ for details.
    ```

# OncoSimulR

Version: 2.10.0

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
       Ending warning-mutPropGrowth tests Tue Oct 23 23:50:58 2018 
        Took  3.07 
      
      
       Starting wide2long tests Tue Oct 23 23:50:58 2018 
      
       Ending wide2long tests Tue Oct 23 23:50:59 2018 
        Took  1.33 
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 3204 SKIPPED: 10 FAILED: 1
      1. Failure: We crash as we should (@test.mutPropGrowth.R#482) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  6.6Mb
      sub-directories of 1Mb or more:
        doc   5.4Mb
    ```

# onemap

Version: 2.1.1

## In both

*   R CMD check timed out
    

*   checking whether package ‘onemap’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: loading Rplot failed
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/onemap/new/onemap.Rcheck/00install.out’ for details.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘maps’
      All declared Imports should be used.
    ```

# openPrimeR

Version: 1.2.0

## In both

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 13.5Mb
      sub-directories of 1Mb or more:
        R         2.1Mb
        extdata  10.2Mb
    ```

# opensensmapr

Version: 0.5.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    Warning in bind_rows_(x, .id) :
      binding character and factor vector, coercing into character vector
    Warning: Removed 5 rows containing missing values (geom_path).
    Don't know how to automatically pick scale for object of type difftime. Defaulting to continuous.
    Don't know how to automatically pick scale for object of type difftime. Defaulting to continuous.
    Don't know how to automatically pick scale for object of type difftime. Defaulting to continuous.
    Loading required package: maps
    Loading required package: maptools
    Loading required package: sp
    Checking rgeos availability: TRUE
    Loading required package: rgeos
    rgeos version: 0.4-1, (SVN revision 579)
     GEOS runtime version: 3.6.1-CAPI-1.10.1 
     Linking to sp version: 1.3-1 
     Polygon checking: TRUE 
    
    Quitting from lines 51-56 (osem-intro.Rmd) 
    Error: processing vignette 'osem-intro.Rmd' failed with diagnostics:
    this functions requires additional packages. install them with
        install.packages(c("sf", "maps", "maptools", "rgeos"))
    Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘sf’
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘sf’
    ```

# orderedLasso

Version: 1.7

## In both

*   checking R code for possible problems ... NOTE
    ```
    ...
    plot.orderedLasso.cv: no visible global function definition for
      ‘abline’
    plot.timeLagLasso.cv: no visible global function definition for ‘par’
    plot.timeLagLasso.cv: no visible global function definition for ‘plot’
    plot.timeLagLasso.cv: no visible global function definition for ‘axis’
    plot.timeLagLasso.cv: no visible global function definition for ‘mtext’
    plot.timeLagLasso.cv: no visible global function definition for
      ‘points’
    plot.timeLagLasso.cv: no visible global function definition for
      ‘abline’
    timeLagLasso: no visible binding for global variable ‘sd’
    timeLagLasso.cv: no visible binding for global variable ‘sd’
    timeLagLasso.cv: no visible binding for global variable ‘var’
    timeLagLasso.path: no visible binding for global variable ‘sd’
    Undefined global functions or variables:
      abline approx axis mtext par plot points sd segments var
    Consider adding
      importFrom("graphics", "abline", "axis", "mtext", "par", "plot",
                 "points", "segments")
      importFrom("stats", "approx", "sd", "var")
    to your NAMESPACE file.
    ```

# ordinalClust

Version: 1.3.1

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘methods’
      All declared Imports should be used.
    ```

# ordinalForest

Version: 2.2

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘ggplot2’
      All declared Imports should be used.
    ```

# ORFik

Version: 1.0.0

## In both

*   checking whether package ‘ORFik’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘DelayedArray’ was built under R version 3.5.1
      Warning: package ‘Rsamtools’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/ORFik/new/ORFik.Rcheck/00install.out’ for details.
    ```

# Organism.dplyr

Version: 1.8.1

## In both

*   checking dependencies in R code ... NOTE
    ```
    Unexported objects imported by ':::' calls:
      ‘AnnotationDbi:::smartKeys’ ‘GenomicFeatures:::.exons_with_3utr’
      ‘GenomicFeatures:::.exons_with_5utr’
      ‘GenomicFeatures:::get_TxDb_seqinfo0’
      ‘S4Vectors:::extract_data_frame_rows’
      See the note in ?`:::` about the use of this operator.
    ```

*   checking R code for possible problems ... NOTE
    ```
    .toGRanges: no visible binding for global variable ‘.’
    intronsByTranscript,src_organism: no visible binding for global
      variable ‘.’
    orgPackageName,src_organism: no visible binding for global variable
      ‘name’
    orgPackageName,src_organism: no visible binding for global variable
      ‘organism’
    orgPackageName,src_organism: no visible binding for global variable
      ‘OrgDb’
    Undefined global functions or variables:
      . OrgDb name organism
    ```

# OriGen

Version: 1.4.3

## In both

*   checking compiled code ... NOTE
    ```
    File ‘OriGen/libs/OriGen.so’:
      Found ‘__gfortran_st_close’, possibly from ‘close’ (Fortran)
        Object: ‘OriGen.o’
      Found ‘__gfortran_st_open’, possibly from ‘open’ (Fortran)
        Object: ‘OriGen.o’
      Found ‘__gfortran_st_read’, possibly from ‘read’ (Fortran)
        Object: ‘OriGen.o’
    
    Compiled code should not call entry points which might terminate R nor
    write to stdout/stderr instead of to the console, nor use Fortran I/O
    nor system RNGs.
    
    See ‘Writing portable packages’ in the ‘Writing R Extensions’ manual.
    ```

# osmplotr

Version: 0.3.0

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  6.5Mb
      sub-directories of 1Mb or more:
        doc   5.9Mb
    ```

# otvPlots

Version: 0.2.1

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘quantreg’
      All declared Imports should be used.
    ```

# outbreaker2

Version: 1.0.1

## In both

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘adegenet’
    ```

# OutbreakTools

Version: 0.1-16

## In both

*   checking examples ... ERROR
    ```
    ...
    > x@trees
    1 phylogenetic trees
    > 
    > ## Plotting the dynamic contact network ##
    > par(mfrow=c(2,2))
    > plot(get.contacts(x),main="Contact network - days 0-3",displaylabels=TRUE)
    > plot(get.contacts(x, from=0, to=1.1), main="Contact network - days 0-1",
    +      displaylabels=TRUE)
    > plot(get.contacts(x, from=2, to=2.1), main="Contact network - day 2",
    +      displaylabels=TRUE)
    > plot(get.contacts(x, from=3, to=3.1), main="Contact network - day 3",
    +      displaylabels=TRUE)
    > 
    > ## Mapping the outbreak (by sex) ##
    > plot(x,'geo',location=c('lon','lat'),zoom=13,colorBy='Sex')
    Warning in download.file(url, destfile = tmp, quiet = !messaging, mode = "wb") :
      cannot open URL 'http://maps.googleapis.com/maps/api/staticmap?center=51.516439,-0.171146&zoom=13&size=640x640&scale=2&maptype=terrain&language=en-EN&sensor=false': HTTP status was '403 Forbidden'
    Error in download.file(url, destfile = tmp, quiet = !messaging, mode = "wb") : 
      cannot open URL 'http://maps.googleapis.com/maps/api/staticmap?center=51.516439,-0.171146&zoom=13&size=640x640&scale=2&maptype=terrain&language=en-EN&sensor=false'
    Calls: plot ... plotGeo -> get_map -> get_googlemap -> download.file
    Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
      ...
    Warning in .local(x, ...) :
      The following sequence IDs are not in the dataset: 311, 222
    Warning in .local(x, ...) :
      The following sequence IDs are not in the dataset: 311, 222
    Warning in (if (out_format(c("latex", "sweave", "listings"))) sanitize_fn else paste0)(path,  :
      replaced special characters in figure filename "figs/OutbreakTools-out.width==".8\\textwidth"" -> "figs/OutbreakTools-out.width___.8\\textwidth_"
    Warning in (if (out_format(c("latex", "sweave", "listings"))) sanitize_fn else paste0)(path,  :
      dots in figure paths replaced with _ ("figs/OutbreakTools-out_width____8//textwidth_")
    Warning: The shape palette can deal with a maximum of 6 discrete values because
    more than 6 becomes difficult to discriminate; you have 11. Consider
    specifying shapes manually if you must have them.
    Warning: Removed 58 rows containing missing values (geom_point).
    Warning: `show_guide` has been deprecated. Please use `show.legend` instead.
    Warning: `show_guide` has been deprecated. Please use `show.legend` instead.
    Warning in download.file(url, destfile = tmp, quiet = !messaging, mode = "wb") :
      cannot open URL 'http://maps.googleapis.com/maps/api/staticmap?center=51.516439,-0.171146&zoom=14&size=640x640&scale=2&maptype=terrain&language=en-EN&sensor=false': HTTP status was '403 Forbidden'
    Quitting from lines 1022-1023 (OutbreakTools.Rnw) 
    Error: processing vignette 'OutbreakTools.Rnw' failed with diagnostics:
    cannot open URL 'http://maps.googleapis.com/maps/api/staticmap?center=51.516439,-0.171146&zoom=14&size=640x640&scale=2&maptype=terrain&language=en-EN&sensor=false'
    Execution halted
    ```

# P2C2M

Version: 0.7.6

## In both

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking:
      ‘genealogicalSorting’ ‘phybase’ ‘Rmpi’
    ```

*   checking R code for possible problems ... NOTE
    ```
    ms.exec: no visible global function definition for ‘tail’
    p2c2m.readstarb: no visible global function definition for ‘head’
    p2c2m.readstarb: no visible binding for global variable ‘head’
    rmOutlrs: no visible global function definition for ‘boxplot.stats’
    rmext: no visible global function definition for ‘tail’
    rowMedians: no visible binding for global variable ‘median’
    stats.acrGenes: no visible binding for global variable ‘sd’
    stats.perGene: no visible binding for global variable ‘sd’
    statshelpers.cv: no visible binding for global variable ‘sd’
    statshelpers.qntls: no visible binding for global variable ‘quantile’
    Undefined global functions or variables:
      boxplot.stats head median quantile sd tail
    Consider adding
      importFrom("grDevices", "boxplot.stats")
      importFrom("stats", "median", "quantile", "sd")
      importFrom("utils", "head", "tail")
    to your NAMESPACE file.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 490 marked UTF-8 strings
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    Warning: Removed 575 rows containing non-finite values
    (stat_density).
    Warning: Removed 2 rows containing missing values (geom_vline).
    Warning: Removed 370 rows containing missing values (geom_point).
    Warning: Removed 370 rows containing missing values (geom_point).
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'P2C2M_Vignette.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `fullpage.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.5 \usepackage
                   {listings}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# pa

Version: 1.2-1

## In both

*   checking dependencies in R code ... NOTE
    ```
    Package in Depends field not imported from: ‘grid’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

*   checking R code for possible problems ... NOTE
    ```
    .formula.make: no visible global function definition for ‘formula’
    regress: no visible global function definition for ‘lm’
    regress: no visible global function definition for ‘model.matrix’
    Undefined global functions or variables:
      formula lm model.matrix
    Consider adding
      importFrom("stats", "formula", "lm", "model.matrix")
    to your NAMESPACE file.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Loading required package: grid
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'pa.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.30 \ifthenelse
                    {\boolean{Sweave@inconsolata}}{%^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# paleofire

Version: 1.2.2

## In both

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    trying URL 'http://blarquez.com/public/data/data_cageo.zip'
    Content type 'application/zip' length 14857 bytes (14 KB)
    ==================================================
    downloaded 14 KB
    
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'paleofire-paper.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `pifont.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.102 \newcounter
                     {tnote}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# paletteer

Version: 0.1.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘jcolors’ ‘oompaBase’ ‘palr’ ‘pals’ ‘viridisLite’
      All declared Imports should be used.
    ```

# pandaR

Version: 1.12.0

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  9.3Mb
      sub-directories of 1Mb or more:
        data   9.0Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    dFunction: no visible global function definition for ‘cor’
    importPandaMatlab: no visible global function definition for
      ‘read.delim’
    panda: no visible global function definition for ‘cor’
    panda: no visible global function definition for ‘aggregate’
    plot.panda: no visible global function definition for ‘hist’
    plotCommunityDetection: no visible global function definition for
      ‘title’
    plotZbyTF: no visible global function definition for ‘aggregate’
    prepResult: no visible global function definition for ‘pnorm’
    Undefined global functions or variables:
      aggregate cor hist pnorm read.delim title
    Consider adding
      importFrom("graphics", "hist", "title")
      importFrom("stats", "aggregate", "cor", "pnorm")
      importFrom("utils", "read.delim")
    to your NAMESPACE file.
    ```

*   checking Rd \usage sections ... NOTE
    ```
    S3 methods shown with full name in documentation object 'plot.panda':
      ‘plot.panda’
    
    The \usage entries for S3 methods should use the \method markup and not
    their full name.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

# panelvar

Version: 0.5.1

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘methods’
      All declared Imports should be used.
    ```

# parallelDist

Version: 0.2.2

## In both

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Loading required package: ggplot2
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'parallelDist.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `microtype.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.21 \usepackage
                    [T1]{fontenc}		   %% cf http://www.khirevich.com/latex/font/^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# parlitools

Version: 0.2.1

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘sf’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# parmsurvfit

Version: 0.0.1

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘flexsurv’
      All declared Imports should be used.
    ```

# parsemsf

Version: 0.1.1

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘dbplyr’
      All declared Imports should be used.
    ```

# pathfindR

Version: 1.2.2

## In both

*   checking whether package ‘pathfindR’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘IRanges’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/pathfindR/new/pathfindR.Rcheck/00install.out’ for details.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 12.2Mb
      sub-directories of 1Mb or more:
        data      1.5Mb
        extdata   9.5Mb
    ```

# pathVar

Version: 1.10.0

## In both

*   checking R code for possible problems ... NOTE
    ```
    ...
    plotOneSample: no visible binding for global variable ‘Number_of_genes’
    plotTwoSamplesCont: no visible binding for global variable ‘PwayName’
    plotTwoSamplesCont: no visible binding for global variable
      ‘NumOfGenesFromDataSetInPathway’
    plotTwoSamplesCont: no visible binding for global variable ‘value’
    plotTwoSamplesCont: no visible binding for global variable
      ‘..density..’
    plotTwoSamplesCont: no visible binding for global variable ‘group’
    plotTwoSamplesDisc: no visible binding for global variable ‘Cluster’
    plotTwoSamplesDisc: no visible binding for global variable
      ‘Number_of_genes’
    sigOneSample: no visible binding for global variable ‘APval’
    sigOneSample: no visible binding for global variable ‘PwayName’
    sigTwoSamplesCont: no visible binding for global variable ‘APval’
    sigTwoSamplesCont: no visible binding for global variable ‘PwayName’
    sigTwoSamplesDisc: no visible binding for global variable ‘APval’
    sigTwoSamplesDisc: no visible binding for global variable ‘PwayName’
    Undefined global functions or variables:
      ..density.. APval Cluster NumOfGenesFromDataSetInPathway
      Number_of_genes PercOfGenesInPway PwayName avg cv group medAbsDev
      standDev value
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    Error in re-building vignettes:
      ...
    Loading required package: ggplot2
    Loading required package: gridExtra
    Using X1, X2 as id variables
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'pathVar.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `a4wide.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.11 \usepackage
                    [utf8x]{inputenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# patPRO

Version: 1.1.0

## In both

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Checking should be performed on sources prepared by ‘R CMD build’.
    ```

# patternplot

Version: 0.2.1

## In both

*   checking for GNU extensions in Makefiles ... NOTE
    ```
    GNU make is a SystemRequirements.
    ```

# pauwels2014

Version: 1.0

## In both

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Title field: should not end in a period.
    ```

*   checking R code for possible problems ... NOTE
    ```
    add_noise: no visible global function definition for ‘rnorm’
    estimate_risk_dream6 : <anonymous>: no visible global function
      definition for ‘var’
    generate_sample: no visible global function definition for ‘rnorm’
    generate_sample: no visible global function definition for ‘runif’
    sample_function: no visible global function definition for ‘runif’
    sample_function : <anonymous>: no visible global function definition
      for ‘var’
    sample_function_multi_mod_weight: no visible global function definition
      for ‘runif’
    sample_function_multi_mod_weight : <anonymous>: no visible global
      function definition for ‘var’
    sample_function_single_mod: no visible global function definition for
      ‘runif’
    Undefined global functions or variables:
      rnorm runif var
    Consider adding
      importFrom("stats", "rnorm", "runif", "var")
    to your NAMESPACE file.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Loading required package: deSolve
    Loading required package: ggplot2
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'pauwels2014.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.30 \ifthenelse
                    {\boolean{Sweave@inconsolata}}{%^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# PAWL

Version: 0.5

## In both

*   checking whether the namespace can be loaded with stated dependencies ... NOTE
    ```
    Warning: no function found corresponding to methods exports from ‘PAWL’ for: ‘show’
    
    A namespace must be able to be loaded with just the base namespace
    loaded: otherwise if the namespace gets loaded by a saved object, the
    session will be unable to start.
    
    Probably some imports need to be declared in the NAMESPACE file.
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' calls to packages already attached by Depends:
      ‘foreach’ ‘ggplot2’ ‘reshape’
      Please remove these calls from your code.
    Packages in Depends field not imported from:
      ‘foreach’ ‘ggplot2’ ‘methods’ ‘mvtnorm’ ‘reshape’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    smcparameters.constructor: no visible global function definition for
      ‘new’
    systematic_resampling: no visible global function definition for
      ‘runif’
    target.constructor: no visible global function definition for ‘new’
    tuningparameters.constructor: no visible global function definition for
      ‘new’
    show,smcparameters: no visible global function definition for ‘tail’
    Undefined global functions or variables:
      %do% abline aes aes_string cov element_text facet_wrap foreach
      geom_density2d geom_line geom_point geom_step geom_vline ggplot hist
      labs mahalanobis melt new quantile rgamma rmultinom rmvnorm rnorm
      runif stat_bin2d tail theme xlab ylab ylim
    Consider adding
      importFrom("graphics", "abline", "hist")
      importFrom("methods", "new")
      importFrom("stats", "cov", "mahalanobis", "quantile", "rgamma",
                 "rmultinom", "rnorm", "runif")
      importFrom("utils", "tail")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

# Pbase

Version: 0.20.0

## In both

*   checking whether package ‘Pbase’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/Pbase/new/Pbase.Rcheck/00install.out’ for details.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.5Mb
      sub-directories of 1Mb or more:
        data      1.7Mb
        extdata   2.2Mb
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Authors@R field gives more than one person with maintainer role:
      Laurent Gatto <lg390@cam.ac.uk> [aut, cre]
      Johannes Rainer <Johannes.Rainer@eurac.edu> [aut, cre]
      Sebastian Gibb <mail@sebastiangibb.de> [aut, cre]
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Failed with error:  'package 'Gviz' could not be loaded'
      Error in .requirePackage(package) : 
        unable to find required package 'Pbase'
      Calls: <Anonymous> ... getClass -> getClassDef -> .classEnv -> .requirePackage
      Execution halted
    ```

# pbcmc

Version: 1.8.0

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/runTests.R’ failed.
    Last 13 lines of output:
          union, unique, unsplit, which, which.max, which.min
      
      Welcome to Bioconductor
      
          Vignettes contain introductory material; view with
          'browseVignettes()'. To cite Bioconductor, see
          'citation("Biobase")', and for packages 'citation("pkgname")'.
      
      No methods found in package 'BiocGenerics' for request: 'unlist' when loading 'pbcmc'
      Error in library("RUnit", quietly = TRUE) : 
        there is no package called 'RUnit'
      Calls: <Anonymous> -> library
      In addition: Warning message:
      package 'limma' was built under R version 3.5.1 
      Execution halted
    ```

*   checking whether package ‘pbcmc’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘limma’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/pbcmc/new/pbcmc.Rcheck/00install.out’ for details.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    
    Global results:
    Assigned 
           5 
    Particular results:
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'pbcmc-vignette.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.55 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# pcadapt

Version: 4.0.3

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  6.6Mb
      sub-directories of 1Mb or more:
        doc       2.9Mb
        extdata   2.7Mb
    ```

# PCADSC

Version: 0.8.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘Matrix’ ‘pander’
      All declared Imports should be used.
    ```

# pcaExplorer

Version: 2.6.0

## In both

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  7.0Mb
      sub-directories of 1Mb or more:
        doc   6.6Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    pcaExplorer: no visible binding for '<<-' assignment to
      ‘pcaexplorer_env’
    pcaExplorer : <anonymous>: no visible binding for global variable
      ‘airway’
    pcaExplorer : <anonymous>: no visible binding for global variable
      ‘pcaexplorer_env’
    Undefined global functions or variables:
      airway pcaexplorer_env
    ```

# pcalg

Version: 2.6-0

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  6.0Mb
      sub-directories of 1Mb or more:
        data   2.1Mb
        doc    1.1Mb
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘unifDAG’
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    
    Warning: allDags() is deprecated and only kept for backward compatibility. Please use pdag2allDags() instead
    
    Warning: allDags() is deprecated and only kept for backward compatibility. Please use pdag2allDags() instead
    
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'pcalgDoc.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.55 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# pcaMethods

Version: 1.72.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    ...
    LaTeX errors found:
    ! Please use \mathaccent for accents in math mode.
    \add@accent ...@spacefactor \spacefactor }\accent 
                                                      #1 #2\egroup \spacefactor ...
    l.603 ...rcum{}2}{}).}{cvstat,pcaRes.Rdash.method}
                                                      
    ! Missing { inserted.
    <to be read again> 
                       \egroup 
    l.603 ...rcum{}2}{}).}{cvstat,pcaRes.Rdash.method}
                                                      
    ! You can't use `\spacefactor' in math mode.
    \add@accent ...}\accent #1 #2\egroup \spacefactor 
                                                      \accent@spacefactor 
    l.603 ...rcum{}2}{}).}{cvstat,pcaRes.Rdash.method}
                                                      
    ! Missing } inserted.
    <inserted text> 
                    }
    l.603 ...rcum{}2}{}).}{cvstat,pcaRes.Rdash.method}
                                                      
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    svdImpute: no visible global function definition for ‘prcomp’
    svdPca: no visible global function definition for ‘prcomp’
    plot,pcaRes: no visible global function definition for ‘gray’
    plot,pcaRes: no visible global function definition for ‘barplot’
    plot,pcaRes: no visible global function definition for ‘legend’
    slplot,pcaRes: no visible global function definition for ‘par’
    slplot,pcaRes: no visible global function definition for ‘layout’
    slplot,pcaRes: no visible global function definition for ‘abline’
    slplot,pcaRes: no visible global function definition for ‘lines’
    Undefined global functions or variables:
      abline barplot cor cov gray layout legend lines median na.omit pairs
      par points prcomp qf rnorm runif setTxtProgressBar text
      txtProgressBar
    Consider adding
      importFrom("grDevices", "gray")
      importFrom("graphics", "abline", "barplot", "layout", "legend",
                 "lines", "pairs", "par", "points", "text")
      importFrom("stats", "cor", "cov", "median", "na.omit", "prcomp", "qf",
                 "rnorm", "runif")
      importFrom("utils", "setTxtProgressBar", "txtProgressBar")
    to your NAMESPACE file.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    
    The following object is masked from ‘package:stats’:
    
        loadings
    
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'missingValues.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.30 \ifthenelse
                    {\boolean{Sweave@inconsolata}}{%^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# pcaPA

Version: 2.0.2

## In both

*   checking whether package ‘pcaPA’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/pcaPA/new/pcaPA.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘pcaPA’ ...
** package ‘pcaPA’ successfully unpacked and MD5 sums checked
** libs
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG `gsl-config --cflags`  -I/usr/local/include   -fPIC  -Wall -g -O2  -c Cpolychoric.cpp -o Cpolychoric.o
/bin/sh: gsl-config: command not found
Cpolychoric.cpp:36:10: fatal error: 'gsl/gsl_math.h' file not found
#include <gsl/gsl_math.h>
         ^~~~~~~~~~~~~~~~
1 error generated.
make: *** [Cpolychoric.o] Error 1
ERROR: compilation failed for package ‘pcaPA’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/pcaPA/new/pcaPA.Rcheck/pcaPA’

```
### CRAN

```
* installing *source* package ‘pcaPA’ ...
** package ‘pcaPA’ successfully unpacked and MD5 sums checked
** libs
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG `gsl-config --cflags`  -I/usr/local/include   -fPIC  -Wall -g -O2  -c Cpolychoric.cpp -o Cpolychoric.o
/bin/sh: gsl-config: command not found
Cpolychoric.cpp:36:10: fatal error: 'gsl/gsl_math.h' file not found
#include <gsl/gsl_math.h>
         ^~~~~~~~~~~~~~~~
1 error generated.
make: *** [Cpolychoric.o] Error 1
ERROR: compilation failed for package ‘pcaPA’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/pcaPA/old/pcaPA.Rcheck/pcaPA’

```
# pcrsim

Version: 1.0.2

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘strvalidator’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# PDN

Version: 0.1.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘glmnet’
      All declared Imports should be used.
    ```

# pdp

Version: 0.7.0

## In both

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘ICEbox’
    ```

# PDQutils

Version: 0.1.6

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    Error in re-building vignettes:
      ...
    large z may result in inaccurate quantiles
    Loading required package: polynom
    Loading required package: reshape2
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'PDQutils.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `paralist.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.98 ^^M
            
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# peakPantheR

Version: 1.2.0

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      18: f(..., self = self)
      19: train_cartesian(scale_x, self$limits$x, "x")
      20: scale$break_info(range)
      21: f(..., self = self)
      22: ggproto_parent(ScaleContinuous, self)$break_info(range)
      23: f(..., self = self)
      24: self$get_breaks_minor(b = major, limits = range)
      25: f(..., self = self)
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 1148 SKIPPED: 0 FAILED: 1
      1. Error: default multiplot (@test_annotationDiagnosticMultiplot.R#180) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

# PedCNV

Version: 0.1

## In both

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Title field: should not end in a period.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Packages in Depends field not imported from:
      ‘Rcpp’ ‘RcppArmadillo’ ‘ggplot2’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    Inde: no visible global function definition for ‘var’
    STE: no visible global function definition for ‘pchisq’
    STIM: no visible global function definition for ‘cor’
    STIM: no visible global function definition for ‘pchisq’
    plot.clust: no visible global function definition for ‘princomp’
    plot.clust: no visible global function definition for ‘loadings’
    plot.clust: no visible global function definition for ‘qplot’
    plot.clust: no visible global function definition for ‘geom_histogram’
    plot.clust: no visible global function definition for ‘aes_string’
    plot.clust: no visible global function definition for ‘barplot’
    plot.clust: no visible global function definition for ‘title’
    plot.clust: no visible global function definition for ‘mtext’
    plot.clust: no visible global function definition for ‘text’
    Undefined global functions or variables:
      aes_string barplot cor cov geom_histogram kmeans lm loadings mtext
      pchisq prcomp princomp qplot rbinom text title var
    Consider adding
      importFrom("graphics", "barplot", "mtext", "text", "title")
      importFrom("stats", "cor", "cov", "kmeans", "lm", "loadings", "pchisq",
                 "prcomp", "princomp", "rbinom", "var")
    to your NAMESPACE file.
    ```

# pepStat

Version: 1.14.0

## In both

*   checking whether package ‘pepStat’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘IRanges’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/pepStat/new/pepStat.Rcheck/00install.out’ for details.
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    The following object is masked from 'package:pepStat':
    
        position
    
    Some peptides have no match in the GRanges object rownames and are removed from the peptideSet!
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'pepStat.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `fullpage.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.60 \usepackage
                    {graphicx}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Description field: should contain one or more complete sentences.
    ```

*   checking whether the namespace can be loaded with stated dependencies ... NOTE
    ```
    Warning: no function found corresponding to methods exports from ‘pepStat’ for: ‘end’, ‘start’
    
    A namespace must be able to be loaded with just the base namespace
    loaded: otherwise if the namespace gets loaded by a saved object, the
    session will be unable to start.
    
    Probably some imports need to be declared in the NAMESPACE file.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    coerce,peptideSet-ExpressionSet: no visible global function definition
      for ‘annotation’
    end,peptideSet: no visible global function definition for ‘end’
    position,peptideSet: no visible global function definition for ‘start’
    position,peptideSet: no visible global function definition for ‘end’
    start,peptideSet: no visible global function definition for ‘start’
    write.pSet,peptideSet: no visible global function definition for
      ‘start’
    write.pSet,peptideSet: no visible global function definition for ‘end’
    write.pSet,peptideSet: no visible global function definition for
      ‘write.csv’
    Undefined global functions or variables:
      annotation dev.flush dev.hold dev.interactive devAskNewPage end
      lm.fit lm.wfit mcols mcols<- median read.csv sd start write.csv
    Consider adding
      importFrom("grDevices", "dev.flush", "dev.hold", "dev.interactive",
                 "devAskNewPage")
      importFrom("stats", "end", "lm.fit", "lm.wfit", "median", "sd",
                 "start")
      importFrom("utils", "read.csv", "write.csv")
    to your NAMESPACE file.
    ```

# perry

Version: 0.2.0

## In both

*   checking R code for possible problems ... NOTE
    ```
    hasMethod: no visible global function definition for ‘getS3method’
    Undefined global functions or variables:
      getS3method
    Consider adding
      importFrom("utils", "getS3method")
    to your NAMESPACE file.
    ```

# petro.One

Version: 0.1.3

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      > library(testthat)
      > library(petro.One)
      > 
      > test_check("petro.One")
      ── 1. Failure: make_search_url matches number of papers from_year to_year (@test
      get_papers_count(my_url) is not more than `expected`. Difference: -1
      
      [1] 2
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 93 SKIPPED: 0 FAILED: 1
      1. Failure: make_search_url matches number of papers from_year to_year (@test_from_year_to_year.R#12) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘Rgraphviz’ ‘cluster’ ‘graph’
      All declared Imports should be used.
    ```

# PGA

Version: 1.10.1

## In both

*   checking whether package ‘PGA’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/PGA/new/PGA.Rcheck/00install.out’ for details.
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    Warning: Ignoring unknown aesthetics: position
    Warning: Ignoring unknown aesthetics: position
    INDEL(DB) didn't exist!
    Warning: Ignoring unknown aesthetics: position
    Warning: Ignoring unknown aesthetics: position
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'PGA.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `beramono.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.87 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.7Mb
      sub-directories of 1Mb or more:
        extdata   1.8Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Unexported objects imported by ':::' calls:
      ‘biomaRt:::martBM’ ‘biomaRt:::martDataset’ ‘biomaRt:::martHost’
      ‘customProDB:::makeTranscriptDbFromBiomart_archive’
      See the note in ?`:::` about the use of this operator.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    reportSNV: no visible binding for global variable ‘isUnique’
    reportSNV: no visible binding for global variable ‘Index’
    reportSNV: no visible binding for global variable ‘aaref’
    reportSNV: no visible binding for global variable ‘aavar’
    reportSNV: no visible binding for global variable ‘genename’
    reportSNV: no visible binding for global variable ‘proname’
    reportSNV: no visible binding for global variable ‘.SD’
    reportSNV: no visible binding for global variable ‘ID’
    reportSNV: no visible binding for global variable ‘Change’
    reportSNV: no visible binding for global variable ‘aapos’
    reportSNV: no visible binding for global variable ‘abc’
    reportSNV: no visible binding for global variable ‘xyz’
    Undefined global functions or variables:
      . .I .N .SD CUFF_ID Change Class Evalue Frame Freq ID Index Mass
      MutNum Query Qvalue Strand Substring Type aapos aaref aavar abc
      alleleCount alleles charge chr chrom cumlen delta_da delta_ppm evalue
      gene_name genename genome<- id isSAP isUnique junType jun_type label
      miss mods mrnaAcc mz name output pep peptide pincoding position
      pro_name proname prot protAcc protein rbindlist readAAStringSet
      readDNAStringSet refbase rsid seqlengths seqlevels seqlevels<- subseq
      transcript tx_name txid txname varbase writeXStringSet x xyz y
    ```

# PGPC

Version: 1.8.0

## In both

*   R CMD check timed out
    

*   checking whether package ‘PGPC’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘EBImage’ was built under R version 3.5.1
      Warning: package ‘limma’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/PGPC/new/PGPC.Rcheck/00install.out’ for details.
    ```

*   checking package dependencies ... NOTE
    ```
    Depends: includes the non-default packages:
      ‘EBImage’ ‘imageHTS’ ‘SearchTrees’ ‘limma’ ‘RColorBrewer’ ‘gplots’
      ‘splots’ ‘ggplot2’ ‘geneplotter’ ‘ChemmineR’ ‘reshape2’ ‘plyr’
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 251.8Mb
      sub-directories of 1Mb or more:
        data     172.8Mb
        doc        7.9Mb
        extdata   70.9Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Packages in Depends field not imported from:
      ‘ChemmineR’ ‘RColorBrewer’ ‘geneplotter’ ‘ggplot2’ ‘gplots’ ‘plyr’
      ‘reshape2’ ‘splots’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    Unexported object imported by a ':::' call: ‘imageHTS:::writeThumbnail’
      See the note in ?`:::` about the use of this operator.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    summarizeWellsExtended: no visible binding for global variable ‘sd’
    summarizeWellsExtended: no visible binding for global variable
      ‘quantile’
    summarizeWellsExtended: no visible global function definition for
      ‘write.table’
    Undefined global functions or variables:
      abline aes as.dist colorRampPalette concdrug1.micro_M. cor data
      dev.off element_blank element_text experiment geom_bar geom_errorbar
      geom_jitter geom_line geom_point geom_vline ggplot gpar grid.raster
      grid.text hclust heatmap.2 identifier inhibition labs lm mad
      medpolish p.adjust plot png points popViewport position_dodge
      position_jitter pushViewport pvalue quantile scale_colour_manual
      scale_fill_manual scale_x_log10 scale_y_continuous sd sem theme
      theme_bw unit value variable viewport write.table xlab ylab
    Consider adding
      importFrom("grDevices", "colorRampPalette", "dev.off", "png")
      importFrom("graphics", "abline", "plot", "points")
      importFrom("stats", "as.dist", "cor", "hclust", "lm", "mad",
                 "medpolish", "p.adjust", "quantile", "sd")
      importFrom("utils", "data", "write.table")
    to your NAMESPACE file.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 28 marked Latin-1 strings
    ```

# PGRdup

Version: 0.2.3.3

## In both

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘RecordLinkage’
    ```

# phantasus

Version: 1.0.2

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 18.0Mb
      sub-directories of 1Mb or more:
        doc   4.6Mb
        www  12.8Mb
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Package listed in more than one of Depends, Imports, Suggests, Enhances:
      ‘svglite’
    A package should be listed in only one of these fields.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Unexported objects imported by ':::' calls:
      'GEOquery:::getDirListing' 'opencpu:::rookhandler'
      'opencpu:::tmp_root' 'opencpu:::win_or_mac'
      See the note in ?`:::` about the use of this operator.
    ```

*   checking R code for possible problems ... NOTE
    ```
    gseaPlot: no visible global function definition for 'setNames'
    read.gct: no visible global function definition for 'head'
    Undefined global functions or variables:
      head setNames
    Consider adding
      importFrom("stats", "setNames")
      importFrom("utils", "head")
    to your NAMESPACE file.
    ```

# phase1RMD

Version: 1.0.8

## In both

*   checking whether package ‘phase1RMD’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/phase1RMD/new/phase1RMD.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘phase1RMD’ ...
** package ‘phase1RMD’ successfully unpacked and MD5 sums checked
** R
** data
** byte-compile and prepare package for lazy loading
Error: package or namespace load failed for ‘rjags’:
 .onLoad failed in loadNamespace() for 'rjags', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/phase1RMD/rjags/libs/rjags.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/phase1RMD/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/phase1RMD/rjags/libs/rjags.so
  Reason: image not found
Error : package ‘rjags’ could not be loaded
ERROR: lazy loading failed for package ‘phase1RMD’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/phase1RMD/new/phase1RMD.Rcheck/phase1RMD’

```
### CRAN

```
* installing *source* package ‘phase1RMD’ ...
** package ‘phase1RMD’ successfully unpacked and MD5 sums checked
** R
** data
** byte-compile and prepare package for lazy loading
Error: package or namespace load failed for ‘rjags’:
 .onLoad failed in loadNamespace() for 'rjags', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/phase1RMD/rjags/libs/rjags.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/phase1RMD/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/phase1RMD/rjags/libs/rjags.so
  Reason: image not found
Error : package ‘rjags’ could not be loaded
ERROR: lazy loading failed for package ‘phase1RMD’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/phase1RMD/old/phase1RMD.Rcheck/phase1RMD’

```
# PhaseType

Version: 0.1.3

## In both

*   checking whether package ‘PhaseType’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      LJMA_arms.c:297:11: warning: comparison of array 'env->p' equal to a null pointer is always false [-Wtautological-pointer-compare]
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/PhaseType/new/PhaseType.Rcheck/00install.out’ for details.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Packages in Depends field not imported from:
      ‘ggplot2’ ‘reshape’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

*   checking R code for possible problems ... NOTE
    ```
    plot.phtMCMC: no visible global function definition for ‘ggplot’
    plot.phtMCMC: no visible global function definition for ‘melt’
    plot.phtMCMC: no visible global function definition for ‘geom_line’
    plot.phtMCMC: no visible global function definition for ‘aes_string’
    plot.phtMCMC: no visible global function definition for ‘geom_smooth’
    plot.phtMCMC: no visible global function definition for ‘facet_wrap’
    plot.phtMCMC: no visible global function definition for ‘ggtitle’
    plot.phtMCMC: no visible global function definition for ‘xlab’
    plot.phtMCMC: no visible global function definition for ‘ylab’
    plot.phtMCMC: no visible global function definition for ‘geom_density’
    Undefined global functions or variables:
      aes_string facet_wrap geom_density geom_line geom_smooth ggplot
      ggtitle melt xlab ylab
    ```

*   checking compiled code ... NOTE
    ```
    File ‘PhaseType/libs/PhaseType.so’:
      Found ‘_rand’, possibly from ‘rand’ (C)
        Object: ‘LJMA_arms.o’
    
    Compiled code should not call entry points which might terminate R nor
    write to stdout/stderr instead of to the console, nor use Fortran I/O
    nor system RNGs.
    
    See ‘Writing portable packages’ in the ‘Writing R Extensions’ manual.
    ```

# phenoTest

Version: 1.28.0

## In both

*   checking whether package ‘phenoTest’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘IRanges’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/phenoTest/new/phenoTest.Rcheck/00install.out’ for details.
    ```

*   checking package dependencies ... NOTE
    ```
    Packages which this enhances but not available for checking:
      ‘org.Ce.eg.db’ ‘org.Rn.eg.db’ ‘org.Dm.eg.db’
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Title field: should not end in a period.
    Packages listed in more than one of Depends, Imports, Suggests, Enhances:
      ‘annotate’ ‘GSEABase’
    A package should be listed in only one of these fields.
    ```

*   checking whether the namespace can be loaded with stated dependencies ... NOTE
    ```
    Warning: no function found corresponding to methods exports from ‘phenoTest’ for: ‘show’
    
    A namespace must be able to be loaded with just the base namespace
    loaded: otherwise if the namespace gets loaded by a saved object, the
    session will be unable to start.
    
    Probably some imports need to be declared in the NAMESPACE file.
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' call to ‘ggplot2’ which was already attached by Depends.
      Please remove these calls from your code.
    Namespace in Imports field not imported from: ‘annotate’
      All declared Imports should be used.
    Packages in Depends field not imported from:
      ‘BMA’ ‘Heatplus’ ‘annotate’ ‘ggplot2’ ‘methods’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    There are ::: calls to the package's namespace in its code. A package
      almost never needs to use ::: for its own objects:
      ‘sortDragHtmlTable’
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      gray hclust heatmap_plus image integrate kruskal.test layout legend
      lines lm median model.matrix new p.adjust pData par pc1 pc2 pchisq
      pdf png pnorm polygon prcomp predict qplot quantile rect rgb sd text
      theme wilcox.test write.csv y
    Consider adding
      importFrom("grDevices", "densCols", "dev.off", "gray", "pdf", "png",
                 "rgb")
      importFrom("graphics", "abline", "axTicks", "axis", "barplot",
                 "boxplot", "dotchart", "image", "layout", "legend", "lines",
                 "par", "polygon", "rect", "text")
      importFrom("methods", "new")
      importFrom("stats", "BIC", "anova", "approx", "approxfun",
                 "as.dendrogram", "as.dist", "binom.test", "chisq.test",
                 "coef", "coefficients", "complete.cases", "confint", "cor",
                 "cov", "cutree", "density", "dist", "hclust", "integrate",
                 "kruskal.test", "lm", "median", "model.matrix", "p.adjust",
                 "pchisq", "pnorm", "prcomp", "predict", "quantile", "sd",
                 "wilcox.test")
      importFrom("utils", "write.csv")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

*   checking Rd \usage sections ... NOTE
    ```
    S3 methods shown with full name in documentation object 'plot.gseaData':
      ‘plot.gseaData’
    
    S3 methods shown with full name in documentation object 'plot.gseaSignatures':
      ‘plot.gseaSignaturesSign’
    
    S3 methods shown with full name in documentation object 'summary.gseaData':
      ‘summary.gseaData’
    
    S3 methods shown with full name in documentation object 'summary.gseaSignificance':
      ‘summary.gseaSignificanceSign’
    
    The \usage entries for S3 methods should use the \method markup and not
    their full name.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
        addNode
    
    Warning in checkGsetSimmetry(x, gsets, test, B) :
      The provided score is strongly not simmetric or is not centered on 0. You should set paremeter 'center' to TRUE.
    
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'phenoTest.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.30 \ifthenelse
                    {\boolean{Sweave@inconsolata}}{%^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# PhenStat

Version: 2.17.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘SmoothWin’
      All declared Imports should be used.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    
    
    
    
    
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'PhenStat.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `times.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.7 \usepackage
                   {a4wide}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# philr

Version: 1.6.0

## In both

*   checking R code for possible problems ... NOTE
    ```
    name.balance: no visible global function definition for ‘as’
    vote.annotation: no visible global function definition for ‘is’
    Undefined global functions or variables:
      as is
    Consider adding
      importFrom("methods", "as", "is")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘compositions’
    ```

# photobiologyFilters

Version: 0.4.4

## In both

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 1 marked UTF-8 string
    ```

# photobiologyLamps

Version: 0.4.2

## In both

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘photobiologyLEDs’
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 1 marked Latin-1 string
    ```

# photobiologyPlants

Version: 0.4.1

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
      Only 'total' transmittance can be meaningfully plotted in a combined plot
    Warning in O_plot(spct = x, w.band = w.band, range = range, pc.out = pc.out,  :
      Only 'total' transmittance can be meaningfully plotted in a combined plot
    Warning in O_plot(spct = x, w.band = w.band, range = range, pc.out = pc.out,  :
      Only 'total' transmittance can be meaningfully plotted in a combined plot
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'dataplotsPlants.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `listings.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.64 \usepackage
                    {booktabs}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# PhyInformR

Version: 1.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘gplots’ ‘phytools’
      All declared Imports should be used.
    ```

# phyloseq

Version: 1.24.2

## In both

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  9.3Mb
      sub-directories of 1Mb or more:
        doc   6.5Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    merge_phyloseq_pair,sample_data-sample_data : <anonymous>: no visible
      binding for global variable ‘X0’
    merge_samples,sample_data: no visible global function definition for
      ‘aggregate’
    plot_phyloseq,phyloseq: no visible binding for global variable
      ‘esophagus’
    Undefined global functions or variables:
      #OTU ID .SD := Abundance Classification Consensus Lineage J OTU
      OTULabel SE.sim Sample V1 V2 X0 aggregate as.dist as.formula
      as.hclust axis capture.output combn complete.cases count cutree
      dcast.data.table download.file eigenvalue esophagus gap h.adj.index
      head i k label queryID queryString read read.table relevel se tail
      untar unzip value vmax vmin write.table x xdodge xend xfartiplab
      xleft xright y yend
    Consider adding
      importFrom("graphics", "axis")
      importFrom("stats", "aggregate", "as.dist", "as.formula", "as.hclust",
                 "complete.cases", "cutree", "relevel")
      importFrom("utils", "capture.output", "combn", "download.file", "head",
                 "read.table", "tail", "untar", "unzip", "write.table")
    to your NAMESPACE file.
    ```

# phyloseqGraphTest

Version: 0.0.2

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘intergraph’
      All declared Imports should be used.
    ```

# phylosim

Version: 3.0.2

## In both

*   R CMD check timed out
    

# Pi

Version: 1.8.0

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘XGR’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# pifpaf

Version: 1.0.1

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘gridExtra’
      All declared Imports should be used.
    ```

# pitchRx

Version: 1.8.2

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘ggsubplot’
    ```

# pivottabler

Version: 1.0.0

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  6.1Mb
      sub-directories of 1Mb or more:
        data   2.0Mb
        doc    2.9Mb
    ```

# PKgraph

Version: 1.7

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available:
      ‘RGtk2’ ‘gWidgetsRGtk2’ ‘cairoDevice’ ‘rggobi’
    
    Depends: includes the non-default packages:
      ‘RGtk2’ ‘gWidgetsRGtk2’ ‘cairoDevice’ ‘lattice’ ‘rggobi’ ‘ggplot2’
      ‘proto’
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# PKPDmisc

Version: 2.1.1

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘purrr’
      All declared Imports should be used.
    ```

# PKreport

Version: 1.5

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'PKreport.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.30 \ifthenelse
                    {\boolean{Sweave@inconsolata}}{%^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    generate.plot: no visible global function definition for ‘dev.off’
    writeHTML: no visible global function definition for
      ‘packageDescription’
    writeLst: no visible global function definition for
      ‘packageDescription’
    writeLst.tab: no visible global function definition for
      ‘packageDescription’
    writeTable: no visible global function definition for
      ‘packageDescription’
    writeTable: no visible global function definition for ‘quantile’
    initialize,nonmem: no visible global function definition for
      ‘read.table’
    Undefined global functions or variables:
      bmp browseURL dev.cur dev.new dev.off formula heatmap jpeg
      packageDescription png quantile read.table tiff
    Consider adding
      importFrom("grDevices", "bmp", "dev.cur", "dev.new", "dev.off", "jpeg",
                 "png", "tiff")
      importFrom("stats", "formula", "heatmap", "quantile")
      importFrom("utils", "browseURL", "packageDescription", "read.table")
    to your NAMESPACE file.
    ```

# planar

Version: 1.6

## In both

*   checking whether package ‘planar’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      hcubature.c:917:15: warning: comparison of unsigned enum expression < 0 is always false [-Wtautological-compare]
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/planar/new/planar.Rcheck/00install.out’ for details.
    ```

# PlasmaMutationDetector

Version: 1.7.2

## In both

*   checking whether package ‘PlasmaMutationDetector’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘DelayedArray’ was built under R version 3.5.1
      Warning: package ‘Rsamtools’ was built under R version 3.5.1
      Warning: package ‘rtracklayer’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/PlasmaMutationDetector/new/PlasmaMutationDetector.Rcheck/00install.out’ for details.
    ```

# playwith

Version: 0.9-54

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: ‘cairoDevice’ ‘gWidgetsRGtk2’ ‘RGtk2’
    
    Package suggested but not available for checking: ‘latticist’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# plethy

Version: 1.18.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    There are ::: calls to the package's namespace in its code. A package
      almost never needs to use ::: for its own objects:
      ‘csv.to.table’ ‘find.break.ranges.integer’ ‘fix.time’ ‘multi.grep’
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    mvtsplot.data.frame: no visible global function definition for ‘bxp’
    mvtsplot.data.frame : <anonymous>: no visible binding for global
      variable ‘median’
    mvtsplot.data.frame: no visible global function definition for ‘lines’
    mvtsplot.data.frame: no visible global function definition for ‘Axis’
    mvtsplot.data.frame: no visible global function definition for ‘legend’
    retrieveMatrix,BuxcoDB: no visible global function definition for
      ‘terms’
    tsplot,BuxcoDB: no visible binding for global variable ‘Days’
    tsplot,BuxcoDB: no visible binding for global variable ‘Value’
    tsplot,BuxcoDB: no visible binding for global variable ‘Sample_Name’
    Undefined global functions or variables:
      Axis Days Sample_Name Value abline bxp colors layout legend lines
      median mtext packageDescription par plot rnorm strwidth terms
    Consider adding
      importFrom("grDevices", "colors")
      importFrom("graphics", "Axis", "abline", "bxp", "layout", "legend",
                 "lines", "mtext", "par", "plot", "strwidth")
      importFrom("stats", "median", "rnorm", "terms")
      importFrom("utils", "packageDescription")
    to your NAMESPACE file.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    
    The following object is masked from ‘package:base’:
    
        expand.grid
    
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'plethy.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `beramono.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.87 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# plotluck

Version: 1.1.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘hexbin’ ‘quantreg’
      All declared Imports should be used.
    ```

# plotly

Version: 4.8.0

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      [1] "Running test: plotly-alpha-blending"
      [1] "Running test: plotly-alpha-no-color"
      [1] "Running test: plotly-factor-axis"
      [1] "Running test: plotly-character-axis"
      [1] "Running test: plotly-histogram"
      [1] "Running test: plotly-histogram-vert"
      [1] "Running test: plotly-inherit-FALSE"
      [1] "Running test: plotly-time-series-summary"
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 1307 SKIPPED: 30 FAILED: 2
      1. Error: (unknown) (@test-ggplot-sf.R#3) 
      2. Error: (unknown) (@test-plotly-sf.R#3) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘sf’
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.5Mb
      sub-directories of 1Mb or more:
        htmlwidgets   3.1Mb
    ```

# plotrr

Version: 1.0.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘dplyr’ ‘stats’
      All declared Imports should be used.
    ```

# plotscale

Version: 0.1.6

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘graphics’
      All declared Imports should be used.
    ```

# plspm

Version: 0.4.9

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    
    The following object is masked from 'package:plspm':
    
        alpha
    
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'plspm_introduction.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `parskip.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.68 ^^M
            
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# plyranges

Version: 1.0.3

## In both

*   checking whether package ‘plyranges’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/plyranges/new/plyranges.Rcheck/00install.out’ for details.
    ```

# pmc

Version: 1.0.3

## In both

*   R CMD check timed out
    

# pmpp

Version: 0.1.0

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      tests/testthat/test-theme.r:350:50:style :Only use double-quotes. 
      tests/testthat/test-theme.r:351:48:style :Only use double-quotes. 
      tests/testthat/test-theme.r:352:49:style :Only use double-quotes. 
      tests/testthat/test-utilities.r:27:23:style :Put spaces around all infix operators. 
      tests/testthat/test-utilities.r:27:39:style :Put spaces around all infix operators. 
      tests/testthat/test-utilities.r:27:46:style :Put spaces around all infix operators. 
      tests/testthat/test-utilities.r:27:57:style :Put spaces around all infix operators. 
      tests/testthat/test-viridis.R:16:1:style :Trailing whitespace is superfluous. 
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 64 SKIPPED: 0 FAILED: 1
      1. Failure: Package Style (@test-lintr.R#32) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

# pmxTools

Version: 0.1.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘GGally’ ‘PKNCA’ ‘grid’ ‘magrittr’ ‘plyr’ ‘xpose’
      All declared Imports should be used.
    ```

# pogos

Version: 1.0.0

## In both

*   checking R code for possible problems ... NOTE
    ```
    compoundsByCell: no visible binding for global variable ‘cell_lines_v1’
    compoundsByCell: no visible binding for global variable ‘compounds_v1’
    compoundsByCell: no visible binding for global variable ‘datasets_v1’
    lkc: no visible binding for global variable ‘cell_lines_v1’
    lkc: no visible binding for global variable ‘compounds_v1’
    lkc: no visible binding for global variable ‘datasets_v1’
    plot,DRProfSet-missing: no visible binding for global variable ‘dose’
    plot,DRProfSet-missing: no visible binding for global variable
      ‘response’
    plot,DRProfSet-missing: no visible binding for global variable ‘drug’
    plot,DRTraceSet-missing: no visible binding for global variable ‘dose’
    plot,DRTraceSet-missing: no visible binding for global variable
      ‘response’
    plot,DRTraceSet-missing: no visible binding for global variable
      ‘cell_line’
    Undefined global functions or variables:
      cell_line cell_lines_v1 compounds_v1 datasets_v1 dose drug response
    ```

# PopED

Version: 0.4.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘tidyr’
      All declared Imports should be used.
    ```

# PopGenReport

Version: 3.0.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'PopGenReportIntroduction.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `scrartcl.cls' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: cls)
    
    ! Emergency stop.
    <read *> 
             
    l.1 \documentclass[a4paper]{scrartcl}\usepackage
                                                    []{graphicx}\usepackage[]{co...
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘ecodist’
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Error in .requirePackage(package) : 
        unable to find required package 'raster'
      Calls: <Anonymous> ... getClass -> getClassDef -> .classEnv -> .requirePackage
      Execution halted
    ```

# poppr

Version: 2.8.1

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    Warning in replayPlot(x) : "fill" is not a graphical parameter
    Warning in replayPlot(x) : "border" is not a graphical parameter
    Warning in replayPlot(x) : "fill" is not a graphical parameter
    Warning in infinite_vals_replacement(D, warning) :
      Infinite values detected.
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'algo.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `colortbl.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.268 \long
               \def\@secondoffive#1#2#3#4#5{#2}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    LaTeX errors found:
    ! pdfTeX error (ext4): \pdfendlink ended up in different nesting level than \pd
    fstartlink.
    \AtBegShi@Output ...ipout \box \AtBeginShipoutBox 
                                                      \fi \fi 
    l.191 \item \code
    !  ==> Fatal error occurred, no output PDF file produced!
    ```

# popReconstruct

Version: 1.0-5

## In both

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    
    Calculating life expectancy at birth ...
    ... done
    Calculating net number of migrants ...
    ... done
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'burkina-faso-females.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `colortbl.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.267 \XC@@pkg
                  ^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# PortfolioEffectHFT

Version: 1.8

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'PortfolioEffectHFT.tex' failed.
    LaTeX errors:
    ! Package babel Error: Unknown option `english'. Either you misspelled it
    (babel)                or the language definition file english.ldf was not foun
    d.
    
    See the babel package documentation for explanation.
    ! LaTeX Error: File `titlesec.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.15 \usepackage
                    {setspace}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  6.3Mb
      sub-directories of 1Mb or more:
        java   5.1Mb
    ```

# postal

Version: 0.1.0

## Newly broken

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      15: purrr::map_dfr(., fetch_and_sleep)
      16: map(.x, .f, ...)
      17: .f(.x[[i]], ...)
      18: readr::write_csv(this, write_to, append = TRUE, col_names = FALSE)
      19: write_delim(x, path, delim = ",", na = na, append = append, col_names = col_names)
      20: stream_delim(x_out, path, delim, col_names = col_names, append = append, na = na)
      21: open(path, "ab")
      22: open.connection(path, "ab")
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 108 SKIPPED: 0 FAILED: 1
      1. Error: (unknown) (@test_fetch_zones.R#144) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

# PowerExplorer

Version: 1.0.1

## In both

*   checking whether package ‘PowerExplorer’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘DelayedArray’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/PowerExplorer/new/PowerExplorer.Rcheck/00install.out’ for details.
    ```

# ppcSpatial

Version: 0.2.0

## In both

*   checking whether package ‘ppcSpatial’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/ppcSpatial/new/ppcSpatial.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘ppcSpatial’ ...
** package ‘ppcSpatial’ successfully unpacked and MD5 sums checked
** R
** inst
** byte-compile and prepare package for lazy loading
Error in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]) : 
  there is no package called ‘sf’
ERROR: lazy loading failed for package ‘ppcSpatial’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/ppcSpatial/new/ppcSpatial.Rcheck/ppcSpatial’

```
### CRAN

```
* installing *source* package ‘ppcSpatial’ ...
** package ‘ppcSpatial’ successfully unpacked and MD5 sums checked
** R
** inst
** byte-compile and prepare package for lazy loading
Error in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]) : 
  there is no package called ‘sf’
ERROR: lazy loading failed for package ‘ppcSpatial’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/ppcSpatial/old/ppcSpatial.Rcheck/ppcSpatial’

```
# PPInfer

Version: 1.6.0

## In both

*   R CMD check timed out
    

*   checking whether package ‘PPInfer’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘graph’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/PPInfer/new/PPInfer.Rcheck/00install.out’ for details.
    ```

*   checking package dependencies ... NOTE
    ```
    Depends: includes the non-default packages:
      ‘biomaRt’ ‘fgsea’ ‘kernlab’ ‘ggplot2’ ‘igraph’ ‘STRINGdb’
      ‘yeastExpData’
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    ORA: no visible global function definition for ‘setTxtProgressBar’
    ORA: no visible global function definition for ‘p.adjust’
    ORA.dotplot: no visible global function definition for ‘p.adjust’
    enrich.net: no visible global function definition for ‘stack’
    enrich.net: no visible global function definition for ‘adjustcolor’
    enrich.net : <anonymous>: no visible global function definition for
      ‘adjustcolor’
    enrich.net: no visible binding for global variable ‘legend’
    net.infer: no visible global function definition for ‘na.omit’
    net.infer.ST: no visible global function definition for ‘na.omit’
    ppi.infer.human: no visible global function definition for ‘na.omit’
    ppi.infer.mouse: no visible global function definition for ‘na.omit’
    Undefined global functions or variables:
      adjustcolor fisher.test legend na.omit p.adjust setTxtProgressBar
      stack txtProgressBar
    Consider adding
      importFrom("grDevices", "adjustcolor")
      importFrom("graphics", "legend")
      importFrom("stats", "fisher.test", "na.omit", "p.adjust")
      importFrom("utils", "setTxtProgressBar", "stack", "txtProgressBar")
    to your NAMESPACE file.
    ```

# PPtreeViz

Version: 2.0.3

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘Rcpp’
      All declared Imports should be used.
    ```

# pqsfinder

Version: 1.8.0

## In both

*   checking whether package ‘pqsfinder’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘IRanges’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/pqsfinder/new/pqsfinder.Rcheck/00install.out’ for details.
    ```

*   checking if this is a source package ... NOTE
    ```
    Found the following apparent object files/libraries:
      src/boost_regex/c_regex_traits.o src/boost_regex/cpp_regex_traits.o
      src/boost_regex/cregex.o src/boost_regex/fileiter.o
      src/boost_regex/icu.o src/boost_regex/instances.o
      src/boost_regex/posix_api.o src/boost_regex/regex.o
      src/boost_regex/regex_debug.o src/boost_regex/regex_raw_buffer.o
      src/boost_regex/regex_traits_defaults.o
      src/boost_regex/static_mutex.o src/boost_regex/usinstances.o
      src/boost_regex/w32_regex_traits.o src/boost_regex/wc_regex_traits.o
      src/boost_regex/wide_posix_api.o src/boost_regex/winstances.o
    Object files/libraries should not be included in a source package.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 12.8Mb
      sub-directories of 1Mb or more:
        lib  11.0Mb
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Versioned 'LinkingTo' value for ‘BH’ is only usable in R >= 3.0.2
    ```

*   checking for GNU extensions in Makefiles ... NOTE
    ```
    GNU make is a SystemRequirements.
    ```

# preprosim

Version: 0.2.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘e1071’
      All declared Imports should be used.
    ```

# prevR

Version: 3.3

## In both

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘sparr’
    ```

# PRISMA

Version: 0.2-7

## In both

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    
        lowess
    
    Loading required package: ggplot2
       [[ suppressing 24 column names ‘line56’, ‘line32’, ‘line6’ ... ]]
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'PRISMA.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.30 \ifthenelse
                    {\boolean{Sweave@inconsolata}}{%^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# Prize

Version: 1.10.0

## In both

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    2 Normal_expression is processed.
    3 Frequency is processed.
    4 Epitopes is processed.
    4.1 Number_of_epitopes is processed.
    4.2 Size_of_epitopes is processed.
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'Prize.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `cite.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.8 \title
              {Prize: an R package for prioritization estimation based on analyt...
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# processmapR

Version: 0.3.2

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘viridis’
      All declared Imports should be used.
    ```

# profr

Version: 0.3.1

## In both

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' call to ‘ggplot2’ in package code.
      Please use :: or requireNamespace() instead.
      See section 'Suggested packages' in the 'Writing R Extensions' manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    ggplot.profr: no visible binding for global variable ‘start’
    ggplot.profr: no visible binding for global variable ‘end’
    ggplot.profr: no visible global function definition for ‘geom_text’
    ggplot.profr: no visible binding for global variable ‘time’
    ggplot.profr: no visible global function definition for
      ‘scale_y_continuous’
    ggplot.profr: no visible global function definition for
      ‘scale_x_continuous’
    plot.profr: no visible global function definition for ‘plot’
    plot.profr: no visible global function definition for ‘rect’
    plot.profr: no visible binding for global variable ‘time’
    plot.profr: no visible global function definition for ‘text’
    profr: no visible global function definition for ‘Rprof’
    Undefined global functions or variables:
      Rprof aes end geom_rect geom_text ggplot hist plot rect
      scale_x_continuous scale_y_continuous start text time
    Consider adding
      importFrom("graphics", "hist", "plot", "rect", "text")
      importFrom("stats", "end", "start", "time")
      importFrom("utils", "Rprof")
    to your NAMESPACE file.
    ```

# ProgGUIinR

Version: 0.0-4

## In both

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking:
      ‘qtbase’ ‘RGtk2’ ‘cairoDevice’ ‘gWidgetsRGtk2’
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Description field: should contain one or more complete sentences.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      definition for ‘apropos’
    showGtkWidgetInfo : getMethodForObject: no visible global function
      definition for ‘help’
    showGtkWidgetInfo: no visible global function definition for
      ‘gSignalConnect’
    Undefined global functions or variables:
      PangoStyle addSpring apropos gSignalConnect gbutton gframe ggroup
      gnotebook gpanedgroup gstatusbar gtable gtext gtkAction
      gtkCellRendererText gtkEntry gtkEntryCompletionNew gtkFrame gtkHBox
      gtkHBoxNew gtkHPaned gtkLabel gtkListStore gtkNotebook
      gtkScrolledWindow gtkSeparatorToolItem gtkStatusbar gtkTextBuffer
      gtkTextView gtkToolButton gtkToolbar gtkTreeView gtkTreeViewColumn
      gtkTypeGetSignals gtkVBox gtkVBoxNew gtkWindow gwindow help
      rGtkDataFrame svalue svalue<- tcl tclvalue tkbind tkconfigure
      tkdelete tkget tkgrid tkgrid.columnconfigure tkgrid.rowconfigure
      tkinsert tkmenu tkpack tksee tkset tktag.add tktag.configure
      tktag.remove tktext tktoplevel tkwm.title tkxview ttkbutton ttkframe
      ttklabel ttkpanedwindow ttkscrollbar ttktreeview visible<-
    Consider adding
      importFrom("utils", "apropos", "help")
    to your NAMESPACE file.
    ```

# projections

Version: 0.3.1

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘distcrete’
      All declared Imports should be used.
    ```

# pRoloc

Version: 1.20.2

## In both

*   R CMD check timed out
    

*   checking whether package ‘pRoloc’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘MSnbase’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/pRoloc/new/pRoloc.Rcheck/00install.out’ for details.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 12.9Mb
      sub-directories of 1Mb or more:
        doc  10.6Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Unexported objects imported by ':::' calls:
      ‘MLInterfaces:::.macroF1’ ‘MLInterfaces:::.precision’
      ‘MLInterfaces:::.recall’ ‘MLInterfaces:::es2df’
      ‘caret:::predict.plsda’
      See the note in ?`:::` about the use of this operator.
    There are ::: calls to the package's namespace in its code. A package
      almost never needs to use ::: for its own objects:
      ‘opt’
    ```

*   checking R code for possible problems ... NOTE
    ```
    Found the following possibly unsafe calls:
    File ‘pRoloc/R/annotation.R’:
      unlockBinding("params", .pRolocEnv)
    ```

# pRolocGUI

Version: 1.14.0

## In both

*   checking whether package ‘pRolocGUI’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘pRoloc’ was built under R version 3.5.1
      Warning: package ‘MSnbase’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/pRolocGUI/new/pRolocGUI.Rcheck/00install.out’ for details.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Authors@R field gives more than one person with maintainer role:
      Lisa Breckels <lms79@cam.ac.uk> [aut, cre]
      Laurent Gatto <lg390@cam.ac.uk> [aut, cre]
    ```

*   checking dependencies in R code ... NOTE
    ```
    Unexported object imported by a ':::' call: ‘pRoloc:::remap’
      See the note in ?`:::` about the use of this operator.
    ```

# proportion

Version: 2.0.0

## In both

*   checking Rd cross-references ... NOTE
    ```
    Packages unavailable to check Rd xrefs: ‘binom’, ‘PropCIs’, ‘BlakerCI’, ‘prevalence’
    ```

# propr

Version: 4.0.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘utils’
      All declared Imports should be used.
    ```

# proteomics

Version: 0.2

## In both

*   checking R code for possible problems ... NOTE
    ```
    addIonSatistics: no visible binding for global variable ‘median’
    adjustOne: no visible binding for global variable ‘median’
    attachModifications: no visible global function definition for
      ‘na.omit’
    avrgLoadingCalculation: no visible binding for global variable ‘median’
    channelResponses : estimate: no visible binding for global variable
      ‘var’
    responseStatisics : func: no visible global function definition for
      ‘aggregate’
    testing: no visible global function definition for ‘aov’
    testingOneshot: no visible global function definition for ‘anova’
    testingTukey: no visible global function definition for ‘TukeyHSD’
    Undefined global functions or variables:
      TukeyHSD aggregate anova aov median na.omit var
    Consider adding
      importFrom("stats", "TukeyHSD", "aggregate", "anova", "aov", "median",
                 "na.omit", "var")
    to your NAMESPACE file.
    ```

# proteoQC

Version: 1.16.0

## In both

*   checking whether package ‘proteoQC’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘MSnbase’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/proteoQC/new/proteoQC.Rcheck/00install.out’ for details.
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error: processing vignette 'proteoQC.Rmd' failed with diagnostics:
    there is no package called ‘prettydoc’
    Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘RforProteomics’
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  7.4Mb
      sub-directories of 1Mb or more:
        doc       2.5Mb
        extdata   3.9Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    plotMS2boxplot: no visible binding for global variable ‘techRep’
    plotMS2boxplot: no visible binding for global variable ‘fraction’
    plotMS2boxplot: no visible binding for global variable ‘MS2QC’
    plotSampleIDResultErrorBar: no visible binding for global variable
      ‘fraction’
    plotSampleIDResultErrorBar: no visible binding for global variable
      ‘val’
    plotSampleIDResultErrorBar: no visible binding for global variable ‘se’
    plotSampleVenn: no visible global function definition for ‘grid.draw’
    plotTechRepVenn : <anonymous>: no visible global function definition
      for ‘grid.draw’
    qcHist: no visible binding for global variable ‘error’
    qcHist: no visible binding for global variable ‘techRep’
    qcHist: no visible binding for global variable ‘bioRep’
    qcHist2: no visible binding for global variable ‘error’
    qcHist2: no visible binding for global variable ‘fractile’
    Undefined global functions or variables:
      ..count.. Intensity MS1QC MS2QC TMT10 TMT6 Tag V1 V2 V3 V4 V5 bioRep
      curenv delta error exprs fractile fraction grid.draw iTRAQ4 iTRAQ8
      label peplength peptide_summary precursorCharge quantify ratio
      readMgfData se techRep val x y
    ```

# PRROC

Version: 1.3.1

## In both

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'PRROC.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `scrartcl.cls' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: cls)
    
    ! Emergency stop.
    <read *> 
             
    l.2 ^^M
           
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# pscore

Version: 0.1-2

## In both

*   checking R code for possible problems ... NOTE
    ```
    mahalanobisComposite: no visible global function definition for
      ‘princomp’
    prepareComposite: no visible global function definition for ‘cov’
    winsorizor : f: no visible global function definition for ‘quantile’
    Undefined global functions or variables:
      cov princomp quantile
    Consider adding
      importFrom("stats", "cov", "princomp", "quantile")
    to your NAMESPACE file.
    ```

# psd

Version: 1.0-1

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    /Library/Frameworks/R.framework/Resources/bin/Rscript -e "if (getRversion() < '3.0.0') knitr::knit2pdf('fftw.Rnw') else tools::texi2pdf('fftw.tex')"
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'fftw.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `float.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.67 \usepackage
                    {natbib}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: <Anonymous> -> texi2dvi
    Execution halted
    make: *** [fftw.pdf] Error 1
    Error in buildVignettes(dir = "/Users/max/github/ggplot2/revdep/checks.noindex/psd/new/psd.Rcheck/vign_test/psd") : 
      running 'make' failed
    Execution halted
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    psdcore.default: no visible global function definition for ‘ts’
    pspectrum.ts: no visible global function definition for ‘frequency’
    spec_confint.default: no visible global function definition for
      ‘pchisq’
    spec_confint.default: no visible global function definition for
      ‘qchisq’
    splineGrad.default: no visible global function definition for ‘par’
    splineGrad.default: no visible global function definition for ‘plot’
    splineGrad.default: no visible global function definition for ‘lines’
    Undefined global functions or variables:
      abline acf as.ts frequency head is.ts layout legend lines lm
      loess.control mtext par pchisq plot qchisq residuals spec.pgram start
      tail title ts var
    Consider adding
      importFrom("graphics", "abline", "layout", "legend", "lines", "mtext",
                 "par", "plot", "title")
      importFrom("stats", "acf", "as.ts", "frequency", "is.ts", "lm",
                 "loess.control", "pchisq", "qchisq", "residuals",
                 "spec.pgram", "start", "ts", "var")
      importFrom("utils", "head", "tail")
    to your NAMESPACE file.
    ```

# psda

Version: 1.0.1

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘raster’
      All declared Imports should be used.
    ```

# psichomics

Version: 1.6.2

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  7.7Mb
      sub-directories of 1Mb or more:
        doc   5.6Mb
    ```

*   checking compiled code ... NOTE
    ```
    File ‘psichomics/libs/psichomics.so’:
      Found ‘___stdoutp’, possibly from ‘stdout’ (C)
        Object: ‘psiFastCalc.o’
      Found ‘_printf’, possibly from ‘printf’ (C)
        Object: ‘psiFastCalc.o’
      Found ‘_putchar’, possibly from ‘putchar’ (C)
        Object: ‘psiFastCalc.o’
    
    Compiled code should not call entry points which might terminate R nor
    write to stdout/stderr instead of to the console, nor use Fortran I/O
    nor system RNGs.
    
    See ‘Writing portable packages’ in the ‘Writing R Extensions’ manual.
    ```

# PSLM2015

Version: 0.2.0

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.3Mb
      sub-directories of 1Mb or more:
        data   5.2Mb
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 26 marked Latin-1 strings
    ```

# psycho

Version: 0.3.7

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘methods’
      All declared Imports should be used.
    ```

# psygenet2r

Version: 1.12.0

## In both

*   checking whether package ‘psygenet2r’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Note: possible error in 'psygenetGene(x, databse = "ALL")': unused argument (databse = "ALL") 
      Note: possible error in 'psygenetGene(x, databse = "ALL", ': unused argument (databse = "ALL") 
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/psygenet2r/new/psygenet2r.Rcheck/00install.out’ for details.
    Information on the location(s) of code generating the ‘Note’s can be
    obtained by re-running with environment variable R_KEEP_PKG_SOURCE set
    to ‘yes’.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.8Mb
      sub-directories of 1Mb or more:
        doc   5.2Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘GO.db’
      All declared Imports should be used.
    package 'methods' is used but not declared
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      ‘Disease1’
    plot,JaccardIndexPsy-ANY: no visible binding for global variable
      ‘Disease2’
    plot,JaccardIndexPsy-ANY: no visible binding for global variable
      ‘JaccardIndex’
    plot,JaccardIndexPsy-ANY: no visible binding for global variable
      ‘value’
    plot,JaccardIndexPsy-ANY: no visible binding for global variable
      ‘variable’
    Undefined global functions or variables:
      Category Disease1 Disease2 JaccardIndex Var1 c0.Number_of_Abstracts
      c0.Score c1.Gene_Symbol c2.DiseaseName c2.PsychiatricDisorder
      category combn database diseases gene new perc phyper pie read.csv
      read.delim universe value variable
    Consider adding
      importFrom("graphics", "pie")
      importFrom("methods", "new")
      importFrom("stats", "phyper")
      importFrom("utils", "combn", "read.csv", "read.delim")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

# PTXQC

Version: 0.92.3

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  8.2Mb
      sub-directories of 1Mb or more:
        doc        4.6Mb
        examples   2.2Mb
    ```

# PureCN

Version: 1.10.0

## In both

*   R CMD check timed out
    

*   checking whether package ‘PureCN’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘DelayedArray’ was built under R version 3.5.1
      Warning: package ‘Rsamtools’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/PureCN/new/PureCN.Rcheck/00install.out’ for details.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  7.3Mb
      sub-directories of 1Mb or more:
        data      1.1Mb
        doc       2.6Mb
        extdata   3.0Mb
    ```

# pysd2r

Version: 0.1.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘knitr’
      All declared Imports should be used.
    ```

# qcmetrics

Version: 1.18.0

## In both

*   checking examples ... ERROR
    ```
    ...
    
    $uni
    [1] "Cabin University"
    
    > show(qcm)
    Object of class "QcMetrics"
     containing 2 QC metrics.
     and 3 metadata variables.
    > 
    > destdir <- tempdir()
    > (report <- file.path(destdir, "testQCReport"))
    [1] "/var/folders/lb/xhxqmcrd7gv302_b1pdfykh80000gn/T//RtmpMooCJG/testQCReport"
    > 
    > ## pdf report
    > qcReport(qcm, reportname = report)
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on '/var/folders/lb/xhxqmcrd7gv302_b1pdfykh80000gn/T//RtmpMooCJG/testQCReport.tex' failed.
    Messages:
    /usr/local/bin/texi2dvi: pdflatex exited with bad status, quitting.
    Calls: qcReport ... qcReport -> .local -> reporting_pdf -> <Anonymous> -> texi2dvi
    Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 51-72 (qcmetrics.Rnw) 
    Error: processing vignette 'qcmetrics.Rnw' failed with diagnostics:
    there is no package called 'RforProteomics'
    Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘RforProteomics’
    ```

*   checking R code for possible problems ... NOTE
    ```
    n15qc: no visible global function definition for ‘fvarLabels’
    n15qc: no visible global function definition for ‘fData’
    n15qc : <anonymous>: no visible global function definition for ‘ggplot’
    n15qc : <anonymous>: no visible global function definition for ‘aes’
    n15qc : <anonymous>: no visible global function definition for
      ‘geom_jitter’
    n15qc : <anonymous>: no visible global function definition for
      ‘geom_boxplot’
    n15qc : <anonymous>: no visible global function definition for
      ‘geom_hline’
    n15qc : <anonymous>: no visible global function definition for ‘labs’
    n15qc: no visible global function definition for ‘fData<-’
    n15qc: no visible global function definition for ‘combineFeatures’
    n15qc: no visible global function definition for ‘exprs’
    n15qc: no visible global function definition for ‘fileNames’
    n15qc: no visible global function definition for ‘experimentData’
    Undefined global functions or variables:
      aes combineFeatures experimentData exprs fData fData<- fileNames
      fvarLabels geom_boxplot geom_hline geom_jitter ggplot labs
    ```

# qdap

Version: 2.3.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'cleaning_and_debugging.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `mathpazo.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.59 \usepackage
                    {xfrac}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.6Mb
      sub-directories of 1Mb or more:
        R   2.1Mb
    ```

# qqplotr

Version: 0.0.3

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘knitr’ ‘purrr’ ‘rmarkdown’
      All declared Imports should be used.
    ```

# qrqc

Version: 1.34.0

## In both

*   checking whether package ‘qrqc’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      io.c:347:14: warning: incompatible pointer types initializing 'gzFile *' (aka 'struct gzFile_s **') with an expression of type 'gzFile' (aka 'struct gzFile_s *') [-Wincompatible-pointer-types]
      io.c:350:21: warning: incompatible pointer types passing 'gzFile *' (aka 'struct gzFile_s **') to parameter of type 'gzFile' (aka 'struct gzFile_s *'); dereference with * [-Wincompatible-pointer-types]
      io.c:452:21: warning: incompatible pointer types passing 'gzFile *' (aka 'struct gzFile_s **') to parameter of type 'gzFile' (aka 'struct gzFile_s *'); dereference with * [-Wincompatible-pointer-types]
      io.c:453:14: warning: incompatible pointer types passing 'gzFile *' (aka 'struct gzFile_s **') to parameter of type 'gzFile' (aka 'struct gzFile_s *'); dereference with * [-Wincompatible-pointer-types]
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘biovizBase’ was built under R version 3.5.1
      Warning: package 'Rsamtools' was built under R version 3.5.1
      Warning: package 'GenomicRanges' was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/qrqc/new/qrqc.Rcheck/00install.out’ for details.
    ```

*   checking package dependencies ... NOTE
    ```
    Depends: includes the non-default packages:
      ‘reshape’ ‘ggplot2’ ‘Biostrings’ ‘biovizBase’ ‘brew’ ‘xtable’
      ‘Rsamtools’ ‘testthat’
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Packages listed in more than one of Depends, Imports, Suggests, Enhances:
      ‘reshape’ ‘ggplot2’ ‘Biostrings’ ‘biovizBase’
    A package should be listed in only one of these fields.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Packages in Depends field not imported from:
      'Rsamtools' 'brew' 'testthat' 'xtable'
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    kmerKLPlot,SequenceSummary: no visible binding for global variable
      'kmer'
    kmerKLPlot,SequenceSummary: no visible binding for global variable
      'position'
    kmerKLPlot,SequenceSummary: no visible binding for global variable 'kl'
    kmerKLPlot,list : <anonymous>: no visible binding for global variable
      'kmer'
    kmerKLPlot,list: no visible binding for global variable 'position'
    kmerKLPlot,list: no visible binding for global variable 'kl'
    kmerKLPlot,list: no visible binding for global variable 'kmer'
    plotGC,SequenceSummary : <local>: no visible global function definition
      for 'aggregate'
    qualPlot,FASTQSummary: no visible binding for global variable
      'position'
    qualPlot,list: no visible binding for global variable 'position'
    Undefined global functions or variables:
      DNAStringSet aggregate base entropy kl kmer na.exclude position
      quantile write.XStringSet
    Consider adding
      importFrom("stats", "aggregate", "na.exclude", "quantile")
    to your NAMESPACE file.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    `geom_smooth()` using method = 'gam' and formula 'y ~ s(x, bs = "cs")'
    Warning: Removed 5900 rows containing non-finite values (stat_smooth).
    Warning: Removed 59 rows containing missing values (geom_linerange).
    Warning: Removed 59 rows containing missing values (geom_linerange).
    Warning: Removed 59 rows containing missing values (geom_point).
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'qrqc.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.30 \ifthenelse
                    {\boolean{Sweave@inconsolata}}{%^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# qsort

Version: 0.2.2

## In both

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 6 marked UTF-8 strings
    ```

# QualInt

Version: 1.0.0

## In both

*   checking R code for possible problems ... NOTE
    ```
    ...
    powerval: no visible global function definition for ‘qnorm’
    powerval: no visible global function definition for ‘pnorm’
    print.expplot: no visible global function definition for ‘gray’
    print.expplot: no visible binding for global variable ‘coef’
    print.outplot: no visible global function definition for ‘gray’
    print.outplot: no visible binding for global variable ‘coef’
    qualint: no visible global function definition for ‘qnorm’
    qualint: no visible global function definition for ‘plot’
    qualval: no visible global function definition for ‘qnorm’
    qualval: no visible global function definition for ‘plot’
    surout: no visible global function definition for ‘tail’
    surout: no visible global function definition for ‘coef’
    Undefined global functions or variables:
      coef dbinom gray pchisq plot pnorm qnorm rnorm sd tail
    Consider adding
      importFrom("grDevices", "gray")
      importFrom("graphics", "plot")
      importFrom("stats", "coef", "dbinom", "pchisq", "pnorm", "qnorm",
                 "rnorm", "sd")
      importFrom("utils", "tail")
    to your NAMESPACE file.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Loading required package: survival
    Loading required package: ggplot2
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'qualint.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.30 \ifthenelse
                    {\boolean{Sweave@inconsolata}}{%^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# qualmap

Version: 0.1.1

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘sf’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# quanteda

Version: 1.3.4

## In both

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 71 marked UTF-8 strings
    ```

# quantro

Version: 1.14.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    Loading required package: iterators
    Loading required package: locfit
    locfit 1.5-9.1 	 2013-03-22
    Warning in has_utility("pdfcrop") :
      pdfcrop not installed or not in PATH
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'quantro-vignette.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `beramono.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.87 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# quickPlot

Version: 0.1.5

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘fastshp’
    ```

# quickReg

Version: 1.5.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘psych’
      All declared Imports should be used.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘PredictABEL’
    ```

# quokar

Version: 0.1.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘MCMCpack’ ‘gridExtra’ ‘knitr’
      All declared Imports should be used.
    ```

# qvalue

Version: 2.12.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    	 ==> NOT changing 'width'
    Warning: 'mode(width)' differs between new and previous
    	 ==> NOT changing 'width'
    Warning: 'mode(width)' differs between new and previous
    	 ==> NOT changing 'width'
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'qvalue.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `fullpage.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.59 \usepackage
                    {bibentry}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking R code for possible problems ... NOTE
    ```
    lfdr: no visible global function definition for ‘qnorm’
    lfdr: no visible global function definition for ‘density’
    lfdr: no visible global function definition for ‘smooth.spline’
    lfdr: no visible global function definition for ‘predict’
    lfdr: no visible global function definition for ‘dnorm’
    pi0est: no visible global function definition for ‘smooth.spline’
    pi0est: no visible global function definition for ‘predict’
    pi0est: no visible global function definition for ‘quantile’
    plot.qvalue: no visible global function definition for ‘quantile’
    write.qvalue: no visible global function definition for ‘write.table’
    Undefined global functions or variables:
      density dnorm predict qnorm quantile smooth.spline write.table
    Consider adding
      importFrom("stats", "density", "dnorm", "predict", "qnorm", "quantile",
                 "smooth.spline")
      importFrom("utils", "write.table")
    to your NAMESPACE file.
    ```

# R2admb

Version: 0.7.16

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'R2admb.tex' failed.
    LaTeX errors:
    ! Package babel Error: Unknown option `american'. Either you misspelled it
    (babel)                or the language definition file american.ldf was not fou
    nd.
    
    See the babel package documentation for explanation.
    ! LaTeX Error: File `listings.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.67 \usepackage
                    {verbatim}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# r2glmm

Version: 0.1.2

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘data.table’ ‘dplyr’ ‘lmerTest’
      All declared Imports should be used.
    ```

# R3CPET

Version: 1.12.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    loading HPRD network, with 9616 nodes
    Warning in has_utility("pdfcrop") :
      pdfcrop not installed or not in PATH
    plotting networks
    plots will be also available on file AllGraphs.pdf
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'R3CPET.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `beramono.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.87 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  6.0Mb
      sub-directories of 1Mb or more:
        data      3.0Mb
        example   1.0Mb
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Package listed in more than one of Depends, Imports, Suggests, Enhances:
      ‘methods’
    A package should be listed in only one of these fields.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: 'BiocGenerics'
      All declared Imports should be used.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    .formatDAVIDResult: no visible global function definition for
      'formatList'
    .formatDAVIDResult: no visible global function definition for
      'formatGene2Gene'
    .formatDAVIDResult: no visible global function definition for
      'formatAnnotationReport'
    .get.NetworksGenes: no visible global function definition for
      'annotatePeakInBatch'
    EnsemblToHGNC: no visible global function definition for 'useMart'
    EnsemblToHGNC: no visible global function definition for 'useDataset'
    EnsemblToHGNC: no visible global function definition for 'getBM'
    EntrezToHGNC: no visible global function definition for 'useMart'
    EntrezToHGNC: no visible global function definition for 'useDataset'
    EntrezToHGNC: no visible global function definition for 'getBM'
    createServer,ChiapetExperimentData-NetworkCollection-ChromMaintainers:
      no visible global function definition for 'runApp'
    Undefined global functions or variables:
      TxDb.Hsapiens.UCSC.hg19.knownGene annotatePeakInBatch
      formatAnnotationReport formatGene2Gene formatGeneReport
      formatGeneReportFull formatList getBM org.Hs.eg.db org.Hs.egUCSCKG
      runApp select toTable useDataset useMart
    ```

# radiant.data

Version: 0.9.7

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘shinyFiles’
      All declared Imports should be used.
    ```

# raincpc

Version: 0.4

## In both

*   checking R code for possible problems ... NOTE
    ```
    cpc_get_rawdata: no visible global function definition for
      ‘download.file’
    Undefined global functions or variables:
      download.file
    Consider adding
      importFrom("utils", "download.file")
    to your NAMESPACE file.
    ```

# randomForestExplainer

Version: 0.9

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘MASS’ ‘dtplyr’
      All declared Imports should be used.
    ```

# randomizeR

Version: 1.4.2

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
      Solution may be nonunique
    Warning in rq.fit.br(wx, wy, tau = tau, ...) :
      Solution may be nonunique
    Warning in rq.fit.br(wx, wy, tau = tau, ...) :
      Solution may be nonunique
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'article.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.55 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# randomUniformForest

Version: 1.1.5

## In both

*   checking R code for possible problems ... NOTE
    ```
    ...
      ‘rnorm’
    update.unsupervised: no visible global function definition for
      ‘predict’
    weightedVoteModel: no visible global function definition for ‘lm’
    which.is.duplicate: no visible global function definition for ‘na.omit’
    Undefined global functions or variables:
      abline aggregate barplot cor dev.new dev.off dist graphics.off grid
      head heat.colors kmeans legend lm median memory.limit model.frame
      model.matrix model.response mosaicplot na.omit par pbinom persp plot
      points predict qnorm quantile rnorm runif sd title
    Consider adding
      importFrom("grDevices", "dev.new", "dev.off", "graphics.off",
                 "heat.colors")
      importFrom("graphics", "abline", "barplot", "grid", "legend",
                 "mosaicplot", "par", "persp", "plot", "points", "title")
      importFrom("stats", "aggregate", "cor", "dist", "kmeans", "lm",
                 "median", "model.frame", "model.matrix", "model.response",
                 "na.omit", "pbinom", "predict", "qnorm", "quantile",
                 "rnorm", "runif", "sd")
      importFrom("utils", "head", "memory.limit")
    to your NAMESPACE file.
    ```

# raptr

Version: 0.1.2

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘gurobi’
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.7Mb
      sub-directories of 1Mb or more:
        data   3.6Mb
    ```

# Rariant

Version: 1.16.0

## In both

*   checking whether package ‘Rariant’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘DelayedArray’ was built under R version 3.5.1
      Warning: package ‘Rsamtools’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/Rariant/new/Rariant.Rcheck/00install.out’ for details.
    ```

*   checking examples ... WARNING
    ```
    Found the following significant warnings:
    
      Warning: 'rbind_all' is deprecated.
    Deprecated functions may be defunct as soon as of the next release of
    R.
    See ?Deprecated.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  7.8Mb
      sub-directories of 1Mb or more:
        doc       2.3Mb
        extdata   5.2Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    tallyBamRegion: no visible global function definition for 'PileupParam'
    tallyBamRegion: no visible global function definition for
      'ScanBamParam'
    tallyBamRegion: no visible global function definition for 'pileup'
    Undefined global functions or variables:
      PileupParam ScanBamParam pileup
    ```

*   checking installed files from ‘inst/doc’ ... NOTE
    ```
    The following files should probably not be installed:
      ‘rariant-inspect-ci.png’, ‘rariant-inspect-shift.png’
    
    Consider the use of a .Rinstignore file: see ‘Writing R Extensions’,
    or move the vignette sources from ‘inst/doc’ to ‘vignettes’.
    ```

# RATest

Version: 0.1.2

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
      non-integer #successes in a binomial glm!
    Warning in (if (out_format(c("latex", "sweave", "listings"))) sanitize_fn else paste0)(path,  :
      dots in figure paths replaced with _ ("figure/panel_b")
    Warning in (if (out_format(c("latex", "sweave", "listings"))) sanitize_fn else paste0)(path,  :
      dots in figure paths replaced with _ ("figure/hist_cdf")
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'RDperm.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.30 \ifthenelse
                    {\boolean{Sweave@inconsolata}}{%^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# rattle

Version: 5.2.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'rattle.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `lastpage.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.13 \usepackage
                    {algorithm2e}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking:
      ‘cairoDevice’ ‘gWidgetsRGtk2’ ‘playwith’ ‘rggobi’ ‘RGtk2’
      ‘RGtk2Extras’
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  7.9Mb
      sub-directories of 1Mb or more:
        data   3.1Mb
        etc    1.9Mb
        po     1.2Mb
    ```

# rbefdata

Version: 0.3.5

## In both

*   checking R code for possible problems ... NOTE
    ```
    ...
    bef.portal.get.attachments_for: no visible global function definition
      for ‘flush.console’
    bef.portal.get.categories_for: no visible binding for global variable
      ‘id’
    bef.portal.get.categories_for: no visible global function definition
      for ‘read.csv’
    bef.portal.get.dataset: no visible global function definition for
      ‘read.csv’
    bef.portal.get.dataset_by: no visible global function definition for
      ‘read.csv’
    bef.portal.get.datasets.for_keyword : <anonymous>: no visible global
      function definition for ‘read.csv’
    bef.portal.get.datasets.for_proposal: no visible global function
      definition for ‘read.csv’
    upload_file: no visible global function definition for ‘write.csv’
    Undefined global functions or variables:
      browseURL flush.console id read.csv write.csv
    Consider adding
      importFrom("utils", "browseURL", "flush.console", "read.csv",
                 "write.csv")
    to your NAMESPACE file.
    ```

# RBesT

Version: 1.3-4

## In both

*   checking for GNU extensions in Makefiles ... NOTE
    ```
    GNU make is a SystemRequirements.
    ```

# rbokeh

Version: 0.5.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘ggplot2’
      All declared Imports should be used.
    ```

# rcartocolor

Version: 0.0.22

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘sf’
    ```

# RCAS

Version: 1.6.0

## In both

*   R CMD check timed out
    

*   checking whether package ‘RCAS’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘graph’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘rtracklayer’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/RCAS/new/RCAS.Rcheck/00install.out’ for details.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘DBI’
      All declared Imports should be used.
    ```

# rcellminer

Version: 2.2.0

## In both

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .BBSoptions
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 10.6Mb
      sub-directories of 1Mb or more:
        data   3.0Mb
        doc    6.5Mb
    ```

*   checking whether the namespace can be loaded with stated dependencies ... NOTE
    ```
    Warning: no function found corresponding to methods exports from ‘rcellminer’ for: ‘initialize’
    
    A namespace must be able to be loaded with just the base namespace
    loaded: otherwise if the namespace gets loaded by a saved object, the
    session will be unable to start.
    
    Probably some imports need to be declared in the NAMESPACE file.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    plotDrugSets: no visible binding for global variable ‘sd’
    plotDrugSets: no visible global function definition for ‘pdf’
    plotDrugSets: no visible global function definition for ‘par’
    plotDrugSets: no visible global function definition for ‘axis’
    plotDrugSets: no visible global function definition for ‘segments’
    plotDrugSets: no visible global function definition for ‘dev.off’
    plotStructures: no visible global function definition for ‘par’
    rcdkplot: no visible global function definition for ‘par’
    rcdkplot: no visible global function definition for ‘get.depictor’
    rcdkplot: no visible global function definition for ‘plot’
    rcdkplot: no visible global function definition for ‘rasterImage’
    Undefined global functions or variables:
      IQR axis dev.off element_text get.depictor layout lcm lm na.exclude
      par pdf plot quantile rasterImage residuals sd segments xlim ylim
    Consider adding
      importFrom("grDevices", "dev.off", "pdf")
      importFrom("graphics", "axis", "layout", "lcm", "par", "plot",
                 "rasterImage", "segments")
      importFrom("stats", "IQR", "lm", "na.exclude", "quantile", "residuals",
                 "sd")
    to your NAMESPACE file.
    ```

# rCGH

Version: 1.10.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    
    Dataset with 15848 probes and 8 columns.
    Array information:
    
    
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'rCGH.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `a4wide.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.35 \usepackage
                    [utf8]{inputenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.2Mb
      sub-directories of 1Mb or more:
        data      2.8Mb
        extdata   1.2Mb
    ```

*   checking files in ‘vignettes’ ... NOTE
    ```
    The following directory looks like a leftover from 'knitr':
      ‘figure’
    Please remove from your package.
    ```

# rclimateca

Version: 1.0.2

## In both

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 24 marked UTF-8 strings
    ```

# RcmdrPlugin.FuzzyClust

Version: 1.1

## In both

*   R CMD check timed out
    

# rcongresso

Version: 0.4.6

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      16: withVisible(function_list[[k]](value))
      17: function_list[[k]](value)
      18: dplyr::do(., .congresso_api(API_path, .))
      19: do.rowwise_df(., .congresso_api(API_path, .))
      20: overscope_eval_next(overscope, args[[j]])
      21: .congresso_api(API_path, .)
      22: .get_from_api(path, query)
      23: stop(.ERRO_RETORNO_JSON, call. = FALSE)
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 72 SKIPPED: 0 FAILED: 1
      1. Error: Quantidade de itens por requisição (@test_proposicoes.R#82) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

# RDAVIDWebService

Version: 1.18.0

## In both

*   checking whether package ‘RDAVIDWebService’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘graph’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/RDAVIDWebService/new/RDAVIDWebService.Rcheck/00install.out’ for details.
    ```

*   checking examples ... WARNING
    ```
    Found the following significant warnings:
      Warning: working directory was changed to ‘/private/var/folders/lb/xhxqmcrd7gv302_b1pdfykh80000gn/T/Rtmp9tGE10’, resetting
      Warning: working directory was changed to ‘/private/var/folders/lb/xhxqmcrd7gv302_b1pdfykh80000gn/T/Rtmp9tGE10’, resetting
      Warning: working directory was changed to ‘/private/var/folders/lb/xhxqmcrd7gv302_b1pdfykh80000gn/T/Rtmp9tGE10’, resetting
      Warning: working directory was changed to ‘/private/var/folders/lb/xhxqmcrd7gv302_b1pdfykh80000gn/T/Rtmp9tGE10’, resetting
      Warning: working directory was changed to ‘/private/var/folders/lb/xhxqmcrd7gv302_b1pdfykh80000gn/T/Rtmp9tGE10’, resetting
      Warning: working directory was changed to ‘/private/var/folders/lb/xhxqmcrd7gv302_b1pdfykh80000gn/T/Rtmp9tGE10’, resetting
      Warning: working directory was changed to ‘/private/var/folders/lb/xhxqmcrd7gv302_b1pdfykh80000gn/T/Rtmp9tGE10’, resetting
      Warning: working directory was changed to ‘/private/var/folders/lb/xhxqmcrd7gv302_b1pdfykh80000gn/T/Rtmp9tGE10’, resetting
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 24.1Mb
      sub-directories of 1Mb or more:
        java  21.7Mb
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Title field: should not end in a period.
    Malformed Description field: should contain one or more complete sentences.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      ‘callNextMethod’
    terms,DAVIDGODag: no visible global function definition for ‘nodeData’
    terms,DAVIDGODag: no visible global function definition for ‘goDag’
    uniqueIds,DAVIDGenes: no visible global function definition for
      ‘validObject’
    upsideDown,graph: no visible global function definition for ‘nodes’
    upsideDown,graph: no visible global function definition for ‘edges’
    upsideDown,graph : <anonymous>: no visible global function definition
      for ‘edges’
    upsideDown,graph : <anonymous> : <anonymous>: no visible global
      function definition for ‘addEdge’
    Undefined global functions or variables:
      GOGraph Term addEdge callNextMethod edges fill getFromNamespace goDag
      inEdges na.omit nodeData nodeData<- nodeDataDefaults<- nodes read.csv
      removeNode type.convert validObject x y
    Consider adding
      importFrom("methods", "callNextMethod", "validObject")
      importFrom("stats", "na.omit")
      importFrom("utils", "getFromNamespace", "read.csv", "type.convert")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Warning: package 'graph' was built under R version 3.5.1
    Warning: package 'IRanges' was built under R version 3.5.1
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'RDavidWS-vignette.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.30 \ifthenelse
                    {\boolean{Sweave@inconsolata}}{%^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# RDML

Version: 0.9-9

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.5Mb
      sub-directories of 1Mb or more:
        R     2.0Mb
        doc   2.4Mb
    ```

# re2r

Version: 0.2.0

## In both

*   checking for GNU extensions in Makefiles ... NOTE
    ```
    GNU make is a SystemRequirements.
    ```

# recoup

Version: 1.8.0

## In both

*   checking whether package ‘recoup’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘DelayedArray’ was built under R version 3.5.1
      Warning: package ‘Rsamtools’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/recoup/new/recoup.Rcheck/00install.out’ for details.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    recoup : <anonymous>: no visible global function definition for
      ‘runValue’
    recoupCorrelation: no visible binding for global variable ‘Index’
    recoupCorrelation: no visible binding for global variable ‘Coverage’
    recoupCorrelation: no visible binding for global variable ‘Condition’
    recoupCorrelation: no visible binding for global variable ‘Design’
    recoupHeatmap : <anonymous>: no visible global function definition for
      ‘grid.text’
    recoupProfile: no visible binding for global variable ‘Signal’
    recoupProfile: no visible binding for global variable ‘Condition’
    recoupProfile: no visible binding for global variable ‘Design’
    reduceExons : <anonymous>: no visible global function definition for
      ‘DataFrame’
    splitVector: no visible global function definition for ‘Rle’
    Undefined global functions or variables:
      Condition Coverage DataFrame Design IRanges Index Rle ScanBamParam
      Seqinfo Signal alphabetFrequency bamWhich<- biocLite dbConnect
      dbDisconnect dbDriver dbGetQuery dbWriteTable f flankedSexon gene
      get.transcript.utr.attributes getBSgenome grid.text indexBam
      installed.genomes mclapply mcmapply runValue seqlevels seqlevels<-
      sexon sortBam subjectHits warnwrap
    ```

# rEDM

Version: 0.7.1

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
      ...
    Error: processing vignette 'rEDM-algorithms.ltx' failed with diagnostics:
    Running 'texi2dvi' on 'rEDM-algorithms.ltx' failed.
    LaTeX errors:
    ! Package babel Error: Unknown option `english'. Either you misspelled it
    (babel)                or the language definition file english.ldf was not foun
    d.
    
    See the babel package documentation for explanation.
    ! LaTeX Error: File `cite.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.15 ^^M
            
    !  ==> Fatal error occurred, no output PDF file produced!
    Execution halted
    ```

# ReinforcementLearning

Version: 1.0.2

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘ggplot2’
      All declared Imports should be used.
    ```

# repijson

Version: 0.1.0

## In both

*   checking examples ... ERROR
    ```
    ...
    > toyll$date.of.admission <- as.POSIXct(toyll$date.of.admission)
    > toyll$date.of.discharge <- as.POSIXct(toyll$date.of.discharge)
    > toyll$contact1.date <- as.POSIXct(toyll$contact1.date)
    > toyll$contact2.date <- as.POSIXct(toyll$contact2.date)
    > toyll$contact3.date <- as.POSIXct(toyll$contact3.date)
    > 
    > ind.fields <- c(names(toyll)[1:5], "hospital", "fever", "sleepy")
    > x <- as.ejObject(toyll,
    +                  recordAttributes=ind.fields,
    +                  eventDefinitions=list(
    +                  define_ejEvent(name="admission", date="date.of.admission"),
    +                  define_ejEvent(name="discharge", date="date.of.discharge"),
    +                  define_ejEvent(name="contact1", date="contact1.date", attributes="contact1.id"),
    +                  define_ejEvent(name="contact2", date="contact2.date", attributes="contact2.id"),
    +                  define_ejEvent(name="contact3", date="contact3.date", attributes="contact3.id")
    +                  ))
    > print(x)
    Error in loadNamespace(j <- i[[1L]], c(lib.loc, .libPaths()), versionCheck = vI[[j]]) : 
      there is no package called ‘sf’
    Calls: print ... tryCatch -> tryCatchList -> tryCatchOne -> <Anonymous>
    Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 68-80 (repijson.Rmd) 
    Error: processing vignette 'repijson.Rmd' failed with diagnostics:
    there is no package called 'sf'
    Execution halted
    ```

# replyr

Version: 0.9.8

## In both

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘rquery’
    ```

# ReporteRs

Version: 0.8.10

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘ReporteRsjars’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# ReportingTools

Version: 2.20.0

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/runTests.R’ failed.
    Last 13 lines of output:
      Test files with failing tests
      
         test_DESeqDataSet.R 
           /Users/max/github/ggplot2/revdep/checks.noindex/ReportingTools/new/ReportingTools.Rcheck/ReportingTools/unitTests/test_DESeqDataSet.R 
      
         test_DESeqResults.R 
           /Users/max/github/ggplot2/revdep/checks.noindex/ReportingTools/new/ReportingTools.Rcheck/ReportingTools/unitTests/test_DESeqResults.R 
      
      
      Error in BiocGenerics:::testPackage("ReportingTools") : 
        unit tests failed for package ReportingTools
      In addition: Warning messages:
      1: package 'graph' was built under R version 3.5.1 
      2: package 'edgeR' was built under R version 3.5.1 
      Execution halted
    ```

*   checking whether package ‘ReportingTools’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package 'ggbio' was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/ReportingTools/new/ReportingTools.Rcheck/00install.out’ for details.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      'fData'
    .marrayLM.to.html: no visible global function definition for
      'featureNames'
    .marrayLM.to.html: no visible global function definition for 'fData'
    check.ids: no visible binding for global variable 'org.Hs.eg.db'
    check.ids: no visible global function definition for 'keys'
    custHeaderPanel : <anonymous>: no visible binding for global variable
      'tags'
    custHeaderPanel : <anonymous>: no visible global function definition
      for 'HTML'
    custHeaderPanel: no visible global function definition for 'tagList'
    custHeaderPanel: no visible global function definition for 'tag'
    custHeaderPanel: no visible global function definition for 'div'
    custHeaderPanel: no visible global function definition for 'h1'
    publish,trellis-HTMLReport: no visible binding for global variable
      'htmlRep'
    toReportDF,DESeqDataSet: no visible global function definition for
      'mcols'
    Undefined global functions or variables:
      HTML columns description div exprs fData featureNames h1 htmlRep keys
      keytype mcols org.Hs.eg.db tag tagList tags
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'basicReportingTools.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `times.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.8 \usepackage
                   {hyperref}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# rerddap

Version: 0.4.2

## In both

*   checking package dependencies ... NOTE
    ```
    Package which this enhances but not available for checking: ‘taxize’
    ```

# rexposome

Version: 1.2.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
      Categorical exposures will be droped and not used in the analysis.
    Warning: from glmnet Fortran code (error code -97); Convergence for 97th lambda value not reached after maxit=100000 iterations; solutions for larger lambdas returned
    Warning in mexwas(exp_std, phenotype = "wheezing", family = "binomial") :
      Categorical exposures will be droped and not used in the analysis.
    Loading required package: lattice
    
    Attaching package: 'mice'
    
    The following objects are masked from 'package:BiocGenerics':
    
        cbind, rbind
    
    The following objects are masked from 'package:base':
    
        cbind, rbind
    
    convert: profile 'icc': 'RGB ': RGB color space not permitted on grayscale PNG `mutiple_imputation_data_analysis_files/figure-html/plot_family_continuous-1.png' @ warning/png.c/MagickPNGWarningHandler/1744.
    Quitting from lines 229-231 (mutiple_imputation_data_analysis.Rmd) 
    Error: processing vignette 'mutiple_imputation_data_analysis.Rmd' failed with diagnostics:
    'names' attribute [4] must be the same length as the vector [0]
    Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 11.0Mb
      sub-directories of 1Mb or more:
        data      2.8Mb
        doc       5.1Mb
        extdata   2.2Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    imputeLOD: multiple local function definitions for ‘faux’ with
      different formal arguments
    plotHistogram,ExposomeSet: no visible binding for global variable
      ‘..density..’
    Undefined global functions or variables:
      ..density..
    ```

# rfigshare

Version: 0.3.7

## In both

*   checking R code for possible problems ... NOTE
    ```
    fs_author_ids : <anonymous>: no visible global function definition for
      ‘select.list’
    fs_download : <anonymous>: no visible global function definition for
      ‘download.file’
    Undefined global functions or variables:
      download.file select.list
    Consider adding
      importFrom("utils", "download.file", "select.list")
    to your NAMESPACE file.
    ```

# RforProteomics

Version: 1.18.1

## In both

*   R CMD check timed out
    

*   checking whether package ‘RforProteomics’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘MSnbase’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/RforProteomics/new/RforProteomics.Rcheck/00install.out’ for details.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 11.7Mb
      sub-directories of 1Mb or more:
        doc  10.8Mb
    ```

# RGCCA

Version: 2.1.2

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    tlmgr search --file --global '/algorithm2e.sty'
    Unknown directive ...l
    Warning in system2("tlmgr", args, ...) :
      running command ''tlmgr' search --file --global '/algorithm2e.sty'' had status 25
    ! LaTeX Error: File `algorithm2e.sty' not found.
    
    ! Emergency stop.
    <read *> 
    
    Error: processing vignette 'vignette_RGCCA.Rmd' failed with diagnostics:
    Failed to compile vignette_RGCCA.tex. See vignette_RGCCA.log for more info.
    Execution halted
    ```

# RGraphics

Version: 2.0-14

## In both

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking:
      ‘gWidgetsRGtk2’ ‘playwith’ ‘pmg’ ‘rggobi’ ‘RGraphics’ ‘SVGAnnotation’
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  9.9Mb
      sub-directories of 1Mb or more:
        extra   9.4Mb
    ```

# RiboProfiling

Version: 1.10.0

## In both

*   R CMD check timed out
    

*   checking whether package ‘RiboProfiling’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: replacing previous import ‘BiocGenerics::Position’ by ‘ggplot2::Position’ when loading ‘RiboProfiling’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/RiboProfiling/new/RiboProfiling.Rcheck/00install.out’ for details.
    ```

*   checking R code for possible problems ... NOTE
    ```
    applyShiftFeature: no visible global function definition for 'is'
    aroundPromoter: no visible global function definition for 'is'
    codonInfo: no visible global function definition for 'is'
    countShiftReads: no visible global function definition for 'is'
    histMatchLength: no visible global function definition for 'is'
    orfRelativePos: no visible global function definition for 'is'
    plotSummarizedCov: no visible global function definition for 'is'
    readStartCov: no visible global function definition for 'is'
    readsToStartOrEnd: no visible global function definition for 'is'
    riboSeqFromBAM: no visible global function definition for 'is'
    Undefined global functions or variables:
      is
    Consider adding
      importFrom("methods", "is")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

# RITAN

Version: 1.4.2

## Newly broken

*   R CMD check timed out
    

## In both

*   checking Rd \usage sections ... WARNING
    ```
    Documented arguments not in \usage in documentation object 'resource_reduce':
      ‘mutual_overlap’
    
    Functions with \usage entries need to have the appropriate \alias
    entries, and all their arguments documented.
    The \usage entries must correspond to syntactically valid R code.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Package listed in more than one of Depends, Imports, Suggests, Enhances:
      ‘sqldf’
    A package should be listed in only one of these fields.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘BgeeDB’ ‘dynamicTreeCut’ ‘gsubfn’
      All declared Imports should be used.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    load_geneset_symbols: no visible binding for global variable
      ‘geneset_list’
    load_geneset_symbols: no visible binding for global variable ‘f’
    load_geneset_symbols: no visible binding for global variable
      ‘active_genesets’
    network_overlap : <anonymous>: no visible binding for global variable
      ‘network_list’
    network_overlap: no visible binding for global variable ‘network_list’
    network_overlap : select_edges: no visible binding for global variable
      ‘network_list’
    plot.term_enrichment_by_subset: no visible binding for global variable
      ‘Var2’
    plot.term_enrichment_by_subset: no visible binding for global variable
      ‘Var1’
    show_active_genesets_hist: no visible binding for global variable
      ‘active_genesets’
    term_enrichment : process_source: no visible binding for global
      variable ‘active_genesets’
    Undefined global functions or variables:
      Var1 Var2 active_genesets all_net all_symbols f geneset_list
      network_list ss2
    ```

# rlfsm

Version: 0.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    ...
                                                      
    ! Missing } inserted.
    <inserted text> 
                    }
    l.1153 ...cum{}{-1-p}U\_{g,h}(x,y) dxdy}{}}{theta}
                                                      
    ! Missing $ inserted.
    <inserted text> 
                    $
    l.1153 ...cum{}{-1-p}U\_{g,h}(x,y) dxdy}{}}{theta}
                                                      
    ! Display math should end with $$.
    <to be read again> 
                       \endgroup 
    l.1153 ...cum{}{-1-p}U\_{g,h}(x,y) dxdy}{}}{theta}
                                                      
    ! Missing } inserted.
    <inserted text> 
                    }
    l.1153 ...cum{}{-1-p}U\_{g,h}(x,y) dxdy}{}}{theta}
                                                      
    ```

# RMCriteria

Version: 0.1.1

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘dplyr’
      All declared Imports should be used.
    ```

# rms

Version: 5.1-2

## In both

*   checking Rd cross-references ... NOTE
    ```
    Packages unavailable to check Rd xrefs: ‘multiwayvcov’, ‘treatSens’
    ```

# RNAdecay

Version: 1.0.2

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 98.9Mb
      sub-directories of 1Mb or more:
        data   2.7Mb
        libs  95.8Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    decay_plot: no visible binding for global variable ‘text’
    decay_plot: no visible binding for global variable ‘x’
    decay_plot: no visible binding for global variable ‘y’
    decay_plot: no visible binding for global variable ‘t.decay’
    decay_plot: no visible binding for global variable ‘value’
    decay_plot: no visible binding for global variable ‘treatment’
    Undefined global functions or variables:
      t.decay text treatment value x y
    Consider adding
      importFrom("graphics", "text")
    to your NAMESPACE file.
    ```

# RnBeads

Version: 1.12.1

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/runTests.R’ failed.
    Last 13 lines of output:
      ERROR in /Users/max/github/ggplot2/revdep/checks.noindex/RnBeads/new/RnBeads.Rcheck/RnBeads/unitTests/test_differential.R: Error while sourcing  /Users/max/github/ggplot2/revdep/checks.noindex/RnBeads/new/RnBeads.Rcheck/RnBeads/unitTests/test_differential.R : Error in meth(rnb.set.example) : object 'rnb.set.example' not found
      ERROR in /Users/max/github/ggplot2/revdep/checks.noindex/RnBeads/new/RnBeads.Rcheck/RnBeads/unitTests/test_others.R: Error while sourcing  /Users/max/github/ggplot2/revdep/checks.noindex/RnBeads/new/RnBeads.Rcheck/RnBeads/unitTests/test_others.R : Error in meth(rnb.set.example) : object 'rnb.set.example' not found
      
      Test files with failing tests
      
         test_differential.R 
           /Users/max/github/ggplot2/revdep/checks.noindex/RnBeads/new/RnBeads.Rcheck/RnBeads/unitTests/test_differential.R 
      
         test_others.R 
           /Users/max/github/ggplot2/revdep/checks.noindex/RnBeads/new/RnBeads.Rcheck/RnBeads/unitTests/test_others.R 
      
      
      Error in BiocGenerics:::testPackage("RnBeads") : 
        unit tests failed for package RnBeads
      Execution halted
    ```

*   checking whether package ‘RnBeads’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘limma’ was built under R version 3.5.1
      Warning: package ‘GenomicFeatures’ was built under R version 3.5.1
      Warning: package ‘DelayedArray’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/RnBeads/new/RnBeads.Rcheck/00install.out’ for details.
    ```

*   checking running R code from vignettes ...
    ```
       ‘RnBeads.Rnw’ ... failed
       ‘RnBeads_Annotations.Rnw’ ... OK
     WARNING
    Errors in running code in vignettes:
    when running code in ‘RnBeads.Rnw’
      ...
    > rnb.options(filtering.sex.chromosomes.removal = TRUE, 
    +     identifiers.column = "Sample_ID")
    
    > rnb.options(differential = FALSE)
    
    > library(RnBeads.hg19)
    
      When sourcing ‘RnBeads.R’:
    Error: there is no package called ‘RnBeads.hg19’
    Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘RnBeads.hg19’
    
    Depends: includes the non-default packages:
      ‘BiocGenerics’ ‘S4Vectors’ ‘GenomicRanges’ ‘MASS’ ‘cluster’ ‘ff’
      ‘fields’ ‘ggplot2’ ‘gplots’ ‘gridExtra’ ‘limma’ ‘matrixStats’
      ‘illuminaio’ ‘methylumi’ ‘plyr’
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  8.5Mb
      sub-directories of 1Mb or more:
        R     3.1Mb
        doc   3.2Mb
    ```

*   checking whether the namespace can be loaded with stated dependencies ... NOTE
    ```
    Warning: no function found corresponding to methods exports from ‘RnBeads’ for: ‘samples’
    
    A namespace must be able to be loaded with just the base namespace
    loaded: otherwise if the namespace gets loaded by a saved object, the
    session will be unable to start.
    
    Probably some imports need to be declared in the NAMESPACE file.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Unexported objects imported by ':::' calls:
      'Gviz:::.getBMFeatureMap' 'doParallel:::.options'
      'grDevices:::.smoothScatterCalcDensity'
      'minfi:::.default.450k.annotation' 'minfi:::.extractFromRGSet450k'
      'minfi:::.normalizeFunnorm450k'
      See the note in ?`:::` about the use of this operator.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      Density Deviance Difference DoISVA Error EstDimRMT GenomeAxisTrack ID
      IdeogramTrack IlluminaHumanMethylation450kmanifest
      IlluminaHumanMethylationEPICmanifest Index Intensity Measure
      PairsBootRefFreeEwasModel Probe RGChannelSet Rd2HTML RefFreeEwasModel
      SNP Sample Slide Target Term UcscTrack Value addSex as.profileCGH
      assayDataElement assayDataElementNames barcode bv chrom combinedRank
      combinedRank.var comma covgMedian covgPercLow covgPercUp cv.glmnet
      daglad diffmeth diffmeth.p.adj.fdr diffmeth.p.val
      dinucleotideFrequency eps expectedCounts featureData featureData<-
      featureNames featureNames<- foreach geneCounts genome<- getCN
      getDoParWorkers getGreen getManifest getMeth getRed getSex getUnmeth
      getVarCov glmnet grid.draw grid.newpage group group1 group2 i
      impute.knn intensities is.subsegmentation k letterFrequency lme
      loadRegionDB log10FDR log10P mapToGenome mean.diff mean.quot.log2
      melt mergeRegionDBs muted n.sites ng.2 num.sites numSites
      numeric.names oddsRatios p.vals.t.na.adj pData percent_format
      phenoData phenoData<- plotTracks preprocessSWAN pvalues qvalue
      refText reg.type region.size registerDoParallel relative.coord report
      runLOLA samples seqlengths seqlevels<- sigCategories sites2ignore
      size solve.QP stopCluster sva target tsne type types universeCounts
      useMart v var.diff varFit varLabels x xmlValue y yint
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Warning: package ‘GenomicRanges’ was built under R version 3.5.1
    Warning: package ‘IRanges’ was built under R version 3.5.1
    Warning: package ‘limma’ was built under R version 3.5.1
    Warning: package ‘GenomicFeatures’ was built under R version 3.5.1
    Warning: package ‘DelayedArray’ was built under R version 3.5.1
    
    Error: processing vignette 'RnBeads.Rnw' failed with diagnostics:
     chunk 8 
    Error in library(RnBeads.hg19) : 
      there is no package called ‘RnBeads.hg19’
    Execution halted
    ```

# RndTexExams

Version: 1.5

## In both

*   checking examples ... ERROR
    ```
    ...
    > # Build pdfs
    > result.out <- rte.build.rdn.test(list.in = list.out,
    +                                  f.out = f.out,
    +                                  n.test = n.test,
    +                                  n.question = n.question,
    +                                  latex.dir.out = latex.dir.out,
    +                                  pdf.dir.out = pdf.dir.out)
    
    rte: Checking for error in inputs... Done
    rte: pdflatex flavor: texlive
    rte: Type of OS: Darwin
    rte: Latex compile function: custom
    rte: Type of exam template: examdesign
    rte: Number of mchoice questions: 4
    rte: Building Test #1...This is pdfTeX, Version 3.14159265-2.6-1.40.19 (TeX Live 2018) (preloaded format=pdflatex)
     restricted \write18 enabled.
    entering extended mode
    Error in rte.compile.latex(f.temp.tex, pdf.dir.out = pdf.dir.out, latex.compile.fct = latex.compile.fct) : 
      Cant find compiled pdf file. You should check for sintax errors in the tex file and whether your OS has a flavor of latex installed.
    Calls: rte.build.rdn.test -> rte.compile.latex
    Execution halted
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      ── 1. Error: (unknown) (@test_rte.R#24)  ───────────────────────────────────────
      Cant find compiled pdf file. You should check for sintax errors in the tex file and whether your OS has a flavor of latex installed.
      1: rte.build.rdn.test(list.in = list.out, f.out = f.out, n.test = n.test, n.question = n.question, 
             latex.dir.out = latex.dir.out, pdf.dir.out = pdf.dir.out, do.randomize.questions = T, 
             latex.compile.fct = "custom", do.randomize.answers = T, do.clean.up = T) at testthat/test_rte.R:24
      2: rte.compile.latex(f.temp.tex, pdf.dir.out = pdf.dir.out, latex.compile.fct = latex.compile.fct)
      3: stop(paste("Cant find compiled pdf file. You should check for sintax errors in the tex file and whether your", 
             "OS has a flavor of latex installed."))
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 1 SKIPPED: 0 FAILED: 1
      1. Error: (unknown) (@test_rte.R#24) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    This is pdfTeX, Version 3.14159265-2.6-1.40.19 (TeX Live 2018) (preloaded format=pdflatex)
     restricted \write18 enabled.
    entering extended mode
    Quitting from lines 121-147 (rte-vignette_creating_exams.Rmd) 
    Error: processing vignette 'rte-vignette_creating_exams.Rmd' failed with diagnostics:
    Cant find compiled pdf file. You should check for sintax errors in the tex file and whether your OS has a flavor of latex installed.
    Execution halted
    ```

# Rnightlights

Version: 0.2.3

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
             gadmVersion = gadmVersion, custPolyPath = custPolyPath), method = "auto", mode = "wb", 
             extra = "-c")
      
      PING google.com (172.217.8.14): 56 data bytes
      64 bytes from 172.217.8.14: icmp_seq=0 ttl=53 time=25.606 ms
      
      --- google.com ping statistics ---
      1 packets transmitted, 1 packets received, 0.0% packet loss
      round-trip min/avg/max/stddev = 25.606/25.606/25.606/0.000 ms
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 50 SKIPPED: 1 FAILED: 1
      1. Error: admLevel lookups work (@test-admlevels.R#10) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

# Rnits

Version: 1.14.0

## In both

*   checking whether package ‘Rnits’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘limma’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/Rnits/new/Rnits.Rcheck/00install.out’ for details.
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    ==================================================
    downloaded 502 KB
    
    Warning in has_utility("pdfcrop") :
      pdfcrop not installed or not in PATH
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'Rnits-vignette.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `beramono.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.87 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Description field: should contain one or more complete sentences.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      ‘scale_color_brewer’
    plotResults,Rnits: no visible global function definition for
      ‘facet_wrap’
    plotResults,Rnits: no visible global function definition for ‘dev.off’
    summary,Rnits: no visible global function definition for ‘hist’
    timeAlign,Rnits: no visible global function definition for ‘quantile’
    timeAlign,Rnits: no visible global function definition for ‘mvfft’
    timeAlign,Rnits: no visible global function definition for ‘abline’
    Undefined global functions or variables:
      Sample Time abline aes dev.off facet_wrap gaussian geom_point
      geom_smooth glm hat hist kmeans mad median mvfft p.adjust par predict
      quantile rnorm scale_color_brewer sd setNames setTxtProgressBar
      smooth.spline theme theme_bw txtProgressBar value ylab
    Consider adding
      importFrom("grDevices", "dev.off")
      importFrom("graphics", "abline", "hist", "par")
      importFrom("stats", "gaussian", "glm", "hat", "kmeans", "mad",
                 "median", "mvfft", "p.adjust", "predict", "quantile",
                 "rnorm", "sd", "setNames", "smooth.spline")
      importFrom("utils", "setTxtProgressBar", "txtProgressBar")
    to your NAMESPACE file.
    ```

# robCompositions

Version: 2.0.8

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'imputation.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `scrartcl.cls' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: cls)
    
    ! Emergency stop.
    <read *> 
             
    l.3 \documentclass{scrartcl}\usepackage
                                           []{graphicx}\usepackage[]{color}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘StatDA’
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 5701 marked UTF-8 strings
    ```

# robustbase

Version: 0.93-3

## In both

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘robustX’
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Loading required package: robustbase
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'fastMcd-kmini.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `mathtools.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.12 \usepackage
                    [utf8]{inputenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# robustlmm

Version: 2.2-1

## In both

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    Loading required package: Matrix
    Loading required package: ggplot2
    Warning in compare(fmUncontam, fm, rfm, rfm2, rfm3, show.rho.functions = FALSE) :
      Comparison for objects not fitted to the same dataset
    Loading required package: reshape2
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'rlmer.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.55 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# rODE

Version: 0.99.6

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘data.table’
      All declared Imports should be used.
    ```

# Roleswitch

Version: 1.18.0

## In both

*   checking whether package ‘Roleswitch’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘IRanges’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/Roleswitch/new/Roleswitch.Rcheck/00install.out’ for details.
    ```

*   checking running R code from vignettes ...
    ```
       ‘Roleswitch.Rnw’ ... failed
     WARNING
    Errors in running code in vignettes:
    when running code in ‘Roleswitch.Rnw’
      ...
    +     "hsa-miR-27b", "hsa-miR-2 ..." ... [TRUNCATED] 
    
    > rs <- roleswitch(eset, mirna.expr)
    Loading required package: hgu95av2.db
    Warning in library(package, lib.loc = lib.loc, character.only = TRUE, logical.return = TRUE,  :
      there is no package called ‘hgu95av2.db’
    
      When sourcing ‘Roleswitch.R’:
    Error: hgu95av2.db package must be installed
    Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Depends: includes the non-default packages:
      ‘pracma’ ‘reshape’ ‘plotrix’ ‘microRNA’ ‘biomaRt’ ‘Biostrings’
      ‘Biobase’ ‘DBI’
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking R code for possible problems ... NOTE
    ```
    diagnosticPlot: no visible global function definition for ‘par’
    diagnosticPlot: no visible global function definition for ‘axis’
    diagnosticPlot: no visible global function definition for ‘plot’
    getSeedMatrix: no visible global function definition for ‘data’
    getTranscriptIDwithLongest3UTR: no visible global function definition
      for ‘aggregate’
    roleswitch: no visible global function definition for ‘aggregate’
    Undefined global functions or variables:
      aggregate axis data par plot
    Consider adding
      importFrom("graphics", "axis", "par", "plot")
      importFrom("stats", "aggregate")
      importFrom("utils", "data")
    to your NAMESPACE file.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    3: max(p.x-p.x.prev)=0.00009
    4: max(p.x-p.x.prev)=0.00000
    
    Start roleswitch with 365 miRNA and 10892 mRNA
    1: max(p.x-p.x.prev)=0.12596
    2: max(p.x-p.x.prev)=0.00009
    3: max(p.x-p.x.prev)=0.00003
    4: max(p.x-p.x.prev)=0.00000
    Some genes or miRNA are left out in calculation
    b/c they have zero target sites or targets!
    Their probabilities are set to zero in the output matrices
    Loading required package: ggplot2
    Loading required package: hgu95av2.db
    Warning in library(package, lib.loc = lib.loc, character.only = TRUE, logical.return = TRUE,  :
      there is no package called ‘hgu95av2.db’
    
    Error: processing vignette 'Roleswitch.Rnw' failed with diagnostics:
     chunk 10 (label = eset) 
    Error in roleswitch(eset, mirna.expr) : 
      hgu95av2.db package must be installed
    Execution halted
    ```

# rolypoly

Version: 0.1.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘matrixcalc’
      All declared Imports should be used.
    ```

# rotations

Version: 1.5

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'rotations-intro.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `microtype.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.43 ^^M
            
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.5Mb
      sub-directories of 1Mb or more:
        R      2.0Mb
        data   2.3Mb
    ```

# rpanel

Version: 1.1-4

## In both

*   checking whether package ‘rpanel’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/rpanel/new/rpanel.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘rpanel’ ...
** package ‘rpanel’ successfully unpacked and MD5 sums checked
** R
** data
*** moving datasets to lazyload DB
** demo
** inst
** byte-compile and prepare package for lazy loading
Warning in fun(libname, pkgname) : couldn't connect to display ""
Error in structure(.External(.C_dotTcl, ...), class = "tclObj") : 
  [tcl] can't find package BWidget.

Error : unable to load R code in package ‘rpanel’
ERROR: lazy loading failed for package ‘rpanel’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/rpanel/new/rpanel.Rcheck/rpanel’

```
### CRAN

```
* installing *source* package ‘rpanel’ ...
** package ‘rpanel’ successfully unpacked and MD5 sums checked
** R
** data
*** moving datasets to lazyload DB
** demo
** inst
** byte-compile and prepare package for lazy loading
Warning in fun(libname, pkgname) : couldn't connect to display ""
Error in structure(.External(.C_dotTcl, ...), class = "tclObj") : 
  [tcl] can't find package BWidget.

Error : unable to load R code in package ‘rpanel’
ERROR: lazy loading failed for package ‘rpanel’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/rpanel/old/rpanel.Rcheck/rpanel’

```
# RPPanalyzer

Version: 1.4.5

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'RPPanalyzer.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `times.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.4 \usepackage
                   {hyperref}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.5Mb
      sub-directories of 1Mb or more:
        extdata   4.5Mb
    ```

# rPref

Version: 1.2

## In both

*   checking for GNU extensions in Makefiles ... NOTE
    ```
    GNU make is a SystemRequirements.
    ```

# rprev

Version: 1.0.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘doParallel’ ‘foreach’
      All declared Imports should be used.
    ```

# rpsftm

Version: 1.2.3

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘rootSolve’
    
    Package suggested but not available for checking: ‘tableone’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# Rqc

Version: 1.14.0

## In both

*   checking whether package ‘Rqc’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘Rsamtools’ was built under R version 3.5.1
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘DelayedArray’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/Rqc/new/Rqc.Rcheck/00install.out’ for details.
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

# rrr

Version: 1.0.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘Rcpp’
      All declared Imports should be used.
    ```

# RSA

Version: 0.9.12

## In both

*   checking whether package ‘RSA’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: loading Rplot failed
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/RSA/new/RSA.Rcheck/00install.out’ for details.
    ```

# rSARP

Version: 1.0.0

## In both

*   checking files in ‘vignettes’ ... NOTE
    ```
    Files named as vignettes but with no recognized vignette engine:
       ‘vignettes/Untitled.Rnw’
    (Is a VignetteBuilder field missing?)
    ```

# RSDA

Version: 2.0.8

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘randomcoloR’
      All declared Imports should be used.
    ```

# rsinaica

Version: 0.5.0

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘rsinaica-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: sinaica_station_dates
    > ### Title: Dates supported by a station
    > ### Aliases: sinaica_station_dates
    > 
    > ### ** Examples
    > 
    > ## id 271 is Xalostoc. See `stations_sinaica`
    > df <- sinaica_station_dates(271, "Manual")
    Error: parse error: premature EOF
                                           
                         (right here) ------^
    Execution halted
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 467 marked UTF-8 strings
    ```

# rsMove

Version: 0.2.5

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘gdalUtils’ ‘igraph’ ‘lattice’ ‘lubridate’ ‘randomForest’ ‘rgdal’
      ‘spatialEco’
      All declared Imports should be used.
    ```

# rsoi

Version: 0.3.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘utils’
      All declared Imports should be used.
    ```

# rstan

Version: 2.18.1

## In both

*   R CMD check timed out
    

*   checking installed package size ... NOTE
    ```
      installed size is 10.8Mb
      sub-directories of 1Mb or more:
        libs   8.3Mb
    ```

# rstanarm

Version: 2.18.1

## In both

*   R CMD check timed out
    

*   checking installed package size ... NOTE
    ```
      installed size is 19.3Mb
      sub-directories of 1Mb or more:
        R      2.0Mb
        doc    3.4Mb
        libs  13.3Mb
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘biglm’
    ```

*   checking for GNU extensions in Makefiles ... NOTE
    ```
    GNU make is a SystemRequirements.
    ```

# rsvd

Version: 0.9

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  6.2Mb
      sub-directories of 1Mb or more:
        data   6.1Mb
    ```

# rsvg

Version: 1.3

## In both

*   checking whether package ‘rsvg’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/rsvg/new/rsvg.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘rsvg’ ...
** package ‘rsvg’ successfully unpacked and MD5 sums checked
Package librsvg-2.0 was not found in the pkg-config search path.
Perhaps you should add the directory containing `librsvg-2.0.pc'
to the PKG_CONFIG_PATH environment variable
No package 'librsvg-2.0' found
Homebrew 1.7.7
Homebrew/homebrew-core (git revision 1a54; last commit 2018-10-12)
Homebrew/homebrew-cask (git revision 1745; last commit 2018-10-12)
Using PKG_CFLAGS=-I/usr/local/opt/librsvg/include
Using PKG_LIBS=-L/usr/local/opt/librsvg/lib -lrsvg
------------------------- ANTICONF ERROR ---------------------------
Configuration failed because librsvg-2.0 was not found. Try installing:
 * deb: librsvg2-dev (Debian, Ubuntu, etc)
 * rpm: librsvg2-devel (Fedora, EPEL)
 * csw: librsvg_dev, sunx11_devel (Solaris)
 * brew: librsvg (OSX)
If librsvg-2.0 is already installed, check that 'pkg-config' is in your
PATH and PKG_CONFIG_PATH contains a librsvg-2.0.pc file. If pkg-config
is unavailable you can set INCLUDE_DIR and LIB_DIR manually via:
R CMD INSTALL --configure-vars='INCLUDE_DIR=... LIB_DIR=...'
--------------------------------------------------------------------
ERROR: configuration failed for package ‘rsvg’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/rsvg/new/rsvg.Rcheck/rsvg’

```
### CRAN

```
* installing *source* package ‘rsvg’ ...
** package ‘rsvg’ successfully unpacked and MD5 sums checked
Package librsvg-2.0 was not found in the pkg-config search path.
Perhaps you should add the directory containing `librsvg-2.0.pc'
to the PKG_CONFIG_PATH environment variable
No package 'librsvg-2.0' found
Homebrew 1.7.7
Homebrew/homebrew-core (git revision 1a54; last commit 2018-10-12)
Homebrew/homebrew-cask (git revision 1745; last commit 2018-10-12)
Using PKG_CFLAGS=-I/usr/local/opt/librsvg/include
Using PKG_LIBS=-L/usr/local/opt/librsvg/lib -lrsvg
------------------------- ANTICONF ERROR ---------------------------
Configuration failed because librsvg-2.0 was not found. Try installing:
 * deb: librsvg2-dev (Debian, Ubuntu, etc)
 * rpm: librsvg2-devel (Fedora, EPEL)
 * csw: librsvg_dev, sunx11_devel (Solaris)
 * brew: librsvg (OSX)
If librsvg-2.0 is already installed, check that 'pkg-config' is in your
PATH and PKG_CONFIG_PATH contains a librsvg-2.0.pc file. If pkg-config
is unavailable you can set INCLUDE_DIR and LIB_DIR manually via:
R CMD INSTALL --configure-vars='INCLUDE_DIR=... LIB_DIR=...'
--------------------------------------------------------------------
ERROR: configuration failed for package ‘rsvg’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/rsvg/old/rsvg.Rcheck/rsvg’

```
# RSwissMaps

Version: 0.1.0

## In both

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 18627 marked UTF-8 strings
    ```

# RTCGA

Version: 1.10.0

## In both

*   checking examples ... ERROR
    ```
    ...
    + 	MET = `MET|4233`) %>%  
    + 	#cancer samples
    + 	filter(substr(bcr_patient_barcode, 14, 15) == "01") -> ACC_BLCA_BRCA_OV.rnaseq
    > 	
    > 
    > boxplotTCGA(ACC_BLCA_BRCA_OV.rnaseq, "cohort", "MET")
    > boxplotTCGA(ACC_BLCA_BRCA_OV.rnaseq, "cohort", "log1p(MET)")
    > boxplotTCGA(ACC_BLCA_BRCA_OV.rnaseq, "reorder(cohort,log1p(MET), median)", "log1p(MET)")
    > boxplotTCGA(ACC_BLCA_BRCA_OV.rnaseq, "reorder(cohort,log1p(MET), max)", "log1p(MET)")
    > boxplotTCGA(ACC_BLCA_BRCA_OV.rnaseq, "reorder(cohort,log1p(MET), median)", "log1p(MET)",
    + xlab = "Cohort Type", ylab = "Logarithm of MET")
    > boxplotTCGA(ACC_BLCA_BRCA_OV.rnaseq, "reorder(cohort,log1p(MET), median)", "log1p(MET)", 
    + xlab = "Cohort Type", ylab = "Logarithm of MET", legend.title = "Cohorts")
    > boxplotTCGA(ACC_BLCA_BRCA_OV.rnaseq, "reorder(cohort,log1p(MET), median)", "log1p(MET)", 
    + xlab = "Cohort Type", ylab = "Logarithm of MET", legend.title = "Cohorts", legend = "bottom")
    > 
    > ## facet example
    > library(RTCGA.mutations)
    Error in library(RTCGA.mutations) : 
      there is no package called ‘RTCGA.mutations’
    Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking:
      ‘RTCGA.clinical’ ‘RTCGA.mutations’ ‘RTCGA.RPPA’ ‘RTCGA.mRNA’
      ‘RTCGA.miRNASeq’ ‘RTCGA.methylation’ ‘RTCGA.CNV’ ‘RTCGA.PANCAN12’
    ```

*   checking R code for possible problems ... NOTE
    ```
    availableDates: no visible binding for global variable ‘.’
    downloadTCGA: no visible binding for global variable ‘.’
    ggbiplot: no visible binding for global variable ‘xvar’
    ggbiplot: no visible binding for global variable ‘yvar’
    ggbiplot: no visible global function definition for ‘muted’
    ggbiplot: no visible binding for global variable ‘varname’
    ggbiplot: no visible binding for global variable ‘angle’
    ggbiplot: no visible binding for global variable ‘hjust’
    read.mutations: no visible binding for global variable ‘.’
    read.rnaseq: no visible binding for global variable ‘.’
    survivalTCGA: no visible binding for global variable ‘times’
    whichDateToUse: no visible binding for global variable ‘.’
    Undefined global functions or variables:
      . angle hjust muted times varname xvar yvar
    ```

*   checking Rd cross-references ... NOTE
    ```
    Packages unavailable to check Rd xrefs: ‘RTCGA.clinical’, ‘RTCGA.mutations’, ‘RTCGA.CNV’, ‘RTCGA.RPPA’, ‘RTCGA.mRNA’, ‘RTCGA.miRNASeq’, ‘RTCGA.methylation’
    ```

# rtf

Version: 0.4-13

## In both

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'rtf.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.30 \ifthenelse
                    {\boolean{Sweave@inconsolata}}{%^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# rtimicropem

Version: 1.3

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘R6’
      All declared Imports should be used.
    ```

# rtrek

Version: 0.1.0

## In both

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 992 marked UTF-8 strings
    ```

# rTRM

Version: 1.18.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Read 8734 items
    Read 8734 items
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'rTRM_Introduction.tex' failed.
    LaTeX errors:
    ! Package babel Error: Unknown option `english'. Either you misspelled it
    (babel)                or the language definition file english.ldf was not foun
    d.
    
    See the babel package documentation for explanation.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# RtutoR

Version: 1.2

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘DT’ ‘rmarkdown’
      All declared Imports should be used.
    ```

# rtweet

Version: 0.6.8

## In both

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 113868 marked UTF-8 strings
    ```

# rubias

Version: 0.1.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘ggplot2’
      All declared Imports should be used.
    ```

# rvinecopulib

Version: 0.3.0.1.1

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  7.9Mb
      sub-directories of 1Mb or more:
        include   2.5Mb
        libs      4.9Mb
    ```

# rWBclimate

Version: 0.1.3

## In both

*   checking R code for possible problems ... NOTE
    ```
    check_ISO_code: no visible binding for global variable ‘NoAm_country’
    check_ISO_code: no visible binding for global variable ‘SoAm_country’
    check_ISO_code: no visible binding for global variable ‘Oceana_country’
    check_ISO_code: no visible binding for global variable ‘Africa_country’
    check_ISO_code: no visible binding for global variable ‘Asia_country’
    check_ISO_code: no visible binding for global variable ‘Eur_country’
    climate_map: no visible binding for global variable ‘data’
    date_correct: no visible global function definition for ‘tail’
    Undefined global functions or variables:
      Africa_country Asia_country Eur_country NoAm_country Oceana_country
      SoAm_country data tail
    Consider adding
      importFrom("utils", "data", "tail")
    to your NAMESPACE file.
    ```

# RxODE

Version: 0.8.0-8

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘n1qn1’
      All declared Imports should be used.
    ```

# rzeit2

Version: 0.2.0

## In both

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 841 marked UTF-8 strings
    ```

# sadists

Version: 0.2.3

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    `binwidth`.
    large z may result in inaccurate quantiles
    `stat_bin()` using `bins = 30`. Pick better value with
    `binwidth`.
    large z may result in inaccurate quantiles
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'sadists.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `paralist.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.91 ^^M
            
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# samExploreR

Version: 1.4.0

## In both

*   checking whether package ‘samExploreR’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘Rsubread’ was built under R version 3.5.1
      Warning: package ‘edgeR’ was built under R version 3.5.1
      Warning: package ‘limma’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/samExploreR/new/samExploreR.Rcheck/00install.out’ for details.
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .BBSoptions
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    plotsamExplorer: no visible binding for global variable ‘f’
    plotsamExplorer: no visible binding for global variable ‘value’
    plotsamExplorer: no visible binding for global variable ‘group’
    Undefined global functions or variables:
      f group value
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    Sub sample N: 7
    Sub sampling: X.Users.max.github.ggplot2.revdep.library.noindex.samExploreR.RNAseqData.HNRNPC.bam.chr14.extdata.ERR127305_chr14.bam
    Sub sample N: 8
    ANOVA test for label 'New, Gene' and f values 0.85, 0.9, 0.95
    ANOVA test for labels 'New, Gene', 'Old, Gene', 'New, Exon' and f value 0.9
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'Manual.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `psfrag.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.10 \RequirePackage
                        []{/Users/max/github/ggplot2/revdep/library.noindex/samE...
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# sampleClassifier

Version: 1.4.0

## In both

*   R CMD check timed out
    

*   checking whether package ‘sampleClassifier’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘IRanges’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/sampleClassifier/new/sampleClassifier.Rcheck/00install.out’ for details.
    ```

# SamplerCompare

Version: 1.3.0

## In both

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'glue.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `setspace.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.9 \usepackage
                   {fancyvrb}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# sand

Version: 1.0.3

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘mixer’
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 6 marked UTF-8 strings
    ```

# santaR

Version: 1.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘gridExtra’
      All declared Imports should be used.
    ```

# savR

Version: 1.18.0

## In both

*   checking R code for possible problems ... NOTE
    ```
    buildReports,savProject-character: no visible global function
      definition for ‘dev.off’
    qualityHeatmap,savProject-integer-integer-logical: no visible global
      function definition for ‘quantile’
    Undefined global functions or variables:
      dev.off quantile
    Consider adding
      importFrom("grDevices", "dev.off")
      importFrom("stats", "quantile")
    to your NAMESPACE file.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Loading required package: ggplot2
    Warning: `position` is deprecated
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'savR.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `memoir.cls' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: cls)
    
    ! Emergency stop.
    <read *> 
             
    l.2 \usepackage
                   {microtype}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# SC3

Version: 1.8.0

## In both

*   R CMD check timed out
    

*   checking installed package size ... NOTE
    ```
      installed size is  6.3Mb
      sub-directories of 1Mb or more:
        data   2.6Mb
        doc    3.0Mb
    ```

# scater

Version: 1.8.4

## In both

*   checking whether package ‘scater’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘DelayedArray’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/scater/new/scater.Rcheck/00install.out’ for details.
    ```

*   checking examples ... WARNING
    ```
    Found the following significant warnings:
    
      Warning: 'read10xResults' is deprecated.
      Warning: 'downsampleCounts' is deprecated.
      Warning: 'normalizeExprs' is deprecated.
      Warning: 'normalizeExprs' is deprecated.
      Warning: 'normalizeExprs' is deprecated.
      Warning: 'normalizeExprs' is deprecated.
      Warning: 'normalizeExprs' is deprecated.
      Warning: 'normalizeExprs' is deprecated.
      Warning: 'read10xResults' is deprecated.
    Deprecated functions may be defunct as soon as of the next release of
    R.
    See ?Deprecated.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 14.8Mb
      sub-directories of 1Mb or more:
        doc       5.4Mb
        extdata   2.9Mb
        libs      4.8Mb
    ```

# scatr

Version: 1.0.1

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘R6’ ‘cowplot’ ‘ggplot2’ ‘ggridges’ ‘ggstance’ ‘jmvcore’
      All declared Imports should be used.
    ```

# scDD

Version: 1.4.0

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    > ### ** Examples
    > 
    >  
    > # load toy simulated example SingleCellExperiment object to find DD genes  
    > data(scDatExSim)
    > 
    > # set arguments to pass to scDD function
    > 
    > prior_param=list(alpha=0.01, mu0=0, s0=0.01, a0=0.01, b0=0.01)
    > 
    > # call the scDD function to perform permutations and classify DD genes
    > 
    > scDatExSim <- scDD(scDatExSim, prior_param=prior_param, testZeroes=FALSE)
    Setting up parallel back-end using 18 cores
    Clustering observed expression data for each gene
    Warning in socketConnection(host, port, TRUE, TRUE, "a+b", timeout = timeout) :
      port 11399 cannot be opened
    Error in socketConnection(host, port, TRUE, TRUE, "a+b", timeout = timeout) : 
      cannot open the connection
    Calls: scDD ... .local -> .bpfork -> .bpforkConnect -> socketConnection
    Execution halted
    ```

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    
    The following object is masked from 'package:BiocGenerics':
    
        combine
    
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'scDD.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `underscore.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.33 ^^M
            
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘grDevices’ ‘graphics’ ‘stats’
      All declared Imports should be used.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      ‘binomial’
    validation : <anonymous>: no visible global function definition for
      ‘var’
    validation: no visible binding for global variable ‘var’
    validation: no visible global function definition for ‘par’
    validation: no visible global function definition for ‘plot’
    validation: no visible global function definition for ‘abline’
    validation: no visible global function definition for ‘points’
    Undefined global functions or variables:
      abline axis binomial density dev.off fisher.test hcl hist ks.test
      lines lm model.matrix p.adjust par pchisq pdf plot points quantile
      rbinom rect residuals rnbinom rt runif t.test var
    Consider adding
      importFrom("grDevices", "dev.off", "hcl", "pdf")
      importFrom("graphics", "abline", "axis", "hist", "lines", "par",
                 "plot", "points", "rect")
      importFrom("stats", "binomial", "density", "fisher.test", "ks.test",
                 "lm", "model.matrix", "p.adjust", "pchisq", "quantile",
                 "rbinom", "residuals", "rnbinom", "rt", "runif", "t.test",
                 "var")
    to your NAMESPACE file.
    ```

# scFeatureFilter

Version: 1.0.0

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  6.4Mb
      sub-directories of 1Mb or more:
        data   3.7Mb
        doc    2.4Mb
    ```

# scmap

Version: 1.2.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘Biobase’
      All declared Imports should be used.
    ```

# scmeth

Version: 1.0.1

## In both

*   R CMD check timed out
    

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘ggplot2’
      All declared Imports should be used.
    ```

# SCnorm

Version: 1.2.1

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    11:                   PrintProgressPlots = TRUE                
    12:                   FilterCellNum
                          ^
    Warning in highr::hilight(x, format, prompt = options$prompt, markup = opts$markup) :
      the syntax of the source code is invalid; the fallback mode is used
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'SCnorm.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `setspace.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.33 \usepackage
                    {natbib}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# scone

Version: 1.4.0

## In both

*   R CMD check timed out
    

*   checking whether package ‘scone’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘DelayedArray’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/scone/new/scone.Rcheck/00install.out’ for details.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    sconeReport : server: no visible global function definition for ‘theme’
    sconeReport : server: no visible global function definition for
      ‘element_blank’
    sconeReport : server: no visible global function definition for
      ‘ggplotly’
    sconeReport : server: no visible global function definition for
      ‘geom_violin’
    sconeReport : server: no visible global function definition for
      ‘coord_cartesian’
    sconeReport : server: no visible global function definition for
      ‘scale_fill_manual’
    sconeReport : server: no visible global function definition for
      ‘geom_point’
    sconeReport : server: no visible global function definition for
      ‘guides’
    Undefined global functions or variables:
      %>% aes coord_cartesian element_blank geom_bar geom_point geom_violin
      ggplot ggplotly guides labs plot_ly plotlyOutput renderVisNetwork
      scale_fill_manual theme visEdges visGroups visHierarchicalLayout
      visLegend visNetwork visNetworkOutput visNetworkProxy visOptions
      visSelectNodes ylim
    ```

# scoper

Version: 0.1.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘shazam’
      All declared Imports should be used.
    ```

# scPipe

Version: 1.2.1

## In both

*   checking whether package ‘scPipe’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘DelayedArray’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/scPipe/new/scPipe.Rcheck/00install.out’ for details.
    ```

*   checking whether the namespace can be loaded with stated dependencies ... WARNING
    ```
    Error in dyn.load(file, DLLpath = DLLpath, ...) : 
      unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/scPipe/Rhtslib/libs/Rhtslib.so':
      dlopen(/Users/max/github/ggplot2/revdep/library.noindex/scPipe/Rhtslib/libs/Rhtslib.so, 6): Symbol not found: _lzma_easy_buffer_encode
      Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/scPipe/Rhtslib/libs/Rhtslib.so
      Expected in: flat namespace
     in /Users/max/github/ggplot2/revdep/library.noindex/scPipe/Rhtslib/libs/Rhtslib.so
    Calls: <Anonymous> ... namespaceImport -> loadNamespace -> library.dynam -> dyn.load
    Execution halted
    
    A namespace must be able to be loaded with just the base namespace
    loaded: otherwise if the namespace gets loaded by a saved object, the
    session will be unable to start.
    
    Probably some imports need to be declared in the NAMESPACE file.
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 66-73 (scPipe_tutorial.Rmd) 
    Error: processing vignette 'scPipe_tutorial.Rmd' failed with diagnostics:
    there is no package called 'Rsubread'
    Execution halted
    ```

*   checking R code for possible problems ... NOTE
    ```
    plot_demultiplex: no visible binding for global variable ‘status’
    plot_demultiplex: no visible binding for global variable ‘count’
    plot_demultiplex: no visible binding for global variable ‘label_y’
    plot_demultiplex: no visible binding for global variable ‘label_tx’
    Undefined global functions or variables:
      count label_tx label_y status
    ```

*   checking for unstated dependencies in vignettes ... NOTE
    ```
    '::' or ':::' import not declared from: ‘Rsubread’
    ```

# scran

Version: 1.8.4

## In both

*   checking whether package ‘scran’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘DelayedArray’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/scran/new/scran.Rcheck/00install.out’ for details.
    ```

*   checking examples ... WARNING
    ```
    Found the following significant warnings:
    
      Warning: 'exploreData' is deprecated.
      Warning: 'selectorPlot' is deprecated.
    Deprecated functions may be defunct as soon as of the next release of
    R.
    See ?Deprecated.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  6.1Mb
      sub-directories of 1Mb or more:
        libs   5.0Mb
    ```

# scsR

Version: 1.16.0

## In both

*   checking whether package ‘scsR’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘IRanges’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/scsR/new/scsR.Rcheck/00install.out’ for details.
    ```

*   checking package dependencies ... NOTE
    ```
    Depends: includes the non-default packages:
      ‘STRINGdb’ ‘BiocGenerics’ ‘Biostrings’ ‘IRanges’ ‘plyr’ ‘tcltk’
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    seed_correction_pooled: no visible global function definition for
      ‘txtProgressBar’
    seed_correction_pooled: no visible global function definition for
      ‘setTxtProgressBar’
    seed_removal: no visible global function definition for
      ‘txtProgressBar’
    seed_removal: no visible global function definition for
      ‘setTxtProgressBar’
    seeds_analysis : <anonymous>: no visible global function definition for
      ‘ks.test’
    seeds_analysis: no visible global function definition for ‘phyper’
    transcribe_seqs: no visible global function definition for
      ‘txtProgressBar’
    transcribe_seqs: no visible global function definition for
      ‘setTxtProgressBar’
    Undefined global functions or variables:
      cor.test heatmap.2 ks.test lm phyper setTxtProgressBar txtProgressBar
    Consider adding
      importFrom("stats", "cor.test", "ks.test", "lm", "phyper")
      importFrom("utils", "setTxtProgressBar", "txtProgressBar")
    to your NAMESPACE file.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    
    The following objects are masked from ‘package:STRINGdb’:
    
        delColDf, renameColDf
    
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'scsR.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.30 \ifthenelse
                    {\boolean{Sweave@inconsolata}}{%^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# SDaA

Version: 0.1-3

## In both

*   checking R code for possible problems ... NOTE
    ```
    lahiri.design: no visible global function definition for ‘runif’
    Undefined global functions or variables:
      runif
    Consider adding
      importFrom("stats", "runif")
    to your NAMESPACE file.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    
    The following object is masked from ‘package:SDaA’:
    
        seals
    
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'SDaA_using_survey.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `memoir.cls' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: cls)
    
    ! Emergency stop.
    <read *> 
             
    l.2 \usepackage
                   {Sweave}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# sdcMicro

Version: 5.3.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'sdc_guidelines.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `scrartcl.cls' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: cls)
    
    ! Emergency stop.
    <read *> 
             
    l.1 \documentclass[12pt]{scrartcl}\usepackage
                                                 []{graphicx}\usepackage[]{color...
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# SDEFSR

Version: 0.7.1.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    ! Undefined control sequence.
    l.1326 \textbackslash{}providecommand\{\doi
                                               \}\{doi: 
    Here is how much of TeX's memory you used:
     10863 strings out of 494585
     155031 string characters out of 6177194
     224799 words of memory out of 5000000
     14224 multiletter control sequences out of 15000+600000
     56791 words of font info for 48 fonts, out of 8000000 for 9000
     14 hyphenation exceptions out of 8191
     28i,7n,33p,672b,467s stack positions out of 5000i,500n,10000p,200000b,80000s
    
    Error: processing vignette 'SDEFSRpackage.Rmd' failed with diagnostics:
    Failed to compile SDEFSRpackage.tex. See SDEFSRpackage.log for more info.
    Execution halted
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘grDevices’
      All declared Imports should be used.
    ```

# sdmvspecies

Version: 0.3.2

## In both

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    [3] "/Users/max/github/ggplot2/revdep/checks.noindex/sdmvspecies/new/sdmvspecies.Rcheck/sdmvspecies/external/env/bio12.bil"
    [4] "/Users/max/github/ggplot2/revdep/checks.noindex/sdmvspecies/new/sdmvspecies.Rcheck/sdmvspecies/external/env/bio14.bil"
    [5] "/Users/max/github/ggplot2/revdep/checks.noindex/sdmvspecies/new/sdmvspecies.Rcheck/sdmvspecies/external/env/bio16.bil"
    [6] "/Users/max/github/ggplot2/revdep/checks.noindex/sdmvspecies/new/sdmvspecies.Rcheck/sdmvspecies/external/env/bio5.bil" 
    [7] "/Users/max/github/ggplot2/revdep/checks.noindex/sdmvspecies/new/sdmvspecies.Rcheck/sdmvspecies/external/env/bio7.bil" 
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'sdmvspecies.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `hanging.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.9 \usepackage
                   [utf8]{inputenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# sdStaf

Version: 1.0.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘rgdal’ ‘rgeos’ ‘tidyr’
      All declared Imports should be used.
    ```

# seaaroundus

Version: 1.2.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘rgdal’
      All declared Imports should be used.
    ```

# seewave

Version: 2.1.0

## In both

*   checking whether package ‘seewave’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/seewave/new/seewave.Rcheck/00install.out’ for details.
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘rpanel’
    ```

## Installation

### Devel

```
* installing *source* package ‘seewave’ ...
** package ‘seewave’ successfully unpacked and MD5 sums checked
** libs
/usr/local/clang4/bin/clang -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG   -I/usr/local/include   -fPIC  -Wall -g -O2  -c seewave_init.c -o seewave_init.o
/usr/local/clang4/bin/clang -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG   -I/usr/local/include   -fPIC  -Wall -g -O2  -c stft.c -o stft.o
stft.c:3:11: fatal error: 'sndfile.h' file not found
# include <sndfile.h> //Library for write or read sound file
          ^~~~~~~~~~~
1 error generated.
make: *** [stft.o] Error 1
make: *** Waiting for unfinished jobs....
ERROR: compilation failed for package ‘seewave’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/seewave/new/seewave.Rcheck/seewave’

```
### CRAN

```
* installing *source* package ‘seewave’ ...
** package ‘seewave’ successfully unpacked and MD5 sums checked
** libs
/usr/local/clang4/bin/clang -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG   -I/usr/local/include   -fPIC  -Wall -g -O2  -c seewave_init.c -o seewave_init.o
/usr/local/clang4/bin/clang -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG   -I/usr/local/include   -fPIC  -Wall -g -O2  -c stft.c -o stft.o
stft.c:3:11: fatal error: 'sndfile.h' file not found
# include <sndfile.h> //Library for write or read sound file
          ^~~~~~~~~~~
1 error generated.
make: *** [stft.o] Error 1
make: *** Waiting for unfinished jobs....
ERROR: compilation failed for package ‘seewave’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/seewave/old/seewave.Rcheck/seewave’

```
# selfea

Version: 1.0.1

## In both

*   checking R code for possible problems ... NOTE
    ```
    get_statistics_from_file: no visible global function definition for
      ‘read.csv’
    glm_anova: no visible global function definition for ‘aov’
    glm_anova: no visible global function definition for ‘glm’
    glm_anova: no visible global function definition for ‘gaussian’
    glm_anova: no visible global function definition for ‘anova’
    glm_anova: no visible global function definition for ‘quasipoisson’
    glm_anova: no visible global function definition for ‘sd’
    glm_anova: no visible global function definition for ‘p.adjust’
    ttest_cohens_d: no visible global function definition for ‘t.test’
    ttest_cohens_d: no visible global function definition for ‘sd’
    Undefined global functions or variables:
      anova aov gaussian glm p.adjust quasipoisson read.csv sd t.test
    Consider adding
      importFrom("stats", "anova", "aov", "gaussian", "glm", "p.adjust",
                 "quasipoisson", "sd", "t.test")
      importFrom("utils", "read.csv")
    to your NAMESPACE file.
    ```

# SentimentAnalysis

Version: 1.3-2

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘SnowballC’ ‘XML’ ‘mgcv’
      All declared Imports should be used.
    ```

# sentometrics

Version: 0.5.1

## In both

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 4436 marked UTF-8 strings
    ```

*   checking for GNU extensions in Makefiles ... NOTE
    ```
    GNU make is a SystemRequirements.
    ```

# SEPA

Version: 1.10.0

## In both

*   checking R code for possible problems ... NOTE
    ```
    ...
      definition for ‘t.test’
    truetimepattern : <anonymous>: no visible global function definition
      for ‘p.adjust’
    truetimevisualize : <anonymous> : <anonymous>: no visible global
      function definition for ‘median’
    truetimevisualize: no visible binding for global variable ‘time’
    truetimevisualize: no visible binding for global variable ‘expmean’
    truetimevisualize: no visible binding for global variable ‘Gene’
    windowGOanalysis: no visible global function definition for ‘new’
    windowGOvisualize: no visible binding for global variable ‘Var1’
    windowGOvisualize: no visible binding for global variable ‘value’
    Undefined global functions or variables:
      Gene Var1 Var2 confint expmean fitted lines lm median new p.adjust
      pseudotime t.test time value xend yend
    Consider adding
      importFrom("graphics", "lines")
      importFrom("methods", "new")
      importFrom("stats", "confint", "fitted", "lm", "median", "p.adjust",
                 "t.test", "time")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

# seqbias

Version: 1.28.0

## In both

*   checking whether package ‘seqbias’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/seqbias/new/seqbias.Rcheck/00install.out’ for details.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Versioned 'LinkingTo' value for ‘Rsamtools’ is only usable in R >= 3.0.2
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' call to ‘GenomicRanges’ which was already attached by Depends.
      Please remove these calls from your code.
    Packages in Depends field not imported from:
      ‘Biostrings’ ‘GenomicRanges’ ‘methods’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      function definition for ‘runif’
    random.intervals: no visible global function definition for ‘GRanges’
    random.intervals: no visible global function definition for ‘IRanges’
    seqbias.fit: no visible global function definition for ‘new’
    seqbias.load: no visible global function definition for ‘new’
    seqbias.predict: no visible global function definition for ‘is’
    seqbias.predict : <anonymous>: no visible global function definition
      for ‘seqnames’
    seqbias.predict : <anonymous>: no visible global function definition
      for ‘start’
    seqbias.predict : <anonymous>: no visible global function definition
      for ‘end’
    seqbias.predict : <anonymous>: no visible global function definition
      for ‘strand’
    Undefined global functions or variables:
      GRanges IRanges end is new runif seqnames start strand
    Consider adding
      importFrom("methods", "is", "new")
      importFrom("stats", "end", "runif", "start")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

*   checking compiled code ... NOTE
    ```
    File ‘seqbias/libs/seqbias.so’:
      Found ‘_rand’, possibly from ‘rand’ (C)
        Objects: ‘sequencing_bias.o’, ‘twobitseq.o’
    
    Compiled code should not call entry points which might terminate R nor
    write to stdout/stderr instead of to the console, nor use Fortran I/O
    nor system RNGs.
    
    See ‘Writing portable packages’ in the ‘Writing R Extensions’ manual.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    
        strsplit
    
    Warning: package ‘Rsamtools’ was built under R version 3.5.1
    Loading required package: ggplot2
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'overview.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.30 \ifthenelse
                    {\boolean{Sweave@inconsolata}}{%^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# seqCAT

Version: 1.2.1

## In both

*   checking whether package ‘seqCAT’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘DelayedArray’ was built under R version 3.5.1
      Warning: package ‘Rsamtools’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/seqCAT/new/seqCAT.Rcheck/00install.out’ for details.
    ```

# SeqFeatR

Version: 0.2.7

## In both

*   checking whether package ‘SeqFeatR’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/SeqFeatR/new/SeqFeatR.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘SeqFeatR’ ...
** package ‘SeqFeatR’ successfully unpacked and MD5 sums checked
** R
** inst
** byte-compile and prepare package for lazy loading
Warning in fun(libname, pkgname) : couldn't connect to display ""
Warning: package ‘IRanges’ was built under R version 3.5.1
Error : .onLoad failed in loadNamespace() for 'rjags', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/SeqFeatR/rjags/libs/rjags.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/SeqFeatR/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/SeqFeatR/rjags/libs/rjags.so
  Reason: image not found
ERROR: lazy loading failed for package ‘SeqFeatR’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/SeqFeatR/new/SeqFeatR.Rcheck/SeqFeatR’

```
### CRAN

```
* installing *source* package ‘SeqFeatR’ ...
** package ‘SeqFeatR’ successfully unpacked and MD5 sums checked
** R
** inst
** byte-compile and prepare package for lazy loading
Warning in fun(libname, pkgname) : couldn't connect to display ""
Warning: package ‘IRanges’ was built under R version 3.5.1
Error : .onLoad failed in loadNamespace() for 'rjags', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/SeqFeatR/rjags/libs/rjags.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/SeqFeatR/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/SeqFeatR/rjags/libs/rjags.so
  Reason: image not found
ERROR: lazy loading failed for package ‘SeqFeatR’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/SeqFeatR/old/SeqFeatR.Rcheck/SeqFeatR’

```
# seqplots

Version: 1.18.0

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 14.9Mb
      sub-directories of 1Mb or more:
        doc        3.7Mb
        seqplots  10.1Mb
    ```

*   checking foreign function calls ... NOTE
    ```
    Foreign function call to a different package:
      .Call("BWGFile_summary", ..., PACKAGE = "rtracklayer")
    See chapter ‘System and foreign language interfaces’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    plotHeatmap,list: no visible global function definition for ‘kmeans’
    plotHeatmap,list: no visible global function definition for ‘hclust’
    plotHeatmap,list: no visible global function definition for ‘dist’
    plotHeatmap,list: no visible global function definition for ‘cutree’
    plotHeatmap,list: no visible global function definition for
      ‘as.dendrogram’
    plotHeatmap,list: no visible global function definition for ‘title’
    Undefined global functions or variables:
      Var1 Var2 abline adjustcolor approx as.dendrogram axis box
      capture.output colorRampPalette cutree dist hclust image kmeans
      layout lines mtext par plot.new qt rainbow rect rgb text title value
    Consider adding
      importFrom("grDevices", "adjustcolor", "colorRampPalette", "rainbow",
                 "rgb")
      importFrom("graphics", "abline", "axis", "box", "image", "layout",
                 "lines", "mtext", "par", "plot.new", "rect", "text",
                 "title")
      importFrom("stats", "approx", "as.dendrogram", "cutree", "dist",
                 "hclust", "kmeans", "qt")
      importFrom("utils", "capture.output")
    to your NAMESPACE file.
    ```

# SeqSQC

Version: 1.2.0

## In both

*   checking whether package ‘SeqSQC’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘ExperimentHub’ was built under R version 3.5.1
      Warning: package ‘AnnotationHub’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/SeqSQC/new/SeqSQC.Rcheck/00install.out’ for details.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.5Mb
      sub-directories of 1Mb or more:
        doc       1.9Mb
        extdata   3.3Mb
    ```

# Seurat

Version: 2.3.4

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘loomR’
    ```

# sf

Version: 0.7-0

## In both

*   checking whether package ‘sf’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/sf/new/sf.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘sf’ ...
** package ‘sf’ successfully unpacked and MD5 sums checked
configure: CC: /usr/local/clang4/bin/clang
configure: CXX: /usr/local/clang4/bin/clang++ -std=gnu++11
checking for gdal-config... no
no
configure: error: gdal-config not found or not executable.
ERROR: configuration failed for package ‘sf’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/sf/new/sf.Rcheck/sf’

```
### CRAN

```
* installing *source* package ‘sf’ ...
** package ‘sf’ successfully unpacked and MD5 sums checked
configure: CC: /usr/local/clang4/bin/clang
configure: CXX: /usr/local/clang4/bin/clang++ -std=gnu++11
checking for gdal-config... no
no
configure: error: gdal-config not found or not executable.
ERROR: configuration failed for package ‘sf’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/sf/old/sf.Rcheck/sf’

```
# sglr

Version: 0.7

## In both

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Description field: should contain one or more complete sentences.
    ```

*   checking R code for possible problems ... NOTE
    ```
    .computePStar: no visible global function definition for ‘uniroot’
    plotBoundary: no visible global function definition for
      ‘scale_y_continuous’
    Undefined global functions or variables:
      scale_y_continuous uniroot
    Consider adding
      importFrom("stats", "uniroot")
    to your NAMESPACE file.
    ```

# sgmcmc

Version: 0.2.3

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 6 SKIPPED: 0 FAILED: 32
      1. Error: Check SGLD chain reasonable for 1d Gaussian (@test1dGauss.R#90) 
      2. Error: Check SGLDCV chain reasonable for 1d Gaussian (@test1dGauss.R#104) 
      3. Error: Check SGHMC chain reasonable for 1d Gaussian (@test1dGauss.R#117) 
      4. Error: Check SGHMCCV chain reasonable for 1d Gaussian (@test1dGauss.R#130) 
      5. Error: Check SGNHT chain reasonable for 1d Gaussian (@test1dGauss.R#143) 
      6. Error: Check SGNHTCV chain reasonable for 1d Gaussian (@test1dGauss.R#156) 
      7. Error: Check SGLD chain reasonable for 3d Gaussian (@test3dGauss.R#93) 
      8. Error: Check SGLDCV chain reasonable for 3d Gaussian (@test3dGauss.R#109) 
      9. Error: Check SGHMC chain reasonable for 3d Gaussian (@test3dGauss.R#124) 
      1. ...
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking S3 generic/method consistency ... WARNING
    ```
    /Users/max/.virtualenvs/r-tensorflow/lib/python2.7/site-packages/h5py/__init__.py:36: FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`.
      from ._conv import register_converters as _register_converters
    See section ‘Generic functions and methods’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking replacement functions ... WARNING
    ```
    /Users/max/.virtualenvs/r-tensorflow/lib/python2.7/site-packages/h5py/__init__.py:36: FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`.
      from ._conv import register_converters as _register_converters
    The argument of a replacement function which corresponds to the right
    hand side must be named ‘value’.
    ```

*   checking for missing documentation entries ... WARNING
    ```
    /Users/max/.virtualenvs/r-tensorflow/lib/python2.7/site-packages/h5py/__init__.py:36: FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`.
      from ._conv import register_converters as _register_converters
    All user-level objects in a package should have documentation entries.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking for code/documentation mismatches ... WARNING
    ```
    /Users/max/.virtualenvs/r-tensorflow/lib/python2.7/site-packages/h5py/__init__.py:36: FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`.
      from ._conv import register_converters as _register_converters
    /Users/max/.virtualenvs/r-tensorflow/lib/python2.7/site-packages/h5py/__init__.py:36: FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`.
      from ._conv import register_converters as _register_converters
    /Users/max/.virtualenvs/r-tensorflow/lib/python2.7/site-packages/h5py/__init__.py:36: FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`.
      from ._conv import register_converters as _register_converters
    ```

*   checking dependencies in R code ... NOTE
    ```
    /Users/max/.virtualenvs/r-tensorflow/lib/python2.7/site-packages/h5py/__init__.py:36: FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`.
      from ._conv import register_converters as _register_converters
    ```

*   checking foreign function calls ... NOTE
    ```
    /Users/max/.virtualenvs/r-tensorflow/lib/python2.7/site-packages/h5py/__init__.py:36: FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`.
      from ._conv import register_converters as _register_converters
    See chapter ‘System and foreign language interfaces’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    /Users/max/.virtualenvs/r-tensorflow/lib/python2.7/site-packages/h5py/__init__.py:36: FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`.
      from ._conv import register_converters as _register_converters
    ```

*   checking Rd \usage sections ... NOTE
    ```
    /Users/max/.virtualenvs/r-tensorflow/lib/python2.7/site-packages/h5py/__init__.py:36: FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`.
      from ._conv import register_converters as _register_converters
    The \usage entries for S3 methods should use the \method markup and not
    their full name.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

# sgmodel

Version: 0.1.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘tidyverse’
      All declared Imports should be used.
    ```

# shiny

Version: 1.1.0

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  9.4Mb
      sub-directories of 1Mb or more:
        R     2.0Mb
        www   6.4Mb
    ```

# ShinyItemAnalysis

Version: 1.2.8

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘DT’ ‘data.table’ ‘gridExtra’ ‘knitr’ ‘latticeExtra’ ‘msm’ ‘plotly’
      ‘shinyBS’ ‘shinydashboard’ ‘xtable’
      All declared Imports should be used.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘WrightMap’
    ```

# shinyKGode

Version: 1.0.5

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘KGode’ ‘XML’ ‘ggplot2’ ‘gridExtra’ ‘mvtnorm’ ‘reshape2’ ‘shinyjs’
      All declared Imports should be used.
    ```

# SIBER

Version: 2.1.3

## In both

*   checking examples ... ERROR
    ```
    ...
    > ### ** Examples
    > 
    > x <- stats::rnorm(50)
    > y <- stats::rnorm(50)
    > parms <- list()
    > parms$n.iter <- 2 * 10^3
    > parms$n.burnin <- 500
    > parms$n.thin <- 2     
    > parms$n.chains <- 2    
    > priors <- list()
    > priors$R <- 1 * diag(2)
    > priors$k <- 2
    > priors$tau.mu <- 1.0E-3
    > fitEllipse(x, y, parms, priors)
    Error: .onLoad failed in loadNamespace() for 'rjags', details:
      call: dyn.load(file, DLLpath = DLLpath, ...)
      error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/SIBER/rjags/libs/rjags.so':
      dlopen(/Users/max/github/ggplot2/revdep/library.noindex/SIBER/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
      Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/SIBER/rjags/libs/rjags.so
      Reason: image not found
    Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 74-93 (Centroid-Vectors.Rmd) 
    Error: processing vignette 'Centroid-Vectors.Rmd' failed with diagnostics:
    .onLoad failed in loadNamespace() for 'rjags', details:
      call: dyn.load(file, DLLpath = DLLpath, ...)
      error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/SIBER/rjags/libs/rjags.so':
      dlopen(/Users/max/github/ggplot2/revdep/library.noindex/SIBER/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
      Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/SIBER/rjags/libs/rjags.so
      Reason: image not found
    Execution halted
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘dplyr’ ‘ggplot2’ ‘viridis’
      All declared Imports should be used.
    ```

# sicegar

Version: 0.2.2

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘dplyr’
      All declared Imports should be used.
    ```

# SigFuge

Version: 1.18.0

## In both

*   checking whether package ‘SigFuge’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/SigFuge/new/SigFuge.Rcheck/00install.out’ for details.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    SFfigure: no visible global function definition for ‘hcl’
    SFfigure: no visible binding for global variable ‘cluster’
    SFfigure: no visible binding for global variable ‘median’
    SFfigure: no visible binding for global variable ‘x’
    SFfigure: no visible binding for global variable ‘y1’
    SFfigure: no visible binding for global variable ‘label’
    SFfigure: no visible binding for global variable ‘value’
    SFfigure: no visible binding for global variable ‘Clusters’
    SFfigure: no visible binding for global variable ‘y2’
    SFfigure: no visible global function definition for ‘pdf’
    SFfigure: no visible global function definition for ‘dev.off’
    SFlabels: no visible global function definition for ‘kmeans’
    SFnormalize : <anonymous>: no visible global function definition for
      ‘median’
    Undefined global functions or variables:
      Clusters base cluster dev.off e hcl kmeans label median pdf rgb s
      value x y1 y2
    Consider adding
      importFrom("grDevices", "dev.off", "hcl", "pdf", "rgb")
      importFrom("stats", "kmeans", "median")
    to your NAMESPACE file.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    
        strsplit
    
    Warning: `show_guide` has been deprecated. Please use `show.legend` instead.
    Warning: `show_guide` has been deprecated. Please use `show.legend` instead.
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'SigFuge.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `beramono.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.87 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# SIMAT

Version: 1.12.1

## In both

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    when loading mzR. If you encounter such issues, please send a report,
    including the output of sessionInfo() to the Bioc support forum at 
    https://support.bioconductor.org/. For details see also
    https://github.com/sneumann/mzR/wiki/mzR-Rcpp-compiler-linker-issue.
    Warning: Removed 2 rows containing missing values (geom_bar).
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'SIMAT-vignette.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.30 \ifthenelse
                    {\boolean{Sweave@inconsolata}}{%^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# simcausal

Version: 0.5.4

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    
    The following objects are masked from 'package:base':
    
        format.pval, units
    
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'simcausal_vignette.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.55 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘tmlenet’
    ```

# SimCorrMix

Version: 0.1.1

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘MASS’ ‘grid’
      All declared Imports should be used.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘PoisNor’
    ```

# SimDesign

Version: 1.11

## Newly fixed

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    See ?SimFunctions to get started with SimDesign
    
    Simulation complete. Total execution time: 0.31s
    Warning in socketConnection("localhost", port = port, server = TRUE, blocking = TRUE,  :
      port 11637 cannot be opened
    Quitting from lines 58-62 (Fixed_obj_fun.Rmd) 
    Error: processing vignette 'Fixed_obj_fun.Rmd' failed with diagnostics:
    cannot open the connection
    Execution halted
    ```

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘doMPI’
    ```

# simmr

Version: 0.3

## In both

*   checking whether package ‘simmr’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/simmr/new/simmr.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘simmr’ ...
** package ‘simmr’ successfully unpacked and MD5 sums checked
** R
** inst
** byte-compile and prepare package for lazy loading
Error: package or namespace load failed for ‘rjags’:
 .onLoad failed in loadNamespace() for 'rjags', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/simmr/rjags/libs/rjags.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/simmr/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/simmr/rjags/libs/rjags.so
  Reason: image not found
Error : package ‘rjags’ could not be loaded
ERROR: lazy loading failed for package ‘simmr’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/simmr/new/simmr.Rcheck/simmr’

```
### CRAN

```
* installing *source* package ‘simmr’ ...
** package ‘simmr’ successfully unpacked and MD5 sums checked
** R
** inst
** byte-compile and prepare package for lazy loading
Error: package or namespace load failed for ‘rjags’:
 .onLoad failed in loadNamespace() for 'rjags', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/simmr/rjags/libs/rjags.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/simmr/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/simmr/rjags/libs/rjags.so
  Reason: image not found
Error : package ‘rjags’ could not be loaded
ERROR: lazy loading failed for package ‘simmr’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/simmr/old/simmr.Rcheck/simmr’

```
# SimMultiCorrData

Version: 0.2.2

## In both

*   checking Rd cross-references ... NOTE
    ```
    Packages unavailable to check Rd xrefs: ‘BinNonNor’, ‘PoisNor’, ‘PoisBinOrdNor’, ‘PoisBinOrdNonNor’
    ```

# simPH

Version: 1.3.10

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    Doing a final clean up.
    All Xl set to 0.
    All Xl set to 0.
    `geom_smooth()` using method = 'gam' and formula 'y ~ s(x, bs = "cs")'
    `geom_smooth()` using method = 'gam' and formula 'y ~ s(x, bs = "cs")'
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'simPH-overview.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.55 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# SimRepeat

Version: 0.1.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘BB’ ‘MASS’ ‘ggplot2’ ‘grid’ ‘triangle’
      All declared Imports should be used.
    ```

# simulator

Version: 0.2.0

## In both

*   R CMD check timed out
    

# sincell

Version: 1.12.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    Loading required package: DDRTree
    Loading required package: irlba
    convert: profile 'icc': 'RGB ': RGB color space not permitted on grayscale PNG `figure/EigenValues-1.png' @ warning/png.c/MagickPNGWarningHandler/1744.
    convert: profile 'icc': 'RGB ': RGB color space not permitted on grayscale PNG `figure/plot_cellsLowDimensionalSpace_ICA-1.png' @ warning/png.c/MagickPNGWarningHandler/1744.
    convert: profile 'icc': 'RGB ': RGB color space not permitted on grayscale PNG `figure/Comparisson_graphs-1.png' @ warning/png.c/MagickPNGWarningHandler/1744.
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'sincell-vignette.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `beramono.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.87 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    sc_InitializingSincellObject: no visible binding for global variable
      ‘var’
    sc_StatisticalSupportByGeneSubsampling: no visible global function
      definition for ‘cor’
    sc_StatisticalSupportByReplacementWithInSilicoCellsReplicates: no
      visible global function definition for ‘cor’
    sc_clusterObj: no visible global function definition for ‘hclust’
    sc_clusterObj: no visible global function definition for ‘cutree’
    sc_distanceObj: no visible global function definition for ‘cor’
    sc_marker2color: no visible global function definition for
      ‘colorRampPalette’
    Undefined global functions or variables:
      cmdscale colorRampPalette combn cor cutree hclust plot prcomp
      quantile rnbinom rnorm runif var
    Consider adding
      importFrom("grDevices", "colorRampPalette")
      importFrom("graphics", "plot")
      importFrom("stats", "cmdscale", "cor", "cutree", "hclust", "prcomp",
                 "quantile", "rnbinom", "rnorm", "runif", "var")
      importFrom("utils", "combn")
    to your NAMESPACE file.
    ```

*   checking compiled code ... NOTE
    ```
    File ‘sincell/libs/sincell.so’:
      Found ‘_rand’, possibly from ‘rand’ (C)
        Objects: ‘pseudoreplicatesbymodel.o’, ‘pseudoreplicatesbynoise.o’,
          ‘pseudoreplicatesbynoise_cv2.o’
      Found ‘_srand’, possibly from ‘srand’ (C)
        Objects: ‘pseudoreplicatesbymodel.o’, ‘pseudoreplicatesbynoise.o’,
          ‘pseudoreplicatesbynoise_cv2.o’
    
    Compiled code should not call entry points which might terminate R nor
    write to stdout/stderr instead of to the console, nor use Fortran I/O
    nor system RNGs.
    
    See ‘Writing portable packages’ in the ‘Writing R Extensions’ manual.
    ```

# Single.mTEC.Transcriptomes

Version: 1.8.0

## In both

*   R CMD check timed out
    

*   checking data for ASCII and uncompressed saves ... WARNING
    ```
      Warning: package needs dependence on R (>= 2.10)
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 916.1Mb
      sub-directories of 1Mb or more:
        data  906.6Mb
        doc     9.3Mb
    ```

# singleCellTK

Version: 1.0.3

## In both

*   checking whether package ‘singleCellTK’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘DelayedArray’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/singleCellTK/new/singleCellTK.Rcheck/00install.out’ for details.
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .BBSoptions
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.3Mb
      sub-directories of 1Mb or more:
        data   1.0Mb
        doc    3.7Mb
    ```

# singscore

Version: 1.0.0

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘singscore-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: projectScoreLandscape
    > ### Title: Project data on the landscape plot obtained from
    > ###   'plotScoreLandscape()'
    > ### Aliases: projectScoreLandscape
    > 
    > ### ** Examples
    > 
    > ranked <- rankGenes(toy_expr_se)
    > scoredf1 <- simpleScore(ranked, upSet = toy_gs_up, downSet = toy_gs_dn)
    > scoredf2 <- simpleScore(ranked, upSet = toy_gs_up)
    > psl <- plotScoreLandscape(scoredf1, scoredf2)
    > projectScoreLandscape(psl,scoredf1, scoredf2)
    Warning: Ignoring unknown aesthetics: text
    Error in FUN(X[[i]], ...) : object 'Signature 1' not found
    Calls: <Anonymous> ... ggplot_build.ggplot -> by_layer -> f -> <Anonymous> -> f -> lapply -> FUN
    Execution halted
    ```

*   checking whether package ‘singscore’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘graph’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/singscore/new/singscore.Rcheck/00install.out’ for details.
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    
    Loading required package: XML
    
    Attaching package: 'XML'
    
    The following object is masked from 'package:tools':
    
        toHTML
    
    Loading required package: graph
    
    Attaching package: 'graph'
    
    The following object is masked from 'package:XML':
    
        addNode
    
    Quitting from lines 174-182 (singscore.Rmd) 
    Error: processing vignette 'singscore.Rmd' failed with diagnostics:
    object 'tcga-EPI' not found
    Execution halted
    ```

# SISPA

Version: 1.10.0

## In both

*   checking files in ‘vignettes’ ... WARNING
    ```
    Files in the 'vignettes' directory but no files in 'inst/doc':
      ‘SISPA.Rmd’, ‘SISPA.md’, ‘SISPA_data.Rda’,
        ‘SISPA_files/figure-html/unnamed-chunk-2-1.png’,
        ‘SISPA_files/figure-html/unnamed-chunk-3-1.png’,
        ‘SISPA_files/figure-html/unnamed-chunk-4-1.png’,
        ‘SISPA_files/figure-html/unnamed-chunk-5-1.png’,
        ‘SISPA_files/figure-html/unnamed-chunk-6-1.png’,
        ‘SISPA_files/figure-html/unnamed-chunk-7-1.png’,
        ‘sispa_overview.png’
    Files named as vignettes but with no recognized vignette engine:
       ‘vignettes/SISPA.Rmd’
    (Is a VignetteBuilder field missing?)
    ```

*   checking R code for possible problems ... NOTE
    ```
    cptSamples : cptsPlot: no visible global function definition for
      ‘abline’
    cptSamples : cptsPlot: no visible global function definition for ‘text’
    Undefined global functions or variables:
      abline text
    Consider adding
      importFrom("graphics", "abline", "text")
    to your NAMESPACE file.
    ```

# SixSigma

Version: 0.9-52

## In both

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'HelicopterInstructions.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.30 \ifthenelse
                    {\boolean{Sweave@inconsolata}}{%^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# skynet

Version: 1.2.2

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘maps’
      All declared Imports should be used.
    ```

# slalom

Version: 1.2.0

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 23.3Mb
      sub-directories of 1Mb or more:
        data      3.5Mb
        extdata  18.3Mb
    ```

# SmartEDA

Version: 0.3.0

## In both

*   checking Rd cross-references ... NOTE
    ```
    Packages unavailable to check Rd xrefs: ‘InformationValue’, ‘DataExplorer’
    ```

# SmarterPoland

Version: 1.7

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  8.2Mb
      sub-directories of 1Mb or more:
        data   8.1Mb
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 1122 marked UTF-8 strings
    ```

# SMFI5

Version: 1.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' calls to packages already attached by Depends:
      ‘ggplot2’ ‘reshape’
      Please remove these calls from your code.
    Packages in Depends field not imported from:
      ‘corpcor’ ‘ggplot2’ ‘reshape’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    sim.cir: no visible global function definition for ‘aes’
    sim.cir: no visible global function definition for ‘geom_line’
    sim.cir: no visible global function definition for ‘ggtitle’
    sim.n.chi2: no visible global function definition for ‘rchisq’
    sim.n.chi2: no visible global function definition for ‘rnorm’
    sim.n.chi2: no visible global function definition for ‘rpois’
    sim.vasicek: no visible global function definition for ‘rnorm’
    sim.vasicek: no visible global function definition for ‘melt’
    sim.vasicek: no visible global function definition for ‘ggplot’
    sim.vasicek: no visible global function definition for ‘aes’
    sim.vasicek: no visible global function definition for ‘geom_line’
    sim.vasicek: no visible global function definition for ‘ggtitle’
    Undefined global functions or variables:
      acf aes cor cov dchisq geom_line ggplot ggtitle head melt optim plot
      pseudoinverse qnorm rchisq rnorm rpois sd title
    Consider adding
      importFrom("graphics", "plot", "title")
      importFrom("stats", "acf", "cor", "cov", "dchisq", "optim", "qnorm",
                 "rchisq", "rnorm", "rpois", "sd")
      importFrom("utils", "head")
    to your NAMESPACE file.
    ```

# SMITE

Version: 1.8.0

## In both

*   checking whether package ‘SMITE’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/SMITE/new/SMITE.Rcheck/00install.out’ for details.
    ```

# snht

Version: 1.0.5

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'pairwiseSNHT.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.55 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# SNPhood

Version: 1.10.0

## In both

*   R CMD check timed out
    

*   checking whether package ‘SNPhood’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘Rsamtools’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/SNPhood/new/SNPhood.Rcheck/00install.out’ for details.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 12.0Mb
      sub-directories of 1Mb or more:
        data   4.7Mb
        doc    6.6Mb
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    BugReports field is not a suitable URL but appears to contain an email address
      not specified by mailto: nor contained in < >
    ```

*   checking R code for possible problems ... NOTE
    ```
    .calcBinomTestVector: no visible binding for global variable ‘pp’
    Undefined global functions or variables:
      pp
    ```

# soc.ca

Version: 0.7.3

## In both

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 833 marked UTF-8 strings
    ```

# socialmixr

Version: 0.1.3

## In both

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 1 marked UTF-8 string
    ```

# soGGi

Version: 1.12.0

## In both

*   checking whether package ‘soGGi’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘DelayedArray’ was built under R version 3.5.1
      Warning: package ‘Rsamtools’ was built under R version 3.5.1
      Warning: replacing previous import ‘ggplot2::Position’ by ‘BiocGenerics::Position’ when loading ‘soGGi’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/soGGi/new/soGGi.Rcheck/00install.out’ for details.
    ```

*   checking for code/documentation mismatches ... WARNING
    ```
    Codoc mismatches from documentation object 'c,ChIPprofile-method':
    \S4method{c}{ChIPprofile}
      Code: function(x, ...)
      Docs: function(x, ..., recursive = FALSE)
      Argument names in docs not in code:
        recursive
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    Scale for 'x' is already present. Adding another scale for 'x', which will
    replace the existing scale.
    No id variables; using all as measure variables
    Scale for 'x' is already present. Adding another scale for 'x', which will
    replace the existing scale.
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'soggi.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `beramono.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.87 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking foreign function calls ... NOTE
    ```
    Foreign function call to a different package:
      .Call("rle_sum_any", ..., PACKAGE = "chipseq")
    See chapter ‘System and foreign language interfaces’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      definition for ‘weighted.mean’
    getShifts: no visible global function definition for
      ‘readGAlignmentsFromBam’
    getSummitScore: no visible global function definition for
      ‘readGAlignmentsFromBam’
    plotRegion.ChIPprofile: no visible global function definition for
      ‘formula’
    runFindSummit: no visible global function definition for
      ‘readGAlignmentsFromBam’
    runRegionPlot : <anonymous>: no visible global function definition for
      ‘spline’
    summitPipeline: no visible global function definition for
      ‘readGAlignmentsFromBam’
    plotRegion,ChIPprofile: no visible global function definition for
      ‘formula’
    Undefined global functions or variables:
      formula read.delim readGAlignmentsFromBam spline weighted.mean
    Consider adding
      importFrom("stats", "formula", "spline", "weighted.mean")
      importFrom("utils", "read.delim")
    to your NAMESPACE file.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Failed with error:  'package 'DelayedArray' could not be loaded'
      Error in .requirePackage(package) : 
        unable to find required package 'soGGi'
      Calls: <Anonymous> ... getClass -> getClassDef -> .classEnv -> .requirePackage
      Execution halted
    ```

# soilcarbon

Version: 1.2.0

## In both

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 145 marked UTF-8 strings
    ```

# SomaticCancerAlterations

Version: 1.16.0

## In both

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Description field: should contain one or more complete sentences.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    .load_dataset: no visible global function definition for ‘data’
    .maf2gr: no visible binding for global variable ‘Chromosome’
    .maf2gr: no visible binding for global variable ‘Start_position’
    .maf2gr: no visible binding for global variable ‘End_position’
    .read_maf: no visible global function definition for ‘read.delim’
    hg2ncbi: no visible global function definition for ‘seqnameStyle<-’
    hg2ncbi: no visible global function definition for ‘genome<-’
    mutationDensity: no visible global function definition for
      ‘keepSeqlevels’
    mutationDensity: no visible global function definition for ‘as’
    ncbi2hg: no visible global function definition for ‘seqnameStyle<-’
    ncbi2hg: no visible global function definition for ‘genome<-’
    scaListDatasets: no visible global function definition for ‘data’
    Undefined global functions or variables:
      Chromosome End_position Start_position as data genome<- keepSeqlevels
      read.delim seqnameStyle<-
    Consider adding
      importFrom("methods", "as")
      importFrom("utils", "data", "read.delim")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

# SomaticSignatures

Version: 2.16.0

## In both

*   checking whether package ‘SomaticSignatures’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘DelayedArray’ was built under R version 3.5.1
      Warning: package ‘Rsamtools’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/SomaticSignatures/new/SomaticSignatures.Rcheck/00install.out’ for details.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Package listed in more than one of Depends, Imports, Suggests, Enhances:
      ‘NMF’
    A package should be listed in only one of these fields.
    ```

# sorvi

Version: 0.7.26

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    convert bootstrapped spaghettis to long format
    Computing density estimates for each vertical cut ...
    vertical cross-sectional density estimate
    Tile approach
    Build ggplot figure ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'sorvi.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `float.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.15 ^^M
            
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    regression_plot: no visible global function definition for
      ‘colorRampPalette’
    regression_plot: no visible global function definition for
      ‘loess.control’
    regression_plot: no visible global function definition for ‘predict’
    regression_plot : <anonymous>: no visible global function definition
      for ‘quantile’
    regression_plot : <anonymous>: no visible global function definition
      for ‘pnorm’
    regression_plot: no visible global function definition for
      ‘flush.console’
    regression_plot: no visible global function definition for ‘density’
    Undefined global functions or variables:
      colorRampPalette density flush.console loess loess.control pnorm
      predict quantile read.csv
    Consider adding
      importFrom("grDevices", "colorRampPalette")
      importFrom("stats", "density", "loess", "loess.control", "pnorm",
                 "predict", "quantile")
      importFrom("utils", "flush.console", "read.csv")
    to your NAMESPACE file.
    ```

# SourceSet

Version: 0.1.1

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘r2cytoscape’
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘simPATHy’
    ```

# sparklyr

Version: 0.9.2

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.1Mb
      sub-directories of 1Mb or more:
        R      2.1Mb
        java   1.9Mb
    ```

# sparseMVN

Version: 0.2.1.1

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'sparseMVN.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.55 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# SparseSignatures

Version: 1.0.3

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Warning in has_utility("pdfcrop") :
      pdfcrop not installed or not in PATH
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'SparseSignatures.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `beramono.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.87 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking R code for possible problems ... NOTE
    ```
    import.counts.data: no visible global function definition for ‘.’
    patient.plot: no visible binding for global variable ‘Context’
    patient.plot: no visible binding for global variable ‘alt’
    patient.plot: no visible binding for global variable ‘N’
    signatures.plot: no visible binding for global variable ‘Context’
    signatures.plot: no visible binding for global variable ‘alt’
    signatures.plot: no visible binding for global variable ‘value’
    Undefined global functions or variables:
      . Context N alt value
    ```

# SpatialBall

Version: 0.1.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘lubridate’
      All declared Imports should be used.
    ```

# SpatialEpiApp

Version: 0.3

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘INLA’
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘RColorBrewer’ ‘SpatialEpi’ ‘dplyr’ ‘dygraphs’ ‘ggplot2’
      ‘htmlwidgets’ ‘knitr’ ‘leaflet’ ‘mapproj’ ‘maptools’ ‘rgdal’ ‘rgeos’
      ‘rmarkdown’ ‘shinyjs’ ‘spdep’ ‘xts’
      All declared Imports should be used.
    ```

# spbabel

Version: 0.4.8

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘spbabel-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: as_tibble.sfg
    > ### Title: Individual geometries as tibbles.
    > ### Aliases: as_tibble.sfg
    > 
    > ### ** Examples
    > 
    > 
    > tibble::as_tibble(sf::st_point(c(1, 1, 1)))
    Error in loadNamespace(name) : there is no package called ‘sf’
    Calls: <Anonymous> ... tryCatch -> tryCatchList -> tryCatchOne -> <Anonymous>
    Execution halted
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      1: library(sf) at testthat/test-sf.R:3
      2: stop(txt, domain = NA)
      
      ── 2. Error: (unknown) (@test-sfFromTable.R#3)  ────────────────────────────────
      there is no package called 'sf'
      1: library(sf) at testthat/test-sfFromTable.R:3
      2: stop(txt, domain = NA)
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 60 SKIPPED: 3 FAILED: 2
      1. Error: (unknown) (@test-sf.R#3) 
      2. Error: (unknown) (@test-sfFromTable.R#3) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 50-55 (sfbabel.Rmd) 
    Error: processing vignette 'sfbabel.Rmd' failed with diagnostics:
    there is no package called 'sf'
    Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘sf’
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘sf’
    ```

# speaq

Version: 2.3.3

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.2Mb
      sub-directories of 1Mb or more:
        data   1.5Mb
        doc    3.4Mb
    ```

# specmine

Version: 2.0.3

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘rcytoscapejs’
    ```

# spew

Version: 1.3.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Loading required package: sp
    Warning in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y,  :
      no font could be found for family "10"
    Warning in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y,  :
      no font could be found for family "10"
    Quitting from lines 45-64 (spew-quickstart.Rmd) 
    Error: processing vignette 'spew-quickstart.Rmd' failed with diagnostics:
    polygon edge not found
    Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘Rmpi’
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘maptools’
      All declared Imports should be used.
    ```

# spFSR

Version: 1.0.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘class’ ‘mlbench’
      All declared Imports should be used.
    ```

# SpidermiR

Version: 1.10.0

## In both

*   checking whether package ‘SpidermiR’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Note: possible error in 'simpleNetwork(NetworkData, ': unused argument (textColour = "black") 
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/SpidermiR/new/SpidermiR.Rcheck/00install.out’ for details.
    Information on the location(s) of code generating the ‘Note’s can be
    obtained by re-running with environment variable R_KEEP_PKG_SOURCE set
    to ‘yes’.
    ```

*   checking R code for possible problems ... NOTE
    ```
    .SpidermiRvisualize_gene: possible error in simpleNetwork(NetworkData,
      linkColour = "gray", textColour = "black", zoom = TRUE): unused
      argument (textColour = "black")
    Case_Study1_loading_3_network: no visible global function definition
      for ‘SpidermiRanalyze_DEnetworkTCGA’
    SpidermiRvisualize_plot_target: no visible binding for global variable
      ‘miRNAs’
    SpidermiRvisualize_plot_target: no visible binding for global variable
      ‘mRNA_target’
    Undefined global functions or variables:
      SpidermiRanalyze_DEnetworkTCGA mRNA_target miRNAs
    ```

# spikeSlabGAM

Version: 1.1-14

## In both

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
      Some 'x' values are beyond 'boundary.knots'; Linear extrapolation used.
    Warning in (function (xnew)  :
      predictions outside fitted range for lin(glucose).
    Warning in (function (xnew)  :
      predictions outside fitted range for sm(glucose).
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'UsingSpikeSlabGAM.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.55 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# splatter

Version: 1.4.3

## In both

*   checking whether package ‘splatter’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘DelayedArray’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/splatter/new/splatter.Rcheck/00install.out’ for details.
    ```

# SPLINTER

Version: 1.6.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    Loading required package: AnnotationDbi
    Loading required package: Biobase
    Welcome to Bioconductor
    
        Vignettes contain introductory material; view with
        'browseVignettes()'. To cite Bioconductor, see
        'citation("Biobase")', and for packages 'citation("pkgname")'.
    
    Import genomic features from the file as a GRanges object ... OK
    Prepare the 'metadata' data frame ... OK
    Make the TxDb object ... OK
    tlmgr search --file --global '/beramono.sty'
    Unknown directive ...l
    ! LaTeX Error: File `beramono.sty' not found.
    
    ! Emergency stop.
    <read *> 
    
    Error: processing vignette 'vignette.Rmd' failed with diagnostics:
    Failed to compile vignette.tex. See vignette.log for more info.
    Execution halted
    ```

*   checking R code for possible problems ... NOTE
    ```
    metaremove: no visible global function definition for 'is'
    metaremove: no visible global function definition for 'endoapply'
    Undefined global functions or variables:
      endoapply is
    Consider adding
      importFrom("methods", "is")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

# SPONGE

Version: 1.2.0

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  9.8Mb
      sub-directories of 1Mb or more:
        data   8.0Mb
        doc    1.5Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      for ‘head’
    sponge_plot_simulation_results: no visible binding for global variable
      ‘mscor’
    sponge_run_benchmark: no visible binding for global variable
      ‘precomputed_cov_matrices’
    sponge_run_benchmark: no visible binding for global variable
      ‘elastic.net’
    sponge_run_benchmark: no visible binding for global variable
      ‘each.miRNA’
    sponge_subsampling: no visible binding for global variable ‘sub.n’
    sponge_subsampling: no visible binding for global variable ‘geneA’
    sponge_subsampling: no visible binding for global variable ‘geneB’
    Undefined global functions or variables:
      . .EACHI .I := J alpha attach.big.matrix chunk cor_cut cov.matrices.k
      cov.matrices.m cov.matrix df_cut dt.m each.miRNA elastic.net g_expr
      g_expr_batch gene geneA geneA_idx geneB geneB_idx gene_combis ginv
      head i k m mir mirna mscor mwhich p.adj p.val
      precomputed_cov_matrices solution sub.n
    Consider adding
      importFrom("utils", "head")
    to your NAMESPACE file.
    ```

# sport

Version: 0.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    LaTeX errors found:
    ! LaTeX Error: Command \k unavailable in encoding OT1.
    
    See the LaTeX manual or LaTeX Companion for explanation.
    Type  H <return>  for immediate help.
     ...                                              
    ! LaTeX Error: Command \k unavailable in encoding OT1.
    
    See the LaTeX manual or LaTeX Companion for explanation.
    Type  H <return>  for immediate help.
     ...                                              
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 6863 marked UTF-8 strings
    ```

# sppmix

Version: 1.0.2

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  7.1Mb
      sub-directories of 1Mb or more:
        data   5.4Mb
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 17 marked UTF-8 strings
    ```

# ssdtools

Version: 0.0.2

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘actuar’ ‘pryr’
      All declared Imports should be used.
    ```

# ssviz

Version: 1.14.0

## In both

*   checking whether package ‘ssviz’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘Rsamtools’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/ssviz/new/ssviz.Rcheck/00install.out’ for details.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Description field: should contain one or more complete sentences.
    ```

# stacomiR

Version: 0.5.4.0

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: ‘gWidgetsRGtk2’ ‘RGtk2’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# staRdom

Version: 1.0.8

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘readr’ ‘tools’
      All declared Imports should be used.
    ```

# starmie

Version: 0.1.2

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  7.0Mb
      sub-directories of 1Mb or more:
        doc       1.1Mb
        extdata   4.9Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘MCMCpack’
      All declared Imports should be used.
    ```

# stars

Version: 0.1-1

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘sf’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# STATegRa

Version: 1.16.1

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  6.1Mb
      sub-directories of 1Mb or more:
        data   2.4Mb
        doc    2.9Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    modelSelection,list-numeric-character: no visible binding for global
      variable ‘components’
    modelSelection,list-numeric-character: no visible binding for global
      variable ‘mylabel’
    plotVAF,caClass: no visible binding for global variable ‘comp’
    plotVAF,caClass: no visible binding for global variable ‘VAF’
    plotVAF,caClass: no visible binding for global variable ‘block’
    selectCommonComps,list-numeric: no visible binding for global variable
      ‘comps’
    selectCommonComps,list-numeric: no visible binding for global variable
      ‘block’
    selectCommonComps,list-numeric: no visible binding for global variable
      ‘comp’
    selectCommonComps,list-numeric: no visible binding for global variable
      ‘ratio’
    Undefined global functions or variables:
      VAF block comp components comps mylabel ratio
    ```

# states

Version: 0.2.1

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘lubridate’
      All declared Imports should be used.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 3 marked UTF-8 strings
    ```

# statsDK

Version: 0.1.1

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘dplyr’ ‘ggplot2’ ‘stringr’
      All declared Imports should be used.
    ```

# stminsights

Version: 0.2.2

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘huge’ ‘readr’ ‘scales’ ‘shinyjs’
      All declared Imports should be used.
    ```

# stormwindmodel

Version: 0.1.1

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    Attaching package: 'dplyr'
    
    The following objects are masked from 'package:stats':
    
        filter, lag
    
    The following objects are masked from 'package:base':
    
        intersect, setdiff, setequal, union
    
    
    Attaching package: 'gridExtra'
    
    The following object is masked from 'package:dplyr':
    
        combine
    
    Quitting from lines 220-229 (Details.Rmd) 
    Error: processing vignette 'Details.Rmd' failed with diagnostics:
    there is no package called 'sf'
    Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘sf’
    ```

# stpp

Version: 2.0-3

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘rpanel’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# strataG

Version: 2.0.2

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘survival’
      All declared Imports should be used.
    ```

# StroupGLMM

Version: 0.1.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘broom’ ‘car’ ‘lmerTest’ ‘pbkrtest’
      All declared Imports should be used.
    ```

# structSSI

Version: 1.1.1

## In both

*   checking R code for possible problems ... NOTE
    ```
    PlotHypTree: no visible global function definition for ‘browseURL’
    treePValues: no visible global function definition for ‘lm’
    treePValues: no visible global function definition for ‘pf’
    plot,GBH-ANY: no visible binding for global variable ‘sorted.hyp’
    plot,GBH-ANY: no visible binding for global variable ‘pval’
    plot,GBH-ANY: no visible binding for global variable ‘group’
    plot,GBH-ANY: no visible binding for global variable ‘type’
    Undefined global functions or variables:
      browseURL group lm pf pval sorted.hyp type
    Consider adding
      importFrom("stats", "lm", "pf")
      importFrom("utils", "browseURL")
    to your NAMESPACE file.
    ```

# strvalidator

Version: 2.1.0

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: ‘gWidgets2RGtk2’ ‘RGtk2’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# stUPscales

Version: 1.0.3.3

## In both

*   checking examples ... ERROR
    ```
    ...
    +            )
    > 
    > # Defining deterministic input:
    > library(EmiStatR)
    > # data(P1)
    > 
    > input.det <- input(spatial = 0, zero = 1e-5, 
    +                     folder = system.file("shiny", package = "EmiStatR"),
    +                     cores = 0,
    +                     ww = list(qs = 150, CODs = 104, NH4s = 4.7),
    +                     inf = list(qf= 0.04, CODf = 0, NH4f = 0),
    +                     rw = list(CODr = 71, NH4r = 1, stat = "Dahl"),
    +                     P1 = P1, st = list(E1=E1), export = 0)
    > 
    > # Invoking `EmiStatR` with the deterministic input: 
    > sim.det   <- EmiStatR(input.det)
    Warning in file(file, "rt") :
      cannot open file '/Users/max/github/forks/ggplot2/revdep/library.noindex/stUPscales/EmiStatR/shiny/EmiStatR_inputCSO/inputCSO/pe_factor.csv': No such file or directory
    Error in file(file, "rt") : cannot open the connection
    Calls: EmiStatR ... EmiStatR -> CInp2TS -> read.csv -> read.table -> file
    Execution halted
    ```

# SubgrPlots

Version: 0.1.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘UpSetR’ ‘alluvial’ ‘geoR’ ‘gridBase’
      All declared Imports should be used.
    ```

# subSeq

Version: 1.10.0

## In both

*   checking R code for possible problems ... NOTE
    ```
    ...
    summary.subsamples: no visible binding for global variable ‘o.padj’
    summary.subsamples: no visible binding for global variable
      ‘significant’
    summary.subsamples: no visible binding for global variable ‘estFDP’
    summary.subsamples: no visible binding for global variable ‘rFDP’
    summary.subsamples: no visible binding for global variable ‘metric’
    summary.subsamples: no visible binding for global variable ‘value’
    summary.subsamples: no visible binding for global variable ‘percent’
    voomLimma: no visible global function definition for ‘model.matrix’
    Undefined global functions or variables:
      . ID average.depth average.value coefficient cor count cov depth
      estFDP method metric model.matrix o.coefficient o.lfdr o.padj
      p.adjust padj percent plot proportion pvalue rFDP rbinom replication
      selectMethod significant valid value var
    Consider adding
      importFrom("graphics", "plot")
      importFrom("methods", "selectMethod")
      importFrom("stats", "cor", "cov", "model.matrix", "p.adjust", "rbinom",
                 "var")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

# subspaceMOA

Version: 0.6.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘fields’
      All declared Imports should be used.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘subspace’
    ```

# SummarizedBenchmark

Version: 1.0.4

## In both

*   checking whether package ‘SummarizedBenchmark’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘DelayedArray’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/SummarizedBenchmark/new/SummarizedBenchmark.Rcheck/00install.out’ for details.
    ```

*   checking package dependencies ... NOTE
    ```
    Depends: includes the non-default packages:
      ‘tidyr’ ‘SummarizedExperiment’ ‘S4Vectors’ ‘BiocGenerics’ ‘UpSetR’
      ‘rlang’ ‘stringr’ ‘BiocParallel’ ‘ggplot2’ ‘mclust’ ‘dplyr’
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 12.7Mb
      sub-directories of 1Mb or more:
        data   9.0Mb
        doc    3.3Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Unexported object imported by a ':::' call: ‘BiocGenerics:::replaceSlots’
      See the note in ?`:::` about the use of this operator.
    ```

*   checking R code for possible problems ... NOTE
    ```
    .list2mat : <anonymous>: no visible binding for global variable
      ‘.method’
    .list2mat : <anonymous>: no visible binding for global variable ‘.val’
    .list2mat : <anonymous>: no visible binding for global variable ‘.id’
    plotROC: no visible binding for global variable ‘FDR’
    plotROC: no visible binding for global variable ‘TPR’
    plotROC: no visible binding for global variable ‘method’
    Undefined global functions or variables:
      .id .method .val FDR TPR method
    ```

# SUMMER

Version: 0.2.1

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘INLA’
    ```

# SuperLearner

Version: 2.0-24

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'SuperLearnerPresent.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `pdfpages.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.8 \usepackage
                   {Sweave}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# sure

Version: 0.2.0

## In both

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘truncdist’
    ```

# surveillance

Version: 1.16.2

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘INLA’
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  7.3Mb
      sub-directories of 1Mb or more:
        R      2.0Mb
        doc    2.3Mb
        help   1.0Mb
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘scoringRules’
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    glrnb: Fitting Poisson model because alpha == 0
    glrnb: Fitting Poisson model because alpha == 0
    glrnb: Fitting Poisson model because alpha == 0
    glrnb: Fitting Poisson model because alpha == 0
    glrnb: Fitting Poisson model because alpha == 0
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'glrnb.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.30 \ifthenelse
                    {\boolean{Sweave@inconsolata}}{%^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# survivALL

Version: 0.9.3

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘cowplot’
      All declared Imports should be used.
    ```

# survminer

Version: 0.4.3

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.6Mb
      sub-directories of 1Mb or more:
        doc   5.1Mb
    ```

# survMisc

Version: 0.5.5

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'plots.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `mathtools.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.62 \usepackage
                    [sc]{mathpazo}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# survsup

Version: 0.0.1

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘gridExtra’ ‘stats’ ‘survival’ ‘utils’
      All declared Imports should be used.
    ```

# sValues

Version: 0.1.6

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'sValues.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.30 \ifthenelse
                    {\boolean{Sweave@inconsolata}}{%^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# SVAPLSseq

Version: 1.6.0

## In both

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    
    Warning in if (surr.select == "automatic") { :
      the condition has length > 1 and only the first element will be used
    Warning in if (surr.select == "manual") index = opt.surrs :
      the condition has length > 1 and only the first element will be used
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'SVAPLSseq.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `beramono.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.87 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# svdvis

Version: 0.1

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘svdvis-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: radviz
    > ### Title: Radial Coordinates Plots
    > ### Aliases: radviz
    > 
    > ### ** Examples
    > 
    > set.seed(1234)
    > dat = matrix(rnorm(9*4), 9, 4, dimnames=list(paste(1:9),letters[1:4]))
    > radviz(dat, group=c(rep("Group 1",3),rep("Group 2",3),rep("Group 3",3)))
    Error: `data` must be uniquely named but has duplicate columns
    Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 78-80 (svdvis-vignette.Rmd) 
    Error: processing vignette 'svdvis-vignette.Rmd' failed with diagnostics:
    `data` must be uniquely named but has duplicate columns
    Execution halted
    ```

# SVMMaj

Version: 0.2-8

## In both

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    
    The following object is masked from ‘package:ggplot2’:
    
        alpha
    
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'paper.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.55 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# SWATH2stats

Version: 1.10.2

## In both

*   checking package dependencies ... NOTE
    ```
    Packages which this enhances but not available for checking:
      ‘imsbInfer’ ‘MSstats’
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    Optimizing parameters
    Calculating p-values
    Calculating FDR
    Aggregating statistics
    Done
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'SWATH2stats_vignette.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `authblk.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.5 \makeatletter
                     ^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# swfdr

Version: 1.6.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Warning in has_utility("pdfcrop") :
      pdfcrop not installed or not in PATH
    tlmgr search --file --global '/beramono.sty'
    Unknown directive ...l
    Warning in system2("tlmgr", args, ...) :
      running command ''tlmgr' search --file --global '/beramono.sty'' had status 25
    ! LaTeX Error: File `beramono.sty' not found.
    
    ! Emergency stop.
    <read *> 
    
    Error: processing vignette 'swfdrTutorial.Rmd' failed with diagnostics:
    Failed to compile swfdrTutorial.tex. See swfdrTutorial.log for more info.
    Execution halted
    ```

*   checking R code for possible problems ... NOTE
    ```
    lm_pi0: no visible global function definition for ‘glm’
    lm_pi0: no visible binding for global variable ‘binomial’
    Undefined global functions or variables:
      binomial glm
    Consider adding
      importFrom("stats", "binomial", "glm")
    to your NAMESPACE file.
    ```

# switchde

Version: 1.6.0

## In both

*   checking whether package ‘switchde’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘DelayedArray’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/switchde/new/switchde.Rcheck/00install.out’ for details.
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

# swmmr

Version: 0.8.1

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    
    Attaching package: 'dplyr'
    
    The following objects are masked from 'package:stats':
    
        filter, lag
    
    The following objects are masked from 'package:base':
    
        intersect, setdiff, setequal, union
    
    Quitting from lines 26-31 (How_to_convert_between_GIS_data_and_SWMM_with_swmmr.Rmd) 
    Error: processing vignette 'How_to_convert_between_GIS_data_and_SWMM_with_swmmr.Rmd' failed with diagnostics:
    there is no package called 'sf'
    Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘sf’
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘sf’
    ```

# SWMPrExtension

Version: 0.3.16

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘rgeos’
      All declared Imports should be used.
    ```

# synergyfinder

Version: 1.6.1

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'synergyfinder.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `authblk.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.53 \usepackage
                    {grffile}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    In addition: Warning message:
    In replayPlot(x) : `mgp[1:3]' are of differing sign
    Execution halted
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    License components with restrictions not permitted:
      Mozilla Public License 2.0 + file LICENSE
    ```

*   checking R code for possible problems ... NOTE
    ```
    PlotSynergy: no visible binding for global variable ‘pred.cory’
    PlotSynergy: no visible binding for global variable ‘pred.corx’
    Undefined global functions or variables:
      pred.corx pred.cory
    ```

# synlet

Version: 1.10.0

## In both

*   checking R code for possible problems ... NOTE
    ```
    ...
    siRNAPlot: no visible global function definition for ‘pdf’
    siRNAPlot: no visible global function definition for ‘dev.off’
    tTest: no visible global function definition for ‘p.adjust’
    zFactor: no visible binding for global variable ‘condition’
    zFactor: no visible binding for global variable ‘sd’
    zFactor: no visible binding for global variable ‘median’
    zFactor: no visible global function definition for ‘complete.cases’
    Undefined global functions or variables:
      COL_NAME EXPERIMENT_MODIFICATION EXPERIMENT_TYPE MASTER_PLATE PLATE
      READOUT ROW_NAME Var1 WELL_CONTENT_NAME colorRampPalette
      complete.cases condition dev.off experiments is mad median medpolish
      p.adjust pdf phyper rainbow sd siRNA t.test value write.table
    Consider adding
      importFrom("grDevices", "colorRampPalette", "dev.off", "pdf",
                 "rainbow")
      importFrom("methods", "is")
      importFrom("stats", "complete.cases", "mad", "median", "medpolish",
                 "p.adjust", "phyper", "sd", "t.test")
      importFrom("utils", "write.table")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

# synthpop

Version: 1.5-0

## In both

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
      ...
    Loading required package: lattice
    Loading required package: MASS
    Loading required package: nnet
    Loading required package: ggplot2
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'inference.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `authblk.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.11 ^^M
            
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# systemPipeR

Version: 1.14.0

## In both

*   checking whether package ‘systemPipeR’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘Rsamtools’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘DelayedArray’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/systemPipeR/new/systemPipeR.Rcheck/00install.out’ for details.
    ```

*   checking package subdirectories ... WARNING
    ```
    Invalid citation information in ‘inst/CITATION’:
      Error in library(knitcitations): there is no package called ‘knitcitations’
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'systemPipeChIPseq.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `beramono.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.87 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      inst/extdata/.BatchJobs.R
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  6.9Mb
      sub-directories of 1Mb or more:
        R         1.0Mb
        doc       3.9Mb
        extdata   1.7Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘BiocGenerics’ ‘VariantAnnotation’
      All declared Imports should be used.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      ‘close.screen’
    writeTargetsRef: no visible global function definition for ‘read.delim’
    show,INTERSECTset: no visible binding for global variable ‘vennset’
    Undefined global functions or variables:
      AllVariants Base Comparisons Counts Coverage Cycle DataFrame Feature
      Frequency IRanges Intersect_Sets Length Level Method Outliers Percent
      Quality RelDiv Rle Sample SampleMatch Strand Type aggregate alt
      altDepth<- asVCF boxplot chunk close.screen combn dev.off first
      import.bed last locateVariants low mid minQuality model.matrix
      na.omit pdf phyper plot predictCoding qwidth read.delim
      readGAlignmentPairs readGAlignments readVcf ref refDepth<- screen
      seqlengths seqlengths<- split.screen subsetByOverlaps
      summarizeOverlaps symbols test_sample text top tophatargs
      totalDepth<- vennset write.table writeVcf
    Consider adding
      importFrom("grDevices", "dev.off", "pdf")
      importFrom("graphics", "boxplot", "close.screen", "plot", "screen",
                 "split.screen", "symbols", "text")
      importFrom("stats", "aggregate", "model.matrix", "na.omit", "phyper")
      importFrom("utils", "combn", "read.delim", "write.table")
    to your NAMESPACE file.
    ```

*   checking for unstated dependencies in vignettes ... NOTE
    ```
    'library' or 'require' call not declared from: ‘GenomicAlignments’
    ```

# tableone

Version: 0.9.3

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘methods’
      All declared Imports should be used.
    ```

# tadaatoolbox

Version: 0.16.0

## In both

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 3 marked UTF-8 strings
    ```

# taRifx

Version: 1.0.6.1

## In both

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' calls in package code:
      ‘gdata’ ‘ggplot2’ ‘grid’ ‘lattice’ ‘xtable’
      Please use :: or requireNamespace() instead.
      See section 'Suggested packages' in the 'Writing R Extensions' manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    xtable.CrossTable: no visible global function definition for ‘label<-’
    xtable.CrossTable: no visible global function definition for ‘align<-’
    xtable.CrossTable: no visible global function definition for ‘digits<-’
    xtable.CrossTable: no visible global function definition for
      ‘display<-’
    xtable.summary.lme: no visible global function definition for
      ‘caption<-’
    xtable.summary.lme: no visible global function definition for ‘label<-’
    xtable.summary.lme: no visible global function definition for ‘align<-’
    xtable.summary.lme: no visible global function definition for
      ‘digits<-’
    xtable.summary.lme: no visible global function definition for
      ‘display<-’
    xtablelm: no visible global function definition for ‘xtable’
    Undefined global functions or variables:
      aes align<- caption<- convertUnit coord_cartesian digits<- display<-
      ggplot gpar grid.layout grid.lines grid.newpage grid.points
      grid.polyline grid.rect grid.segments grid.text interleave label<-
      latticeParseFormula opts panel.densityplot panel.lines panel.xyplot
      popViewport pushViewport seekViewport stat_summary theme_text unit
      upViewport viewport xtable
    ```

# TarSeqQC

Version: 1.10.0

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    > bamFile<-system.file("extdata", "mybam.bam", package="TarSeqQC", 
    +     mustWork=TRUE)
    > bedFile<-system.file("extdata", "mybed.bed", package="TarSeqQC",
    +     mustWork=TRUE)
    > fastaFile<-system.file("extdata", "myfasta.fa", package="TarSeqQC", 
    +     mustWork=TRUE)
    > 
    > ## Creating a TargetExperiment object
    > 
    > # Defining feature parameter
    > feature<-"amplicon"
    > # Defining attribute parameter
    > attribute<-"coverage"
    > ampliPanel<-TargetExperiment(bedFile, bamFile, fastaFile, attribute=attribute,
    +     feature=feature)
    Warning in socketConnection(host, port, TRUE, TRUE, "a+b", timeout = timeout) :
      port 11681 cannot be opened
    Error in socketConnection(host, port, TRUE, TRUE, "a+b", timeout = timeout) : 
      cannot open the connection
    Calls: TargetExperiment ... .local -> .bpfork -> .bpforkConnect -> socketConnection
    Execution halted
    ```

## In both

*   checking whether package ‘TarSeqQC’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘Rsamtools’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/TarSeqQC/new/TarSeqQC.Rcheck/00install.out’ for details.
    ```

*   checking R code for possible problems ... NOTE
    ```
    plotNtdPercentage,TargetExperiment: no visible global function
      definition for ‘path’
    plotRegion,TargetExperiment: no visible global function definition for
      ‘path’
    readFrequencies,TargetExperiment: no visible global function definition
      for ‘path’
    Undefined global functions or variables:
      path
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Error in .requirePackage(package) : 
        unable to find required package 'TarSeqQC'
      Calls: <Anonymous> ... getClass -> getClassDef -> .classEnv -> .requirePackage
      Execution halted
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    Error in re-building vignettes:
      ...
    Warning: package 'GenomicRanges' was built under R version 3.5.1
    Warning: package 'IRanges' was built under R version 3.5.1
    Warning: package 'Rsamtools' was built under R version 3.5.1
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'TarSeqQC-vignette.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `times.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.12 \usepackage
                    [colorlinks=false, urlcolor=black, pdfborder={0 0 0}]{hyperr...
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# TCGAbiolinks

Version: 2.8.4

## In both

*   R CMD check timed out
    

*   checking dependencies in R code ... WARNING
    ```
    '::' or ':::' import not declared from: ‘tidyr’
    ```

*   checking for missing documentation entries ... WARNING
    ```
    Warning: package ‘maftools’ was built under R version 3.5.1
    All user-level objects in a package should have documentation entries.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 72.6Mb
      sub-directories of 1Mb or more:
        R      2.4Mb
        data   3.5Mb
        doc   66.4Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    TCGAanalyze_networkInference: no visible global function definition for
      ‘minet’
    TCGAquery_recount2: no visible binding for global variable ‘rse_gene’
    TCGAtumor_purity: no visible binding for global variable ‘Tumor.purity’
    TCGAvisualize_SurvivalCoxNET: no visible global function definition for
      ‘dNetInduce’
    TCGAvisualize_SurvivalCoxNET: no visible global function definition for
      ‘dNetPipeline’
    TCGAvisualize_SurvivalCoxNET: no visible global function definition for
      ‘dCommSignif’
    TCGAvisualize_SurvivalCoxNET: no visible global function definition for
      ‘visNet’
    TCGAvisualize_oncoprint: no visible binding for global variable ‘value’
    readExonQuantification: no visible binding for global variable ‘exon’
    readExonQuantification: no visible binding for global variable
      ‘coordinates’
    Undefined global functions or variables:
      TabSubtypesCol_merged Tumor.purity barcode c3net clinical coordinates
      dCommSignif dNetInduce dNetPipeline exon knnmi.cross
      limmacontrasts.fit limmamakeContrasts minet portions rse_gene value
      visNet
    ```

# TCGAbiolinksGUI

Version: 1.6.1

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 20.8Mb
      sub-directories of 1Mb or more:
        app   1.1Mb
        doc  19.5Mb
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Description field: should contain one or more complete sentences.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      'DT' 'IlluminaHumanMethylation27kanno.ilmn12.hg19'
      'IlluminaHumanMethylation27kmanifest'
      'IlluminaHumanMethylation450kanno.ilmn12.hg19'
      'IlluminaHumanMethylation450kmanifest'
      'IlluminaHumanMethylationEPICanno.ilm10b2.hg19'
      'IlluminaHumanMethylationEPICmanifest' 'minfi'
      All declared Imports should be used.
    Package in Depends field not imported from: 'TCGAbiolinksGUI.data'
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

# tcR

Version: 2.2.2

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  6.0Mb
      sub-directories of 1Mb or more:
        data   1.4Mb
        doc    3.9Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘scales’
      All declared Imports should be used.
    ```

# TCseq

Version: 1.4.0

## In both

*   checking R code for possible problems ... NOTE
    ```
    DBresult: no visible global function definition for ‘as’
    DBresult.cluster: no visible global function definition for ‘as’
    TCA: no visible global function definition for ‘is’
    TCA: no visible global function definition for ‘as’
    TCAFromSummarizedExperiment: no visible global function definition for
      ‘is’
    TCAFromSummarizedExperiment: no visible global function definition for
      ‘as’
    countReads: no visible global function definition for
      ‘createAnnotationFile’
    countReads: no visible global function definition for ‘featureCounts’
    timeclustplot: no visible binding for global variable ‘group’
    timecourseTable: no visible global function definition for ‘as’
    Undefined global functions or variables:
      as createAnnotationFile featureCounts group is
    Consider adding
      importFrom("methods", "as", "is")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    
    The following objects are masked from ‘package:base’:
    
        aperm, apply
    
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'TCseq.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `a4wide.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.8 \usepackage
                   [utf8]{inputenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# TeachingDemos

Version: 2.10

## In both

*   R CMD check timed out
    

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘R2wd’
    ```

# TELP

Version: 1.0

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘TELP-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: telp
    > ### Title: The Base Function of This Package The Free Evocation of Words
    > ###   Technique
    > ### Aliases: telp
    > ### Keywords: function
    > 
    > ### ** Examples
    > 
    > telp()
    Error in structure(.External(.C_dotTclObjv, objv), class = "tclObj") : 
      [tcl] invalid command name "toplevel".
    Calls: telp ... tktoplevel -> tkwidget -> tcl -> .Tcl.objv -> structure
    Execution halted
    ```

# TFEA.ChIP

Version: 1.0.0

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/runTests.R’ failed.
    Last 13 lines of output:
      Attaching package: 'AnnotationDbi'
      
      The following object is masked from 'package:dplyr':
      
          select
      
      
      Error in library("RUnit", quietly = TRUE) : 
        there is no package called 'RUnit'
      Calls: <Anonymous> -> library
      In addition: Warning messages:
      1: package 'GenomicFeatures' was built under R version 3.5.1 
      2: package 'IRanges' was built under R version 3.5.1 
      3: package 'GenomicRanges' was built under R version 3.5.1 
      Execution halted
    ```

*   checking whether package ‘TFEA.ChIP’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘GenomicFeatures’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/TFEA.ChIP/new/TFEA.ChIP.Rcheck/00install.out’ for details.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Failed with error:  'package 'DelayedArray' could not be loaded'
      Error in .requirePackage(package) : 
        unable to find required package 'DESeq2'
      Calls: <Anonymous> ... getClass -> getClassDef -> .classEnv -> .requirePackage
      Execution halted
    ```

# tfestimators

Version: 1.9.0

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      Instructions for updating:
      Please switch to tf.train.get_global_step
      WARNING:tensorflow:Casting <dtype: 'float32'> labels to bool.
      WARNING:tensorflow:Casting <dtype: 'float32'> labels to bool.
      WARNING:tensorflow:Casting <dtype: 'float32'> labels to bool.
      WARNING:tensorflow:Casting <dtype: 'float32'> labels to bool.
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 102 SKIPPED: 3 FAILED: 4
      1. Failure: input_fn can be constructed through formula interface (@helper-utils.R#21) 
      2. Failure: input_fn can be constructed correctly from data.frame objects (@helper-utils.R#21) 
      3. Failure: input_fn can be constructed correctly from matrix objects (@helper-utils.R#21) 
      4. Failure: R factors are coerced appropriately (@helper-utils.R#21) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

# TFutils

Version: 1.0.0

## In both

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 62 marked UTF-8 strings
    ```

# theseus

Version: 0.1.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘gridExtra’ ‘splancs’ ‘tidyverse’
      All declared Imports should be used.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Packages unavailable to check Rd xrefs: ‘DESeq2’, ‘dada2’
    ```

# tidybayes

Version: 1.0.3

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
             modules = modules, factories = factories, jags = jags, call.setup = TRUE, method = method, 
             mutate = mutate)
      10: setup.jags(model = outmodel, monitor = outmonitor, data = outdata, n.chains = n.chains, 
             inits = outinits, modules = modules, factories = factories, response = response, 
             fitted = fitted, residual = residual, jags = jags, method = method, mutate = mutate)
      11: loadandcheckrjags()
      12: stop("Loading the rjags package failed (diagnostics are given above this error message)", 
             call. = FALSE)
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 269 SKIPPED: 2 FAILED: 1
      1. Error: tidy_draws works with runjags (@test.tidy_draws.R#87) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

# tidycensus

Version: 0.8.1

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘sf’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# tidyinftheo

Version: 0.2.1

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      12: shannon_entropy(.)
      13: reduce_data(.data, !!enquo(X), numvars = 1, na.rm = na.rm)
      14: tidyselect::vars_select(names(reduced_tab), !!!quos(...))
      15: vars_select_eval(.vars, quos)
      16: map_lgl(quos, quo_is_helper)
      17: .f(.x[[i]], ...)
      18: extract_expr(quo)
      19: is_call(expr, paren_sym)
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 37 SKIPPED: 0 FAILED: 1
      1. Error: right number of columns given (@test-errors.R#10) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘tibble’ ‘tidyverse’
      All declared Imports should be used.
    ```

# tidymodels

Version: 0.0.1

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘broom’ ‘ggplot2’ ‘infer’ ‘pillar’ ‘recipes’ ‘rsample’
      ‘tidyposterior’ ‘tidypredict’ ‘tidytext’ ‘yardstick’
      All declared Imports should be used.
    ```

# tidyverse

Version: 1.2.1

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘dbplyr’ ‘reprex’ ‘rlang’
      All declared Imports should be used.
    ```

# tidyxl

Version: 1.0.3

## In both

*   checking compiled code ... WARNING
    ```
    File ‘tidyxl/libs/tidyxl.so’:
      Found ‘_abort’, possibly from ‘abort’ (C)
        Object: ‘xlex.o’
    
    Compiled code should not call entry points which might terminate R nor
    write to stdout/stderr instead of to the console, nor use Fortran I/O
    nor system RNGs.
    
    See ‘Writing portable packages’ in the ‘Writing R Extensions’ manual.
    ```

# tigris

Version: 0.7

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘sf’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# tikzDevice

Version: 0.12

## In both

*   checking examples ... ERROR
    ```
    ...
    
    	{\\tiny Hello \\LaTeX!}
    
    Run the following commands for diagnosis:
    
    	tikzTest()
    	tikzTest("{\\\\tiny Hello \\\\LaTeX!}")
    
    Common reasons for failure include:
      * The string contains a character which is special to LaTeX unless
        escaped properly, such as % or $.
      * The string makes use of LaTeX commands provided by a package and
        the tikzDevice was not told to load the package.
    
    The TeX and log files used for the calculation can help diagnose the
    problem. If these files are missing, rerun the plot and make sure to
    keep the R session open.
    TeX file: /var/folders/lb/xhxqmcrd7gv302_b1pdfykh80000gn/T//Rtmp3QHrqH/tikzDevice1094e55ec8d2a/tikzStringWidthCalc.tex
    Log file: /var/folders/lb/xhxqmcrd7gv302_b1pdfykh80000gn/T//Rtmp3QHrqH/tikzDevice1094e55ec8d2a/tikzStringWidthCalc.log
    Calls: getLatexStrWidth -> getMetricsFromLatex
    Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    TeX was unable to calculate metrics for:
    
    	m
    
    Run the following commands for diagnosis:
    
    	tikzTest()
    	tikzTest("m")
    
    Common reasons for failure include:
      * The string contains a character which is special to LaTeX unless
        escaped properly, such as % or $.
      * The string makes use of LaTeX commands provided by a package and
        the tikzDevice was not told to load the package.
    
    The TeX and log files used for the calculation can help diagnose the
    problem. If these files are missing, rerun the plot and make sure to
    keep the R session open.
    TeX file: /var/folders/lb/xhxqmcrd7gv302_b1pdfykh80000gn/T//RtmpEm8o3q/tikzDevice109b63d120b1b/tikzStringWidthCalc.tex
    Log file: /var/folders/lb/xhxqmcrd7gv302_b1pdfykh80000gn/T//RtmpEm8o3q/tikzDevice109b63d120b1b/tikzStringWidthCalc.log
    Execution halted
    ```

# timeline

Version: 0.9

## In both

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' call to ‘shiny’ in package code.
      Please use :: or requireNamespace() instead.
      See section 'Suggested packages' in the 'Writing R Extensions' manual.
    Package in Depends field not imported from: ‘ggplot2’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

*   checking R code for possible problems ... NOTE
    ```
    timeline: no visible global function definition for ‘ggplot’
    timeline: no visible global function definition for ‘geom_segment’
    timeline: no visible global function definition for ‘aes_string’
    timeline: no visible global function definition for ‘geom_rect’
    timeline: no visible global function definition for ‘geom_text’
    timeline: no visible global function definition for ‘theme’
    timeline: no visible global function definition for ‘element_blank’
    timeline: no visible global function definition for ‘xlab’
    timeline: no visible global function definition for ‘ylab’
    timeline: no visible global function definition for ‘xlim’
    timeline: no visible global function definition for
      ‘scale_y_continuous’
    timeline: no visible global function definition for ‘geom_point’
    timeline: no visible global function definition for ‘scale_color_grey’
    timeline: no visible global function definition for ‘geom_hline’
    Undefined global functions or variables:
      aes_string element_blank geom_hline geom_point geom_rect geom_segment
      geom_text ggplot scale_color_grey scale_y_continuous theme xlab xlim
      ylab
    ```

# timelineS

Version: 0.1.1

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘base’
      All declared Imports should be used.
    ```

# TimeProjection

Version: 0.2.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' calls in package code:
      ‘ggplot2’ ‘plyr’
      Please use :: or requireNamespace() instead.
      See section 'Suggested packages' in the 'Writing R Extensions' manual.
    Packages in Depends field not imported from:
      ‘Matrix’ ‘lubridate’ ‘timeDate’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    plotCalendarHeatmap: no visible global function definition for ‘ddply’
    plotCalendarHeatmap: no visible global function definition for ‘.’
    plotCalendarHeatmap: no visible binding for global variable ‘year’
    plotCalendarHeatmap: no visible binding for global variable ‘month’
    plotCalendarHeatmap: no visible binding for global variable ‘week’
    plotCalendarHeatmap: no visible global function definition for ‘ggplot’
    plotCalendarHeatmap: no visible global function definition for ‘aes’
    plotCalendarHeatmap: no visible binding for global variable ‘monthweek’
    plotCalendarHeatmap: no visible binding for global variable ‘weekday’
    plotCalendarHeatmap: no visible global function definition for
      ‘geom_tile’
    plotCalendarHeatmap: no visible global function definition for
      ‘facet_grid’
    plotCalendarHeatmap: no visible global function definition for
      ‘scale_fill_gradientn’
    projectDate: no visible global function definition for ‘holidayNYSE’
    projectDate: no visible global function definition for
      ‘sparse.model.matrix’
    Undefined global functions or variables:
      . aes ddply facet_grid geom_tile ggplot holidayNYSE isWeekday month
      monthweek scale_fill_gradientn sparse.model.matrix week weekday year
    ```

# TimerQuant

Version: 1.10.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
      no non-missing arguments to min; returning Inf
    Warning in min(x, na.rm = TRUE) :
      no non-missing arguments to min; returning Inf
    Warning in min(x, na.rm = TRUE) :
      no non-missing arguments to min; returning Inf
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'genPaperFigures.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `float.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.30 \date
              {}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      ‘rainbow’
    plotPrimordiumProfile: no visible binding for global variable ‘median’
    plotPrimordiumProfile: no visible binding for global variable ‘mad’
    plotPrimordiumProfile: no visible global function definition for ‘par’
    plotPrimordiumProfile: no visible global function definition for ‘plot’
    plotPrimordiumProfile: no visible global function definition for ‘axis’
    plotPrimordiumProfile: no visible global function definition for
      ‘points’
    plotPrimordiumProfile: no visible global function definition for
      ‘polygon’
    plotPrimordiumProfile: no visible global function definition for ‘rgb’
    simulatedRatio: no visible global function definition for ‘rnorm’
    Undefined global functions or variables:
      approxfun axis mad median optimize par plot points polygon predict
      rainbow rgb rnorm
    Consider adding
      importFrom("grDevices", "rainbow", "rgb")
      importFrom("graphics", "axis", "par", "plot", "points", "polygon")
      importFrom("stats", "approxfun", "mad", "median", "optimize",
                 "predict", "rnorm")
    to your NAMESPACE file.
    ```

# tint

Version: 0.1.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    tlmgr search --file --global '/tufte-handout.cls'
    Unknown directive ...l
    Warning in system2("tlmgr", args, ...) :
      running command ''tlmgr' search --file --global '/tufte-handout.cls'' had status 25
    ! LaTeX Error: File `tufte-handout.cls' not found.
    
    ! Emergency stop.
    <read *> 
    
    Error: processing vignette 'tintPDF.Rmd' failed with diagnostics:
    Failed to compile tintPDF.tex. See tintPDF.log for more info.
    Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.1Mb
      sub-directories of 1Mb or more:
        doc         3.7Mb
        rmarkdown   1.3Mb
    ```

# TissueEnrich

Version: 1.0.7

## In both

*   checking whether package ‘TissueEnrich’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘DelayedArray’ was built under R version 3.5.1
      Warning: package ‘graph’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/TissueEnrich/new/TissueEnrich.Rcheck/00install.out’ for details.
    ```

# tmap

Version: 2.1-1

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘sf’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# toaster

Version: 0.5.5

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      ── 1. Failure: format 'boxplot' works (@test-showData.R#12)  ───────────────────
      setdiff(p$layout$panel_ranges[[1]]$x.labels, getNumericColumns(notNullNumData)) not equal to character(0).
      target is NULL, current is character
      
      ── 2. Failure: format 'boxplot' with facets works (@test-showData.R#21)  ───────
      nrow(p$layout$panel_layout) not equal to length(cols).
      target is NULL, current is numeric
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 496 SKIPPED: 0 FAILED: 2
      1. Failure: format 'boxplot' works (@test-showData.R#12) 
      2. Failure: format 'boxplot' with facets works (@test-showData.R#21) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

# topdownr

Version: 1.2.0

## In both

*   checking whether package ‘topdownr’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘IRanges’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/topdownr/new/topdownr.Rcheck/00install.out’ for details.
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 31-36 (analysis.Rmd) 
    Error: processing vignette 'analysis.Rmd' failed with diagnostics:
    there is no package called 'topdownrdata'
    Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘topdownrdata’
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .BBSoptions
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘topdownrdata’
    ```

# TOSTER

Version: 0.3.4

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘R6’ ‘jmvcore’
      All declared Imports should be used.
    ```

# totalcensus

Version: 0.3.0

## In both

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 42 marked Latin-1 strings
      Note: found 360 marked UTF-8 strings
    ```

# tourrGui

Version: 0.4

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘RGtk2’
    
    Package suggested but not available for checking: ‘rggobi’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# toxboot

Version: 0.1.1

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘rmongodb’
    ```

# toxplot

Version: 0.1.1

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘tidyr’
      All declared Imports should be used.
    ```

# TPMplt

Version: 0.1.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘qpcR’
      All declared Imports should be used.
    ```

# TPP

Version: 3.8.5

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available:
      ‘biobroom’ ‘mefa’ ‘nls2’ ‘sme’ ‘VennDiagram’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# trackeR

Version: 1.2.0

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 13.7Mb
      sub-directories of 1Mb or more:
        data      2.1Mb
        doc       2.5Mb
        extdata   8.4Mb
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    
    The following object is masked from 'package:graphics':
    
        matplot
    
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'trackeR.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.55 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# trackr

Version: 0.10.2

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    Error in re-building vignettes:
      ...
    Loading required package: histry
    Error in get(invar, envir = .GlobalEnv) : object 'score' not found
    Error in get(invar, envir = .GlobalEnv) : object 'score' not found
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'Extending-trackr.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `times.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.60 \usepackage
                    {hyperref}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# TrafficBDE

Version: 0.1.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘neuralnet’
      All declared Imports should be used.
    ```

# transcriptogramer

Version: 1.2.1

## In both

*   R CMD check timed out
    

*   checking installed package size ... NOTE
    ```
      installed size is  6.2Mb
      sub-directories of 1Mb or more:
        data   4.4Mb
        doc    1.2Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    differentiallyExpressed,Transcriptogram: no visible binding for global
      variable ‘x’
    differentiallyExpressed,Transcriptogram: no visible binding for global
      variable ‘y’
    differentiallyExpressed,Transcriptogram : <anonymous>: no visible
      binding for global variable ‘x’
    differentiallyExpressed,Transcriptogram : <anonymous>: no visible
      binding for global variable ‘y’
    Undefined global functions or variables:
      x y
    ```

# transcriptR

Version: 1.8.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    Loading required package: Biobase
    Welcome to Bioconductor
    
        Vignettes contain introductory material; view with
        'browseVignettes()'. To cite Bioconductor, see
        'citation("Biobase")', and for packages 'citation("pkgname")'.
    
    Warning in has_utility("pdfcrop") :
      pdfcrop not installed or not in PATH
    tlmgr search --file --global '/beramono.sty'
    Unknown directive ...l
    Warning in system2("tlmgr", args, ...) :
      running command ''tlmgr' search --file --global '/beramono.sty'' had status 25
    ! LaTeX Error: File `beramono.sty' not found.
    
    ! Emergency stop.
    <read *> 
    
    Error: processing vignette 'transcriptR.Rmd' failed with diagnostics:
    Failed to compile transcriptR.tex. See transcriptR.log for more info.
    Execution halted
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Error in .requirePackage(package) : 
        unable to find required package 'transcriptR'
      Calls: <Anonymous> ... getClass -> getClassDef -> .classEnv -> .requirePackage
      Execution halted
    ```

# trapezoid

Version: 2.0-0

## In both

*   checking R code for possible problems ... NOTE
    ```
    rtrapezoid: no visible global function definition for ‘runif’
    Undefined global functions or variables:
      runif
    Consider adding
      importFrom("stats", "runif")
    to your NAMESPACE file.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    Error in re-building vignettes:
      ...
    Loading required package: trapezoid
    Loading required package: plyr
    Loading required package: ggplot2
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'trapezoid.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `times.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.9 \usepackage
                   {hyperref}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# treeclim

Version: 2.0.0

## In both

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘dplR’
    ```

# treecm

Version: 1.2.2

## In both

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'treecm.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.30 \ifthenelse
                    {\boolean{Sweave@inconsolata}}{%^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# treespace

Version: 1.1.3

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  6.2Mb
      sub-directories of 1Mb or more:
        doc   4.7Mb
    ```

# trialr

Version: 0.0.3

## In both

*   checking whether package ‘trialr’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/trialr/new/trialr.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘trialr’ ...
** package ‘trialr’ successfully unpacked and MD5 sums checked
** libs
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I"`"/Library/Frameworks/R.framework/Resources/bin/Rscript" --vanilla -e "cat(system.file('include', 'src', package = 'StanHeaders'))"`" -DBOOST_RESULT_OF_USE_TR1 -DBOOST_NO_DECLTYPE -DBOOST_DISABLE_ASSERTS -DEIGEN_NO_DEBUG -I"/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/trialr/BH/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/trialr/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c Modules.cpp -o Modules.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I"`"/Library/Frameworks/R.framework/Resources/bin/Rscript" --vanilla -e "cat(system.file('include', 'src', package = 'StanHeaders'))"`" -DBOOST_RESULT_OF_USE_TR1 -DBOOST_NO_DECLTYPE -DBOOST_DISABLE_ASSERTS -DEIGEN_NO_DEBUG -I"/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/trialr/BH/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/trialr/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c RcppExports.cpp -o RcppExports.o
/usr/local/clang4/bin/clang -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I"`"/Library/Frameworks/R.framework/Resources/bin/Rscript" --vanilla -e "cat(system.file('include', 'src', package = 'StanHeaders'))"`" -DBOOST_RESULT_OF_USE_TR1 -DBOOST_NO_DECLTYPE -DBOOST_DISABLE_ASSERTS -DEIGEN_NO_DEBUG -I"/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/trialr/BH/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/trialr/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c trialr-init.c -o trialr-init.o
In file included from RcppExports.cpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/RcppEigen.h:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/RcppEigenForward.h:30:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Dense:1:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Core:531:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from RcppExports.cpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/RcppEigen.h:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/RcppEigenForward.h:30:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Dense:2:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/LU:47:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from RcppExports.cpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/RcppEigen.h:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/RcppEigenForward.h:30:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Dense:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Cholesky:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Jacobi:29:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from RcppExports.cpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/RcppEigen.h:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/RcppEigenForward.h:30:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Dense:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Cholesky:43:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from RcppExports.cpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/RcppEigen.h:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/RcppEigenForward.h:30:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Dense:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/QR:17:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Householder:27:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from RcppExports.cpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/RcppEigen.h:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/RcppEigenForward.h:30:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Dense:5:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/SVD:48:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from RcppExports.cpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/RcppEigen.h:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/RcppEigenForward.h:30:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Dense:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Geometry:58:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from RcppExports.cpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/RcppEigen.h:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/RcppEigenForward.h:30:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Dense:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Eigenvalues:58:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from RcppExports.cpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/RcppEigen.h:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/RcppEigenForward.h:31:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Sparse:26:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/SparseCore:66:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from RcppExports.cpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/RcppEigen.h:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/RcppEigenForward.h:31:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Sparse:27:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/OrderingMethods:71:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from RcppExports.cpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/RcppEigen.h:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/RcppEigenForward.h:31:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Sparse:29:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/SparseCholesky:43:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from RcppExports.cpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/RcppEigen.h:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/RcppEigenForward.h:31:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Sparse:32:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/SparseQR:35:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from RcppExports.cpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/RcppEigen.h:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/RcppEigenForward.h:31:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Sparse:33:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/IterativeLinearSolvers:46:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from RcppExports.cpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/RcppEigen.h:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/RcppEigenForward.h:32:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/CholmodSupport:45:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from RcppExports.cpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/RcppEigen.h:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/RcppEigenForward.h:35:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/unsupported/Eigen/KroneckerProduct:34:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/unsupported/Eigen/../../Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from RcppExports.cpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/RcppEigen.h:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/RcppEigenForward.h:39:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/unsupported/Eigen/Polynomials:135:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/unsupported/Eigen/../../Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from RcppExports.cpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/RcppEigen.h:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/RcppEigenForward.h:40:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/unsupported/Eigen/SparseExtra:51:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/unsupported/Eigen/../../Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:5:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/stan_fit.hpp:20:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/stan_args.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/BH/include/boost/lexical_cast.hpp:32:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/BH/include/boost/lexical_cast/try_lexical_convert.hpp:42:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/BH/include/boost/lexical_cast/detail/converter_lexical.hpp:52:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/BH/include/boost/container/container_fwd.hpp:61:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/BH/include/boost/container/detail/std_fwd.hpp:27:1: warning: inline namespaces are a C++11 feature [-Wc++11-inline-namespace]
BOOST_MOVE_STD_NS_BEG
^
/Users/max/github/ggplot2/revdep/library.noindex/trialr/BH/include/boost/move/detail/std_ns_begin.hpp:18:34: note: expanded from macro 'BOOST_MOVE_STD_NS_BEG'
   #define BOOST_MOVE_STD_NS_BEG _LIBCPP_BEGIN_NAMESPACE_STD
                                 ^
/usr/local/clang4/bin/../include/c++/v1/__config:388:52: note: expanded from macro '_LIBCPP_BEGIN_NAMESPACE_STD'
#define _LIBCPP_BEGIN_NAMESPACE_STD namespace std {inline namespace _LIBCPP_NAMESPACE {
                                                   ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:5:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/core.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/core/autodiffstackstorage.hpp:4:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/memory/stack_alloc.hpp:145:10: warning: 'auto' type specifier is a C++11 extension [-Wc++11-extensions]
    for (auto& block : blocks_)
         ^
/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/memory/stack_alloc.hpp:145:22: warning: range-based for loop is a C++11 extension [-Wc++11-extensions]
    for (auto& block : blocks_)
                     ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:5:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/core.hpp:4:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/core/autodiffstackstorage.hpp:47:69: warning: deleted function definitions are a C++11 extension [-Wc++11-extensions]
    AutodiffStackStorage &operator=(const AutodiffStackStorage &) = delete;
                                                                    ^
/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/core/autodiffstackstorage.hpp:60:30: warning: deleted function definitions are a C++11 extension [-Wc++11-extensions]
  AutodiffStackSingleton() = delete;
                             ^
/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/core/autodiffstackstorage.hpp:61:71: warning: deleted function definitions are a C++11 extension [-Wc++11-extensions]
  explicit AutodiffStackSingleton(AutodiffStackSingleton_t const &) = delete;
                                                                      ^
/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/core/autodiffstackstorage.hpp:62:73: warning: deleted function definitions are a C++11 extension [-Wc++11-extensions]
  AutodiffStackSingleton &operator=(const AutodiffStackSingleton_t &) = delete;
                                                                        ^
17 warnings generated.
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:5:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Dense:1:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Core:531:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:5:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Dense:2:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/LU:47:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:5:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Dense:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Cholesky:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Jacobi:29:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:5:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Dense:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Cholesky:43:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:5:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Dense:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/QR:17:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Householder:27:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:5:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Dense:5:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/SVD:48:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:5:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Dense:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Geometry:58:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:5:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Dense:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Eigenvalues:58:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:5:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/core.hpp:42:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/core/recover_memory.hpp:25:8: warning: 'auto' type specifier is a C++11 extension [-Wc++11-extensions]
  for (auto &x : ChainableStack::instance().var_alloc_stack_) {
       ^
/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/core/recover_memory.hpp:25:16: warning: range-based for loop is a C++11 extension [-Wc++11-extensions]
  for (auto &x : ChainableStack::instance().var_alloc_stack_) {
               ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:5:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/core.hpp:44:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/core/set_zero_all_adjoints.hpp:15:8: warning: 'auto' type specifier is a C++11 extension [-Wc++11-extensions]
  for (auto &x : ChainableStack::instance().var_stack_)
       ^
/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/core/set_zero_all_adjoints.hpp:15:16: warning: range-based for loop is a C++11 extension [-Wc++11-extensions]
  for (auto &x : ChainableStack::instance().var_stack_)
               ^
/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/core/set_zero_all_adjoints.hpp:17:8: warning: 'auto' type specifier is a C++11 extension [-Wc++11-extensions]
  for (auto &x : ChainableStack::instance().var_nochain_stack_)
       ^
/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/core/set_zero_all_adjoints.hpp:17:16: warning: range-based for loop is a C++11 extension [-Wc++11-extensions]
  for (auto &x : ChainableStack::instance().var_nochain_stack_)
               ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:5:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat.hpp:22:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:19:59: error: expected expression
class ops_partials_edge<ViewElt, Eigen::Matrix<Op, R, C>> {
                                                          ^
/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:35:2: error: expected a type
};
 ^
/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:38:70: error: expected expression
class ops_partials_edge<ViewElt, std::vector<Eigen::Matrix<Op, R, C>>> {
                                                                     ^
/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:38:72: error: expected a type
class ops_partials_edge<ViewElt, std::vector<Eigen::Matrix<Op, R, C>>> {
                                                                       ^
/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:53:2: error: expected a type
};
 ^
/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:56:60: error: a space is required between consecutive right angle brackets (use '> >')
class ops_partials_edge<ViewElt, std::vector<std::vector<Op>>> {
                                                           ^~
                                                           > > 
/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:58:68: error: a space is required between consecutive right angle brackets (use '> >')
  typedef empty_broadcast_array<ViewElt, std::vector<std::vector<Op>>>
                                                                   ^~
                                                                   > > 
/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:61:63: error: a space is required between consecutive right angle brackets (use '> >')
  empty_broadcast_array<partials_t, std::vector<std::vector<Op>>> partials_vec_;
                                                              ^~
                                                              > > 
/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:63:62: error: a space is required between consecutive right angle brackets (use '> >')
  explicit ops_partials_edge(const std::vector<std::vector<Op>> ops) {}
                                                             ^~
                                                             > >
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:5:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat.hpp:96:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/fun/csr_extract_u.hpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Sparse:26:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/SparseCore:66:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:5:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat.hpp:96:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/fun/csr_extract_u.hpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Sparse:27:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/OrderingMethods:71:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:5:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat.hpp:96:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/fun/csr_extract_u.hpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Sparse:29:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/SparseCholesky:43:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:5:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat.hpp:96:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/fun/csr_extract_u.hpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Sparse:32:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/SparseQR:35:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:5:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat.hpp:96:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/fun/csr_extract_u.hpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Sparse:33:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/IterativeLinearSolvers:46:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:5:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat.hpp:99:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/fun/csr_matrix_times_vector.hpp:91:14: warning: range-based for loop is a C++11 extension [-Wc++11-extensions]
  for (int i : v)
             ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:5:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat.hpp:100:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/fun/csr_to_dense_matrix.hpp:48:14: warning: range-based for loop is a C++11 extension [-Wc++11-extensions]
  for (int i : v)
             ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:5:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat.hpp:156:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/fun/log_mix.hpp:178:26: error: expected expression
              .unaryExpr([](T_partials_return x) { return exp(x); });
                         ^
/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/fun/log_mix.hpp:262:24: error: expected expression
            .unaryExpr([](T_partials_return x) { return exp(x); });
                       ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:5:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat.hpp:161:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp.hpp:5:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_2x2.hpp:21:13: warning: alias declarations are a C++11 extension [-Wc++11-extensions]
  using T = typename Mtype::Scalar;
            ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:5:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat.hpp:162:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_multiply.hpp:5:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_action_handler.hpp:20:10: error: unknown type name 'constexpr'
  static constexpr int p_max = 8;
         ^
/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_action_handler.hpp:20:20: error: expected member name or ';' after declaration specifiers
  static constexpr int p_max = 8;
  ~~~~~~~~~~~~~~~~ ^
/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_action_handler.hpp:21:10: error: unknown type name 'constexpr'
  static constexpr int m_max = 55;
         ^
/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_action_handler.hpp:21:20: error: expected member name or ';' after declaration specifiers
  static constexpr int m_max = 55;
  ~~~~~~~~~~~~~~~~ ^
/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_action_handler.hpp:22:10: error: unknown type name 'constexpr'
  static constexpr double tol = 1.1e-16;
         ^
/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_action_handler.hpp:22:20: error: expected member name or ';' after declaration specifiers
  static constexpr double tol = 1.1e-16;
  ~~~~~~~~~~~~~~~~ ^
/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_action_handler.hpp:25:29: error: function definition does not declare parameters
  const std::vector<double> theta_m_single_precision{
                            ^
/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_action_handler.hpp:28:29: error: function definition does not declare parameters
  const std::vector<double> theta_m_double_precision{
                            ^
fatal error: too many errors emitted, stopping now [-ferror-limit=]
29 warnings and 20 errors generated.
make: *** [Modules.o] Error 1
ERROR: compilation failed for package ‘trialr’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/trialr/new/trialr.Rcheck/trialr’

```
### CRAN

```
* installing *source* package ‘trialr’ ...
** package ‘trialr’ successfully unpacked and MD5 sums checked
** libs
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I"`"/Library/Frameworks/R.framework/Resources/bin/Rscript" --vanilla -e "cat(system.file('include', 'src', package = 'StanHeaders'))"`" -DBOOST_RESULT_OF_USE_TR1 -DBOOST_NO_DECLTYPE -DBOOST_DISABLE_ASSERTS -DEIGEN_NO_DEBUG -I"/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/trialr/BH/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/trialr/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c Modules.cpp -o Modules.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I"`"/Library/Frameworks/R.framework/Resources/bin/Rscript" --vanilla -e "cat(system.file('include', 'src', package = 'StanHeaders'))"`" -DBOOST_RESULT_OF_USE_TR1 -DBOOST_NO_DECLTYPE -DBOOST_DISABLE_ASSERTS -DEIGEN_NO_DEBUG -I"/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/trialr/BH/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/trialr/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c RcppExports.cpp -o RcppExports.o
/usr/local/clang4/bin/clang -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I"`"/Library/Frameworks/R.framework/Resources/bin/Rscript" --vanilla -e "cat(system.file('include', 'src', package = 'StanHeaders'))"`" -DBOOST_RESULT_OF_USE_TR1 -DBOOST_NO_DECLTYPE -DBOOST_DISABLE_ASSERTS -DEIGEN_NO_DEBUG -I"/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/trialr/BH/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/trialr/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c trialr-init.c -o trialr-init.o
In file included from RcppExports.cpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/RcppEigen.h:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/RcppEigenForward.h:30:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Dense:1:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Core:531:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from RcppExports.cpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/RcppEigen.h:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/RcppEigenForward.h:30:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Dense:2:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/LU:47:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from RcppExports.cpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/RcppEigen.h:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/RcppEigenForward.h:30:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Dense:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Cholesky:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Jacobi:29:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from RcppExports.cpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/RcppEigen.h:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/RcppEigenForward.h:30:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Dense:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Cholesky:43:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from RcppExports.cpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/RcppEigen.h:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/RcppEigenForward.h:30:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Dense:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/QR:17:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Householder:27:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from RcppExports.cpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/RcppEigen.h:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/RcppEigenForward.h:30:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Dense:5:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/SVD:48:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from RcppExports.cpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/RcppEigen.h:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/RcppEigenForward.h:30:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Dense:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Geometry:58:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from RcppExports.cpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/RcppEigen.h:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/RcppEigenForward.h:30:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Dense:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Eigenvalues:58:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from RcppExports.cpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/RcppEigen.h:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/RcppEigenForward.h:31:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Sparse:26:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/SparseCore:66:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from RcppExports.cpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/RcppEigen.h:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/RcppEigenForward.h:31:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Sparse:27:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/OrderingMethods:71:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from RcppExports.cpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/RcppEigen.h:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/RcppEigenForward.h:31:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Sparse:29:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/SparseCholesky:43:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from RcppExports.cpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/RcppEigen.h:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/RcppEigenForward.h:31:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Sparse:32:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/SparseQR:35:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from RcppExports.cpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/RcppEigen.h:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/RcppEigenForward.h:31:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Sparse:33:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/IterativeLinearSolvers:46:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from RcppExports.cpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/RcppEigen.h:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/RcppEigenForward.h:32:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/CholmodSupport:45:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from RcppExports.cpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/RcppEigen.h:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/RcppEigenForward.h:35:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/unsupported/Eigen/KroneckerProduct:34:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/unsupported/Eigen/../../Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from RcppExports.cpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/RcppEigen.h:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/RcppEigenForward.h:39:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/unsupported/Eigen/Polynomials:135:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/unsupported/Eigen/../../Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from RcppExports.cpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/RcppEigen.h:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/RcppEigenForward.h:40:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/unsupported/Eigen/SparseExtra:51:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/unsupported/Eigen/../../Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:5:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/stan_fit.hpp:20:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/stan_args.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/BH/include/boost/lexical_cast.hpp:32:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/BH/include/boost/lexical_cast/try_lexical_convert.hpp:42:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/BH/include/boost/lexical_cast/detail/converter_lexical.hpp:52:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/BH/include/boost/container/container_fwd.hpp:61:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/BH/include/boost/container/detail/std_fwd.hpp:27:1: warning: inline namespaces are a C++11 feature [-Wc++11-inline-namespace]
BOOST_MOVE_STD_NS_BEG
^
/Users/max/github/ggplot2/revdep/library.noindex/trialr/BH/include/boost/move/detail/std_ns_begin.hpp:18:34: note: expanded from macro 'BOOST_MOVE_STD_NS_BEG'
   #define BOOST_MOVE_STD_NS_BEG _LIBCPP_BEGIN_NAMESPACE_STD
                                 ^
/usr/local/clang4/bin/../include/c++/v1/__config:388:52: note: expanded from macro '_LIBCPP_BEGIN_NAMESPACE_STD'
#define _LIBCPP_BEGIN_NAMESPACE_STD namespace std {inline namespace _LIBCPP_NAMESPACE {
                                                   ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:5:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/core.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/core/autodiffstackstorage.hpp:4:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/memory/stack_alloc.hpp:145:10: warning: 'auto' type specifier is a C++11 extension [-Wc++11-extensions]
    for (auto& block : blocks_)
         ^
/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/memory/stack_alloc.hpp:145:22: warning: range-based for loop is a C++11 extension [-Wc++11-extensions]
    for (auto& block : blocks_)
                     ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:5:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/core.hpp:4:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/core/autodiffstackstorage.hpp:47:69: warning: deleted function definitions are a C++11 extension [-Wc++11-extensions]
    AutodiffStackStorage &operator=(const AutodiffStackStorage &) = delete;
                                                                    ^
/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/core/autodiffstackstorage.hpp:60:30: warning: deleted function definitions are a C++11 extension [-Wc++11-extensions]
  AutodiffStackSingleton() = delete;
                             ^
/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/core/autodiffstackstorage.hpp:61:71: warning: deleted function definitions are a C++11 extension [-Wc++11-extensions]
  explicit AutodiffStackSingleton(AutodiffStackSingleton_t const &) = delete;
                                                                      ^
/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/core/autodiffstackstorage.hpp:62:73: warning: deleted function definitions are a C++11 extension [-Wc++11-extensions]
  AutodiffStackSingleton &operator=(const AutodiffStackSingleton_t &) = delete;
                                                                        ^
17 warnings generated.
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:5:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Dense:1:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Core:531:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:5:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Dense:2:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/LU:47:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:5:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Dense:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Cholesky:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Jacobi:29:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:5:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Dense:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Cholesky:43:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:5:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Dense:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/QR:17:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Householder:27:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:5:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Dense:5:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/SVD:48:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:5:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Dense:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Geometry:58:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:5:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/core.hpp:14:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/core/matrix_vari.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/mat/fun/Eigen_NumTraits.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/fun/Eigen.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Dense:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Eigenvalues:58:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:5:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/core.hpp:42:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/core/recover_memory.hpp:25:8: warning: 'auto' type specifier is a C++11 extension [-Wc++11-extensions]
  for (auto &x : ChainableStack::instance().var_alloc_stack_) {
       ^
/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/core/recover_memory.hpp:25:16: warning: range-based for loop is a C++11 extension [-Wc++11-extensions]
  for (auto &x : ChainableStack::instance().var_alloc_stack_) {
               ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:5:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/mat.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/core.hpp:44:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/core/set_zero_all_adjoints.hpp:15:8: warning: 'auto' type specifier is a C++11 extension [-Wc++11-extensions]
  for (auto &x : ChainableStack::instance().var_stack_)
       ^
/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/core/set_zero_all_adjoints.hpp:15:16: warning: range-based for loop is a C++11 extension [-Wc++11-extensions]
  for (auto &x : ChainableStack::instance().var_stack_)
               ^
/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/core/set_zero_all_adjoints.hpp:17:8: warning: 'auto' type specifier is a C++11 extension [-Wc++11-extensions]
  for (auto &x : ChainableStack::instance().var_nochain_stack_)
       ^
/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/core/set_zero_all_adjoints.hpp:17:16: warning: range-based for loop is a C++11 extension [-Wc++11-extensions]
  for (auto &x : ChainableStack::instance().var_nochain_stack_)
               ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:5:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat.hpp:22:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:19:59: error: expected expression
class ops_partials_edge<ViewElt, Eigen::Matrix<Op, R, C>> {
                                                          ^
/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:35:2: error: expected a type
};
 ^
/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:38:70: error: expected expression
class ops_partials_edge<ViewElt, std::vector<Eigen::Matrix<Op, R, C>>> {
                                                                     ^
/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:38:72: error: expected a type
class ops_partials_edge<ViewElt, std::vector<Eigen::Matrix<Op, R, C>>> {
                                                                       ^
/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:53:2: error: expected a type
};
 ^
/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:56:60: error: a space is required between consecutive right angle brackets (use '> >')
class ops_partials_edge<ViewElt, std::vector<std::vector<Op>>> {
                                                           ^~
                                                           > > 
/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:58:68: error: a space is required between consecutive right angle brackets (use '> >')
  typedef empty_broadcast_array<ViewElt, std::vector<std::vector<Op>>>
                                                                   ^~
                                                                   > > 
/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:61:63: error: a space is required between consecutive right angle brackets (use '> >')
  empty_broadcast_array<partials_t, std::vector<std::vector<Op>>> partials_vec_;
                                                              ^~
                                                              > > 
/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/meta/operands_and_partials.hpp:63:62: error: a space is required between consecutive right angle brackets (use '> >')
  explicit ops_partials_edge(const std::vector<std::vector<Op>> ops) {}
                                                             ^~
                                                             > >
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:5:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat.hpp:96:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/fun/csr_extract_u.hpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Sparse:26:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/SparseCore:66:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:5:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat.hpp:96:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/fun/csr_extract_u.hpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Sparse:27:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/OrderingMethods:71:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:5:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat.hpp:96:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/fun/csr_extract_u.hpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Sparse:29:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/SparseCholesky:43:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:5:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat.hpp:96:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/fun/csr_extract_u.hpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Sparse:32:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/SparseQR:35:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:5:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat.hpp:96:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/fun/csr_extract_u.hpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/Sparse:33:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/IterativeLinearSolvers:46:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/RcppEigen/include/Eigen/src/Core/util/ReenableStupidWarnings.h:10:30: warning: pragma diagnostic pop could not pop, no matching push [-Wunknown-pragmas]
    #pragma clang diagnostic pop
                             ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:5:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat.hpp:99:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/fun/csr_matrix_times_vector.hpp:91:14: warning: range-based for loop is a C++11 extension [-Wc++11-extensions]
  for (int i : v)
             ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:5:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat.hpp:100:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/fun/csr_to_dense_matrix.hpp:48:14: warning: range-based for loop is a C++11 extension [-Wc++11-extensions]
  for (int i : v)
             ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:5:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat.hpp:156:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/fun/log_mix.hpp:178:26: error: expected expression
              .unaryExpr([](T_partials_return x) { return exp(x); });
                         ^
/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/fun/log_mix.hpp:262:24: error: expected expression
            .unaryExpr([](T_partials_return x) { return exp(x); });
                       ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:5:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat.hpp:161:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp.hpp:5:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_2x2.hpp:21:13: warning: alias declarations are a C++11 extension [-Wc++11-extensions]
  using T = typename Mtype::Scalar;
            ^
In file included from Modules.cpp:3:
In file included from ./include/models.hpp:5:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/rstaninc.hpp:3:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/rstan/include/rstan/stan_fit.hpp:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/services/diagnose/diagnose.hpp:10:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/test_gradients.hpp:7:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/src/stan/model/log_prob_grad.hpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/rev/mat.hpp:12:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat.hpp:162:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_multiply.hpp:5:
/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_action_handler.hpp:20:10: error: unknown type name 'constexpr'
  static constexpr int p_max = 8;
         ^
/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_action_handler.hpp:20:20: error: expected member name or ';' after declaration specifiers
  static constexpr int p_max = 8;
  ~~~~~~~~~~~~~~~~ ^
/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_action_handler.hpp:21:10: error: unknown type name 'constexpr'
  static constexpr int m_max = 55;
         ^
/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_action_handler.hpp:21:20: error: expected member name or ';' after declaration specifiers
  static constexpr int m_max = 55;
  ~~~~~~~~~~~~~~~~ ^
/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_action_handler.hpp:22:10: error: unknown type name 'constexpr'
  static constexpr double tol = 1.1e-16;
         ^
/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_action_handler.hpp:22:20: error: expected member name or ';' after declaration specifiers
  static constexpr double tol = 1.1e-16;
  ~~~~~~~~~~~~~~~~ ^
/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_action_handler.hpp:25:29: error: function definition does not declare parameters
  const std::vector<double> theta_m_single_precision{
                            ^
/Users/max/github/ggplot2/revdep/library.noindex/trialr/StanHeaders/include/stan/math/prim/mat/fun/matrix_exp_action_handler.hpp:28:29: error: function definition does not declare parameters
  const std::vector<double> theta_m_double_precision{
                            ^
fatal error: too many errors emitted, stopping now [-ferror-limit=]
29 warnings and 20 errors generated.
make: *** [Modules.o] Error 1
ERROR: compilation failed for package ‘trialr’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/trialr/old/trialr.Rcheck/trialr’

```
# tricolore

Version: 1.2.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 64-75 (choropleth_maps_with_tricolore.Rmd) 
    Error: processing vignette 'choropleth_maps_with_tricolore.Rmd' failed with diagnostics:
    there is no package called 'sf'
    Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘sf’
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘dplyr’
      All declared Imports should be used.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 88 marked UTF-8 strings
    ```

# tRNAscanImport

Version: 1.0.1

## In both

*   checking whether package ‘tRNAscanImport’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/tRNAscanImport/new/tRNAscanImport.Rcheck/00install.out’ for details.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Package listed in more than one of Depends, Imports, Suggests, Enhances:
      ‘GenomicRanges’
    A package should be listed in only one of these fields.
    ```

# tRophicPosition

Version: 0.7.5

## In both

*   checking examples ... ERROR
    ```
    ...
    Running examples in ‘tRophicPosition-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: TPmodel
    > ### Title: Function to create a JAGS-based Bayesian model to calculate
    > ###   trophic position
    > ### Aliases: TPmodel
    > 
    > ### ** Examples
    > 
    > isotopeData <- generateTPData()
    > model.string <- jagsBayesianModel()
    > model <- TPmodel(data = isotopeData, model.string = model.string,
    + n.adapt = 500)
    Error: .onLoad failed in loadNamespace() for 'rjags', details:
      call: dyn.load(file, DLLpath = DLLpath, ...)
      error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/tRophicPosition/rjags/libs/rjags.so':
      dlopen(/Users/max/github/ggplot2/revdep/library.noindex/tRophicPosition/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
      Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/tRophicPosition/rjags/libs/rjags.so
      Reason: image not found
    Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 108-110 (Multiple_model_calculation_of_trophic_position_in_R.Rmd) 
    Error: processing vignette 'Multiple_model_calculation_of_trophic_position_in_R.Rmd' failed with diagnostics:
    .onLoad failed in loadNamespace() for 'rjags', details:
      call: dyn.load(file, DLLpath = DLLpath, ...)
      error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/tRophicPosition/rjags/libs/rjags.so':
      dlopen(/Users/max/github/ggplot2/revdep/library.noindex/tRophicPosition/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
      Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/tRophicPosition/rjags/libs/rjags.so
      Reason: image not found
    Execution halted
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘SIBER’
    ```

# tropr

Version: 0.1.2

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      Failed to find tvtrope content
      1: as.data.frame(trope_content(.url)) at testthat/test-tropr.R:26
      2: trope_content(.url)
      3: stop("Failed to find tvtrope content")
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 3 SKIPPED: 0 FAILED: 5
      1. Error: trope_data (@test-data.R#6) 
      2. Error: trope_cache (@test-data.R#19) 
      3. Error: trope_content, as.data.frame (@test-tropr.R#5) 
      4. Error: trope_history, aggr (@test-tropr.R#14) 
      5. Error: More cases (@test-tropr.R#26) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 41-47 (quickstart.Rmd) 
    Error: processing vignette 'quickstart.Rmd' failed with diagnostics:
    Failed to find tvtrope content
    Execution halted
    ```

# TSCAN

Version: 1.18.0

## In both

*   checking R code for possible problems ... NOTE
    ```
    ...
      ‘lm’
    exprmclust: no visible global function definition for ‘dist’
    plotmclust: no visible binding for global variable ‘pca_dim_1’
    plotmclust: no visible binding for global variable ‘pca_dim_2’
    plotmclust: no visible binding for global variable ‘sample_name’
    preprocess: no visible binding for global variable ‘sd’
    preprocess: no visible global function definition for ‘hclust’
    preprocess: no visible global function definition for ‘dist’
    preprocess: no visible global function definition for ‘cutree’
    preprocess: no visible global function definition for ‘aggregate’
    singlegeneplot: no visible global function definition for
      ‘fitted.values’
    singlegeneplot: no visible binding for global variable ‘predict’
    Undefined global functions or variables:
      aggregate cutree dist fitted.values hclust lm p.adjust pca_dim_1
      pca_dim_2 pchisq prcomp predict sample_name sd
    Consider adding
      importFrom("stats", "aggregate", "cutree", "dist", "fitted.values",
                 "hclust", "lm", "p.adjust", "pchisq", "prcomp", "predict",
                 "sd")
    to your NAMESPACE file.
    ```

# TSMining

Version: 1.0

## In both

*   checking R code for possible problems ... NOTE
    ```
    Func.SAX: no visible global function definition for ‘sd’
    Func.SAX: no visible global function definition for ‘qnorm’
    Func.matrix: no visible global function definition for ‘qnorm’
    Func.motif: no visible global function definition for ‘sd’
    Func.motif: no visible global function definition for ‘na.omit’
    Func.motif: no visible global function definition for ‘combn’
    Func.motif.multivariate: no visible global function definition for
      ‘txtProgressBar’
    Func.motif.multivariate: no visible global function definition for
      ‘setTxtProgressBar’
    Undefined global functions or variables:
      combn na.omit qnorm sd setTxtProgressBar txtProgressBar
    Consider adding
      importFrom("stats", "na.omit", "qnorm", "sd")
      importFrom("utils", "combn", "setTxtProgressBar", "txtProgressBar")
    to your NAMESPACE file.
    ```

# tspmeta

Version: 1.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    ...
    LaTeX errors found:
    ! Please use \mathaccent for accents in math mode.
    \add@accent ...@spacefactor \spacefactor }\accent 
                                                      #1 #2\egroup \spacefactor ...
    l.804 ...asciicircum{}2}{}.}{rescale.Rul.instance}
                                                      
    ! Missing { inserted.
    <to be read again> 
                       \egroup 
    l.804 ...asciicircum{}2}{}.}{rescale.Rul.instance}
                                                      
    ! You can't use `\spacefactor' in math mode.
    \add@accent ...}\accent #1 #2\egroup \spacefactor 
                                                      \accent@spacefactor 
    l.804 ...asciicircum{}2}{}.}{rescale.Rul.instance}
                                                      
    ! Missing } inserted.
    <inserted text> 
                    }
    l.804 ...asciicircum{}2}{}.}{rescale.Rul.instance}
                                                      
    ```

# TSRchitect

Version: 1.8.9

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    > ### ** Examples
    > 
    > extdata.dir <- system.file("extdata", package="TSRchitect")
    > test.Obj <- loadTSSobj(experimentTitle="Code example", inputDir=extdata.dir,
    + isPairedBAM=TRUE, sampleNames=c("sample1-rep1", "sample1-rep2",
    + "sample2-rep1","sample2-rep2"), replicateIDs=c(1,1,2,2))
    ... loadTSSobj ...
    
    Importing paired-end reads ...
    
    
    TSS data were specified to be paired-end read alignments.
    
    Beginning import of 4 bam files ...
    
    Warning in socketConnection(host, port, TRUE, TRUE, "a+b", timeout = timeout) :
      port 11738 cannot be opened
    Error in socketConnection(host, port, TRUE, TRUE, "a+b", timeout = timeout) : 
      cannot open the connection
    Calls: loadTSSobj ... .local -> .bpfork -> .bpforkConnect -> socketConnection
    Execution halted
    ```

## In both

*   checking index information ... WARNING
    ```
    Empty or missing file ‘demo/00Index’.
    See sections ‘The INDEX file’ and ‘Package subdirectories’ in the
    ‘Writing R Extensions’ manual.
    ```

*   checking package subdirectories ... WARNING
    ```
    Subdirectory ‘demo’ contains no demos.
    ```

*   checking top-level files ... NOTE
    ```
    File
      inst/LICENSE
    will install at top-level and is not mentioned in the DESCRIPTION file.
    ```

# TSS.RESTREND

Version: 0.2.13

## In both

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘gimms’
    ```

# tufterhandout

Version: 1.2.1

## In both

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Description field: should contain one or more complete sentences.
    ```

# TVTB

Version: 1.6.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    - AAF
    - MAF
    Overwriting INFO keys in header:
    - REF
    - HET
    - ALT
    - AAF
    - MAF
    convert: profile 'icc': 'RGB ': RGB color space not permitted on grayscale PNG `Introduction_files/figure-html/pairsInfo-1.png' @ warning/png.c/MagickPNGWarningHandler/1744.
    tlmgr search --file --global '/beramono.sty'
    Unknown directive ...l
    Warning in system2("tlmgr", args, ...) :
      running command ''tlmgr' search --file --global '/beramono.sty'' had status 25
    ! LaTeX Error: File `beramono.sty' not found.
    
    ! Emergency stop.
    <read *> 
    
    Error: processing vignette 'tSVE.Rmd' failed with diagnostics:
    Failed to compile tSVE.tex. See tSVE.log for more info.
    Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.2Mb
      sub-directories of 1Mb or more:
        R     2.0Mb
        doc   2.2Mb
    ```

# tweet2r

Version: 1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    ...
    LaTeX errors found:
    ! LaTeX Error: Command \guillemotleft unavailable in encoding OT1.
    
    See the LaTeX manual or LaTeX Companion for explanation.
    Type  H <return>  for immediate help.
     ...                                              
    ! LaTeX Error: Command \guillemotright unavailable in encoding OT1.
    
    See the LaTeX manual or LaTeX Companion for explanation.
    Type  H <return>  for immediate help.
     ...                                              
    ! LaTeX Error: Command \guillemotleft unavailable in encoding OT1.
    
    See the LaTeX manual or LaTeX Companion for explanation.
    Type  H <return>  for immediate help.
     ...                                              
    ! LaTeX Error: Command \guillemotright unavailable in encoding OT1.
    
    See the LaTeX manual or LaTeX Companion for explanation.
    Type  H <return>  for immediate help.
     ...                                              
    ```

# twoddpcr

Version: 1.4.1

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.9Mb
      sub-directories of 1Mb or more:
        data   1.2Mb
        doc    4.1Mb
    ```

# ukbtools

Version: 0.11.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘plyr’
      All declared Imports should be used.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 5 marked UTF-8 strings
    ```

# umx

Version: 2.8.5

## In both

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘sem’
    ```

# unvotes

Version: 0.2.0

## In both

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 4494 marked UTF-8 strings
    ```

# updog

Version: 1.0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    ...
     ...                                              
    ! LaTeX Error: Something's wrong--perhaps a missing \item.
    
    See the LaTeX manual or LaTeX Companion for explanation.
    Type  H <return>  for immediate help.
     ...                                              
    ! LaTeX Error: Something's wrong--perhaps a missing \item.
    
    See the LaTeX manual or LaTeX Companion for explanation.
    Type  H <return>  for immediate help.
     ...                                              
    ! Missing $ inserted.
    <inserted text> 
                    $
    l.767 ... with respect to \eqn{h}{}.}{dpen.Rul.dh}
                                                      
    ! Missing } inserted.
    <inserted text> 
                    }
    l.767 ... with respect to \eqn{h}{}.}{dpen.Rul.dh}
                                                      
    ```

# UpSetR

Version: 1.3.3

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  8.4Mb
      sub-directories of 1Mb or more:
        doc   7.9Mb
    ```

# usmap

Version: 0.4.0

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  7.0Mb
      sub-directories of 1Mb or more:
        extdata   6.2Mb
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 1 marked UTF-8 string
    ```

# uSORT

Version: 1.6.0

## In both

*   checking whether package ‘uSORT’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Note: break used in wrong context: no loop is visible 
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/uSORT/new/uSORT.Rcheck/00install.out’ for details.
    Information on the location(s) of code generating the ‘Note’s can be
    obtained by re-running with environment variable R_KEEP_PKG_SOURCE set
    to ‘yes’.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.3Mb
      sub-directories of 1Mb or more:
        doc       2.0Mb
        extdata   3.1Mb
    ```

# variancePartition

Version: 1.10.4

## In both

*   R CMD check timed out
    

*   checking whether package ‘variancePartition’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘limma’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/variancePartition/new/variancePartition.Rcheck/00install.out’ for details.
    ```

# VariantAnnotation

Version: 1.26.1

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘VariantAnnotation-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: PolyPhenDb-class
    > ### Title: PolyPhenDb objects
    > ### Aliases: PolyPhen PolyPhenDb class:PolyPhenDb PolyPhenDb-class
    > ###   duplicateRSID metadata,PolyPhenDb-method columns,PolyPhenDb-method
    > ###   keys,PolyPhenDb-method select,PolyPhenDb-method
    > ### Keywords: classes methods
    > 
    > ### ** Examples
    > 
    > library(PolyPhen.Hsapiens.dbSNP131)
    Error in library(PolyPhen.Hsapiens.dbSNP131) : 
      there is no package called ‘PolyPhen.Hsapiens.dbSNP131’
    Execution halted
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/VariantAnnotation_unit_tests.R’ failed.
    Last 13 lines of output:
      6: In valid.GenomicRanges.seqinfo(x, suggest.trim = TRUE) :
        GRanges object contains 7 out-of-bound ranges located on sequence
        70477. Note that ranges located on a sequence whose length is unknown
        (NA) or on a circular sequence are not considered out-of-bound (use
        seqlengths() and isCircular() to get the lengths and circularity flags
        of the underlying sequences). You can use trim() to trim these ranges.
        See ?`trim,GenomicRanges-method` for more information.
      7: In valid.GenomicRanges.seqinfo(x, suggest.trim = TRUE) :
        GRanges object contains 6 out-of-bound ranges located on sequence
        70477. Note that ranges located on a sequence whose length is unknown
        (NA) or on a circular sequence are not considered out-of-bound (use
        seqlengths() and isCircular() to get the lengths and circularity flags
        of the underlying sequences). You can use trim() to trim these ranges.
        See ?`trim,GenomicRanges-method` for more information.
      Execution halted
    ```

*   checking whether package ‘VariantAnnotation’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘DelayedArray’ was built under R version 3.5.1
      Warning: package ‘Rsamtools’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/VariantAnnotation/new/VariantAnnotation.Rcheck/00install.out’ for details.
    ```

*   checking running R code from vignettes ...
    ```
       ‘VariantAnnotation.Rnw’ ... failed
       ‘filterVcf.Rnw’ ... OK
     WARNING
    Errors in running code in vignettes:
    when running code in ‘VariantAnnotation.Rnw’
      ...
    > names(nonsyn) <- nms[idx]
    
    > rsids <- unique(names(nonsyn)[grep("rs", names(nonsyn), 
    +     fixed = TRUE)])
    
    > library(PolyPhen.Hsapiens.dbSNP131)
    
      When sourcing ‘VariantAnnotation.R’:
    Error: there is no package called ‘PolyPhen.Hsapiens.dbSNP131’
    Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking:
      ‘SIFT.Hsapiens.dbSNP132’ ‘SIFT.Hsapiens.dbSNP137’
      ‘PolyPhen.Hsapiens.dbSNP131’
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' call to ‘snpStats’ in package code.
      Please use :: or requireNamespace() instead.
      See section 'Suggested packages' in the 'Writing R Extensions' manual.
    Unexported objects imported by ':::' calls:
      ‘BiocGenerics:::replaceSlots’ ‘BiocGenerics:::testPackage’
      ‘Rsamtools:::.RsamtoolsFile’ ‘Rsamtools:::.RsamtoolsFileList’
      ‘Rsamtools:::.io_check_exists’ ‘S4Vectors:::expandByColumnSet’
      ‘S4Vectors:::labeledLine’ ‘S4Vectors:::recycleVector’
      ‘S4Vectors:::selectSome’
      ‘SummarizedExperiment:::.SummarizedExperiment.charbound’
      ‘SummarizedExperiment:::.cbind.DataFrame’
      See the note in ?`:::` about the use of this operator.
    ```

*   checking R code for possible problems ... NOTE
    ```
    .predictCodingGRangesList: no visible binding for global variable
      ‘GENETIC_CODE’
    VRangesForMatching: no visible binding for global variable ‘REF’
    VRangesForMatching: no visible binding for global variable ‘ALT’
    probabilityToSnpMatrix: no visible global function definition for
      ‘post2g’
    import,VcfFile-ANY-ANY: no visible global function definition for
      ‘checkArgFormat’
    Undefined global functions or variables:
      ALT GENETIC_CODE REF checkArgFormat post2g
    ```

*   checking compiled code ... NOTE
    ```
    File ‘VariantAnnotation/libs/VariantAnnotation.so’:
      Found non-API calls to R: ‘R_GetConnection’, ‘R_WriteConnection’
    
    Compiled code should not call non-API entry points in R.
    
    See ‘Writing portable packages’ in the ‘Writing R Extensions’ manual.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    Warning in valid.GenomicRanges.seqinfo(x, suggest.trim = TRUE) :
      GRanges object contains 2405 out-of-bound ranges located on
      sequences 75253, 74357, 74359, 74360, 74361, 74362, 74363,
      74358, 74364, 74365, 75254, 75259, 74368, 74369, 74366, 74367,
      74370, 74372, 74373, 74374, 74375, 74378, 74377, 74380, 74381,
      75262, 75263, 75265, 75266, 75268, 75269, 75271, 75273, 75276,
      75281, 75282, 75283, 74389, 74383, 74384, 74385, 74386, 74387,
      75287, 75288, 75286, 75289, 74390, 74391, 74392, 74393, 74394,
      75291, 74395, 74396, 74397, 74398, 75302, 75304, 75305, and
      75306. Note that ranges located on a sequence whose length is
      unknown (NA) or on a circular sequence are not considered
      out-of-bound (use seqlengths() and isCircular() to get the
      lengths and circularity flags of the underlying sequences). You
      can use trim() to trim these ranges. See
      ?`trim,GenomicRanges-method` for more information.
    
    Error: processing vignette 'VariantAnnotation.Rnw' failed with diagnostics:
     chunk 32 (label = polyphen) 
    Error in library(PolyPhen.Hsapiens.dbSNP131) : 
      there is no package called ‘PolyPhen.Hsapiens.dbSNP131’
    Execution halted
    ```

# vcdExtra

Version: 0.7-1

## In both

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘alr3’
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    Loading required package: gnm
    The following objects are masked _by_ .GlobalEnv:
    
        A, B, C
    
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'vcd-tutorial.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.30 \ifthenelse
                    {\boolean{Sweave@inconsolata}}{%^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# vdg

Version: 1.2.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Warning in has_utility("pdfcrop") :
      pdfcrop not installed or not in PATH
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'vdg.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.55 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# vdmR

Version: 0.2.6

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘Rdpack’ ‘maptools’ ‘rgeos’
      All declared Imports should be used.
    ```

# VetResearchLMM

Version: 1.0.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘nlme’
      All declared Imports should be used.
    ```

# vidger

Version: 1.0.0

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  6.0Mb
      sub-directories of 1Mb or more:
        data   4.1Mb
        doc    1.7Mb
    ```

# vipor

Version: 0.4.5

## In both

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    Error in re-building vignettes:
      ...
    Loading required package: sm
    Warning in fun(libname, pkgname) : couldn't connect to display ""
    Package 'sm', version 2.2-5.6: type help(sm) for summary information
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'methodComparison.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.55 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# viridis

Version: 0.5.1

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘stats’
      All declared Imports should be used.
    ```

# visdat

Version: 0.5.1

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘plotly’ ‘rlang’
      All declared Imports should be used.
    ```

# vmsbase

Version: 2.1.3

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: ‘cairoDevice’ ‘gWidgetsRGtk2’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# voxel

Version: 1.3.5

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘methods’
      All declared Imports should be used.
    ```

# vqtl

Version: 2.0.4

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘iterators’ ‘knitr’ ‘purrr’ ‘testthat’
      All declared Imports should be used.
    ```

# VSE

Version: 0.99

## In both

*   checking whether package ‘VSE’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/VSE/new/VSE.Rcheck/00install.out’ for details.
    ```

# vsn

Version: 3.48.1

## In both

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
      603 y values <= 0 omitted from logarithmic plot
    convert: profile 'icc': 'RGB ': RGB color space not permitted on grayscale PNG `A-vsn_files/figure-html/nkid-calib4-1.png' @ warning/png.c/MagickPNGWarningHandler/1744.
    convert: profile 'icc': 'RGB ': RGB color space not permitted on grayscale PNG `A-vsn_files/figure-html/nkid-calib5-1.png' @ warning/png.c/MagickPNGWarningHandler/1744.
    convert: profile 'icc': 'RGB ': RGB color space not permitted on grayscale PNG `A-vsn_files/figure-html/vsnQ1-1.png' @ warning/png.c/MagickPNGWarningHandler/1744.
    convert: profile 'icc': 'RGB ': RGB color space not permitted on grayscale PNG `A-vsn_files/figure-html/vsnQ2-1.png' @ warning/png.c/MagickPNGWarningHandler/1744.
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'C-likelihoodcomputations.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `beramono.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.87 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# walker

Version: 0.2.3

## In both

*   R CMD check timed out
    

*   checking installed package size ... NOTE
    ```
      installed size is  6.8Mb
      sub-directories of 1Mb or more:
        libs   5.5Mb
    ```

# walkr

Version: 0.3.4

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'walkr.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `microtype.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.43 ^^M
            
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# walrus

Version: 1.0.3

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘R6’ ‘WRS2’ ‘ggplot2’
      All declared Imports should be used.
    ```

# warbleR

Version: 1.1.15

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘bioacoustics’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# Wats

Version: 0.10.3

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 12.3Mb
      sub-directories of 1Mb or more:
        doc  12.1Mb
    ```

# wavClusteR

Version: 2.14.0

## In both

*   checking whether package ‘wavClusteR’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘Rsamtools’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/wavClusteR/new/wavClusteR.Rcheck/00install.out’ for details.
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' call to ‘doMC’ in package code.
      Please use :: or requireNamespace() instead.
      See section 'Suggested packages' in the 'Writing R Extensions' manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    plotSubstitutions: no visible global function definition for ‘barplot’
    processChunk: no visible global function definition for ‘extractAt’
    processMD: no visible global function definition for ‘registerDoMC’
    readSortedBam: no visible global function definition for ‘scanBamFlag’
    readSortedBam : <anonymous>: no visible binding for global variable
      ‘rname’
    readSortedBam : <anonymous>: no visible binding for global variable
      ‘qwidth’
    Undefined global functions or variables:
      Compartment DNAString DNAStringSet Percentage abline axis barplot
      dbinom extractAt grid hist legend lines pairs panel.smooth par
      polygon qwidth rect registerDoMC rname scanBamFlag seqlevels
      setTxtProgressBar strwidth text txtProgressBar write.table
    Consider adding
      importFrom("graphics", "abline", "axis", "barplot", "grid", "hist",
                 "legend", "lines", "pairs", "panel.smooth", "par",
                 "polygon", "rect", "strwidth", "text")
      importFrom("stats", "dbinom")
      importFrom("utils", "setTxtProgressBar", "txtProgressBar",
                 "write.table")
    to your NAMESPACE file.
    ```

# wbstats

Version: 0.2

## In both

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 1528 marked UTF-8 strings
    ```

# weathercan

Version: 0.2.8

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘sf’
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘xml2’
      All declared Imports should be used.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 25 marked UTF-8 strings
    ```

# webr

Version: 0.1.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘moonBook’
      All declared Imports should be used.
    ```

# WeightedROC

Version: 2018.10.1

## In both

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'Definition.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `fullpage.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.4 \usepackage
                   {verbatim}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# wiggleplotr

Version: 1.4.0

## In both

*   checking for missing documentation entries ... WARNING
    ```
    Warning: package ‘GenomicRanges’ was built under R version 3.5.1
    Warning: package ‘IRanges’ was built under R version 3.5.1
    All user-level objects in a package should have documentation entries.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking for code/documentation mismatches ... WARNING
    ```
    Codoc mismatches from documentation object 'getGenotypePalette':
    getGenotypePalette
      Code: function(old = FALSE)
      Docs: function()
      Argument names in code not in docs:
        old
    ```

*   checking R code for possible problems ... NOTE
    ```
    plotCoverage: no visible global function definition for ‘is’
    plotTranscripts: no visible global function definition for ‘is’
    Undefined global functions or variables:
      is
    Consider adding
      importFrom("methods", "is")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

# windfarmGA

Version: 1.2.1

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘RandomFields’ ‘data.table’
      All declared Imports should be used.
    ```

# wordbankr

Version: 0.3.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘dbplyr’
      All declared Imports should be used.
    ```

# XBSeq

Version: 1.12.0

## In both

*   R CMD check timed out
    

*   checking whether package ‘XBSeq’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘DelayedArray’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/XBSeq/new/XBSeq.Rcheck/00install.out’ for details.
    ```

*   checking whether the namespace can be loaded with stated dependencies ... NOTE
    ```
    Warning: no function found corresponding to methods exports from ‘XBSeq’ for: ‘conditions’, ‘conditions<-’, ‘dispTable’
    
    A namespace must be able to be loaded with just the base namespace
    loaded: otherwise if the namespace gets loaded by a saved object, the
    session will be unable to start.
    
    Probably some imports need to be declared in the NAMESPACE file.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    estimateRealCount,XBSeqDataSet: no visible global function definition
      for ‘assay’
    estimateRealCount,XBSeqDataSet: no visible global function definition
      for ‘assay<-’
    estimateSCV,XBSeqDataSet: no visible global function definition for
      ‘conditions’
    estimateSCV,XBSeqDataSet: no visible global function definition for
      ‘dispTable<-’
    Undefined global functions or variables:
      ..count.. DataFrame Gamma Group Sample SummarizedExperiment assay
      assay<- assays baseMean coefficients complete.cases conditions cor
      data ddelap dispTable dispTable<- dnbinom dpois formula glm
      log2FoldChange median optim p.adjust pbeta predict qbeta quantile
      rnbinom scvBiasCorrectionFits
    Consider adding
      importFrom("stats", "Gamma", "coefficients", "complete.cases", "cor",
                 "dnbinom", "dpois", "formula", "glm", "median", "optim",
                 "p.adjust", "pbeta", "predict", "qbeta", "quantile",
                 "rnbinom")
      importFrom("utils", "data")
    to your NAMESPACE file.
    ```

# xgboost

Version: 0.71.2

## In both

*   checking for GNU extensions in Makefiles ... NOTE
    ```
    GNU make is a SystemRequirements.
    ```

# xkcd

Version: 0.0.6

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'xkcd-intro.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.55 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# XLConnect

Version: 0.2-15

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 12.0Mb
      sub-directories of 1Mb or more:
        java   9.8Mb
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    The following objects are masked from ‘package:base’:
    
        as.Date, as.Date.numeric
    
    Loading required package: ggplot2
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'XLConnect.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `datetime.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.13 ^^M
            
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# xtractomatic

Version: 3.4.2

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    date	lon	lat	lowLon	higLon	lowLat	higLat
    4/23/2003	203.899	19.664	203.899	203.899	19.664	19.664
    4/24/2003	204.151	19.821	203.912597	204.389403	18.78051934	20.86148066
    4/30/2003	203.919	20.351	203.6793669	204.1586331	18.79728188	21.90471812
    5/1/2003	204.229	20.305	203.9943343	204.4636657	18.90440013	21.70559987
    Quitting from lines 818-843 (Usingxtractomatic.Rmd) 
    Error: processing vignette 'Usingxtractomatic.Rmd' failed with diagnostics:
    (converted from warning) Removed 4070 rows containing missing values (geom_raster).
    Execution halted
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘dplyr’
      All declared Imports should be used.
    ```

# xxIRT

Version: 2.1.1

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘utils’
      All declared Imports should be used.
    ```

# YAPSA

Version: 1.6.0

## In both

*   checking whether package ‘YAPSA’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/YAPSA/new/YAPSA.Rcheck/00install.out’ for details.
    ```

# zFactor

Version: 0.1.7

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘rootSolve’
      All declared Imports should be used.
    ```

# zFPKM

Version: 1.2.0

## In both

*   checking R code for possible problems ... NOTE
    ```
    PlotGaussianFitDF: no visible global function definition for ‘dnorm’
    PlotGaussianFitDF: no visible binding for global variable ‘density’
    PlotGaussianFitDF: no visible binding for global variable ‘log2fpkm’
    PlotGaussianFitDF: no visible binding for global variable ‘sample_name’
    zFPKMCalc: no visible global function definition for ‘density’
    zFPKMTransform: no visible global function definition for ‘is’
    Undefined global functions or variables:
      density dnorm is log2fpkm sample_name
    Consider adding
      importFrom("methods", "is")
      importFrom("stats", "density", "dnorm")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

# zinbwave

Version: 1.2.0

## In both

*   checking whether package ‘zinbwave’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: package ‘GenomicRanges’ was built under R version 3.5.1
      Warning: package ‘IRanges’ was built under R version 3.5.1
      Warning: package ‘DelayedArray’ was built under R version 3.5.1
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/zinbwave/new/zinbwave.Rcheck/00install.out’ for details.
    ```

# ZipRadius

Version: 1.0.1

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘ggplot2’ ‘testthat’
      All declared Imports should be used.
    ```

# zonator

Version: 0.5.9

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘zdat’
    ```

# zoo

Version: 1.8-4

## In both

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘stinepack’
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    
    The following objects are masked from ‘package:base’:
    
        as.Date, as.Date.numeric
    
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'zoo-design.tex' failed.
    LaTeX errors:
    ! LaTeX Error: File `ae.sty' not found.
    
    Type X to quit or <RETURN> to proceed,
    or enter new name. (Default extension: sty)
    
    ! Emergency stop.
    <read *> 
             
    l.55 \RequirePackage
                        [T1]{fontenc}^^M
    !  ==> Fatal error occurred, no output PDF file produced!
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# zooaRchGUI

Version: 1.0.2

## In both

*   checking whether package ‘zooaRchGUI’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/zooaRchGUI/new/zooaRchGUI.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘zooaRchGUI’ ...
** package ‘zooaRchGUI’ successfully unpacked and MD5 sums checked
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rjags', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/zooaRchGUI/rjags/libs/rjags.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/zooaRchGUI/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/zooaRchGUI/rjags/libs/rjags.so
  Reason: image not found
ERROR: lazy loading failed for package ‘zooaRchGUI’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/zooaRchGUI/new/zooaRchGUI.Rcheck/zooaRchGUI’

```
### CRAN

```
* installing *source* package ‘zooaRchGUI’ ...
** package ‘zooaRchGUI’ successfully unpacked and MD5 sums checked
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rjags', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/zooaRchGUI/rjags/libs/rjags.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/zooaRchGUI/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/zooaRchGUI/rjags/libs/rjags.so
  Reason: image not found
ERROR: lazy loading failed for package ‘zooaRchGUI’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/zooaRchGUI/old/zooaRchGUI.Rcheck/zooaRchGUI’

```
# zoocat

Version: 0.2.0.1

## In both

*   checking Rd \usage sections ... NOTE
    ```
    S3 methods shown with full name in documentation object 'merge':
      ‘cbind.zoocat’
    
    The \usage entries for S3 methods should use the \method markup and not
    their full name.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

# ztype

Version: 0.1.0

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘dplyr’ ‘ggplot2’ ‘lubridate’
      All declared Imports should be used.
    ```

